## TRANSCRIPT     
0:00     
hello Morgan hell Susan so welcome uh yeah I don't think you're     
0:10     
here last week Susan how how are things how are things going with you well um     
0:15     
I've tried several several different models and none of them work they seem     
0:22     
to um don't quite resolve I can't get the bottom part out of the red sort of     
0:31     
speak no matter what I do yeah like I said I think it thinks that I'm building     
0:37     
a bridge or some sort of uh large structure and I'm     
0:42     
not it's cells float in water their parts are kind     
0:49     
of they float in water so they're not pressing down     
0:57     
on well on on the bottom they don't do     
1:04     
that so I'm having some issues with it it's just lovely pictures with the     
1:10     
bottom red and the only way I can get the images of of these things is     
1:18     
to what they say is called write a report oh     
1:24     
so it doesn't turn up until I say report and so I have uh I don't know over a     
1:30     
dozen reports with different variables to try to fix this and no just still     
1:42     
red so I'm meeting with the professors this afternoon and I'm going to ask the one fellow who knows something about     
1:49     
mechanics what I'm an electrical engineer I've     
1:55     
just been trying to do this program like I don't know AutoCAD really I don't know     
2:01     
that's the only thing I might know and this commo thing is just um got the better of     
2:10     
me so I'm just GNA write it up [Laughter] okay so that's using console want me to     
2:18     
do that pardon that's using console     
2:24     
yeah it it's h very involved like um     
2:30     
I wanted to find the middle of a line so I I uh have to kind of break up the line     
2:36     
that's the easy way to find the middle of a line and then I need it back     
2:42     
together again like I can't have it split and operating independently or my     
2:47     
structure doesn't work how do you do that     
2:52     
so well I I did did try to combine it     
2:58     
but it's still not working so I     
3:03     
don't have no clue right if you can see behind me I have a bunch of stuff I need     
3:09     
to sort through finish writing it up     
3:14     
and yeah well good luck with that yeah sorry that console didn't work out I     
3:21     
don't know yeah I've talked to students about it and other people and they they just     
3:29     
just don't like it right oh yeah how about some of the other     
3:36     
programs like we talked about Morpho and some of those are those like still on the table     
3:43     
or I wanted to try Moro but I couldn't get it downloaded on this computer the     
3:49     
last line like it's um Ino U by line     
3:57     
program um and and I'm willing to try it um but I've never really used used     
4:05     
something like that before yeah well yeah and my professor says no     
4:14     
just use Matlab we just want you to use mat lab and I'm going well I wish I'd used mat lab before like iest this     
4:23     
but right so in that lab you can easily transfer it to python if you need a     
4:31     
programming language although right it's related     
4:38     
anyway and M lab's nice it gives you an error message you take the error message you copy and paste it into Google and it     
4:45     
comes up with all these people who've solved this error message and they did this and that and the other     
4:51     
thing and you go there oh that one and you pluck it out and put it in     
4:58     
your program and it works yeah it's like you look up an error message in     
5:04     
console it refers you to the manual you look in the manual and it's     
5:12     
um yeah or it's in in computer e or     
5:17     
something yeah and you're tring okay well on what in earth does that     
5:22     
mean it takes more than an hour to read a page because you're going yeah great     
5:31     
anyway this is my complaint it's hard to to figure out     
5:36     
error messages and console or like what I'm problem I'm     
5:43     
having I don't know I sent it into the help desk and they get back to me a couple of weeks later     
5:51     
yeah no that's not a help desk sorry that's just the tease     
5:58     
yeah anyway so how are things okay yeah we're     
6:05     
just U Rolling Along sort of at the end of the semester and hopefully uh things end uh without     
6:14     
too much incident so and for the term you know for the year yeah yeah that's     
6:20     
that's why I want this written up WR up for the end of the term and my     
6:26     
PowerPoint um started for presentation and just do it     
6:33     
yeah anyway yeah so I've     
6:40     
tried um four or five configurations of of this so that should be     
6:49     
enough they all don't work in some way or     
6:57     
other yeah there's a phone     
7:06     
call yeah good and my phone has been having     
7:12     
trouble oh am     
7:17     
okay um so I guess I can talk about a few things     
7:22     
um yeah there's this let me share my     
7:28     
screen oh yeah I have a paper I should send you oh yeah what is it     
7:37     
the try to remember okay so I just wanted to point well I     
7:43     
did talk about um uh the graph learning on graphs     
7:49     
conference and I have I have yet to like summarize it reasonably but just you     
7:55     
know I've been I've been going through the materials but I haven't been able to summarize it so I'll do that later uh     
8:01     
maybe by the end of the year we'll see um you know it's a lot of stuff so it's it's like you know last year they did uh     
8:10     
they kind of went over the basics of the field and the field is moving pretty fast so they're all sorts of different     
8:16     
approaches that people are using one of the newer approaches of course they're focusing on topological methods things     
8:25     
like that they're also focusing on I think we talked last about different ways that you define a     
8:32     
connection between two nodes and they do a lot of things with message passing to     
8:39     
sort of extract the networks and to build networks from data so there are     
8:44     
different methods you can pull in from like communication networks to help and     
8:51     
you know as for like the types of networks that they had a lot of them were biology some of them were like     
8:57     
proteomic uh there were you know some applications to Medicine there were also a lot of data     
9:03     
science uh techniques so like you know analyzing large data sets with these     
9:09     
kind of methods so it's not just theory that they had     
9:14     
they had a lot of stuff anything on actual physical um networks uh they did have     
9:21     
like a a lecture and a workshop on physics in you know physics but I don't     
9:26     
know if it would have been particularly relevant to what you're were doing so like you know I'm supposed to take these     
9:35     
models large and figure out um what it would be in a large     
9:40     
tissue okay and I think I'm going to go for another approach rather than     
9:46     
building um a t segr to Encompass everything because there's um the vertex     
9:55     
model of cells uses um just the shape of the top apex of the     
10:03     
cell um to determine its behavior and so one might be able to     
10:09     
just work with a simplified model to get most of the behavior of the     
10:16     
self yeah yeah just um I'm kind of fed up with time to get I work so you'll try     
10:25     
other approaches if you have to oh yeah no I've already um looked at I i' what did I say     
10:34     
epithelial cells are mod modifying Kumi um lattice that was it okay yeah     
10:46     
yeah yeah so can I get oh go ahead I I'll     
10:52     
just I'll send you this paper later I found one on C elans I think it was and     
10:58     
then there was another one I thought might like okay yeah that'd be     
11:04     
great okay I'll I'll I'll turn off my camera so that people don't have to look     
11:11     
at my mess     
11:16     
me yeah so I'll straighten it up oh yeah I hope     
11:22     
I can just write this up and straighten up my     
11:27     
mess all right um yeah so that that's learning on graphs there's this other     
11:33     
event coming up early next year this is the virtual sell and development Festival week and this is something I'm     
11:40     
going to submit a abstract for I have to submit it by January 5 so I might talk     
11:47     
about it next week um we have a lot of things in the group that we could submit to this um it just has a lot of     
11:54     
different things that you can uh you know different many posia talks     
12:00     
professional development planner and other types of events so this is something that you know might be of uh     
12:08     
well I'll see what kind of what I think about might be appropriate for this um     
12:14     
this is like 15 minute talks so so would would my Tegrity of cells fit in with     
12:21     
that oh yeah probably that would work I mean you know they have like I     
12:26     
guess it's pretty diverse uh what they're looking for so this is uh just a     
12:32     
virtual thing so it's like you can participate from anywhere in the world     
12:38     
so and it's it's a wide variety of topics in uh cell and developmental     
12:43     
biology so it's the SNB subgroup on cell and developmental biology and so this is     
12:50     
the society for mathematical biology so they're um you know that's exactly what     
12:56     
they're interested in that type of thing     
13:03     
okay so yeah I can send you a link to the thing if you want oh okay     
13:12     
okay so yeah that's that's that and then uh we've had some other types of things     
13:19     
going on uh in the slack so uh gahong     
13:24     
sent this paper along this is an older paper from 2018 but it's it's actually girang ba who of     
13:33     
course has been involved in a number of uh efforts to get uh um cell uh se you     
13:42     
know cell data for seigan embryos uh made public and he's you know     
13:48     
he's working with a number of people here on this paper uh it's called Deep reinforcement learning of cell movements     
13:55     
in the early stage of seelan embryogenesis so I'm going to go through this paper and this is basically where     
14:01     
they're using uh agent based models and they're using deep reinforcement learning along with that to analyze some     
14:08     
of the 3D timelapse microscopy images that we talk about a lot in the     
14:15     
group so I don't know if there was something in the chat here oh yeah I'll send the link to the     
14:21     
virtual cell     
14:27     
meeting so I just put that in the chat for Susan and Morgan this     
14:34     
link okay um cop     
14:41     
yeah so yeah this is uh let's let's look at this paper a little bit and see what     
14:47     
they're doing what kinds of techniques they're using and what their results are so this was published in um     
14:55     
bioinformatics Journal so um so the motivation is that cell movements in the     
15:02     
early phase of cigan development is regulated by a highly complex process in     
15:07     
which a set of rules and connections are formulated at distinct scales previous     
15:13     
efforts have demonstrated that agent-based multiscale modeling systems can integrate both physical and     
15:20     
biological rules and so you know you can have those physical rules of interaction but also the biological rules that you     
15:27     
know we've talked about and provide new avenues to study developmental systems however the     
15:33     
application of these systems to model cell movement is still challenging it requires a comprehensive understanding     
15:39     
of regulatory networks at the right scales and so you know def deciding what     
15:45     
those rules are are you know is it like are they uh Force interactions or are     
15:52     
they um you know different types of uh molecular signaling or what is going on     
16:00     
there what are the rules that we need and remember we don't need to take every single factor into account we just need     
16:06     
to abstract that to some key factors that produce something that looks like     
16:12     
celegans and then work from there so if you have you know physical rules that     
16:17     
are very Bare Bones it's okay as long as it produces a good uh     
16:24     
result uh however the applications of these systems to model sell movement is     
16:29     
still challenging and requires a comprehensive understanding of regulatory networks at the right scales     
16:36     
recent developments in deep learning and reinforcement learning provide an unprecedented opportunity to explore     
16:43     
cell movement using 3D timelapse microscopy images which is what Jang B     
16:49     
does a lot of he's got a lot of experience in acquiring those and analyzing those for you know like     
16:56     
different types of uh molecular signals so in this paper they present a     
17:02     
deep reinforcement learning approach within an agent-based modeling system to     
17:08     
characterize cell movement in the embryonic development of SE elans so this is where they're characterizing the     
17:13     
migration of cells after division they're moving around the embryo and kind of acquiring new positions so a     
17:21     
cell divides one uh you know the daughter cells will move to the left     
17:27     
hemisphere in the right Hemisphere or you'll get cells that migrate from one part of the connectone to another region     
17:34     
the shape changes you know that those involve a lot of movements you know we want to be able to characterize those     
17:40     
but of course those all depend on rules between cells interacting in     
17:47     
space our modeling system captures the complexity of cell movement patterns in the embrio and overcomes the local     
17:54     
optimization problem encountered by traditional rule-based a agent based modeling that uses greedy algorithms so     
18:02     
there are these uh this is the type of agent based modeling that we use where we just simply uh you know use a set of     
18:10     
rules we have a set of cells that are fixed in space and they just interact     
18:15     
based on the state of their neighbors and so you know or you'll have some sort of you know contents of the     
18:22     
cell uh that have to be optimized in some way so a greedy Al algorithms that     
18:31     
kind of goes with robotics think it's a it's based on bolman's constant and I     
18:39     
believe in um uh robot exploration of the     
18:46     
environment works better with uh things that are not greedy like     
18:53     
it yeah anyway yeah that's where I ran into     
18:59     
that was of course on robotics right well yeah it's yeah it's one uh application domain so yeah it's     
19:08     
it's just you know the way they're optimizing the problem and they're kind of considering other types of algorithms     
19:15     
for the some of these movements yeah we tested our model with two real     
19:20     
developmental processes the anterior movement of the CP AAA cell which is in     
19:27     
that C sublineage it's posterior and then it's anterior anterior anterior so     
19:34     
like the divisions and C elegans are often like in these sub uh these uh     
19:39     
sublineages and you know they have they're sort of the nclat talks about or     
19:46     
specifies the direction in which the division occurs so there's a posterior     
19:51     
cell in an anterior cell in in in C and then there's there are three anterior     
19:57     
division ision so when you have a division which cell is anterior which cell is posterior so you get this one     
20:04     
division that's towards the tail and three divisions that go towards the head and then they migrate based on that sort     
20:12     
of orientation so that's why they have these kind of names but basically this     
20:20     
uh cell is moving anteriorly by intercalation which means it's moving in between other cells and     
20:28     
the rearrangement of The Superficial Left Right asymmetry so there's this superficial     
20:33     
Left Right asymmetry that gets rearranged as the cell was migrating um and so they they look at     
20:40     
that so they have basically what they have to do is they have to explore their environment they have to have the cell     
20:46     
explore the space in between other cells so it has to move through the embryo     
20:51     
space it has to sort of squeeze in between other cells and find its uh     
20:57     
adult position position and so this is where things are getting rearranged     
21:02     
because you're moving cells around in the first case the model     
21:07     
results suggested that CP aaa's interpolation is an active directional     
21:13     
cell movement caused by The Continuous effects from a longer distance or further than the length of two adjacent     
21:21     
cells as opposed to a passive movement caused by neighbor cell movements so this is where they kind of uh you know     
21:28     
if you have cells that are next to one another they're kind of moving around and you're getting kind of squeezed or     
21:35     
moved around as a as a result of that and so uh this is like where there's     
21:40     
there adjustments that need to be made based on what your neighbors are doing and that would be like typical of a a     
21:47     
typical agent based modeling uh sort of setting where you know you're basically     
21:53     
responding to your neighbor States um but in this case you're looking at these uh longer distance um     
22:01     
active movements which you know involve the cell actually moving to a new location acquiring new neighbors and     
22:08     
rearranging the Rd that this process is being played out     
22:14     
of so these results showed that our approach to introduce deep reinforcement     
22:19     
learning into the agent based modeling Paradigm can test regulatory mechanisms     
22:25     
by exploring cell migration paths and a reverse engineering perspective this model opens new doors     
22:32     
to explore the large data sets generated by live Imaging so this is the whole point of this was to sort of have a     
22:39     
model for live Imaging data to sort of get some of this realtime process down     
22:46     
and you know it gives us a little bit more flexible approach to this so the     
22:52     
source code is available here at this GitHub repository you look at the GitHub repository     
22:59     
we see that it's uh this is DRL for cell movement uh this just kind of hasn't     
23:05     
really been developed uh that much since but it's basically you know I think it's     
23:11     
supposed to be like a legacy uh repo so this is the paper link to the     
23:18     
paper and so basically tell you how to run it you use pip plot pytorch numpy     
23:25     
Mesa and Python 3 so they're running this on and uh yeah basically it tells you how     
23:33     
to use it and install it so that's the repo um you have the     
23:41     
model and then the Run dqn file this is the Mesa model here you can     
23:47     
see that you have uh all the different things I I don't you know if we want to     
23:54     
get into this code too much but basically they're modeling a series of cells you can see here the state cell     
24:00     
list you have uh you have to Define some of the parameters of the     
24:07     
cells you have to uh adjust the     
24:12     
axes and then you go through and you run the simulation where it's     
24:19     
drawing the plane it's drawing the uh grid for the uh agent based model     
24:26     
and then it's running the program on that uh with these different parameters and you have a grid full of cells and     
24:34     
it's calculating a radius around a cell to determine what its neighbors are and     
24:39     
I guess you're moving the cells around and then you're seeing what the neighbors are you're redrawing this the radius but it's it looks like a pretty     
24:46     
good approach it's using Mesa and it's so let's look at the rest of the paper and see what people uh or what they've     
24:54     
been able to do here so so     
25:01     
uh so they have the they talk a lot about the recordings uh talk about agent based     
25:07     
modeling and directional cell movement which of course is what they're doing here um so directional cell movement is     
25:15     
critical in many physiological processes during seelan development including     
25:20     
morphogenesis structure restoration and nervous system formation it is known     
25:26     
that in these processes cell movements can be guided by gradients of various chemical signals physical interactions     
25:32     
of the cell substrate interface and other mechanisms so this is uh there's a     
25:37     
big literature on this especially in CA elegans it remains an open and interesting challenge is the what and     
25:44     
how one could learn about the rules and mechanisms of cell movement from the movement tracks reported in live Imaging     
25:51     
so you know when we have this live Imaging we have these movies we have these movement tracks that you can     
25:56     
extract we've done this in some of the uh in some of the projects where people     
26:01     
have extracted cell tracks from the data it's basically where the cell moves over time so in each frame the cell moves its     
26:09     
position and you can calculate that track that track then you know there's a division there are new tracks that are     
26:16     
generated and you can actually you know that's very different from the grid that     
26:22     
we usually think of when we think of agent based modeling so you know that's something that has to be incorpor     
26:30     
um and so this paper presents a new approach adopting deeper enforcement     
26:36     
learning approaches within this agent based modeling framework and so this     
26:41     
this is diff this is where we differ from the greedy algorithm that we usually use and so this is the section     
26:47     
where they talk about this deep re enforcement learning is good at dealing with high dimensional inputs and can     
26:53     
optimize complex policies over primitive actions so this is where you you have these policies that you have to select     
27:00     
and uh it's you know it's kind of like a policy for interaction so you select a     
27:06     
policy and you go with it so for example if you had a cell that had a policy for     
27:13     
interacting with other cells it would just be like a set of rules that it would adopt over another set of rules     
27:19     
and then would do that you know it would it would execute on that so these are this is a little bit different way of     
27:25     
thinking about it with a you know regular agent based model you just have a single set of rules usually Implement     
27:32     
over the entire simulation you just kind of choose your rules and say this is     
27:37     
what we're going to do in this case what they're saying is that we can actually create policies that allow us to select     
27:44     
our rule sets especially when we're moving around when we move around we have different rules in different parts     
27:50     
of the embryo or different rules over time so this is     
27:55     
where this natural aligns with the complex cell movement patterns that occur during Sean's erogenesis     
28:03     
uh and so you can characterize this without knowing all the details of the     
28:08     
biology and just kind of you know gives it some flexibility um and so we tested our     
28:16     
model through two represent representative scenarios during celan embryogenesis the first is the anterior     
28:23     
movement of cpaaa by intercalation     
28:28     
and then the rearrangement of The Superficial Left Right asymmetry so these two cases where you have     
28:35     
interpolation and this asymmetry rearrangement in the first case we     
28:40     
propos two hypotheses for the interpolation of cpaaa and simulation results indicated     
28:47     
that cpaaa experienced an active directional movement towards the anterior so it's     
28:53     
moving towards the head uh which is caused by the continuous effects from a longer     
28:59     
distance rather than a passive process which it is squeezed to the Target locations by its neighbor's movement so     
29:06     
this is where you know again your neighbors are moving around you're kind of squeezing on the cellence pushing it     
29:12     
in a new place wherever it fits in in the for the alternate hypothesis it's     
29:19     
that the cell is actually moving somewhere and it's kind of pushing past the rest of the cells it's acquiring new     
29:25     
neighbors and so forth so that's how they kind of think about this um in the     
29:32     
second case the frequently occurring leader     
29:37     
follower mechanism was also supported by the simulation results of the asymmetry     
29:43     
re Arrangement so there's this leader follower mechanism that um people I     
29:49     
guess hav really talked about it but basically if you have cells migrating you might have a leader and you might     
29:55     
have a follower so if you have two neighbors one does something first and     
30:00     
the other follows along or one does an action first and then the second one is impacted by that by that other cell so     
30:07     
they're all sorts of leader follower hype oh go ahead um it's just so much     
30:13     
like the robotics course that I took oh yeah yeah again I guess I guess it's based on     
30:20     
that yeah well yeah it is and then you know there's this also this whole area of game theory that involves like first     
30:27     
mover Dynamics and leader follower mechanisms in that area oh really     
30:33     
yeah so there these the they're these stackle BG games that uh you know     
30:39     
basically Remodel and they do this this is kind of from the economics literature but it's like firms that sort of     
30:46     
innovate first and they come up with an innovation and then the rest of the firms in the market will follow or you     
30:52     
know if there's a leader in innovate innovating some new invention the rest rest of the uh firms will follow and so     
31:00     
that that's an economics but you can apply that to biology where you have these kinds of uh you know hierarchies     
31:08     
of action where if the leader does something it constrains the followers to doing that thing so you could see that     
31:15     
with a physical system of cells where if you move into one space the other cell     
31:20     
can't move there they have to sort of uh move maybe in in or adopt their mve     
31:27     
ments to tailor you know what the other cells have done before them so it really constrains the embryo we actually uh     
31:35     
we've talked about this in a number of papers in our group actually uh we have the paper on the cybernetic embryo from     
31:42     
2018 and then another paper from 2020 on uh connect Dome wearing using the leader     
31:49     
follower game or or leader follower Dynamics I'm glad to see that they're going in this direction as well okay so     
31:56     
in the meeting I mentioned two papers that the group has done on first mover or leader follower Dynamics and so     
32:03     
leader follower Dynamics it's an emerging topic I think in cell biology     
32:10     
thinking about cells that are migrating cells that have sort of move first and     
32:16     
then cells that follow along so theer follower or you can have these first mover Dynamics which means that the sell     
32:23     
that moves first its preference and the constraints are largely geometrical you     
32:28     
know this is something we can map from economics to cell biology we have tools     
32:34     
like stackle work equilibria and other types of tools in economics that we can     
32:40     
transfer to cell biology as well so that being said we have a couple of papers     
32:46     
that we've done in the group on this one from 2018 which I'm showing here and one from 2020 which I'll talk about next so     
32:54     
this paper is in biosystems it's called origins of the embryo self-organization through     
33:00     
cybernetic regulation so this is Robert Stone Tom portuges George mikowski and     
33:08     
myself Tom is a a longer term member of the group who doesn't attend much anymore but he used to attend all the     
33:14     
time early on in our group's history uh Rob stone is a collaborator     
33:21     
he wrote an original paper what he called the cybernetic embryo and then wrote this as sort of     
33:27     
the second version of it and so actually we worked on this all together we cover     
33:33     
a number of things in this paper we talk about sort of this idea of the cybernetic embryo kind of develop some     
33:40     
of the themes of the cybernetic embryo some things about recursivity some things about um     
33:47     
different types of complexity uh talking a little bit about uh systems and then     
33:54     
getting into some of these other issues with uh respect to     
33:59     
thermodynamics one thing that we do talk about in     
34:05     
here is this figure here figure four so this is when we get into this discussion     
34:11     
of first mover Dynamics and embryogenesis we can see that we can map     
34:17     
embryogenesis or very early embryogenesis roughly to a game of Tic-tac-toe and you say what is that     
34:24     
have to do with anything and the answer is of course is that tic-tac-toe has these this first mover Dynamics built     
34:32     
into the game so if I start a game of tict tactoe I put an X in the middle and     
34:38     
then you know in the center square and then if you want to put an O you have to play defense put an o in the lower right     
34:46     
square and then I my possible moves are to block you so now I'm playing defense     
34:52     
but the first mover always has the advantage because they can set the terms of how the space is partitioned what are     
34:59     
the subsequent moves and so this is a game of techto and in England they call     
35:06     
this knots and crosses and Knots and Crosses is actually has a a rich history     
35:11     
in Game Theory and in artificial intelligence research so look that up if you're interested but we can map this     
35:18     
game of tic taaco or NS and process to an embryo and so you can see here at the bottom in B we have this embryo that's     
35:25     
uh dividing so we have a single cell that's fertilized it divides into two     
35:30     
cells and then the first cell at the top uh which would be the anterior pole you     
35:37     
would have a division event and that division event then would constrain sort     
35:42     
of the positions of the second mover this isn't considering migration this is     
35:47     
just considering uh you know a static embryo and migration would influence this model considerably but probably not     
35:54     
because as we talked about in the meeting when you make your move you're sort of leading the rest of the cells so     
36:02     
if one divides first and this is not a conscious process like Tic teac toe but     
36:07     
it's just when the event occurs so cell one divides first it has two cells those     
36:13     
two cells have to be somewhere they can select their position and then two divides and those cells can select their     
36:21     
position and so on and so forth and so you know the way we've proposed this it     
36:26     
can create asymmetries in the embryo structure it can create other types of     
36:33     
uh constraints on the this the positioning of the cells the shape of     
36:38     
the embryo and the ability for cells to migrate so this is a nice model we     
36:45     
haven't really developed it much more than this and then later in the paper we talk about some of the thermodynamic     
36:51     
constraints that might be constraining this as well so two years later I wrote     
36:56     
this payer alone this is raising the connectone the emergence of neuronal activity and behavior and sea Elegance     
37:04     
so in this case you're focusing more just exclusively on the connectone the neuroc connectone this was published in     
37:10     
Frontiers and cellular Neuroscience uh this is the section neuro cellular     
37:16     
neurophysiology this was in a special issue on interplay between ion channels     
37:21     
the nervous system and embryonic development so we also talk about these first mover Dynamics     
37:27     
and in term this in this case in terms of wearing of the connector so this is an example here in     
37:34     
figure two and this is a sort of a model of connecto wiring as these cells are     
37:41     
differentiating into neurons and they start picking their connections to other neurons so this is in cagans where we     
37:49     
have this model of uh connectto Genesis which is what we call it where you have     
37:55     
cells that divide and then they keep dividing and then they start to differentiate into uh in this case     
38:03     
neuronal cells these neuronal cells get turn on at different points in development so these numbers represent     
38:09     
minutes of development at which these cells become neurons or become neuron     
38:15     
likee and so the idea is that there are different sets of rules that are connecting them together so which cells     
38:20     
are most likely to connect to one another and so we review in the paper that there's been some work on     
38:27     
connected you know how cells get connected that there are these hypotheses about Age first and other     
38:34     
types of hypotheses so the earliest cells start to connect and then the other cells join in or if there are     
38:40     
other more complex rules and so you know we kind of recast this question in terms     
38:46     
of the first and second moover Dynamics using different rules so we have these different types of coupling rules these     
38:53     
different types of logic rules and then of course influences of first and second     
38:58     
mover Dynamics so this is you know just a way to kind of understand connector     
39:06     
Genesis based on U the age of the cell but also based on a data set of synaptic     
39:13     
connections during development so this is a good paper that lays out and again     
39:19     
it only lays out briefly this idea of first mover Dynamics but it does put the     
39:25     
recast this in terms of connecting different parts of the emerging connecto     
39:30     
raising the connecto the emergence of neuronal activity behavior in SE elegans infs for cellular neuroscience and     
39:38     
origins of the embryo self-organization through cytic regulation and biosystems     
39:43     
and I look forward maybe to doing another paper on theer follower Dynamics     
39:48     
or first mover Dynamics um maybe next year so if you're interested in     
39:55     
collaborating but so this is where you have this mechanism leader follower that was also supported     
40:01     
by the simulation results of the asymmetry rearrangement so this is good that we have this sort of asymmetry     
40:08     
rearrangement being based on this leader follower mechanism in summary this framework     
40:14     
presents a reverse engineering perspective to investigate regulatory mechanisms behind certain developmental     
40:22     
processes by formulating a reward function as the representation of regulatory mechanisms different     
40:29     
hypotheses can be tested via reinforcement learning procedures by comparing the extent of similarities     
40:36     
between the simulation cell migration paths and the obser observation data     
40:41     
such hypotheses can be either supported or rejected which can facilitate new     
40:46     
explanations of certain cell movement behaviors so this is definitely you know     
40:51     
I think really good paper for framing a lot of this this is the ABM framework     
40:58     
that they talk about where they have the greedy algorithm it's really where you     
41:03     
have um actually I don't know if this is their ABM framework or the the general     
41:09     
ABM framework but this is uh where you start you set up the initialize the     
41:14     
environment you hatch initial cells which means they are populated in in the     
41:20     
simulation so I don't know if they're actually simulating um cell division directly but     
41:26     
they're just populating the uh ABM with cells they're updating the global timer     
41:32     
so there's this timer that goes then there's an is the ending condition met is it in its place that it needs to be     
41:39     
if not then you select the next cell to act you move the the next location you     
41:44     
have a division event you hatch a new cell which means it appears or you don't hatch a new cell     
41:51     
and then you ask have all cells acted and if they haven't then you go back and you select the next cell to act then if     
41:58     
all cells have acted then you update the global timer you maybe select the next cell to     
42:04     
act uh or you say is the ending condition met and then if that's true     
42:09     
then you end the simulation so you go through this round of you go through rounds of time where     
42:16     
you have a bunch of initial cells that populate the environment you check to see if they're in the right location you     
42:23     
update the time you move them again and then you hack new cells when they need to be hatched and then if all cells have     
42:29     
acted then you end the simulation it's pretty simple um and     
42:35     
then the reinforcement learning framework is a little bit different this is the typical diagram for reinforcement     
42:41     
learning where you have the environment which is the embryo all the cells exist in that     
42:46     
environment you have a state and a reward so the state function and the reward function the state is whatever     
42:53     
the whatever position the cell is in and and then if it's updated you have a state t+ one the reward of course is the     
43:01     
current reward so if there's a reward to move then that reward is met if there's     
43:07     
a reward to stay in place the reward is met if it stays in place and then if you have these leader follower Dynamics the     
43:14     
reward fluctuates based on the the first mover of your neighbor or if you're the     
43:19     
first mover and the reward differs there and so then the agent is the cell and     
43:25     
there's an action taken on the environment which is its movement and how it rearranges everything else or it     
43:31     
sets the reward it sort of sets the new set of rewards that that are happening     
43:37     
so you have to select these policies that Define the interaction rules based on the reward based on the current state     
43:44     
and based on actions     
43:50     
so um what does the CN the cpaa     
43:56     
me oh it's just the um the nomenclature so it's like the C     
44:02     
sublineage it's the uh post and then it takes a couple of Divisions it divides     
44:09     
anterior posterior so the anterior cell is one sister or daughter cell the posterior is another daughter cell and     
44:16     
then uh you get another division which is anterior posterior but this time the     
44:22     
cell is anterior so it's towards the head and then there are three divisions towards the head so it's like if you     
44:27     
think about the C sublineage it's dividing along two axes left right and     
44:34     
anterior posterior and usually a lot of the divisions uh in the nomenclature     
44:40     
anterior posterior so you have like the idea is you have these different divisions over time so the daughters     
44:47     
move like towards the tail towards the head and then that's where they end at that's where they divide to and then     
44:54     
they move in addition to that so the movements are additional and they move     
44:59     
wherever they need to be for their final location but you can see that just having the cell divisions like be     
45:06     
spatially explicit they sort of you know set up this this     
45:12     
geometry where you have it's almost like soap bubbles that that expand in     
45:17     
different directions along different axes so the     
45:23     
cpaaa describes like the the daughter and the granddaughter and the great     
45:29     
granddaughter yeah yeah yeah sort of okay     
45:35     
yeah so yeah that's that's uh and so then they have all these cells that they     
45:41     
populate their simulation with uh let's let's look at this figure too this is     
45:47     
figure four this is where we     
45:54     
have uh specific evaluation step for a single action so the way this works is     
46:00     
that you have yourselves this nomenclature this is AB a r p p p a so     
46:08     
this is where you have the ab sublineage and then there's an anterior division a     
46:13     
division to the right a division posterior posterior posterior anterior     
46:19     
then posterior and so when you have that sort of setting you know the cell is going to     
46:25     
be from that AB which is at sort of like the first it's like the two cell State     
46:31     
and that AB is the anterior to that so it's like the first half of the EMB the     
46:38     
front half of the embryo and then you get a couple of Divisions where it's like towards the front towards the right     
46:45     
towards the the tail tail tail then towards the head then towards the     
46:51     
tail and that gives you a three-dimensional location which is here     
46:57     
um and you know this one here is CPAP which means it's posterior and     
47:04     
anterior but it's still located somewhere because the C sublineage divides uh and it it moves to the right     
47:10     
or left so there's this spatial position you can assign to this and so it's a     
47:15     
three-dimensional spatial position and you can put all these positions into a list you put it into a neural network     
47:23     
you get this orientation map which is um you know has his different directions     
47:29     
different amounts of movement so this is a list of cells are pre- selected as     
47:34     
their state cells via the cell neighbor determination model so you have these 3D     
47:40     
positions you determine which ones are neighbors being which ones are the nearest neighbors based on their uh XY Z     
47:48     
position and then you you might use like a ukian distance metric for that just to     
47:54     
determine which ones are are closest um and then a list of cells are pre-selected as the state cells by the     
48:01     
cell neighbor determination model so these are the neighbors here kind of lined up in this row it's like     
48:08     
onedimensional uh row of neighbors here um their locations are concatenated so     
48:16     
you might select a number of one-dimensional uh rows where you know     
48:21     
you might have the x coordinates the y-coordinates and the Zed coordinates and the Neighbors in those dimensions     
48:28     
and then their locations are concatenated and sent to the neural network which is here and the output     
48:34     
action with the maximal probability is selected as the migration cell's Next Movement so this is the movement     
48:41     
Direction and this is I guess the probability not the speed but the probability that this is the right     
48:48     
direction and then you see the model you have these cells that you know they move     
48:53     
in a certain direction and then that determines the the state the this next     
48:59     
state of the uh ABM so you're taking the positions and you're doing this     
49:04     
recursively so once these cells move then you can new locations and you do it     
49:09     
all over and you do it until it ends so that's I'm that's all I'm going     
49:15     
to talk about with this paper um any     
49:20     
questions I think I'll quote it oh yeah yeah     
49:28     
so a nice example of using Mesa and um you know some of these uh     
49:35     
techniques um yeah we' used Mesa in in my other group uh we use Mesa for a     
49:42     
number of different types of models it's pretty good to work with it's you know it's python extensible which means you     
49:49     
can plug other toolboxes into it so you can combine Mesa with reinforcement learning which we've done actually     
49:56     
um we've combined you know agent based models with active inference models and     
50:02     
other things so there other things you can work you can build in Python other types of models and you can make it into     
50:07     
a hybrid model where you have an ABM and some other thing and so that's really powerful     
50:14     
because you know the a standard ABM model isn't necessarily uh you know use as useful as     
50:22     
having other types of algorithms on top of it     
50:29     
okay the next thing I want to talk about today is um so there's been a lot of stuff out     
50:37     
about simulating embryos um there's this set of papers     
50:43     
but there's also this sort of Target article that came out in I think it's     
50:48     
nature methods and it's an article on people building virtual     
50:54     
embryos and so so this is a model uh model that sort of predicts forces in an     
51:01     
embryo this is from the turlier lab they in France uh and they they built this they     
51:08     
developed this technique where you can take microscopy images of an embryo and     
51:14     
estimate the forces using I guess it's some sort of find an element model but     
51:20     
I'm not sure how exactly they're doing it um the the citation of the paper is in this paper     
51:26     
so you know it allows you to estimate the forces across the entire embryo for     
51:32     
all cells and cell interaction spaces so this is where you know you would do like in the last paper we saw that they had     
51:39     
uh characterized the embryo they characterize the cells moving around and the forces and there are all sorts of     
51:45     
interesting interactions there in this case they're just measuring the forces and you can measure it over that you     
51:51     
know dynamically over the period wind cells are dividing and migrating and     
51:57     
getting a sense of what those are so you're actually adding param adding to the parameter     
52:02     
space so to develop computational embryo models takes experimental data Ai and     
52:09     
machine learning ter your lab is developed a method to build a geometric map from Imaging data this is the 390     
52:16     
cell CED embryo which we've worked with the data on that but we haven't talked     
52:22     
about it recently so C squirts are a great model for looking at embryogenesis     
52:27     
C squirts um they have you know there's there's that not only is there data     
52:34     
available uh you know online like we have with seans but it also has the same     
52:39     
rules of seans every Genesis that is that development is a deterministic     
52:45     
thing so we have this nomenclature it's very different from celegans but it's very descriptive in that the cells don't     
52:51     
change their fate or they change their sort of uh you know what they what they do over     
52:59     
different embryos it's very much like if I track like the b sublineage or the a     
53:05     
sublineage I know what the daughter cells will be and roughly where they'll be in the embryo so I I can know this     
53:12     
information and so that makes for that's an advantage for this type of uh you know uh embryo analysis and simulation     
53:21     
so this is the 396 CC squirt embryo um and C squirts are interesting because     
53:29     
they start out kind of in this deterministic mode and then they have a number of different uh phenotypes     
53:36     
throughout development which are also interesting but you know that's not the perview of embryo uh     
53:44     
modeling so how to build a virtual embryo um and so let's see I'm going to     
53:51     
go through some of these it's basically just talking about different groups in doing things we have the computational     
53:58     
embryo models uh this uh researcher Burna soan computational embryo models     
54:04     
will be excellent companions to human embryol like models and natural embryos     
54:10     
themselves she and her lab developed stem cell based models called human extra embryoids so this is where we get     
54:17     
these computational models which I showed in the last image but we also have these other types of embryoid     
54:23     
models and so we've talked about some of embryoid models in humans and in primates and so the embryoid models are     
54:30     
these organoid type models where you have stem cells and you do directed     
54:35     
differentiation on the cells and you develop different tissues and so having     
54:40     
these models you know they're models in the sense that they're not actual embryos but they're embryol like models     
54:48     
uh and in Morgan knows that I'm talking about from Saturday when we talked about some of these embryol like or stemlike     
54:54     
cells uh so you know they're they have different gene expression profiles than     
55:00     
your typical embryo or stem cells but they they behave like them and they have     
55:06     
very similar phenotypes and so you know we can use them as     
55:11     
models uh so yeah this is one example so you can basically use computational     
55:17     
models or you can use embryo like models to build and test theories that 28 new     
55:22     
research questions and make experiment design easier and so it's very hard to     
55:27     
do like you know human embryo models or study development um in you know in sort     
55:34     
of a setting where you can take an embryo and put it in a dish um you know     
55:40     
it's you can use you can do embryo models uh much easier but still you know     
55:45     
computational models are probably even easier than that so you know we have to kind of think     
55:52     
about how these kinds of models fit in in uh computational embryos are you know     
55:59     
good for a lot of things not just looking at uh you know different things     
56:04     
like forces or uh cell migration but you can also do a lot of things with     
56:10     
computational embryos like for example you can use it to assess how drugs affect embryos you can look at model     
56:17     
changes that could lead to congenital diseases or increase disease susceptibility you can also help     
56:24     
understand development in its different guises uh and also use them as teaching tools and you can do a lot of these     
56:31     
things with the embryoid models as well that's why we want to build embryoid models because we want to understand     
56:37     
cells in sort of the context of uh a multi-tissue     
56:42     
embryo so and so these models are getting more complex all the time oops sorry I'll     
56:51     
just so these cells are getting more or these models are getting more complex all the time you're getting uh you have     
56:58     
these integrated models which include extraembryonic tissue uh then there are less complex     
57:04     
non-integrated embryo models that lack this type of extra embryonic tissue so you can build them a different levels of     
57:11     
complexity and build them for different purposes you don't need to build a full embryo model right off the bat you can     
57:18     
study different questions and include things as needed so this uh     
57:25     
let's see if there's anything else in here that I wanted to cover so this is basically one of the     
57:32     
things I mentioned earlier about um sort of soap bubbles and soap films and you     
57:38     
know you can use that as sort of an analogy of the cells moving around in the embryo dividing in different ways     
57:45     
along different axes um so they talk about here embryos are not bubbles but     
57:51     
surface tension shapes the geometry both embryos and Bubbles so there you can actually use bubbles     
57:57     
sort of like this meta model to describe the physics of embryos so you know     
58:02     
bubbles are these spherical things that have to be packed into a larger film The     
58:10     
you know the edges kind of bump against one another so they have this sort of tiling that they achieve this optimal     
58:17     
tiling where they kind of fit in between one another and you can see there's actually a part of an area mathematical     
58:25     
mathematics called soap bubble Theory where they kind of use soap bubbles as a model for um sort of spatial     
58:33     
optimization and and so forth so Bubbles and embryos have a lot of     
58:39     
similarities so we can use some of those sort of insights but you know we don't want to think that one is equivalent to     
58:44     
the other and so they kind of talk about     
58:51     
here where you can integrate AI into these type YP of computational     
58:57     
embryos um so they talk about this platform here     
59:02     
from this group one of the groups that they're talking about they're talking about integrating different types of     
59:07     
data into the model so you can look at things like tension inference so this is     
59:14     
um I guess this is the turlier lab where they they're looking at these four dimensional maps of     
59:20     
tension um and you know you can characterize selling tissue mechanics theoretically and     
59:27     
experimentally um you can also in uh you can extract fluorescence from these     
59:33     
models so you can get a sense of gene expression you can also integrate spatially resolved single cell RNA seek     
59:41     
data so this is all you know kind of part you can do these different types of things the platform that integrates AI     
59:48     
trained on experimental data biophysical models and experimental data will be released soon so they're releasing     
59:54     
reling a model an aiid driven model on some of this as well one module handles     
1:00:00     
forward modeling for simulating with modeling parameters another is for inverse modeling and uses parameters     
1:00:07     
such as data extracted from micrographs so this is where you have the forward modeling which actually     
1:00:13     
makes predictions and the inverse modeling which sort of gets data from a     
1:00:19     
source and is you know sort of things that you can infer things with so you     
1:00:25     
have prediction and inference with these different types of forward and backward modeling     
1:00:30     
techniques um and then you know they kind of talk about extracting cellular     
1:00:36     
geometry from micrographs um there are different types of challenges that you have here uh but     
1:00:43     
you know that's that's part of the game I guess um     
1:00:50     
yeah so I wanted to go on to the other two papers in this group     
1:00:55     
um so this first one is the this is from the turlier group this is on embryo     
1:01:00     
mechanics and the work that they're doing there so they have embryo mechanics     
1:01:06     
cartography inference of 3D force atlases from fluoresence     
1:01:11     
microscopy this is uh kind of talking about how the sort of the methods Behind     
1:01:17     
These maps that they're building so I'll just read through the abstract and then I move to the other paper     
1:01:25     
um so this first the abstract reads tissue morphogenesis results from a     
1:01:31     
tight interplay between gene expression biochemical signaling and mechanics Al     
1:01:37     
the sequencing methods allow the generation of cell resolve spatio temporal maps of gene expression creting     
1:01:43     
similar maps of cell mechanics and threedimensional developing tissues has remained a real challenge so it's you     
1:01:50     
know kind of getting this sequencing data you can get spatio temp Al uh     
1:01:56     
distinctions and you know it's we can do this for single cells but creating     
1:02:01     
similar maps of cell mechanics is harder uh I would argue you know a lot of the     
1:02:06     
nextg sequencing maps are not particularly insightful so that may be     
1:02:12     
the the diff the distance here but um you know exploiting the foam like     
1:02:18     
Arrangements of cells so this is where they compare the arrangement of cells to a foam     
1:02:25     
we propose a robust endtoend computational method called FBO so this     
1:02:30     
is a foam embryo like a foam like embryo to infer spatiotemporal atlases of     
1:02:37     
cellular forces from fluorescent microsi images of cell membranes so they're     
1:02:42     
segmenting the membranes and they're looking at interactions between the     
1:02:48     
membranes our method generates precise 3D meshes of cells geometry and     
1:02:53     
successfully uh successively predicts relative cell surface tensions and pressures we     
1:02:59     
validate it with 3D foam simulations study its noise sensitivity improve its     
1:03:05     
biological relevance in mouse a cdan which is the C squirt model and worm embos that's for three or for     
1:03:12     
celegans uh so yeah they have these I guess in the paper 3D Force inference     
1:03:18     
allows us to recover mechanical features identified previously but also predicts     
1:03:23     
new ones unveiling potential new insights and the spatio temporal regulation of cell mechanics in     
1:03:29     
developing embryos our code is freely available and paves the blade for unraveling the unknown mechanochemical     
1:03:37     
feedbacks that control embryo and tissue morphogenesis so if you look at this uh     
1:03:44     
this is the multimaterial mesh generation algorithm they walk through the algorithm they basically uh you know     
1:03:51     
try to extract the membranes so they're not really interested in the cell centroids or uh you know any of the     
1:03:58     
contents of the cell they're just interested in the edges of the cell or the membranes and then they use a Delani     
1:04:05     
tessellation for you know both the cells and the cell     
1:04:10     
environment these Watershed clustering on a dual voro diagram so they kind of     
1:04:16     
build a model of the cell the morphology and the edge edges of the cell and Define those as the membrane and then     
1:04:22     
they have this multimaterial mesh by a cluster border extraction so they extract out the membranes and so they're     
1:04:29     
extracting out the background they're extracting the membranes and the foreground and then they have this virtual model here which is are two     
1:04:36     
cells that have an interface with each other and then they Benchmark this     
1:04:45     
on uh some uh error and then angle     
1:04:50     
values so they actually characterize the different loads of     
1:04:56     
interaction um they're they're lucky that the actin in the cell actually is     
1:05:02     
on the exterior like it tends to go toward the edges of the cell yeah so um     
1:05:10     
this might be tracking where the actin lies in the cell rather than the cell     
1:05:16     
membrane which doesn't contribute as much as the actin does towards the     
1:05:22     
cell um oh yeah mechanical     
1:05:29     
um I don't know yeah I know what you mean it's like you have these different components of     
1:05:35     
the membrane and the edge of the cell or the body of the cell that are interacting and and giving it shape so     
1:05:42     
they're just extracting it from an image so they don't see the they're not accounting for the three dimensional     
1:05:48     
depth of the of the structure the edges of the cell or whatever     
1:05:55     
yeah well the cell mechanics is determined a lot by its Act of     
1:06:02     
mein right which is on the ex exterior like     
1:06:07     
it tends tends to go toward the exterior of the cell I don't know why yeah you     
1:06:14     
know why I don't know well maybe maybe that's where it's most     
1:06:20     
useful I don't well it doesn't really reason but yeah so I mean yeah so I guess they you     
1:06:28     
know their ideas is building these spherical models that sort of gauge where the interaction points are and     
1:06:35     
again you know it's not like we don't have everything in there and and this may be minimal useful or maybe more     
1:06:41     
useful not really sure but they're able to do that for the so this is just the segmentation step in any case they're     
1:06:48     
going to then simulate these interactions using different parameters so they're going to separate wor it out     
1:06:54     
like the actual uh cell structure with sort of these interaction parameters     
1:07:00     
that you know tell each cell you know you're interacting at a certain Force these are the forces that we expect so     
1:07:07     
there's a surface tension you can map across the cell or the embryo surface so this is a series of cells in an embryo     
1:07:15     
the surface tension is higher at towards the bottom of the embrio and L at the top at the dorsal uh section orus is the     
1:07:24     
ventral section and then there intermediate tensions in the middle uh and then pressure is it's hard to see     
1:07:32     
the colors but seems like their lighter patches of pressure and darker patches of pressure and then you of course you     
1:07:39     
have these other types of things here so this is actually in the aidian     
1:07:46     
embryo this is the 64 cell Aidan embryo so the Aidan embryo at this stage is     
1:07:52     
pretty symmetrical so you know it it sort of controls for asymmetries and shape but you see these cells that have     
1:08:01     
you know um this this sort of structure and you can measure the tension and the     
1:08:06     
pressure and then uh GH and I are down here so G is an exploded view of the     
1:08:13     
surface tension map so this is where we have an e the surface tension map in G     
1:08:19     
it's exploded so all the cells are kind of exploded out and you can see the sort of the cells floating in space so you     
1:08:27     
can see that um it this illustrates cell cell contact tensions within the embryo     
1:08:34     
so it shows like where these on these edges they have different tensions so in     
1:08:39     
this Edge there's a higher tension this other face of this cell so you can see     
1:08:44     
that there Junctions where there's a high surface tension versus not in h you have Force graph     
1:08:51     
representation of the mechanical Atlas so this is a force graph uh Network     
1:08:57     
where you have the forces mapped to these interaction so the nodes are the     
1:09:04     
cells and the edges of this network are the points of interaction so again you     
1:09:09     
can see that these edges these faces of the of individual cells have higher     
1:09:14     
surface tension this corresponds with this edge here in red you can see it so     
1:09:20     
you know they they edges that are red which means very high surface tension and their edges that are blue mean low     
1:09:26     
surface tension so this is almost like not not quite a Tegrity network but it gives us some information that might be     
1:09:33     
useful there to see how that those uh tensions are being     
1:09:39     
regulated any thoughts about that Susan um yeah it looks like a 10 segr     
1:09:45     
Network um I wouldn't have gone with surface tension on this but     
1:09:52     
um yeah I don't know it's I'll read certainly read this yeah     
1:10:01     
yeah and then I here is the 3D stress igen value representation corresponding     
1:10:07     
to stress tensors calculated per cell so this is where you have these stress tensors in the um in the same structure     
1:10:16     
so this is uh you know this they kind of go through for aidian and for     
1:10:22     
seans um and see if they have like a comparison um I was just say surface     
1:10:30     
tension is important yeah yeah sure yeah so they actually show that     
1:10:37     
topology changes as a measure of inference robustness uh this is showing these     
1:10:43     
unstable Junctions between cells so you know if you think about like a soap bubble they sort of smoosh together and     
1:10:51     
they have these interaction points and then as the Soaps burst they kind of come together so you could have like in     
1:10:58     
this case cell death where the cell dies and the other cells kind of take up that space and come together sometimes you     
1:11:06     
have unstable Junctions so it could be like a cell dying or it could be like cells are trying to move against one     
1:11:12     
another they're trying to make those active movements and in that case you get these displacements along the     
1:11:18     
boundary so the topology can change when you have unstable Junctions so at the     
1:11:23     
Junctions between cells in this case you have three cells that have a junction they form a junction they have these     
1:11:30     
components of Y and there are these forces in these different directions and     
1:11:35     
so when these are unstable you can get these shifts in topology so that's what     
1:11:41     
we see here um and then you can have that Tri     
1:11:46     
Junction um disappears yeah yeah Tri Junctions right yeah well they're it's     
1:11:54     
um a construct in a Cell in cells in tissue yeah it's separate     
1:12:02     
it's um yeah they're interesting did I send you any papers on     
1:12:08     
Tri Junction yeah you did we actually talked about it too yeah     
1:12:15     
okay and so that's and then this is of course in uh the Aidan cell here so this is     
1:12:23     
where you have something that's a little bit more sort of elongated like what you     
1:12:29     
would find in you know just kind of showing the polarity anterior     
1:12:35     
posterior uh medial lateral uh then in both anatomical Dimensions what some of     
1:12:43     
these forces look like okay and then there's some     
1:12:49     
references here I don't know how useful they are I don't know if what kind     
1:12:54     
of uh they have if they have their code released or what they're actually     
1:12:59     
working on with this they actually have some tools it looks like they're they     
1:13:04     
kind of mentioned that in the discussion what those tools are um but there are ways you can display these morphologies     
1:13:11     
and forces in multiple ways so you can build Force graphs and things like that so there are actually tools attached to     
1:13:17     
this um and so I don't I don't know where those are exactly but you know as     
1:13:23     
we can find out in the in the work here um and then finally this one here     
1:13:30     
this is the mechanochemical model recapitulating distinct vertebrate Gast gastation modes um this is where they     
1:13:38     
build a mechanochemical model and so this is a virtual embryo gastrulation is     
1:13:44     
a critical event in vertebrate morphogenesis driven by cellular processes and characterized by     
1:13:51     
coordinated multicellular movements that that form robust morphological structures how these structures emerge     
1:13:57     
in a developing organism and vary across vertebrates remains unclear so in the in     
1:14:02     
the last part of that Target paper they talked about will there be one single     
1:14:07     
embryo and answer I would think would be no because embryos are extremely diverse     
1:14:13     
we have different types of embryos different types of modes of development across them we know that like     
1:14:20     
development has like unified modes like a lot of oros have gastrulation and     
1:14:25     
other types of you know uh stages and and types of characterized movements but     
1:14:32     
they vary in how they're executed so you know I don't know if we could have a single embryo model so you know we we     
1:14:38     
want to know though you know at the same time we want to know that we can simulate these across different types of     
1:14:45     
morphological structures so you know to be able to have this kind of you know     
1:14:50     
maybe a module that you can put into different embryo models to say this is like gastation it be you know really     
1:14:58     
nice how these structures emerge in a developing organism and VAR across vertebrates remains unclear inspired by     
1:15:05     
experiments on the chick so this is in chick embryos we derive a theoretical framework that couples actomyosin     
1:15:12     
activity to tissue flow and provides a basis for the Dynamics of gastrulation     
1:15:18     
morphologies our model predicts the onset of development observed experimental pattern of wild type and     
1:15:24     
perturbations of chick gastr relation so this is where we have the wild type and     
1:15:30     
different perturbations of chick gastrulation so it's not just the standard gastrulation mode uh as a     
1:15:37     
spontaneous instability of a uniform state so a lot of you know a lot of     
1:15:43     
times you'll have like mutations in the process or you'll have you know in some     
1:15:49     
not in chick embryo but you'll have to find mutants where you can you know see these effects of the     
1:15:55     
perturbations of this wild type genotype and so this is they characterize this as     
1:16:01     
spontaneous instability so it's interesting during the initial conditions and a parameter     
1:16:06     
in our model allows us to recapitulate the phase space of gastr relation morphologies seen across vertebrates so     
1:16:13     
they're able to take this phase space they're able to build a phase space for     
1:16:18     
all different vertebrate uh species and for this simple gastation     
1:16:24     
orology and this is consistent with experimental observations and there's an accompany accompany paper here I'm not     
1:16:31     
going to talk about but they did some experimental observations and they match altogether this suggests that the early     
1:16:37     
embryonic self organization follows from a minimal predictive theory of acto     
1:16:43     
mechanic mechanosensitive flows so this is sort of this active matter type     
1:16:49     
approach and so they talk about gast relation they talk about this process so     
1:16:57     
gastation is this essential highly conserved process and the development of all vertebrate embryos with the chick     
1:17:04     
being an extensively studied model uh during gastrulation the chick transforms of layer of epithelial cells one a     
1:17:11     
layered structure of three major embryonic tissues at this moment at the moment of     
1:17:16     
eggling the chick embryo contains around 600,000 cells organized in a central     
1:17:23     
Centric epiblast that will give rise to the embryo proper surrounded by a ring of extra Ematic tissue so this is like     
1:17:31     
the when you crack an egg and you have the yolk and everything um and then     
1:17:37     
during the first few hours of development signals from the extraembryonic tissues the epiblast and     
1:17:42     
the developing hypoblast indu Mees uh mesendoderm pre precursor cells located     
1:17:49     
in a CLE shaped domain at the posterior end of the EP epip the     
1:17:54     
epiblast uh so this just talks about some of these uh you know the Dynamics     
1:17:59     
here we have these directed cell interpolations that result in a contraction of the mesendoderm tissue     
1:18:06     
towards the central midline followed by an extension an anterior direction forming the Primitive streak which is     
1:18:12     
going to be important for later structure um and so there all these     
1:18:18     
processes going on and but the important thing to remember is that we have these four generating processes that drive     
1:18:25     
tissue flows and so there are three that they Define the first is the outward migration of the     
1:18:30     
cells uh two is the active intercalation of the mesendoderm precursors generating     
1:18:37     
forces and then three is the ingression of mesendoderm cells in the streak which attract cells towards the streak and     
1:18:43     
dags out the plane motion that's basically like all these different flows uh are driving this     
1:18:50     
process and so the variation of tissue flows morphology seen during gastrulation AC cross vertebrates and     
1:18:56     
they you know B all the cross vertebrates so we they're talking about fishes amphibians and reptiles to     
1:19:04     
amniotes and these are due to changes in embryo geometry and the relative     
1:19:09     
contribution of these three Force generating processes so that's what we have and     
1:19:15     
so this is a a sort of an image of what we have here this is phosph myosin     
1:19:22     
content for this these different structures we have the CLE structure we have this hh2 phase and then we have the     
1:19:30     
Primitive streak that forms from that so it's kind of moving inward along the     
1:19:35     
edges there's pushing outward in the middle and we're going from the sickle     
1:19:40     
to this intermediate structure to this primitive streak which is reinforced laterally inward so this is where we get     
1:19:48     
these these flows and you can see the force Maps here and that's that's that     
1:19:54     
whole process so act that's nice that that's very nice I haven't seen uh     
1:20:00     
someone described the formation of the Primitive Street before like this very     
1:20:06     
nice yeah and then yeah so this is all driven by active active myosin     
1:20:12     
cables and so you can see that they have this set up Susan says very nice yeah     
1:20:21     
um yeah so that that's that's in Chi embryo but they're going to then generalize that outward to other models     
1:20:29     
and so our model of the the sheet like embryo is assumed to be a compressible fluid to accommodate cell ingression     
1:20:37     
this implies that we need a continuity law that accounts for flow Divergence or convergence so they use this linear     
1:20:43     
model and this is kind of talking about some of the Dynamics of the linear model     
1:20:49     
here uh fluid bulk viscosity is characterized by     
1:20:54     
C1 well p 0 characterizes the effect of an active cell ingression and can be     
1:21:00     
thought of as the ratio between the isotropic and anisotropic active stress     
1:21:06     
so they have this more in the methods of basically they're they're modeling this     
1:21:11     
then we may write the equations for local momentum balance of the planer velocity field which is V XY time along     
1:21:19     
with equations for the orientational and intensity dynamics of iosin cables so     
1:21:25     
that's these two uh here uh variations on V this velocity field and here the     
1:21:33     
equations so this is what they're working with so they have you know they've laid this out in terms of a     
1:21:38     
mathematical model and this is uh where they kind of go through gastr relation the Dynamics     
1:21:46     
of Gast relation so they have the sickle and then it goes to this primitive     
1:21:51     
streak so we have they've characterized these Dynamics     
1:21:57     
um so this is uh let's see we have D let's look at D here uh D is modelbased     
1:22:06     
DOA at t0 and the attractor at TF so this is t0 where we have a sickle TF is     
1:22:13     
where you have the Primitive streak so they're modeling between those two uh movie two which is the movie of     
1:22:20     
this which is in the um uh the supplemental materials which is     
1:22:26     
the time evolution of the relevant model based ulian and leian Fields uh oian leian Fields same as D     
1:22:34     
from the experimental velocity that's e so we're looking at here uh model DOA     
1:22:40     
model attractor this is experimental DOA experimental attractor so basically we     
1:22:46     
have a model which is the mathematical model and then we have experimental observations so they match up pretty     
1:22:52     
well um okay um so this yeah this kind of     
1:23:00     
goes through they're actually they're using     
1:23:05     
um they're using ft models which are finite time lipanov exponents so this is     
1:23:13     
uh where you know you model from one time point to another you look at the trajectory between those time points and     
1:23:20     
you do some leian stuff to it and uh it's good     
1:23:25     
stuff um so yeah this just more about the developmental perations so we can     
1:23:32     
actually perturb this in different ways uh we can look and look at model and experiment we can actually match those     
1:23:38     
two together this is now the evolutionary transition in these patterns so we have again we we modeled     
1:23:44     
this in uh single in the chick embryo but we can exp extend this out to other     
1:23:50     
uh vertebrates so we we have fishes frogs lizard and amphibian and a chick     
1:23:56     
basically characterizes across these different um taxa and so you have an initial condition which     
1:24:04     
is the sickle and then you have this ancestral condition which is actually this I guess this nonl it's the     
1:24:11     
spherical structure so basically they give this as the initial condition     
1:24:18     
for our model this is the ancestral condition so this is at the sort of     
1:24:23     
Basil part of the philogyny and then as you sort of get uh when you have     
1:24:30     
Evolution you have different types of initial conditions so this ancestral condition you see this in lizards and     
1:24:37     
amphibians and fish this the spherical thing with the sickle you see that as a     
1:24:43     
the drive condition in a lizards and amphibians and shiks and so that's what     
1:24:49     
they're looking at here and then so they're looking at the different ways in which this primitive Street forms here     
1:24:55     
and then they're looking at the cell ingression process lizards and amphibians and ficient so they're making     
1:25:01     
that connection between two parts of the vertebrate philogyny and then they're     
1:25:06     
looking at the experimental forces and then they're looking at this     
1:25:11     
computational model as an attractor and that's what they get so they have this     
1:25:17     
they're looking at changes in the critical morphogenetic parameter P0 this is the isotropic to an isotropic act     
1:25:24     
stress ratio and regulates the amount of active cell ingression in EMT and then     
1:25:29     
the initial conditions mt0 which is the extent of the Mis mesendoderm precursor     
1:25:35     
territory and this allows us to recapitulate these these events in evolution largely this event between     
1:25:43     
going from this ancestral condition to the reduced mendit so the reduced mendor     
1:25:49     
is the crescent and then it's you know the has these same Dynamics though which     
1:25:54     
is the point yeah well there's there's the where um Richard Gordon's waves oh yeah     
1:26:04     
that's the okay so that yeah and so uh what do     
1:26:12     
we have in the chat yeah drop links to the papers too that would be great I'll put the papers in the uh evor slack but     
1:26:20     
I'll also send them out uh in email form to Susan because I know she's not on     
1:26:25     
slack that's oh well I'm on slack but I don't know how to use it oh well too     
1:26:32     
much so yeah well you know we'll just send an email um yeah so that's no     
1:26:39     
that's good so that's um that's what the and then that uh the first paper I think     
1:26:45     
there's some other papers in that issue of nature methods that are interesting     
1:26:51     
as well so I didn't cover those those but and then the last paper we talked     
1:26:56     
about they had an experimental companion which I also didn't talk about     
1:27:01     
but okay so there's more yeah okay I can maybe have a look I just     
1:27:10     
I need to get my stuff just finalized whatever I've done I've     
1:27:18     
done yeah that that's about it like okay well great well thanks for     
1:27:25     
attending yeah attending the meeting uh I'm here still yeah     
1:27:31     
know any other comments or questions before we go um no you don't know how to run     
1:27:40     
console so not really I haven't met anybody except console employes that     
1:27:46     
know how to run console so um     
1:27:53     
that's just yeah that's a problem yeah all right all right have a good week     
1:28:01     
good luck yeah than bye     
