## TRANSCRIPT

0:00     
morning hello Hi how are you I'm pretty good I'm still doing     
0:06     
graphs but I'm I see the end     
0:12     
good yeah I have to do stress strain graphs     
0:17     
as well as the uh distance Force Ones the force     
0:23     
distance H I don't know okay yeah so     
0:31     
hopefully uh this week I need to hand the whole thing in so oh okay well     
0:37     
that's good good luck thanks     
0:43     
yeah I probably shred something about it so     
0:52     
anyway there's Morgan hello Morgan I hope I don't have any problems again     
1:00     
this week but I will turn off my video after a bit because I don't know     
1:06     
bandwidth or something oh yeah yeah     
1:12     
okay so yeah welcome to the meeting um so let's get started in I think two     
1:20     
weeks I can't remember what the exact date is I think it's a 17 we're going to have the annual meeting for dvo or for     
1:28     
open one and and that's where we usually get together and talk about all the different sub projects and so I have     
1:36     
been working on the dorm 2024 update and so it's actually up on the YouTube     
1:42     
channel right now if you go there and look um and so we have dorm update um     
1:49     
and I kind of go over the 10th uh anniversary of dorm and I talk about     
1:55     
that a little bit I have a separate video on that on the YouTube channel     
2:00     
uh then I also talk about our Google summer of code contributions uh talk a little bit about     
2:06     
picky's project uh talk about the lectures we did this year um talk about the papers     
2:13     
we did this year so we did molecular basis of differentiation wave activity     
2:18     
and EM Genesis um going through that this paper     
2:25     
hypergraphs and then this is Mah's work on spatio temporal decompos     
2:33     
position animal um so it should be up on the YouTube channel I'm going to talk to     
2:40     
the openworm foundation about this uh in a couple weeks so though they're always     
2:45     
uh happy to see our work and what we've been able to do in the year in the past     
2:51     
year so actually I'm going to start with this     
2:57     
article uh from azim off press so this is a really good uh Outlet This is Nico     
3:04     
mccardy who I think runs Asim off press um and this is an essay on the Weavers     
3:09     
for biological progress so you have this uh this is just an article on sort of     
3:17     
how AI is intersecting with uh the idea of you know kind of making progress in     
3:24     
biology uh talking about you know kind of comparing it to the human genome     
3:30     
project and kind of considering how you know maybe if you apply AI to biology     
3:38     
there going to be a number of shortcomings in that area so this is a     
3:43     
this article starts talking about uh Dario emod from anthropic he's the CEO     
3:50     
and he recently published this essay called Machines of Loving race which is     
3:55     
a vision for how AI could radically transform neuroscience economics diplomacy and the meaning of work this     
4:03     
is a pretty ambitious U essay amod also imagines ways AI could     
4:10     
accelerate biological research and yield miraculous cures in the 21st century everything from the prevention and     
4:16     
treatment of mway all infectious and inherited diseases to the elimination of most     
4:21     
cancers so on Saturday we talked about this um in depth with respect to the     
4:27     
Human Genome Project how you know the Human Genome Project kind of promised the same thing like 25 years ago and of     
4:34     
course you know it wasn't probably a fair uh set of promises uh from the     
4:40     
standpoint of what could be accomplished on a time scale that everyone at the     
4:45     
time would be alive to see but you know we're kind of repeating that with AI so     
4:51     
just to frame that in in a framework that you know we can kind of appreciate biology is probably the area     
4:58     
where scientific progress has the greatest potential to directly and unambiguously improve the quality of     
5:04     
human life M mod rights my basic predic is enabled biology and medicine will     
5:10     
allow us to compress the progress that human biologists would have achieved over the next 50 to 100 years into 5 to     
5:18     
10 years so this is like a tenfold increase in the speed of Discovery or of     
5:24     
progress however you want to Define it so you know this is something that     
5:30     
uh you know it's pretty ambitious just to say the least and so this is of     
5:35     
course then in this article they're going to talk about maybe how they     
5:41     
bottlenecks in this process of discovery that probably won't yield the promised     
5:47     
acceleration of progress so it's an inspiring vision but you have to think about the bottlenecks and the road map     
5:55     
ways to solve them so in other words you know they're bottlenecks to making progress     
6:00     
in the Human Genome Project there were bottlenecks in terms of you know bioinformatics in terms of technical     
6:07     
barriers and those had to be overcome at the same time that that technology was being     
6:14     
implemented so you know this at azimov press they published different types of     
6:19     
road maps so they had an article on where's the synthetic blood which was kind of like talking about how easy or     
6:27     
hard it is to get synthetic blood and to grow it in the lab or you know to have     
6:33     
other types of road maps on different topics in biology so you know the the sort of the     
6:40     
the well sort of the Assumption here is that the powerful AI should be smarter     
6:46     
than a Nobel Prize winner but even if you had that kind of super intelligence could you overcome these     
6:52     
bottlenecks so you know we have a lot of bottlenecks in biological progress but those are biophysical rather than     
6:59     
computational so they talk about that here and you know when I talk about biophysical     
7:05     
bottlenecks I mean they're bottom next to understanding different processes so     
7:11     
this holds true for say like Imaging Technologies or sequencing Technologies we have a way to collect data now we     
7:18     
have a better way to collect data but we still don't understand the process that we're working and as a as a consequence     
7:25     
you've seen this with respect to nextg sequencing we generated massive amounts     
7:30     
of data but it kind of sits on a shelf because we can't interpret it properly     
7:36     
and so that's probably not going to speed us up uh even if we have a super     
7:42     
intelligence we might be able to find safe relationships but it's hard to actually verify them and then     
7:48     
incorporate it into actual progress so this is you know some of these bottlenecks are really going to be hard     
7:55     
to overcome maybe with different Technologies that aren't AI but we'll     
8:03     
see so you know again it's kind of hard to know where Solutions will emerge but     
8:10     
you know we don't want to overpromise and underd deliver so that's why researchers hoping to accelerate biology     
8:17     
at large should strive to build platform tools that can apply in multiple contexts So Adam marblestone has written     
8:24     
an article on his substack about building these platform tools     
8:30     
that are di that are versatile rather than narrow solutions to short-term problems and so this is another thing     
8:37     
where you know you can develop a technology and apply it but you know having a platform is better so this is     
8:45     
this Adam marbl Stone article if I can get access to it okay this is called     
8:51     
biotechn technological knowledge hacking this is from the MIT biomaker Workshop     
8:56     
in 2016 Adam marblestone is been associated with openworm in the past so     
9:03     
he's one of uh uh Steven lon's uh colleagues or you know he knows him     
9:09     
pretty well I guess so this is uh you know they have this sort of connection to MIT there other people who were kind     
9:17     
of involved with this such as Ed Boyen and George church so this is kind     
9:24     
of lengthy talk I'm not going to go through it but basically it talks about how     
9:30     
uh bioengineering is not like normal engineering you can't necessarily rely     
9:35     
on analytical calculations the underlying systems are poorly characterized with respect to you     
9:42     
know being able to engineer it so we can engineer say you know an automobile or a     
9:49     
bike but it's hard to engineer a biological system in the same way we have high failure rates so in     
9:56     
engineering we try to have a safety Factor we try to lower the failure rate but in biology you really can't do that     
10:03     
as well uh there's a widely distributed knowledge base so it's hard to know what     
10:08     
we know so you know we have to go into the literature and just find you know the answers that are spread widely     
10:15     
across that literature so that's that's kind of what they're getting at with     
10:22     
this so you know we can use Ai and it can     
10:27     
advance uh knowledge but it's you know it's one of these things we have to manage it's not just something you can     
10:32     
throw at the problem and immediately get a return on your investment so the balance between AI     
10:39     
advances on the one hand and wet lab Innovations on the other is also a bit like the chicken and egg problem yes AI     
10:46     
will accelerate biological progress but first we must make it easier and faster to run experiments while creating better     
10:53     
methods to study biology in a more holistic and less reductionist way so     
10:59     
solving the ladder challenges will oddly enough require both machine learning and wet lab     
11:05     
Innovations So biof Physical bottlenecks are basically you know bottlenecks uh     
11:11     
that we can characterize through this quest to build a virtual cell so there are a bunch of groups that     
11:17     
are trying to do this trying to build working models that can accurately Sim     
11:23     
simulate living organisms and by a virtual cell it's just simulating that one cell a eukariotic cell cell but     
11:30     
maybe even a a procaryotic cell um a virtual cell would enable researchers to     
11:36     
perform biology research on computers rather than actual organisms and you know removing     
11:43     
challenges that arise from the working of the world of atoms so we can't necessarily build a     
11:49     
virtual cell just through simulation we need to understand much more about how cells work and that takes us back to the     
11:57     
realm of experimentation and wet lab research so you can't necessarily     
12:03     
predict the outcome of even simple biological questions such as if I take this Gene from an arabidopsis plant and     
12:10     
insert it into ecoli will a cell Express the Gene and make it functional     
12:16     
protein uh no one really knows this without trying it out in the laboratory and even if we do get like a     
12:23     
meta analysis of studies you know we still have to kind of have all we don't     
12:29     
know what parts we need to abstract away from that system to make the model work and silal so there are a lot of     
12:35     
challenges to this uh kind of approach so in order for 50 to 100 Years     
12:43     
of biological progress to be condensed into 5 to 10 years of work we will need to get much better at running     
12:49     
experiments quickly and also collecting higher quality data sets this essay     
12:54     
focuses on how we might do both specifically for the cell here my focus     
12:59     
in this essay is narrow although my focus in this essay is Arrow I don't discuss bottlenecks and clinical trials     
13:06     
human disease or animal testing I hope others will take on these challenges and similar     
13:11     
essays so that's uh that article um I know that sounds like something that     
13:17     
Morgan could speak to a little bit uh perhaps Susan because she's doing     
13:23     
simulations of or doing kind of engineering of cells in a roundabout     
13:30     
way so any thoughts on that I could certainly use some better     
13:38     
data yeah yeah for my simulations like it's missing like the the elasticity of     
13:48     
um the ACT filaments for instance that are bundled the bundled act in filaments     
13:54     
so they have other molecules included in that and okay okay so what's the     
14:00     
elasticity of this nobody knows that's my     
14:06     
conclusion they've done acting just straight acting but acting bundles like     
14:12     
there's several different molecules involved in different types of active bundles it's     
14:18     
like okay where's the data um not it's not doesn't     
14:25     
exist so that makes my life different     
14:31     
yeah and that gets me into what I'm doing now something called K Factor so     
14:37     
it's the um actual um area of the filament times its uh     
14:49     
elasticity divided by um the rods with     
14:54     
the same thing going on in the denominator and you get a k factor and     
15:01     
that will change a tties um elasticity well how it how it     
15:09     
moves Etc so it's stress strain curve so it would be good to know what     
15:18     
actually is even if I just had one of them even well I would like the K factor     
15:23     
of of the different types of cells and then     
15:29     
then I could do this research easier but there's no     
15:34     
data great yeah what is the diameter of of     
15:42     
the the link um Acton in a cell that     
15:48     
makes these strings that I'm trying to use in my my     
15:54     
tense right what are you talking about what you talk     
16:00     
about yeah it's something like that yeah I mean certainly can you can     
16:07     
sort of use stand-ins for a lot of these things and model them but then the question is how predictive are they or     
16:14     
how successfully can you predict something in the system     
16:20     
so yeah for instance a lot of cells are sensitive to     
16:26     
um the substrate they're on so how a cell is attached to a substrate     
16:34     
is important and it's also important what characteristics the substrate has     
16:41     
very important but I'm just starting to get at modeling     
16:47     
that and of course um my lovely program is     
16:55     
uh won't do everything that I want it to do cells seem to be attached in um points     
17:03     
on the surface of the substrate and you make a MH on the bottom of my pseudo     
17:10     
cell and um I have nodes and then you try to attach the nodes to the     
17:17     
substrate and it doesn't work so somehow this is not     
17:25     
um a proper thing to do if you building a     
17:31     
building which is what I'm up against I'm trying to use software that builds     
17:37     
buildings and um Machinery Machinery Parts okay     
17:45     
yeah but the two should mesh they they they need to come together somehow in     
17:53     
order to to study this right you have to put     
18:00     
proper engineering into it um proper physics and then engineering into your     
18:07     
model or people just aren't going to believe you they're going to say oh well     
18:12     
it's just a theory it's it's just the theory it's not     
18:17     
engineering oh yeah engineering has to work in the real world right     
18:27     
yeah so yeah I'm trying to explain cells in engineering     
18:34     
terms it's giving me a headache yeah yeah I think a lot of approaches to     
18:41     
bioengineering are kind of like that I mean if you're talking about even if you're talking about like a gene circuit     
18:48     
you know they usually use these toy problems that are pretty simple and you know you can do very simple things with     
18:55     
them but then when you try to go beyond that it's a little bit difficult so you     
19:00     
know there's this tendency to and this is true in biology too or you know in in     
19:07     
computational biology there's this tendency to cling to the easier systems or easier applications you can do like a     
19:14     
perfect concept but then does it go beyond that you know that's the kind of     
19:19     
the problem as well I biology is complicated yeah     
19:34     
all right uh what don't we move on then actually this is a paper Susan might enjoy this is forecasting avalanches and     
19:42     
branched actomyosin networks with network science and machine learning so we're kind of following up on this uh     
19:50     
discussion about biological engineering and using different tools to ex you know     
19:56     
um sort of accelerate that process and so this is from the Journal of physical chemistry     
20:03     
B and so this is uh I I'm not familiar with these     
20:08     
authors but they're going to tell us about uh forecasting avalanches in these     
20:14     
active wayos and networks so this is a graphical abstract here we have a     
20:20     
physical Network we transform that into a mathematical     
20:25     
graph and we try to predict these actomyosin Avalanches which uh before reading the     
20:32     
abstract I'm going to assume is some large scale reconfiguration of the physical Network     
20:39     
and you're going to see a large SC scale reconfiguration in the corresponding mathematical graph so you can see that     
20:46     
in the graph you have the structure here and then it transitions to the structure     
20:51     
here so there's something analogous to A first order phase transition in that     
20:57     
graph and so that's what they're trying to predict and you know in these atoosa     
21:05     
networks which remember are these physical networks of actomyosin     
21:10     
so the abstract reads we explored the dynamic and structural effects of actin     
21:16     
related proteins 23 which is ARP 23 as the shorthand on actomyosin     
21:24     
networks using mechanochemical simulations of active matter Networks     
21:29     
and so they were doing these mechanochemical simulations of these networks uh they I guess active matter     
21:37     
means that they're you know they have modeled all the aspects of the Acom meos     
21:42     
and and put them in these kind of interact at these interaction points and     
21:48     
we're looking at the interactions and so you can abstract that to a mathematical graph on the Nano scale the ARP 23     
21:56     
complex Alters topology actomyosin by nucleating a daughter     
22:02     
filament at an angle with respect to a mother filament that a some cellular     
22:07     
scale they orchestrate the formation of the branched actomyosin Network which is here in the physical     
22:13     
n using a corar grain approach we sought to understand how an actom myosa Network     
22:20     
temporally and spatially reorganizes Itself by varying the concentration of     
22:26     
R23 complexes so this is where you get this spatial reorganization of the     
22:32     
network and you get this concentration of R 2.3 comp or two3 complexes so this     
22:39     
is a bit like adding in like something like you know it's kind     
22:45     
of like a transition between ice and water how water transitions between a     
22:51     
solid and liquid so you get this uh sort     
22:56     
of you're adding in instead of adding in temperature adding in a concentration of these     
23:02     
complexes and you end up with this phase transition it's again like a jamming     
23:07     
transition as well where you have a bunch of particles that kind of go     
23:13     
through a pipe and if you get to a certain density the particles cease to     
23:18     
flow and it becomes a solid and so the flow is restricted but it's a a sudden     
23:24     
sort of process it doesn't happen gradually and so that's what they're trying to do they're trying to get these     
23:31     
spatial reorganizations by varying the concentration of these complexes driven by motor Dynamics the     
23:38     
network stalls at a high concentration of rp23 this is where we get this kind of     
23:44     
transition and contracts at a low R23 concentration so you get this remember     
23:52     
this network of course is responding to forces and stresses and strains     
23:59     
and it's it so it has to have some sort of like movement and so when you have a     
24:05     
high concentration of these molecules you have the stalling of the network whereas when you have this low     
24:13     
ARP 23 concentration you get contraction and so this these are two     
24:19     
sort of uh extreme states of the network at an intermediate ar23     
24:25     
concentration however the oios work is formed by Loosely connected clusters     
24:31     
that may collapse suddenly when driven by Motors so you get these kind of clusters here in this mathematical graph     
24:39     
on the left and I guess you can see them here where you have these clusters corresponding in the     
24:45     
network and so these Loosely connected clusters kind of sit there in the sort     
24:50     
of meta stable State and they collapse suddenly when driven by Motors so when     
24:55     
you actually drive the network functionally they can collapse suddenly this physical     
25:03     
phenomena is called an avalanche so this is where we get this rapid shift in the     
25:08     
topology of the network largely due to the marginal instability inherent to the     
25:13     
morphology of a branch damios Network when the R23 complex is present so that     
25:20     
means that you have these sort of clusters that form at this intermediate     
25:25     
concentration of R23 these uh clusters are metast stable yet     
25:34     
they're fragile and if you drive them by some sort of motor or some sort of set     
25:40     
of stresses and strains you can collapse that structure and you can get these     
25:45     
Avalanches or these massive reconfigurations of the network so you see this here so that's what they mean     
25:51     
by an avalanche it's this large scale reconfiguration of the network that can happen suddenly and due to some sort of     
25:59     
perturbation on the not and remember if we have a really high concentration of     
26:05     
R23 the the network ceases to be flexible and it just sits there in sort     
26:12     
of a mass and if we have very low concentrations we end up with this sort     
26:19     
of uh different state where the network contracts and it's also nonfunctional     
26:34     
okay while embracing the data science approaches we unveiled the higher order patterns in the branched acosa networks     
26:42     
and discovered a sudden change in the social topology of actomyosin so they're     
26:47     
talking about uh I don't know I guess they're referring to sort of the way     
26:52     
that this resembles a social network um and what they mean by that is that     
26:58     
social networks of course have been well studied in terms of network properties     
27:04     
uh mainly like small world networks where you have the structure that's     
27:09     
focused on key hubs or highly connected nodes and then a lot of nodes that are     
27:16     
not as highly connected so you get this hierarchical structure you get the small worldness that results in a very short     
27:23     
distance across the graph because you're going from The Edge one Edge the graph to a a hob to the     
27:30     
other edge of the graph pretty quickly and then has consequences for the structure of the     
27:35     
netor so that's what they're getting at with the social network topology of acto     
27:41     
myosa and so what they're referring to here in terms of the actom myosa network     
27:46     
and its Dynamics is that there's a new type of avalanche in addition to the two types of avalanches associated with a     
27:53     
sudden change in the size or shape of the whole actomyosin network as shown in     
27:59     
a previous investigation so they've been studying this for a while they have a previous paper where they show that     
28:06     
there's a sudden change in the size and shape of the whole acosa network which     
28:11     
you see here you can imagine that this network under go some stress or strain or some perturbation something that's     
28:18     
driving it and it can change very quickly the shape and the size of the     
28:24     
network but you also have this social network topology which is this hierarchical     
28:32     
structure that you get that's a sudden change that the network can transition     
28:38     
to our new finding promotes the importance of using network Theory and machine learning models to forecast     
28:45     
avalanches and atoosa networks so this is where they're actually trying to forecast these uh Avalanches instead of     
28:52     
just observing them so you can observe them by you know perturbing the network     
28:58     
by driving the network and letting it sort of move and you can see these changes in the acosa network um so but     
29:07     
you can also then predict maybe where they might occur so you could actually now that you know kind of how these     
29:14     
parameters are you can actually try to make predictions the mechanisms of ARP 23     
29:20     
complexes and shaping the architecture of branch decto myos and networks obtained in this paper will help us     
29:27     
better understand understand the emergent reorganization of the topology in dense atoosa networks that are     
29:35     
difficult to detect in experiments so that's what they're getting at they're actually able to do     
29:41     
these simulations get these graphs look at the topology look at the     
29:46     
behavior and perhaps predict some of these avalanches and of course as     
29:52     
Avalanches would be um useful in terms of understanding quick changes is in the     
29:58     
sort of the composition of um one of these     
30:03     
actomyosin uh networks in the in in the c or in the membrane of     
30:12     
the cell so uh they talk a little bit about muscle cells and how actin filaments and     
30:19     
myosins are organized into a striped circum ear and in non-muscle cells Act of meios networks tend to be     
30:25     
isotropic especially at the edge of cells such as the acting cortex so     
30:30     
they're different depending on the cell type you know they they're located in different places and they have different     
30:37     
functions they're basically contractile but you know you have in muscle cells     
30:42     
you have a very different mod of action than you have in the actin cortex of a of a of a cell     
30:48     
member um and and of course they also have different uh properties uh in terms of you know how     
30:56     
they're organized uh we hypothesize that this Nano structure of the actomyosin network     
31:03     
dictates the structure and dynamics of the entire system so they talked about the rp23     
31:09     
complex uh they talked about coupling this of meios and motors and how this uh controls     
31:18     
contractile tension generation in a Cell now they' different computational     
31:24     
models been used to explore actom myos and contractility and that's references s and 8 a few of these computational     
31:31     
models explore the effect of RP 2/3 complex on the Dynamics of the system so     
31:39     
people have done these they built computational models is looking at the contract contract uh contractility of     
31:46     
the actomyosin themselves but they've not really looked at the effects of the     
31:52     
earth two sary complex on the Dynamics of the system so you know you want to     
31:57     
look at how this network changes over time despite extensive experimental and     
32:05     
computational studies and that's uh references 9 through 12 only recently     
32:10     
has the computational work from our group shown that the presence of the ARP 23 complex causes sudden collapse     
32:17     
dynamics of marginally stable active myosa networks and these are of course     
32:23     
termed Avalanches as we talked about in the abstract     
32:31     
the Avalanche possibly underscores the phenomena of a cyto quake that's reference     
32:37     
14 uh a drastic structural change in the octom myosa network within a short     
32:43     
period of time so the cyto Quake is in reference 14 the biophysical     
32:51     
importance of this phenomena is appreciated since it's deeply related to the structural rearrangement of the the     
32:59     
cytoskeleton however how this phenomena connects to the nanoarchitecture of the branch dcto myosa Network orchestrated     
33:06     
by the ARB 23 complexes remain unclear so they look at the impact of     
33:13     
ARB 2/3 concentration to modulating avalanches in these Branch networks using for scen     
33:19     
simulations they use a software package of median which simulates the organization of actin filaments with     
33:27     
mechanochemical feedback from actin binding proteins in our simulations even though     
33:33     
we only changed the concentration of rp23 complexes while the turnover rates     
33:39     
remain the same in the chemical reactions the patterns of the temporarily evolving networks were     
33:44     
incredibly complex driven by a high concentration     
33:50     
of Motors several new Global features or orders emerged from a locally well-connected     
33:56     
Network and so uh they show this figure figure one this     
34:03     
is the median workflow this is their simulation environment uh uh median consists of     
34:10     
four main steps and its mechanochemical Loop is described in this     
34:17     
figure um the mechanical model is it showing things in     
34:23     
tension okay yeah well I I would like to see it     
34:29     
show um the system intention okay because that's cells are     
34:35     
intention yeah yeah basically all the time right well occasionally not but     
34:45     
um and if they're in tension of course then they're a tensegrity and sometimes     
34:50     
a tensegrity will all of a sudden change shape right so it's kind of a fit     
35:00     
between sort of a Avalanche or a catch Bond or change your shape I don't know     
35:07     
there's some sort of a might be some sort of a fit there um I know my 10     
35:14     
secrity models have some what they call soft modes where all of a sudden it it     
35:22     
collapses and I've I've got graphs to prove that a couple of my models     
35:28     
have that sort of behavior it'll all of a sudden     
35:34     
just there's a sharp change in in the stress strain     
35:41     
graph so it's all kind of interesting yeah um and I ran across a a paper that     
35:50     
uh where they made a Tegrity and then they said um it works really well for     
35:56     
the first part of it but then it does this and I'm going well you built the 10 segr what do you expect     
36:04     
and I maybe send you the     
36:09     
paper they tried but it's it's not good enough they so I wasn't going to include     
36:18     
it in my thesis references even though it's it's or maybe I should just to say     
36:25     
well it's a t secrity so you can observe that this happens to it partway along     
36:32     
Its Behavior stress strain curve     
36:38     
behavior um as you found about just because I think he might be     
36:46     
interested yeah I'll send it to you okay or maybe     
36:51     
I'll I'll look for it right now too yeah yeah     
36:58     
so getting back to the paper we have this uh median workflow median stands     
37:03     
for mechanochemical dynamics of active networks and this was developed by the     
37:09     
pooyan group and they have citations here uh so this is basically to include     
37:15     
the mechanic chemical feedback of the active networks and so this is kind of if they     
37:23     
show okay well that's not what I want     
37:28     
okay so this is the Actos and uh Dynamics here this is the workflow of     
37:34     
Medan so it consists of these four steps the first step is a chemical model uh     
37:40     
where you have an initiation of the chemical model where the system evolves     
37:45     
the actomyosin network with stochastic chemical reaction to Fusion so we have     
37:50     
this chemical reaction diff Fusion model it's instantiated then we have we update the     
37:57     
mechanic chemical configurations in two and that's where the chemical reactions     
38:02     
to form the network locally followed by the formation of a new chem uh     
38:07     
mechanical configuration so we have this chemical model we have this update of the     
38:13     
mechanochemical configurations imagine that this is going through a loop that you know we     
38:19     
instantiate the chemical model at every iteration then we update the mechanical     
38:26     
configurations and three is a a mechanical model where we initiate the     
38:31     
mechanical model and we so we first initiated the chemical model in one and now we     
38:37     
initiate the mechanical model in three so this is where the total energy of the new mechanical configuration is     
38:44     
minimized with the conjugate gradient method by reaching a new equilibrium and then four is update     
38:52     
chemical reaction rates where the chemical reaction rates are mechanic chemical updated at the new equilibrium     
38:59     
state so we cycle through these from 1 to four and then back through 1 to four back through 1 to four so we're always     
39:06     
initiating a new chemical model or a novel chemical model at every iteration     
39:12     
we update those mechanical configurations then we uh update the     
39:17     
mechanical model we then update the chemical reaction rates and then go back to the     
39:23     
chemical so that's how they kind of run their simulation uh they talk about simulation     
39:31     
parameters uh they talk more about this method and then this is sort of the output of the Medan simulation leder     
39:38     
these networks where you have these actomyosin filaments that have contact points where they have interactions and     
39:45     
that's how you can take this physical Network and turn it into a mathematical so this is uh these are     
39:52     
typical snapshots of median simulations for the unbranched actom iOS Works     
39:57     
without herb 23 complexes and uh okay so that's a this     
40:05     
one here on the left this doesn't have the her 2/3 complexes this is just an     
40:11     
unbranched acto myosa Network where there's no actual branching at the at     
40:17     
the contact points so you just see it's like a collection of acos and filaments     
40:23     
kind of in this um in this box and then in B we have these uh branched atoosa     
40:31     
networks with K 23 complexes which allow it to form branches and you have this     
40:38     
network with the R23 complex so if you have too many of them remember we said     
40:44     
that it would freeze up and if we have too few it would uh collapse or Shrink     
40:51     
and so I guess a would be the case where they're too few B would be the case where it's just right     
40:58     
in both snapshots a red cylinder represents an fa actin filament so these red cylinders here fa actin a black bead     
41:06     
represents a plus end of an act F actin filament a white bead represents and I     
41:12     
guess you can see the white beads in there uh represents the minus end of an F faactin filament a blue cylinder which     
41:20     
are these blue lines represent an ensemble of nmia Motors I can consisted     
41:27     
15 to 30 motor heads a green C window represents an alpha actinin cross Linker     
41:34     
you can see the green ones here and in here and a yellow bead represents an R23     
41:39     
complex so we have yellow beads throughout you can see that uh so that's that's where we have this uh model so     
41:49     
this is you know what it looks like in the output of the simulation we have reaction rates that     
41:55     
they've kind of derived here you have uh reaction rates for acting filaments linkers Motors branching and     
42:03     
then destruction you have other parameters in here um and then the data analysis that     
42:11     
they do so they look at they they draw from Network Theory to analyze these     
42:18     
networks and they you know kind of build a mathematical graph out of     
42:25     
this and so so they convert acto myosa networks into mathematical graphs the     
42:32     
confirmation of the mathematical graph from the physical acto myosa Network and the calculation of network Theory order     
42:38     
parameters were performed with the python package Network X so they take these simulation outputs and they turn     
42:45     
them into mathematical graphs uh so they create a standard graph and then they     
42:50     
map everything onto that uh they use Network x uh which is a     
42:55     
standard um approach uh so they talk about the graphs here     
43:02     
where this is uh they kind of show the graphs sort of an introduction to graph     
43:08     
properties which isn't really helpful here um so then they also have order     
43:15     
parameters for these graphs so these graphs are complex systems and one of the things you can do is use order     
43:22     
parameters to sort of understand the structure of the network so uh     
43:30     
yeah so they they actually do a little bit of work on assortativity meaning that they want to see you know how these     
43:37     
networks connect together so they do this uh they establish this Criterium     
43:43     
for assortativity and then they use machine learning models to forecast     
43:54     
Avalanches uh these data analytics for machine learning models so they they do     
43:59     
some analytics here and they get some results in these graphs     
44:05     
here so they look at the time course of these Branch networks with low medium     
44:11     
and high brancher concentrations so I guess the uh I guess low is a medium is B and     
44:21     
high is C so low is a here B is medium C is high     
44:29     
okay so that's what we have there um then we have this graph here     
44:35     
where or this this figure here where we have the branching networks in on the     
44:41     
left and the mathematical networks on the right so this is where we have graph networks a physical atoosa networks at     
44:49     
several arp2 Street concentrations at 500 seconds so this is where AC and E     
44:56     
show snapshot of the simulations that low medium and high R 23 concentrations     
45:02     
so a is here C is here and E is here so these are the physical networks this is a low     
45:10     
concentration of R23 C is a medium concentration and e is a high     
45:16     
concentration you can see this ratio here x b to a is     
45:21     
0.020 z or 02 or 0 2 02     
45:30     
and2 and then your BD and F are the graph representations of those acto     
45:36     
myosa networks so basically you're taking so you're taking this physical     
45:41     
Network that's branching in this case you don't really see a lot of branches in this case you see more branches and     
45:48     
then in E the network is very highly structured in branching so you can see the consequences in the mathematical     
45:55     
note so for ones where you see very little branching you have very few uh     
46:02     
edges between nodes you have these hubs that are sparsely connected and you have     
46:09     
some nodes that aren't connected at all and that makes sense because if you look at the physical Network or the physical     
46:15     
branching it's not very clear that there's a lot of physical     
46:20     
branching in C we have uh more branching and we have this corresponding Network     
46:27     
work where we have these hubs we have edges or peripheral nodes and the edges     
46:33     
are more there are very few things that aren't connected there are a lot more     
46:38     
edges and there's sort of this clustering that you start to see take shape where you get clusters of nodes     
46:45     
that are connected and so like this green cluster here this blue cluster here and then this component around the     
46:52     
edge that's connected and so this is uh indicative of this greater branching and     
46:59     
or 23 then finally in this F you see the     
47:05     
sort of the translation of e is having a lot of branching a profuse branching if     
47:11     
you will and you get this corresponding relationship in the network where you get these clusters this highly     
47:17     
structured uh mathematical network of different clusters it almost look like patches on a Quil so you see that here     
47:25     
so basically Bally is you increase the branching you get more order in the     
47:31     
network and that's really kind of think the Insight here is that they're looking at the amount of order that this brings     
47:38     
to the network they're looking at the connectivity and then they can measure uh they can look at these order     
47:45     
parameters they can look at um the different uh graph uh uh graph     
47:52     
statistics and they can look at how the graph is structured how it's connected     
47:57     
and how it behaves over time     
48:03     
okay so I don't think there's anything else worth talking about here it's just they have a little bit more work on     
48:10     
assortativity and so this is something where you get displacements of fil uh     
48:15     
filaments where you get uh connections that change between filaments and that's what they talk     
48:22     
about at assortativity so there's this uh third type of L which is assortativity I     
48:28     
assume this is what they were talking about when they talked about the social network properties because in social     
48:34     
networks aside from the small world Behavior you get this assortativity where there's a sort of     
48:42     
like attaches to like so there's a way of attaching nodes but through edges to     
48:47     
other nodes and so this attachment has to happen by some process sometimes it's     
48:53     
random sometimes it's what we call assortative which means that like attaches to like or there's some     
49:00     
attachment rule that results in some systematic uh behavior in the network     
49:06     
and so this assortativity is the first CL third classification of avalanche and     
49:12     
so they talk about that in this figure here uh so basically when there's an     
49:20     
increase in insort assortativity in the network node a and node B now form a     
49:25     
center likee clust cluster during the Avalanche the example simulation is a high motor low Linker and medium     
49:32     
brancher concentration so the A and B are here so you have uh so you have in the first case     
49:39     
here you have node a and all of these uh nodes are sort of uh I     
49:46     
guess uh connecting to it in C you have a where all the nodes are connecting to     
49:52     
each other instead of a and F you have where a is connected uh everything's     
49:59     
connecting to a but not b and then in the last example you get B connecting to     
50:04     
the same nodes as a so this is just kind of showing how this these assortativity     
50:10     
rules change okay uh so let's go down to the Fig to     
50:17     
the references because I I thought there were a couple really interesting references in here     
50:27     
so we're looking for so we're looking for reference seven     
50:34     
and reference seven so they talked about 7 through 12 being sort of indicative of     
50:40     
the different types of simulation that are being used currently um so seven is collective Len     
50:48     
dynamics of flexible cytoskeletal fibers that's from the new Journal of physics     
50:54     
there's also a veratile framework for simulating the dynamic mechanical structure of cytoskeletal networks     
51:00     
through biophysical Journal number nine is molecular transport modulates the Adaptive     
51:07     
response a branched acting networks to an external Force that's Journal of     
51:13     
physical chemistry B and then papoian I guess which is the method that they used uh who and propan     
51:22     
uh mechanic mechanochemical feedbacks regulate the acted mesh growth and     
51:27     
lopital protrusions it's biophysics journal and then L and propon uh the     
51:33     
stochastic dynamics of filopodial growth and that's biophysics journal and then 12 is Carlson the     
51:40     
effect of branching on the critical concentration in average filament length of actin so these are all sort of     
51:48     
simulation related studies except for this last one which talks about the effective branching on critical     
51:54     
concentration so this is in biophysics jour uh then then if we go back up to five     
52:00     
actually or well there's five and there's six so six is this paper by Papa     
52:08     
lazaru and macheski the cell pushes back the ARB 23 complex is a key orchestrator     
52:15     
of cellular responses to environmental forces so this talks about this R23     
52:21     
complex in its role in responding to environmental forces so that's kind of the context for you know what the     
52:29     
forcing mechanisms there are in these networks so you can build the network and you can simulate it but unless you     
52:34     
know why it's doing what it's doing it doesn't really mean much um so that's an     
52:41     
interesting reference number five is this reference by Malik garby at all and     
52:47     
this is scaling behavior in steady state Contracting acto myos and networks so     
52:52     
this is important this is a nature physics paper from 2019 this is important understanding these different     
52:58     
modes of behavior and how they change with a growing n so this is where we     
53:04     
want to know something about the steady state and the scaling behavior of the     
53:11     
network okay so then there the other one I wanted to get to here was number 14     
53:17     
so uh actually there's another article on the role of RB 23 complex in shaping     
53:25     
the Dynamics of structures brandao myosa networks that's number 13 that's from     
53:31     
pnas uh so that's another paper on the r 2.3 comp or the 23     
53:36     
complex uh and shaping sort of the Dynamics of the network now number 14 was about these     
53:43     
cyto Quake events so the cyto Quake events are these things that are sort of these U cell-based uh for these     
53:53     
structural avalanches and so this is alanar atol non-equilibrium cyto Quake     
54:00     
Dynamics in cytoskeletal Remodeling and stabilization this is from soft matter     
54:05     
so this describes kind of the cyto Quake phenomenon um so I don't know you know     
54:11     
this is again uh these Avalanches I don't know we not going to go to the     
54:16     
paper right now but that might be something to follow up on okay so     
54:23     
that's good um then there's some other references in here which are nice to     
54:30     
look at uh let's     
54:37     
see there's a paper uh by qu Manos and poolan stochastic ratcheting on a     
54:44     
funneled energy landscape is necessary for highly efficient contractility of     
54:49     
actomyosin force dipoles so this is physics physical review X this is     
54:55     
talking about stochastic ratcheting which I'm not familiar with but I assume that there's some sort of action um on     
55:04     
the structure and thinking about the energy landscape and how that     
55:10     
manages uh the efficient cont contracti contractility of     
55:17     
actomyosin okay so I think that's all for that paper um it was an interesting     
55:23     
set of things a lot of things going on in that paper but basically they're simulating Acom     
55:28     
myosa networks they're looking at these uh these these avalanches and they're     
55:35     
trying to predict them using machine learning from their output of their     
55:43     
simulations how they need to add more chemicals yeah their simulation so you     
55:50     
can see like the bundling of acting right that's the branching of     
55:57     
acum um with ar rp23 complex but there's bundling and     
56:04     
that's a different chemicals so I'd like to see that     
56:11     
yeah of course yeah well I thought it was interesting from the standpoint of     
56:17     
modeling sort of the nonlinear dynamics of the structure now you know one could argue that you need to have more things     
56:24     
in there cuz you're just kind of working with a a network and you you know maybe     
56:30     
have environmental forces so you're just kind of modeling the network and saying we can simulate the forces on the     
56:35     
network and they have this effect but if you had like a cell membrane or a muscle     
56:42     
in real life and you had some force on it you know like maybe another cell or     
56:48     
you know it was pushing against some you know like other cell that was migrating     
56:53     
past it you know some actual real world context where the forces are coming in I     
56:59     
mean the forces are going to be maybe a little bit more especially when you're dealing with the Dynamics they might be     
57:05     
a little bit different than what they're presenting in in the simulations so getting these sort of     
57:11     
isolated modes down I think is important but you also have to have this in     
57:16     
observe behaviors that are sort of naturalistic and that they're actually     
57:22     
happening they're actually things that are biological in nature things happening in in the context of the cell     
57:29     
that's I think it's scaling up that that that's scaling up like adding     
57:37     
um forces over a whole cell would be that would be scaling up right from the     
57:45     
um molecu molecular levels so but I I'm very interested in these branches in us     
57:52     
because that's where my my rods and strings for my Tegrity come from that     
57:59     
that's what they are in real life they're not just a rod they're Branch     
58:04     
Network that goes through the celf yeah so I I need to explain that     
58:10     
actually in my probably in my intro just a bit so maybe I can use some of their     
58:16     
their diagrams and say see there's there's a network in the     
58:22     
cell yeah that' be good yeah yes the other thing I I wonder about with this     
58:29     
is that they're describing branching Dynamics but when they take it and they move it to uh mathematical Network     
58:37     
they're treating it as sort of a connection I don't know if it's a if it's a directional connection or not it     
58:43     
may very well be but there's a difference between sort of like a     
58:49     
interaction between two nodes that's that's non-directional and one that's directional and can have a um     
58:57     
play a role in sort of the Dynamics of the network the mathematical network not the Active network so I think maybe you     
59:04     
know they should have maybe used a little bit more sophisticated uh Network Theory or you know but I don't know how     
59:11     
they actually they I didn't really dig deeply into what they were doing there they just had some demonstrations of     
59:17     
like an ordered network but it didn't appear that there was a lot of like that there was an explicit order in the     
59:24     
mathematical networks but they have oh go ahead um sorry there's a     
59:31     
paper or two about brush bottle networks for cells and uh how they work rather     
59:40     
than the just the straight acting the ones that have branches like a um a a     
59:46     
brush or a bottle on them um are behave     
59:52     
differently than the just this straight up acting so the last diagram showing     
59:59     
all the little branches reminds me of that paper where they said oh there's a     
1:00:05     
brush bottle Network in of acting in the cells so there's there's some some other     
1:00:13     
papers about some of this uh branching and there's also branching that's kind     
1:00:19     
of at 60 and then if it's being pushed on from the top or something it becomes     
1:00:27     
a 90 de um angle yeah so it holds holds better     
1:00:36     
it holds up more Force that's on the um cell     
1:00:43     
membrane I have some maybe I I should show you I have some papers in a um     
1:00:51     
PowerPoint but um you probably don't want to see that     
1:00:56     
today since it is over a 100 um slides long     
1:01:05     
okay yeah I don't know something to do after I've handed in a thesis right yeah gota     
1:01:13     
gotta take it one step at a time well I have to look into these     
1:01:20     
brush bottle networks I had not heard of that before so okay yeah so I'll try to send you a     
1:01:28     
couple of papers to just I I've been thinking of sending new papers and then i w i then I forget because I'm trying     
1:01:36     
to do a bunch of whole bunch of other things and get this done and yeah important thing     
1:01:44     
yeah yeah just keep focused yeah because I can certainly get sidetracked so I've     
1:01:51     
been trying not to right yeah     
1:01:58     
okay well uh any other comments uh dick or Morgan do you have anything before we     
1:02:06     
go great stuff yeah um hard to track all the details when I'm kind of     
1:02:13     
multitasking here but but yeah really like it there was um there was a nice uh     
1:02:23     
intro to something from early there was a nice intro video from um I forget what     
1:02:30     
the the acronym CMB is but it's like mechanobiology I think is the um the the     
1:02:38     
the MB um and and uh yeah just just talking     
1:02:47     
about a number of yeah just interesting interesting Concepts that um are covered     
1:02:55     
in anob biology and yeah um just made me think of now the uh mean they were     
1:03:04     
that's talking about kind of natural structures that that displayed um drain     
1:03:12     
dependent um properties so so they were call them like visco elastic and things     
1:03:19     
like that but just you know again like that interesting intersection of of engineering and biology but     
1:03:26     
yeah I'll try and drop drop links later when I when got the kids to school okay     
1:03:32     
yeah that sounds good yeah a lot of these networks you know they're physical networks but they also exhibit different     
1:03:38     
types of properties when forces are applied so they're not like static they     
1:03:43     
have like you know they deform in different ways and they recover so how do you represent that as a mathematical     
1:03:49     
Network it's a little bit tricky um yeah so yeah it's it's definitely yeah for me     
1:03:56     
I'm I'm just having to do it statically like it's at this place at this point     
1:04:02     
and this is the model of it when it's stressed like this and it's going to be stressed like this a second later     
1:04:10     
because there's a force on it so it changes     
1:04:18     
anyway I don't know I don't know how to get that into um a movie I suppose     
1:04:26     
maybe one could I don't know yeah all right uh well thanks for     
1:04:32     
attending um good luck Susan on your pieces this week thank you yeah just down to the     
1:04:41     
final few things right yeah mostly have it written yeah     
1:04:50     
so yeah I should go do that yeah great all right     
1:05:01     
Moran just um yeah good good job good job     
1:05:08     
s oh I'll just wait till it's finished and they say it's good and then     
1:05:14     
I'll just go yeah enough of this then I can do what I want with it     
1:05:21     
yeah all right thanks talk to you next week okay
