## TRANSCRIPT   
0:00   
Hello Bradley. Hello Lewis. How are you? Good. Good. It's been a long time.   
0:05   
Yeah. Yeah. It's been a long time. So what have you been up to?   
0:12   
So uh so I've been working on that work and uh so Richard uh have been you know   
0:23   
no know uh he's kind of uh sent I mean like a paper to uh to one of the editor   
0:31   
in bios systems so he kind of reached out to us and uh he's suggesting that   
0:37   
there is a deadline timeline around like norm first week uh to submit the work.   
0:43   
So I was preparing the manuscript according to that. So I just wanted to   
0:49   
give a brief out about that work. So what we're going to do about it and a   
0:55   
short presentation where I can you know detail what has been done and you know what the things that should be included   
1:03   
and uh take a review from you all and uh maybe you can add those uh concerns into   
1:10   
that manuscript. So so I thought   
1:16   
yeah that would be great. Um yeah it sounds like it's coming along. So yeah.   
1:22   
Yeah. Um so yeah, do you want to present today or   
1:27   
Yeah. Yeah, I can present. Yeah. Yeah. All right.   
1:34   
Hello Susan and Morgan.   
1:39   
Yeah. So is it visible? Yes. Yes.   
1:47   
So yeah. So this is the work and uh yeah   
1:52   
it's start so the the problem statement of the work uh what we uh you know   
1:58   
started with one of uh this Google summer of code project in 2025.   
2:05   
uh so so it was main to understand how this uh complex multisellular structures   
2:12   
of flexi elegance during embryogenesis how do they evolve and uh so as uh insp   
2:20   
taking an as an inspir inspiration from the previous works of Bradley that uh uh   
2:27   
the anastmosis can be seen in the you know during the uh you know ev   
2:33   
embryogenesis of the scale elegance. So not just uh uh fusion it does but also   
2:39   
there are like certain fusion interactions that can be handled and we can also see uh certain centium cells   
2:46   
which are responsible to form tissue level organizations and all. So so we   
2:52   
were very intrigued by understanding what what could be seen at a computational level. So so we call this   
2:59   
as like a group or high higher order interaction group level interactions. Because general standard graphs has a   
3:06   
single pair wise interaction whereas hypographs has like higher order interactions. So so we have a cellto   
3:14   
cell interaction such as a parent it's been divided into like two daughter cells but uh they do form a small   
3:22   
community where there is a group level interactions. In order to see that so what we do is so we have uh the data   
3:28   
which is represented into two uh certain manner. So one is the spatial proximity   
3:34   
or the spatial relationships where we have the XYZ coordinates of where exactly the parent still has started and   
3:41   
the how it was divided and we have the lineage information where from which how   
3:46   
the from which parent to which daughter and from that daughter acts as like a parent to the nether I mean like while   
3:54   
during the fish and how it grows. So, so this this uh higher order interactions   
4:00   
we wanted to model with uh uh hyperraphs. So, and we wanted to study   
4:06   
this uh evolution process of embryogenesis and uh see how over time   
4:12   
this evolution happens. So in short, so we develop a dynamic   
4:18   
hypograph neural network which integrates both postpatial and the lineage interactions to to understand   
4:24   
and observe and predict the behavior of clans embryoenesis. So this is the main idea of like uh the   
4:32   
the workflow of the way which uh we see. So here we have a singular single   
4:37   
cellular organism and later it divides into multisellular and it becomes an adult one. So during this we have a   
4:43   
single cell and at all the time steps we have like uh the way in which it has been divided. So and also this   
4:51   
information including the lineage information that these are the entire   
4:57   
the temporal uh so we model this as a graph and uh using that graph we take   
5:04   
the lineage and temporal information and then feed it to the hyperraph neural network and this network will help us   
5:12   
understand that what kind of associations it actually forms to make it more uh reliable and more correct   
5:19   
predictions. Suppose we see here initially it starts with understanding that uh you know uh these three do form   
5:26   
a certain hyper edge because they have a certain type of associations. So this might be because of they form a same   
5:34   
they belong to the all the same tissue cells or from which you know a single centumium has been started and and so it   
5:41   
evolved. So this is the the pipeline that we have observed. I mean like so so   
5:47   
drawing these hyper edges on top of that is the the main challenge here. I mean   
5:53   
like how do we say that you know these edges are correct. So we design a complete neural network workflow which   
6:00   
actually does that uh you know tries to understand and optimize and capture the   
6:05   
best higher order interactions in in this uh in this uh emergence of C   
6:11   
elegance. So the this pipeline model is basically we call it as uh dynamic   
6:17   
hypograph uh neural network and uh at each time step we have a 3D point cloud   
6:23   
or like a 3D point cell and over the spatial and the bird times and uh so we   
6:30   
build two kinds of hyper edges. So we don't just rely on spatial hyper edges   
6:35   
by finding the proximity but also we have the lineage information and we see how these connected sibling cells the   
6:43   
the parent and daughter relationship and how they have been acquired. So we formulated it in a way and uh going to   
6:51   
that so we then combined this into a uh you know common hypograph with the both   
6:57   
spatial and lineage relationships and this we give it to a neural network. So we have here hypograph neural network.   
7:04   
So this tries to find out the associations and we have a you know uh   
7:10   
uh a neural network a layer which is called gated recurrent units which tries   
7:16   
to understand how this uh time temporal uh during the time evolution which time   
7:23   
step is actually trying to acquire. So the temporal uh uh features have been acquired by this   
7:31   
layer and the spatial features have been acquired by the hypograph. So this mixture all together will give us a   
7:38   
better associations in the uh how I mean like you know which can be grouped   
7:44   
together which cells can be grouped together. So so this is the final uh   
7:50   
projection head which gives uh an embedding layer. So based on this uh we   
7:56   
acquired the features from like neural network understanding the both spial and lineage information. This layer has all   
8:02   
the features regarding the old time series and the information that it has. So based on that we actually use these   
8:10   
features as like a lower level representations and this help us to map on to the uh the the the data that we   
8:18   
have so that it will help us to understand. So we have a visualization   
8:24   
uh that uh probably Mashid can show.   
8:32   
Uh   
8:39   
uh so we actually ran the experiments. So when we ran the experiments we are   
8:44   
able to get uh the complete embedding space how the you know the neural   
8:49   
networks tries to understand this whole embryoenesis process. So this we   
8:57   
decided in the GSO project and Basher was able to reproduce it and uh he'll   
9:04   
show them   
9:39   
So I think this is a representation space. Now when we try to point out at a certain embedding any embedding. So we   
9:46   
will be able to see that uh what it actually is. So that is the lineage information and uh the XYZ coordinates   
9:53   
and the cluster information that it associates with. So from this we will be able to find a grouped clusters with a   
10:00   
certain behavior and uh then from which we will be able to get uh a good   
10:06   
understanding of uh okay which uh type of cells have been come together which   
10:12   
have been you know drifted apart and uh which are actually forming associations   
10:17   
and which are forming clusters. So we'll be able to see that. Uh so this is just the last time step I'm reading. So we   
10:24   
can we can use all the time step embeddings I mean like when it started   
10:30   
and when it ended at every time we can you know map these embeddings and see   
10:35   
like uh how are they grouping how are they associating so   
10:41   
yeah I'll get back to the presentation you know okay so this is um this is like a two   
10:48   
dimensional um that you're producing off of the yes basically fivedimensional data   
10:54   
And it's that space. Uh so what do you how do you interpret that space? I'm curious.   
11:02   
So there are like I mean like uh so whatever the information that we acquire from a neural network. So we use that   
11:10   
neural network to embed it into a certain space so that uh it uh actually   
11:16   
has the entire representations of the five-dimensional information into that small space. So that we will be able to   
11:23   
say that each data point that we have in the UMAP embedding associates with the   
11:29   
the the relationship that each and every cell has. Suppose they come grouped   
11:34   
together then we can say that okay uh these types of cells have a similar   
11:39   
behavior and these type of cells might have certain characteristics. So that that is   
11:46   
the the key challenge that we are trying to understand here. So   
11:52   
yeah, the reason I asked that is because uh it's very popular to use this in like   
11:58   
genomics and that and people will misinterpret it and say that they're clusters.   
12:04   
I mean, but they're not technically clusters, but they do have this spatial or they do have this relationship where   
12:11   
if they're together in space in the XY space, then they have some something   
12:16   
there, you know? I mean yeah it's a subtle relationship and I don't think anyone really explains   
12:22   
it you know what I mean like when you read papers. Yes. Yeah. Yeah. So yeah we were trying to   
12:29   
understand the similar because uh this was a last time step embedding when we're trying to plot it in a first time   
12:35   
step we have only single cell. So we are trying to understand that okay in a single if there is during the evolution   
12:42   
at least for like 10 time steps we'll be able to see that two two cells have come together then next did the two cells   
12:50   
which were came together in the first time step are they drifting apart in the next part I mean like during the next   
12:57   
successive uh you know uh time periods are they drifting apart if they're drifting apart what does that mean so we   
13:04   
are trying to understand that part So, so I I what what uh I and Basha disc I   
13:11   
mean discussed was uh uh it would be more meaningful if we actually not just   
13:17   
visualize them but we make an uh an interactive tool and uh make it online   
13:24   
so that you know when we click it then we'll be able to see how the changes are   
13:29   
happening so that you know people might like uh suppose we have a certain   
13:34   
perspective If uh two to three different people see that and you know they might have a different perspective. So from   
13:40   
that uh visualizations maybe we can draw some good conclusions is what we   
13:46   
believe. So that that that's that's going to be in our next uh step. So   
13:54   
yeah yeah so I'll uh carry on with the presentation with   
14:06   
Yeah. So this uh so what what we are trying to do is   
14:11   
right now is we have our uh model which is uh the temporal block is the GRU so   
14:17   
which is gated recurrent unit and we have our hyperraph convolution so which is like uh the spatial and we don't have   
14:24   
a transformer model as of now. So we have the default configurations that that is actually working from the paper.   
14:30   
So this is what vet is done in the first. So what we thought was we can mix with certain base models and uh we can   
14:38   
mix with different types of temporal blocks and we can use attention layers and the hyperraphs and all. So we can do   
14:44   
a good sorry baseline runs and from that we can able to test it and second we thought that uh it would be great if we   
14:52   
actually include uh uh the uh I mean like interactive visualization tool. So   
14:58   
that is the second and uh third part is we are still trying to uh you know plot   
15:04   
visualizations in 3D and 2D and we are able to you know bring some good interpretations out of it. So   
15:12   
this is ongoing and uh so if you find an update then probably we'll mail it and   
15:18   
so if you can give us a review on that would be great. Uh review on what?   
15:25   
Uh review on the manuscript. uh that we are trying to submit. Yeah. But like what is the what is the   
15:31   
action item like what would you need? I'm sorry. What would you need for this uh like in   
15:39   
terms of a review you said? So in terms of review I would uh uh I   
15:44   
mean like I would it would I would I would I would appreciate if there is a you know thorough understanding of   
15:50   
whether the interpretation as you have mentioned. So we are trying to interpret something out of it. if if the if uh it   
15:57   
is uh you know it might be uh constricted to a certain perspective whether it can be broadened or you I   
16:04   
mean like uh whether you know more biological insights that could be added into it to make it more enhanced. So I I   
16:13   
mean like I'm not really from like a deep biology background so I would appreciate certain things.   
16:21   
Oh yeah that's definitely the case. Um yeah, I think probably the best way to   
16:26   
go about this is to have like a you know write up some of these results and then   
16:33   
I you know if you send me like a copy of it I can figure out where I can contra   
16:39   
you know contribute a review or some insight and put it in and then you know   
16:44   
that'll be uh I think like you know it looks sol the work looks solid and I   
16:50   
think you know the best way to really kind of go about this especially for   
16:55   
like biosystems is to have visualizations have a lot of maybe about six or seven   
17:01   
graphs where you you know and there's an economy of graphs where you you want to   
17:06   
get like the most descriptive graph possible. So you know one might be   
17:13   
the first one you showed where you show the process of going from the embryo down to   
17:19   
the uh hyper networks and then you know kind of explain in the paper what those   
17:26   
what that transformation is why it's important and then uh the next one being   
17:31   
maybe like the embedding space with uh map and then say why that's important and   
17:38   
then you know maybe have a few more results else. I like the table you had at the end and I kind of wanted to go   
17:44   
back to that bit. Um, can you go back to that slide?   
17:50   
Sure. Sure. Okay. Yeah. So, this is like your your   
17:56   
this is like all the different models that you can kind of have.   
18:01   
Okay. And this is hyper edge convolution temporal block just variations on on   
18:07   
different technical things. And then Yes. Yeah. No. So, do you have like a metric for this   
18:13   
like where you're testing like benchmarking the different methods?   
18:18   
Yeah. Yeah. I I have the all the evolution metrics and I'm trying to uh   
18:24   
you do it in a standard uh the machine learning pipeline like how we do it in   
18:30   
following the similar protocols like uh 80% of the data to testing 202 or 80%   
18:37   
data for training 20 for testing and see like how uh the predictions are well   
18:43   
close with the you know the actual ground truths because we all we have the   
18:48   
entire I label data. So I have made it in.   
18:53   
So yeah, that that's good. That would be good to have the the table and then some of the maybe one statistic you don't   
19:00   
want to overload because this is not a machine learning journal you're targeting   
19:07   
and then like you know is there a way to get like I think that the final graph   
19:12   
would be something like taking these different methods and showing some output from them. So it could be like   
19:19   
the map uh visualization for the different methods and then like either having   
19:25   
panels where people can visually compare them or like you know I don't know overlaying   
19:31   
them is a little messy because you there's going to be a lot of overlap. Yes. Yes. I can have different panels   
19:37   
which can visualize. Yes. Yeah. Yeah. That's great. And then you know   
19:43   
this is something where we've kind of have this st you know we've I've had   
19:49   
some presentations where I give a talk about like the different levels of   
19:55   
dimensionality in uh kind of our approach is like to take   
20:01   
the three-dimensional like you get the threedimensional data the xyz data from   
20:08   
uh cell tracking and then you can uh add in other dimensions.   
20:15   
So like you have all sorts of different dimensions. You can add in time, you can add in um angular data, you can add in   
20:25   
uh cell state, you can add in all sorts of things and then it makes the model more and more higher and higher   
20:31   
dimensional. So like you know you could have a fivedimensional model where you have three dimensions of space from your   
20:38   
cell tracking one dimensional time from your cell tracking and then sort of derivative variables that you get from   
20:46   
uh analyzing the data and then you have like this eightdimensional space   
20:52   
and then that eight-dimensional space instead of the point cloud which is three-dimensional you have this   
20:57   
eightdimensional space which you can then reduce to you know using some sort of dimens functionality reduction or you   
21:04   
can um you know do things like do time series like what like what giadratha did   
21:10   
this summer all sorts of things. So I wanted to kind of put that in the paper too like kind of that formal uh   
21:17   
formalism for that and discussion as we do this and we have other papers where we do this and so I just want to have a   
21:24   
continuation of that um and then you know having this I think it's nice to have the hyperraphs   
21:31   
approach where you have like basically a statement of we can take things from a   
21:38   
lineage tree and then turn it into this sort representation where you can recombine   
21:45   
the nodes into different categories. Um the only well one thing maybe we   
21:52   
could see is more actual hyperraph sort of representations.   
21:58   
So like you know is do you have a good method for showing the hyperraphs   
22:03   
as like these uh you know like showing the nodes within another hyper node and   
22:10   
then showing the hyper edges because that's going to be very hard for people to understand and visualize.   
22:19   
uh there are few type of visualizations where I tried it in 2D but uh in 3D I   
22:24   
think I came up with an approach which is actually feasible. This is called a star method. So the star method what it   
22:32   
does is uh uh the interactions it will place it as a small point and from that   
22:37   
point whatever the I mean like the the nodes are connected hyper nodes so it   
22:43   
will be connected. So it's like uh it doesn't look like we are drawing an edge on top hyper edge on top of all these   
22:50   
things but it look like uh the way the associations have been formed. I can do   
22:55   
that. I mean okay so it doesn't look like the nested groups that you I mean the nested groups   
23:00   
are nice um but how does what does it look like? Do you could you describe it a little bit or   
23:07   
uh I think I have plotted it in long while like I don't have right now but I can   
23:12   
Okay. Uh I can show you it's it was long back I have to   
23:20   
so maybe in the next meeting I'll show how it it it was there I mean so it it   
23:25   
looks more like a cluster and uh there is a point which actually from which   
23:32   
that point all these points are connected to that cluster saying that you know this these two belong to the   
23:37   
same this a hyper edge which is actually connecting all these points.   
23:43   
So, so I mean like all the points suppose let's say hyper nodes are I mean like uh   
23:49   
in circular color we put it as a square box and say that this square box is the   
23:55   
point I mean like uh this is the uh association you know this the hyper edge which   
24:02   
brings all these things together it carries all the information into this we   
24:07   
can it's not very intuitive as drawing a top uh line which is on top of that but it's   
24:15   
more like we can do it in a different fashion but yeah   
24:20   
okay in Python visualizations are quite little hard yeah I know it's uh yeah you like   
24:27   
they're specific visualization tools which usually do a lot better but um that no this is great this is great and   
24:34   
then you know so that's the idea for the paper like really kind of having   
24:39   
you know want to be accessible to the bios systems audience. So it's just a matter of figuring out how that looks   
24:47   
and I think visualizations are the key you know not having a lot of statistics without interpretation or like methods   
24:55   
without interpretation. Um I I yeah the best way to do this is   
25:00   
just to kind of go through and with those things in mind and then we'll review it and see where it might be a   
25:06   
little hard to understand because it's you know it's very hard for people to wrap their heads around it   
25:11   
if they're not used to the Yes. Yes. Yeah. Yes. Yes. So it it should be more uh convincing   
25:18   
and easily understandable too. So uh rather than high level technical   
25:24   
details have to be you know should be meaningful that you know we are uh   
25:29   
trying to make it more to you know applying it to a place where uh this is   
25:35   
more like an applied topic. We are applying the embryoenesis. It should I mean like the   
25:41   
writing should actually makes make more sense about like uh how biologically   
25:48   
this is meaningful. Yeah. Yeah. So that that that consumes little time   
25:54   
to do that. Yeah. Right.   
26:01   
Did you have any other anything else or I mean like I've been uh meeting   
26:08   
monthly once or twice with uh Richard Gordon and I am worshiped both early   
26:16   
meeting and uh we are uh we have submitted a paper where it was halfway   
26:22   
done by uh Richard on uh the exponential of the exponential.   
26:29   
So there are like this there are number of cell types that are increasing from like long back long time and uh so   
26:37   
during the the uh the evolution this neurons have been seen as like when we   
26:42   
try to plot it in an uh logarithmic graph it's exponential the number of cells are increasing exponential in a   
26:49   
logarithmic scale. Yeah. So, so that's one of the interesting thing that we have observed   
26:57   
and so yeah the hyperraph and this   
27:03   
all right that sounds great. Thank you. Thank you.   
27:08   
Yeah, that's great. So yeah, Leworth, congratulations on getting some making   
27:14   
some headway on this and you know having enthusiasm for continuing G-S stuff. Um   
27:22   
yeah, so I'm gonna be giving a lecture to the openworm foundation next month um   
27:29   
on like the yearly report on evil. So I'm going to include some of this into   
27:35   
it. Sure. All right, that would be great. Yeah.   
27:41   
Uh so this is a paper um   
27:46   
where let's see. Okay, so this is a paper uh   
27:52   
that was published last year and this was a natural structure molecular biology   
27:58   
and the title of this is precise and scalable self-organization and mamalian pseudo embryos. So this is   
28:06   
an interesting topic about like creating embryo models for analysis. So a lot of   
28:14   
times it's hard to work with mamalian embryos and we're interested in a lot of the   
28:19   
properties of those embryos but we can't for maybe sometimes ethical reasons or a   
28:25   
lot of times technical reasons work with actual embryos. And so, you know, if   
28:30   
we're working with sea elegans, it's actually fairly easy to work with the embryos, working with something like   
28:36   
zebra fish or uh xenopus, which is the frog. And it's, you know, we can do   
28:42   
experiments there. And that's why they're model organisms because they're very easy to work with in terms of the   
28:48   
embryo. But in mamalian embryos, it gets a bit harder to do certain things in   
28:53   
those models. And so this is where they're actually uh presenting this   
28:59   
model of a gastrloid which they're you calling pseudo embryos. I think this   
29:05   
work um people were calling them virtual embryos for a while but that's kind of   
29:11   
like implies like computer simulations. So um that's   
29:17   
Oh yeah I got to share my screen. I got knocked off my   
29:27   
All right. So, um this is the paper from natural structure nature structural   
29:33   
molecular biology. Okay.   
29:38   
So, this is the paper from nature structural and molecular biology. This is precise and scalable   
29:44   
self-organization and mamalian pseudo embryos. um and authors are here and this includes   
29:50   
Thomas Greger who sort of leads this group in this research.   
29:55   
So uh the I'll read the abstract. Uh gene expression is inherently noisy   
30:02   
posing a challenge to understanding how precise and reproducible patterns of gene expression emerge in mammals. So   
30:09   
again we can study gene expression in like Drosophila embryos. This is where we got a lot of information about uh   
30:17   
sort of morphogenesis and and the sort of colinearity of hawk genes and how those are expressed in the   
30:23   
embryo. So we have some good model systems for looking at gene expression in embryogenesis.   
30:29   
Um but it's harder to do in mamalian embryos and it's very noisy as a process   
30:36   
and so you want to minimize that noise as much as possible. So what they're trying to do is minimize the noisiness   
30:41   
of gene expression. So oftent times you get um what they call biological noise   
30:47   
and then they call technical noise. Biological noise is the noise inherent in gene expression. So genes are   
30:54   
expressed usually when when a cell is stimulated or an environment changes   
31:00   
so that the expression pattern you know the gene is upregulated or downregulated. So if you change the   
31:07   
rearing temperature or if you change its exposure to light or if you uh you know   
31:13   
it it kind of experiences some chemical signaling from other cells or from the   
31:19   
environment gene the gene in question will either go uh it'll either rise in   
31:25   
terms of how it's being transcribed or it will be uh quieted in terms of how it's transcribed. And so uh without   
31:33   
getting into too much technical detail, this is something that involves the   
31:38   
regulatory machinery of the gene. It involves upstream promoters, enhancers,   
31:44   
silencers, things like that. It involves the opening and closing of chromatin and   
31:49   
all of that. So there are a lot of sources for noise there. And so you get this noisiness in terms of the   
31:56   
biological expression. Um you also get biological expression   
32:01   
when you know during development. So you go through different stages of development. Sometimes genes are turned on, sometimes they're turned off at   
32:08   
different phases of development. And if you go to something like cell cycle, the uh switching between upregulation and   
32:15   
downregulation of genes is very precise and it has to be very precise in order for that process to work.   
32:22   
And so that's that's biological noise. And then technical noise is where you get noisiness in gene expression not   
32:28   
because of the biology but because of the technical methods you're using. So   
32:34   
if you have something that you're working with in culture and you put the cells under a lot of stress they will   
32:40   
express certain genes or most genes at at a level that is not characteristic of   
32:47   
their biological background. In other words, if I, you know, have a cell culture and I, you know, have it under   
32:53   
stressful conditions, if I raise it in a high heat environment or if they don't   
32:58   
get enough nutrients in cell culture, then they will express genes there. There are all sorts of stress pathways   
33:05   
in the cell that turn on genes kind of randomly and it's not, you know, in   
33:11   
experimental uh settings, it's not good.   
33:16   
So this is this is a a challenge to understanding precise and reproducible patterns of gene expression. When they   
33:23   
say precise and reproducible, they mean like you know if we're looking for like pattern formation and we're looking at   
33:30   
the expression of genes that lead to pattern formation. It's going to be harder to get that sort of sequence of   
33:37   
events. it's going to be harder to see the effect if we have noise obscuring   
33:42   
the u the process. And so we want to have an experimental   
33:48   
setting where we can high you know have a highly controlled process where we   
33:53   
don't have a lot of extrinsic noise or you know noise that's coming from the   
33:59   
outside usually technical noise. Sometimes that extrinsic noise is biological and so we we want to minimize   
34:06   
that. We only want the sort of noise that you see in the biology.   
34:11   
Uh here we investigate this phenomena using gastroloids three-dimensional in vitro model for early mamalian   
34:18   
development. So they're using gastrloids. So we we talked about organoids before and there are these   
34:23   
different classes of organoid. Um you have gastroids uh assemblids   
34:29   
uh you know neural uh uh you know neuralorgganoids things like   
34:36   
that and gastroloids are a class of them. Uh so this is like kind of a model   
34:43   
of gastrolation if you want to be technical about it. uh that it's a bunch of cells that you have that are sort of   
34:51   
undergoing this gastro uh gastrolation like process.   
34:56   
Uh and so this you know this this model encompasses growth dynamics and gene expression patterns. We observe a   
35:03   
remarkable degree of control over gene expression along the main body axis. So this is anterior to posterior. You see   
35:11   
you should see reproducible sort of pattern formation from anterior   
35:16   
to posterior and in gastroloids you also observe this control over gene   
35:22   
expression uh with pattern boundaries position with single cell precision. So again, if you   
35:27   
look at a normal embryo embryo, you look at it like in in in D.Vo, you you see   
35:34   
these patterns forming. You see them from cell to cell. And you can see this in gastroloids. The idea behind   
35:41   
gastroloids is we can produce this experimental test bed that reproduces   
35:47   
the biology. We're not sure that we get every part of the process right. But if   
35:52   
it's producing a certain output and then we can measure that output that looks   
35:58   
very similar to embry malian embryos, then that's that's good enough. And   
36:04   
that's what we're trying to do here. Um, and so we have a remarkable degree   
36:10   
of control over gene expression along the main body axis with pattern boundaries positioned with single cell   
36:16   
precision. Furthermore, as gastrloids grow, both their physical proportions   
36:21   
and gene expression patterns scale proportionally of system size. So this is the other part about gastroids is   
36:29   
that they're not just like uh precise with respect to some of these developmental processes but they also   
36:37   
grow you know embryos will grow in size as the uh embryo as the embryionic stage   
36:44   
uh progresses. Uh and you should see this in gastroids as well. In other   
36:49   
words, you shouldn't just see like um that you know you get like these kind of   
36:56   
immature systems that grow and the processes of uh you know maintaining   
37:02   
pattern formation with growth. So the more cells you get you you know you   
37:08   
don't get this sort of uh diffusion I don't know how else to put it but   
37:14   
diffusion of pattern formation. In other words, if I have a a system with a small number of cells, I get these excellent   
37:20   
sort of striping patterns. But if the cell if the embryo grows to like three   
37:25   
times that size, the striping patterns will break down and you get this sort of smear. And that's not what we want. What   
37:32   
we want is to maintain that striping pattern no matter what the size. And that requires of course for those new   
37:39   
cells those you know because we're as we grow we gain cells we gain mass to have   
37:46   
that that that space and those cells express the same patterns and ultimately   
37:52   
to express their genes in that coordinating way. So notably these properties emerge   
37:59   
spontaneously in self-organizing cell aggregates distinct from many invivo   
38:04   
systems constrained by fixed boundary conditions. Our findings shed light on the intricacies of developmental   
38:10   
precision reproducibility size scaling within a mamalian embryo   
38:16   
suggesting that these phenomena might constitute fundamental features of multisellularity. So what they're doing   
38:22   
here is they're trying to get a handle on sort of this multisellular self-organization   
38:29   
and you know in like I said in C elegance uh embryogenesis is very precise we see   
38:35   
this in the model that uh Leith was showing that we can actually derive these higher order structures from   
38:42   
embryogenesis and sea elegance precisely because it is very ordered and predictable and   
38:51   
you A lot of people uh for the most part uh C elegance embryogenesis is what we   
38:58   
call deterministic which means that we don't have a lot of variation across   
39:03   
organisms or across the process. You get cells where you know what fate its   
39:10   
daughter cells will have or you have these sublineages where you know the ultimate fate will be of that cell   
39:15   
lineage or sublineage. And so we don't have that in mamalian cells. What we   
39:21   
have is sort of an activity dependence within the embryo. In other words,   
39:28   
different cells are exposed to different signals in the embryo. And depending on those signals, they get uh that that   
39:36   
contributes to pattern formation. So it's a very different model. It's regulative development instead of what   
39:42   
we call mosaic development. And we see mosaic development in in rsophila. We   
39:47   
see it to a certain extent in xenopus. We start to get regulative development. Different parts of the embryo warm   
39:54   
regulative and then in mamalian embryos the entire embryo is regulative and you   
40:02   
end up with these you know it's hard to actually get a lineage tree in mamalian   
40:07   
embryos that is stable. In other words, if I take one embryo I get one lineage   
40:13   
tree. If I go to another embryo, I get another lineage tree. So, let's say I've sampled a bunch of embryos from like a   
40:21   
cow or you know a sheep or or a mouse and I can get those embryos and I can   
40:28   
trace the lineage tree. And the lineage trees are always different. If I were able to track each cell and track their   
40:35   
descendant cells, you would get these different lineage trees. are totally different, but it's uh basically   
40:42   
producing the same set of patterns, the same set of uh forms. And the reason for   
40:47   
that is because it isn't the cells that are deterministic. It's the the chemical   
40:53   
signaling that basically sets itself up in the same way within the embryo   
40:58   
leading to these patterns. And so what they're doing here is they're using gastroloids to reproduce this   
41:05   
environment where you have you can control sort of the growth and the uh   
41:11   
shape of the of the culture and then you can use the same kind of system to   
41:17   
produce a sort of embryo like uh pattern formation. So that's really you know   
41:23   
this is really interesting work. um they they kind of go into some of the details   
41:30   
of what a gastroid is here. So multisellular development entails the meticulous organization of cellular   
41:37   
identities and body proportions in both spatial and temporal dimensions. So as   
41:44   
uh number of cells grows in an embryo, we want to maintain the spatial   
41:49   
relationships between different parts of the body that are emerging. So when we have an anterior to posterior axis which   
41:57   
is front to back, we have a head that's emerging. We have a brain and you know   
42:03   
the different parts of the head. we have, you know, regions like the tail and those things will later   
42:09   
differentiate and so but we don't want to just have like a system where or I   
42:15   
don't we don't want to have but you know I'm anthropomorphizing that a bit but you don't want to have like   
42:22   
evolutionarily you don't want to have a system that you know is kind of like   
42:30   
this space is going to form a head and then the head you know the embryo scales in terms of size and so then the head is   
42:38   
kind of misshapen because you know it doesn't scale with   
42:44   
the system. Now in in bi in vivo biological systems this problem is   
42:50   
solved by coordinated gene expression and signaling between cells. But when   
42:55   
you have something like a gastroid, which is sort of like a collection of cells that you grow in culture, you   
43:02   
know, you need to be able to reproduce the system and have it work. And you know, the the thinking of course is that   
43:10   
um embryogenesis is self-organizing, meaning that it doesn't necessarily matter   
43:17   
uh what the sort of arrangement of cells are necessarily. the the process proceeds a pace unless the organization   
43:26   
of cells is really out of whack then you don't get a viable memory. So that you   
43:32   
know this is an important point that we need to kind of reproduce this multisellular aspect uh where we have   
43:39   
sort of this scalable system and this reproducible system.   
43:44   
Now they they're mimicking gastrolation and gastroloids. So these are gastrloids are organoids. They're these cell   
43:51   
cultures where you threedimensional cell culture where you have these cells that form higher order structures.   
43:57   
Gastrolation is a key event in the process of development. It's uh where the body plan and the   
44:04   
subsequent establishment of asymmetric body axes occur. So you get this um sort of some of the   
44:12   
first sort of patterns or some of the first shape differences that form. So   
44:18   
you have this sort of sphere before then and then gastrolation gives you this   
44:24   
sort of first u asymmetric transformation of the   
44:29   
shape. Coordinated gene expression during this stage leads to reproducible patterns between individuals despite the   
44:37   
noisiness of the underlying molecular events of gene regulation.   
44:42   
So this is important because you know if you had a system where it wasn't reproducible between individuals you   
44:49   
know you would have individuals that were very different. So if you had like say a species of fruitfly   
44:58   
um you would have the fruitfly would lay eggs and if you didn't have this kind of   
45:04   
coordinated gene expression you would have during different stages of development you would have fruit flies   
45:11   
that looked all different in in in all different ways hatching from those eggs. When I mean all different ways, you   
45:17   
would have fruit flies that had heads in the middle of their body and heads on the back of their body and tails in the   
45:25   
middle of their body and tails in the front of their body. And it wouldn't, you know, none of not very many of these   
45:31   
uh egg eggs would be viable, but also they would just not have any sort of   
45:37   
species identity. That's not what we see. And of course, species identity is important for it being able to behave in   
45:45   
a natural environment have a high fitness and so forth. And so   
45:51   
this is important an important mechanism in in development. Uh it's worth noting   
45:56   
that if you go to yeast uh yeast actually exhibit some sort of transient   
46:02   
multisellularity. They form these clumps of cells and this is often an   
46:08   
environmental thing. physics of the environment and you know other properties of the environment sort of   
46:14   
allow for these multisellular aggregates to form. We've talked about them in other uh like I think in archa as well   
46:21   
that this happens. Um but of course these multisellular aggregates don't have these same properties of   
46:28   
coordinated gene expression. That is the multisellular aggregates have different shapes. They're not really regular. And   
46:35   
if one of these multisellular aggregates buds off, the buds or the descendant   
46:42   
clusters will look different. And it doesn't matter because they're not reproducing themselves in the   
46:48   
conventional sense. They're just forming these functional clusters, these functional multisellular structures for   
46:56   
different reasons. And so this is important because you're reproducing yourself. You want to have this sort of   
47:01   
system. Um and so uh the challenge of translating   
47:07   
transcriptional variability into precise and reproducible gene expression patterns has captivated research across   
47:13   
the spectrum of animal models from nematodes to vertebrates. And then they site papers showing these model systems.   
47:21   
Developmental processes have been conceptualized as a sequence of steps aimed at mitigating and correcting   
47:27   
errors in the face of molecular noise. So this is how people have thought of development especially in model   
47:32   
organisms that you know each step in development is about error correction or   
47:39   
it's about mitigating noise and so you have this molecular noise you have this   
47:44   
variation that occurs across embryos and the idea is in development that needs to   
47:50   
be minimized in order to get embryos that are basically look the same or   
47:56   
behave the same or reproduce that species form with enough fitness to   
48:01   
propagate the species then that's the idea and so these mechanisms are kind of   
48:07   
built into developmental systems can you reproduce this in a gastroid is the   
48:13   
question so invertebrates mechanisms such as differential specification rates   
48:18   
and cell sorting and these are again you know like ways in which cells   
48:23   
differentiate and uh form different structures   
48:29   
And those mechanisms have been described as error correction strategies.   
48:34   
So there are different ways in which they're higher order mechanisms in vertebrate embryos. We can revisit these   
48:41   
in another meeting, but this is just kind of a point of interest here in these references 13 and 14 where there   
48:49   
you know people are interpreting these as error correction strategies. So, how does an embryo get around having this   
48:55   
noise or being in this inherently noisy environment? And there are different mechanisms for doing that that sort of   
49:01   
work in the favor of order as opposed to chaos.   
49:07   
However, in the context of the early fly embryo, the precision of macroscopic body plant features can be traced back   
49:13   
to the precision of maternal inputs. So again, this is where you know there's this maternal input.   
49:20   
you have like an egg where you have a live embryo. Um, you have these maternal   
49:27   
inputs and you don't necessarily have this in a gastrloid. The gastrloid is in culture. The cells that you get in a   
49:34   
gastroloid have sort of a lineage, but they're not they're separated from the   
49:39   
environment which they are. So when a uh a mother lays eggs or when a mother has   
49:45   
a live birth, they're inputs from the mother placental inputs that are sort of   
49:52   
important to this uh process exemplified with the morphagen gradient bitcoid.   
49:58   
This precision is transmitted at the single cell level along the major body axis to zygotic genes before   
50:05   
gastrolation. Such precision in flies challenges the limits of molecular noise suggesting it   
50:11   
successive developmental stages may have evolved to minimize noise transmission at each step both across evolutionary   
50:19   
time scales and within the spatiotemporal boundaries of individual work.   
50:24   
Um so an intriguing consequence of this precision is that the scaling of gene expression patterns relative to system   
50:31   
size. So this is kind of one of the consequences is that you have as the   
50:36   
embryo grows these gene expression patterns scale. So if you have an embryo   
50:41   
that grows maybe three or four sizes um those th those patterns maybe like   
50:48   
striping will retain themselves. They won't break down. the stripes will just   
50:54   
get wider and the stripes will get larger instead of breaking down or like   
51:01   
you know becoming maybe like aliased in some way have stripes that are overlapping.   
51:08   
Scaling observed in both invertebrates and vertebrates entails the preservation of body plan proportions among different   
51:15   
individuals during development. scaling manifests at various levels encompassing morphagenetic movements, gene expression   
51:22   
domains and other phenomena reflecting the intric interplay of regulatory. So   
51:27   
again, you know, we're dealing with gene expression, we're dealing with these regulator gene regulatory networks and   
51:35   
uh there's this interplay because gene regulatory networks for one thing overlap. So you can get certain networks   
51:42   
turned on at certain stages of development and they produce different outcomes.   
51:47   
You also have cells that interact with one another. So cell cell communication being what it is, sometimes cells   
51:54   
recruit other cells nearby to express a certain gene expression network or gene   
52:00   
regulatory network. Um other times you get uh gene regulatory networks in all   
52:06   
you know in different parts of the cell maybe like the descendant cells of a   
52:11   
certain mother cell to express this uh same gene regulatory network. And so you   
52:18   
have these mechanisms for sort of these these regulatory mechanisms. Sometimes   
52:23   
they're coordinated, sometimes they're not, but they need to be coordinated for some of these morphagen larger scale   
52:29   
morphagenetic um phenomena. In contrast to organisms well-   
52:35   
definfined developmental boundaries such as flies, frogs or worms, the male development relies on self-organization   
52:42   
and continuous growth. So there, you know, they're organizers in the embryo,   
52:48   
but there isn't like this top down uh a regulation where there's a signal   
52:55   
that's sort of it's instructing the cells sometimes, but it's not determining what the cells do. It's just   
53:01   
giving them a signal that says this is what we do in this region of the embryo.   
53:06   
And then if the cell migrates to another region of the embryo, they might um follow another uh instructive signal or   
53:14   
set of instructive signals. And if you do experiments where you displace different parts of the embryo, you can   
53:21   
get interesting results with respect to the embryo and and embryogenesis.   
53:28   
So this is something especially in mammals where you have a lot of these instructive signals and they dominate   
53:34   
over sort of like the deterministic model of sea elegance. This is an important thing to think about. In   
53:41   
contrast to organisms of well- definfined developmental boundaries such as flies, frogs and worms. Okay, we did that. Um, quantitative assessments of   
53:49   
reproducibility, precision, and scaling of malian systems has been limited prompting inquiries into whether the   
53:55   
precision observed in flies is even necessary uh as mammals rely on different developmental mechanisms. So it may be   
54:02   
that like what we see in Drosophila, which is this highly coordinated gene expression that produces things like   
54:10   
striping patterns and body segments, things like that. Um what they're   
54:16   
basically saying here is that uh you know maybe you don't need that kind of   
54:22   
precision in mammals as opposed to flies for some reason. uh mammals have different developmental mechanisms and   
54:29   
maybe we don't have to have this kind of system or maybe it doesn't exist in mammals   
54:35   
and so um there are different types of self-organizing systems there's of course scaling in flatworms during   
54:41   
regeneration so in a flatworm which is the if you're familiar with Michael Leven's work on flatworms you know he's   
54:50   
done this work where you can take a flatworm and flatworms are interesting Any cell is what they call toad, which   
54:58   
means a single cell, any single cell in a flatworm, you can take that cell, remove it, put it into its own culture,   
55:05   
and it can grow an entirely new flatworm. This is not true of most other   
55:11   
uh developmental modes, but in flatworms, it it it's the case. And uh   
55:17   
Michael Leven has done these experiments where you know you can ablate cells in the flatworm and you can observe massive   
55:25   
regeneration. So he's done experiments where he's obliterated all but one cell in the flatworm and then that one cell   
55:31   
will grow a new flat. And the idea is that there's the self-organizing system based on bio electricity and other   
55:39   
things um that reproduce that sort of structure for the flatworm so that the   
55:46   
cells that are that proliferate post uh post ablation or post um you know   
55:54   
transplantation produces the shape of the flatwork. So this is the kind of   
56:00   
self-organization and scaling that you see can see in biological systems. Now of course with that being said you go to   
56:06   
different systems you go to like mammals or flatworms or sea elegans or fruit flies or whatever and you see different   
56:14   
modes of development. Sometimes you see this this toy potency. Sometimes you see   
56:20   
plur potency where you know certain cells that descend from a mother cell have a range of fates and sometimes you   
56:27   
have this deterministic model see elegance where the daughter cells have very specific fate that they will um   
56:33   
they will default to. And so then so they get into progress in   
56:39   
in vitro models and some of the attempts at looking at in vitro models include   
56:45   
included gastroloids derived from mouse embryionic stem cells.   
56:50   
And those those kinds of in vitro models are promising in terms of giving us this   
56:55   
virtual embryo or this this pseudo embryo. And so we see a lot of different types of properties of these   
57:02   
gastroloids. um they mimic critical events in mamalian gastrolation   
57:08   
uh through self-organized patterning. For example, they break symmetry. So they they exhibit these asymmetrical   
57:14   
events. They elongate along an axis that resembles the most posterior part of the mouse's em mouse embryo's anterior   
57:22   
posterior axis and they can be cultivated in substantial quantities rendering them conducive to quantitative   
57:28   
approaches. So they they don't replicate the entire embryo. They replicate   
57:34   
critical features of the embryo. They have the symmetry breaking and this axial organization. Those are the two   
57:41   
main things that they're going to investigate in this paper. So in the interest of time, I'm not   
57:47   
going to go through much more of the text, but I do want to go through some of the figures. This is an example here   
57:55   
in figure one of these this reproducible gastroid growth.   
58:02   
So we see in this uh in in panel A we have the gastroloid we have it's sort of   
58:10   
reproducing this embryo. They show this inset here where they show the anterior   
58:15   
to posterior organization of the gastrloid and this uh recapitulates the   
58:21   
anterior to posterior organization of the embryo itself. But it's not an embryo, it's a gastroloid. So it's going   
58:28   
to look a little bit different, but basically it has the structure where you get this difference between one end and   
58:34   
the other. And so you can see that here. And then this graph shows the length in   
58:40   
micrometers and microns. And then t in and against time. So over time you get   
58:47   
this length increase and you get the scaling with the length increase of   
58:52   
pattern formation. And so then you have this I think this is volume. in microns cubed and then you   
59:00   
have um I don't know n is 10 the 4th and so you get this um oh I guess it's a   
59:07   
number of them and then over time you get this change here and so this is also   
59:13   
over time this I think is the uh the number of embryo or number of cells and   
59:19   
then the volume over the number of cells and this shows like different increases   
59:26   
and then this This is another derivative of I guess number of cells and this is   
59:31   
an increase over time. So just showing that there's a scalability in the embryo   
59:36   
in different ways to measure. So okay so I guess these notations of n   
59:42   
and notations of v. So you know the uh you have uh variability which is v and   
59:51   
then n is the number of gastroids with identical age and so it's age of the   
59:57   
gastroid and then the volume or the variation actually no volume. So they   
1:00:03   
did some investigations of these systems. Figure two this is where we see   
1:00:09   
scaling of anterior to posterior gene expression pattern the asteroids. So in   
1:00:14   
embryos, we expect there to be this difference in gene expression between the anterior end and the posterior end.   
1:00:22   
And remember, we had it kind of turned on its side, but here the anterior end   
1:00:27   
is at the top, the posterior end is at the bottom of the image. And we're staining for these different genes and   
1:00:33   
their expression. So we have sock 2, CDX2, raw, fox1. So these are all important   
1:00:41   
genes in develop mamalian development and you see that there are these   
1:00:47   
differences in terms of polarity. So you have certain gene well they're all like   
1:00:54   
broad cdx2 and sock 2 all have enhanced expression at the anterior end of the   
1:01:01   
embryo in different ways. You can see this in the color in the heat map here. And then in the bottom fox C1 that's   
1:01:08   
actually expressed posteriorly. Uh and this is just kind of showing this over I   
1:01:14   
think time. So this is where you know we have this change in also in terms of   
1:01:20   
size. So it's growing in size over time and you get this expression and you can see the expression gradient is   
1:01:27   
increasing in size even though it's the same proportion of the embryo. Uh and so   
1:01:32   
you can see that here as well. Uh this one here is just showing the   
1:01:38   
expression for this uh normalized area over time. So you have socks 2, CDX2,   
1:01:45   
bra and fox c1. And you can see actually this is uh x over length. So this is the   
1:01:52   
length of the embryo here. So this is uh anterior end here 0.1 posterior and 0.9.   
1:01:59   
So you can see that this is the uh length of the embryo. you have   
1:02:05   
uh actually I think this is anterior and this is posterior. It's flipped around. So 0.9 is anterior, 0.01 is posterior.   
1:02:13   
So you see the anterior for socks, CDX2 and bra. Then Foxy1 it's totally   
1:02:18   
reversed. You have a different gene expression pattern. And then over time there's a shift.   
1:02:24   
You can see a little bit of a shift, but it maintains that sort of pattern.   
1:02:30   
And then of course this is just kind of showing this uh sort of correlational analysis or Russian analysis here for   
1:02:39   
these data. Figure three just shows single pattern boundaries and gastro precision and gastrloids. So again we   
1:02:45   
want to have precise boundaries. We don't want to just have um you know boundaries that are kind of loose. We   
1:02:52   
want to really model the system and get the kind of precision we get in actual embryos. So here we have um an example   
1:03:00   
here of these different genes. This is the axis from 0.8 to 0.4   
1:03:06   
and then we have the variation here over length. So we can see that there's this   
1:03:12   
uh variability that we see but it's also quite precise in terms of sort of the   
1:03:18   
region in where things are expressed. So you get some variation but you also get this you know tight regulation of   
1:03:26   
spatial restriction. So you get like this green curve here which is socks 2.   
1:03:33   
You get this uh as opposed to this other curve which is bra which is this yellow   
1:03:39   
orange color. And then uh fox uh C1 is kind of in different you know it's kind   
1:03:45   
of showing up in different ways. So it varies by the gene, but you get these   
1:03:50   
highly precise uh results um at least for some genes and you you   
1:03:57   
know you don't get them all over the place usually. So that's the idea.   
1:04:02   
Okay. So I think that's it for that paper. Um, I wanted to go over that to show kind of the um, you know, this this   
1:04:10   
idea of these virtual embryos or these pseudo embryos and using gastroloids,   
1:04:18   
how that can be achieved and how technically they're kind of can be very   
1:04:23   
similar to embryos than malian embryos, which are actually harder to analyze than sea elegance embryos. And I'm not   
1:04:29   
familiar with any artificial systems for sea elegance embryos primarily because we don't really need them. Segans   
1:04:35   
embryos are very easy to work with relatively speaking and so people just use those.   
1:04:42   
So that's that's uh what I had for that paper. Did we have any questions?   
1:04:49   
What was the title of the paper again? It was precise and scalable   
1:04:54   
self-organization and mamalian pseudo. This is in nature structural and   
1:05:00   
molecular biology believe from 2024. So   
1:05:09   
anyway the differences are are interesting between different types of   
1:05:14   
embryos. Yeah. Yeah. I think and like you know   
1:05:20   
this is a system where it's just kind of a generalized mamalian embryo. So I don't mean to say that there are no   
1:05:26   
differences between mamalian embryos. I mean in terms of the mechanisms uh they're they're similar. I mean you   
1:05:33   
don't see like the sort of thing you see in flatworms in mammals. You don't see   
1:05:38   
that kind of toy potency. You see toad potency right at the beginning of our genesis from the one cell stage but   
1:05:44   
beyond that it's all pur potent to an extent. So you get this restrict   
1:05:50   
increasingly restrictive pur potency and then you get differentiated cells which   
1:05:56   
actually sometimes can be plastic even into adult but that varies widely across   
1:06:02   
mammals too. You don't just have Sorry, the dog. Um, yeah. Uh, if you   
1:06:08   
look at blood development, that's that's interesting. Arithrop   
1:06:13   
proesis there's no probably pronouncing that wrong, but anyway. Um, if you look   
1:06:20   
at that, the pur potency in that is is interesting. Yeah, very interesting stuff. Uh, yeah.   
1:06:28   
So, we don't have any more comments. Uh why don't we call it a day for a meeting and um yeah mean maybe we can follow up   
1:06:36   
on this later and thank you to um Leilith for giving a presentation on the   
1:06:42   
hyperraph stuff. I'm looking forward to seeing the paper.   
1:06:49   
Okay, thank you. Have a good week. So in light of the paper we did today, I'd   
1:06:54   
like to talk about two other papers. This one is from Lucy Han, Michael Stump   
1:07:02   
and colleagues. This is I think it's from nature and   
1:07:08   
it's called a stochastic versus deterministic perspective on the timing of some events. So we talked about sort   
1:07:16   
of stochasticity versus determinism in embryogenesis before and there's this   
1:07:22   
whole literature on how we can model developmental cells or cells in   
1:07:29   
embryionic development as either being stochastic or deterministic. So in C   
1:07:34   
elegance we have a deterministic model where a cell has   
1:07:40   
lineage or sublineage it belongs to depending on that lineage which is the   
1:07:46   
history for a mother to daughter cell you have this switch that happens and   
1:07:53   
those cells can only have a certain defined set of   
1:07:58   
fates. So you have say like a sublineage where you have a mother cell and that   
1:08:05   
mother cell proliferates into daughter cells or daughter cells and it might produce something like muscle and those   
1:08:12   
cells can't be their fate can't change suddenly in the middle of the lineage tree even with external perturbation   
1:08:20   
that produces gene expression noise. Usually that noise is buffered to attain   
1:08:25   
that deterministic fate. But there are also uh cell types where   
1:08:30   
there's a stochastic aspect to um sulfate. And so in a lot of cellular   
1:08:37   
reprogramming experiments for example or in mamalian embryos more generally there   
1:08:42   
is this stochastic sort of cell differentiation meaning that you can have uh different   
1:08:50   
sulfates. The cell can be pur potent or even to potent but the determination of   
1:08:56   
that is not switchlike but it's this sort of random process by which   
1:09:02   
depending on the signaling uh that occurs in the cell or the genes that are   
1:09:07   
expressed it produces a certain number. So there's this model of stochasticity   
1:09:13   
versus deterministic uh cell differentiation and um people   
1:09:18   
have applied mathematical models to it. So that's what they're referring to here. So if you read the abstract of   
1:09:24   
this paper um cells are the fundamental units of life and like all life forms   
1:09:29   
they change over time. Changes in cell state are driven by molecular processes.   
1:09:35   
because many are initiated when molecule numbers or the stochometry of a of a   
1:09:40   
cell reaches exceeding a certain threshold. So there's a stochometry   
1:09:46   
threshold meaning that if you have say a certain copy number of mRNA molecules of   
1:09:53   
a certain gene that's expressed you can get this you can hit this threshold and   
1:09:58   
it changes this uh maybe it changes the state of the cell. uh there's certain genes in   
1:10:06   
malian cells that are sort of uh the key pur potency cells and if you overexpress   
1:10:13   
them you can induce chlor potency in a cell um you know things like this um a   
1:10:20   
characteristic that can be described as a digital cellular object. So they're proposing that the whole area of   
1:10:28   
reprogramming or the whole area of sort of stochastic   
1:10:33   
um gene expression is sort of can form this digital cellular logic. Here we   
1:10:39   
show how molecular and cellular noise profoundly influence the time across a critical threshold first passage time   
1:10:47   
and map out scenarios which stochastic dynamics result in shorter or longer a   
1:10:53   
average passage times compared to noiseless dynamics. We illustrate the   
1:10:58   
dependence of mean first passage time and noise for a set of exemplar models of gene expression autoregulatory   
1:11:04   
feedback control and enzyme media catalysis. Our theory provides intuitive insight   
1:11:11   
into the origin of these effects and underscores two important insights. So the first is that deterministic   
1:11:19   
predictions for cellular event timing can be highly inaccurate when molecule   
1:11:24   
numbers are within the range known for many cells. And then two molecular noise can   
1:11:30   
significantly shift mean first passage times particularly with an autoregulatory gene gen genetic   
1:11:37   
feedback. So you know this is kind of speaking to   
1:11:42   
this idea of using stoeometry as a switch to determine fate. So if I   
1:11:49   
have a certain gene that maybe is a key gene in determining frequency or   
1:11:54   
determining self or determining maybe the ability to switch a selfie,   
1:12:02   
I have to have a certain amount of that in the cell. And so this is of course   
1:12:09   
has to be at a threshold of some type. So the accumulation of this molecule has   
1:12:14   
to be you know it's not an even process. Sometimes if the process of gene   
1:12:20   
expression is uniform you can reach that threshold within a certain amount of time. But of course they're they're   
1:12:28   
recycling dynamics. uh you know the molecules degrade the   
1:12:33   
molecules are produced um and and sometimes they're bound by other things in the cell and so the amount of that   
1:12:41   
molecule is going to the the sort of the net amount of that molecule is going to change and so you have this time to   
1:12:49   
threshold that can vary across cells especially when you're talking about uh   
1:12:55   
you know noise and gene expression like we talked about in the last paper where gene expression this sort of constant   
1:13:01   
pulse of production but it's cyclical and it can be stochastic in the sense   
1:13:08   
that it's random in in that cyclical nature so it can pulse at different   
1:13:14   
scales. Uh so this is the kind of thing they're they're thinking about here. Um this   
1:13:20   
first point here uh deterministic predictions for cellular event timing can be highly inaccurate. So if we use a   
1:13:27   
deterministic model to say um that a certain if we have a certain population   
1:13:33   
of cells and we overexpress a gene that those cells will all become a certain   
1:13:39   
type of cell or all become very potent. You can't assume deterministic model   
1:13:44   
fit. you have to assume that not only do uh cells have to reach this threshold   
1:13:51   
stoometric threshold but that they're different in every cell and so you know   
1:13:56   
I remember when I used to work in um cellular reprogramming we spent a lot of   
1:14:03   
time thinking about this problem it seems like a very trivial problem but if you introduce genes into a cell and you   
1:14:10   
overexpress certain genes um you know how how what's a timed sort   
1:14:16   
of like a threshold where those cells can become equivalent. And it was not an   
1:14:22   
easy task because it's not like you just introduce   
1:14:28   
the gene into each cell and then the cells would become potent after a certain amount of time. Some cells will   
1:14:34   
become very potent quickly. Some cells will become parapotent later and some will never become very potent because   
1:14:40   
the process underlying is stochastic. The second point here is that molecular   
1:14:47   
noise can significantly shift mean first passage times particularly with an autoregulatory genetic feedback.   
1:14:55   
So molecular noise plays a lot of large role in some of this variation. But also   
1:15:03   
when you have gene circuits that are autoregulatory meaning that they are   
1:15:08   
regulating themselves especially in the presence of some signal that pushes.   
1:15:15   
So this is all to say that cellular dynamics are not deterministic.   
1:15:20   
observation of RNA or protein number in single cell show that variation with time is stochastic.   
1:15:27   
And so they get in with the idea of expression noise and biomolecular noise   
1:15:33   
and how that can affect copy number and stoometry within the cell. So we can use   
1:15:41   
the chemical master equation to model probabilistic descriptions of chemical reaction kinetics. And this has helped   
1:15:48   
us understand some of these mean behaviors of cells. But the mean behaviors of cells are just that they're   
1:15:55   
mean behaviors. They don't represent every cell all the time. There's a lot of variation around that mean. That's   
1:16:01   
what we kind of want to understand. And so this is where this this whole area comes from about thinking about the   
1:16:08   
stochasticity of these processes and how they deviate from the chemical master equation and   
1:16:14   
how we can get um you know we can at least appreciate that. I don't know if   
1:16:21   
we can really find a different predictive model that will help us because stochastic models generally   
1:16:27   
aren't predictive and that's the whole point. They're stochastic. So uh this gives a little example from   
1:16:34   
cell cycle dynamics. So for example in cell with development and differentiation progression through   
1:16:40   
different phases of the cell cycle only occur when certain cyclic proteins surpass a prescribed threshold.   
1:16:48   
Many physiological processes are also triggered by a certain threshold. And so we need to have sort of these thresholds   
1:16:56   
and we need to figure out what the dynamics are leading up to these thresholds. And that's kind of how we   
1:17:01   
the best we can do in terms of predicting this from cell to cell. So this is an example here first passage   
1:17:07   
time which is to say that you know you have this uh you know once once the   
1:17:13   
amount of this molecule hits the threshold it triggers some event and   
1:17:19   
then the amount of this protein or mRNA concentration can fall below the   
1:17:25   
threshold and that event has been triggered and it doesn't matter after just needs to hit the threshold once and   
1:17:32   
the question is how fast can it hit the threshold given these stoastic dynamics. So this   
1:17:38   
is where we have protein number y ais time on the x- axis and protein number   
1:17:44   
builds over time and it's fluctuating and if it hits the threshold it triggers this apotic event if it never reaches   
1:17:52   
the threshold which is possible it goes into cellular arrest. Sometimes there's   
1:17:57   
a time limit on this process. So it's time limiting in the sense that if this   
1:18:03   
concentration never reaches the threshold before it reaches a certain time limit then it'll by default become   
1:18:10   
go into cell for rest and if it hits the threshold before that time then it goes into atosis but once it hits the   
1:18:18   
threshold makes a decision about fate sometimes it's time limited sometimes it's not   
1:18:24   
this example here in B shows that there's this high v uh high degree of   
1:18:30   
variation in the uh first passage time. So like if you again we have protein   
1:18:37   
number versus time. So we're dealing with p-53 which is a protein um and you   
1:18:43   
know we can use any protein and there are different profiles for this but we have a number of different cells where   
1:18:49   
p-53 is expressed and you see that it forms this distribution around the first   
1:18:55   
passage time. In other words, um, you know, there are different times when it hits the first passage time because of   
1:19:01   
the stochastic dynamics, but you can actually look at those that variation   
1:19:07   
and plot out a I guess a Gaussian curve which describes sort of the mean time   
1:19:13   
the first passage and then some variation. So when you have the   
1:19:19   
when you you're talking about variability and noise, you're talking about this uh the stochometry here this   
1:19:27   
this amount of molecular you know the number of molecules in a in a sample or   
1:19:33   
in a cell but also it's time to first passage which is also so we have this   
1:19:38   
process we have this gene that's being expressed we have these kinetics to produce proteins of different sizes   
1:19:45   
there's this burstiness in protein production uh and then we have this target number of degraded proteins which   
1:19:52   
is this you know we have to basically ally wait a certain amount of time to get a certain number of proteins. And so   
1:19:59   
this is the model here. And so we have in this case we have protein number over   
1:20:04   
time. So we have our protein number and we're looking at variation. We have   
1:20:09   
these uh stochastic dynamics. We have protein number over time here which is   
1:20:15   
showing this. We have the log. We have sort of the   
1:20:22   
power law dynamics here where we have ADA over log k and then we have this   
1:20:30   
other set of dynamics characterized by this power law it over log k where we   
1:20:36   
can see that it's not a linear process it's actually a nonlinear process.   
1:20:43   
So one of the core parts of their model here is this bursty birth death process. So we consider the bursty birth death   
1:20:49   
process with fixed birth size r. So a birth death process is where you look at a pool of some molecule cells organisms   
1:20:59   
and this the number of constituents in that group is governed by two things   
1:21:07   
births and deaths. So those constituents have to be born at some and then they   
1:21:12   
have to persist and as they persist they're still there as as constituents   
1:21:18   
are born that adds to the number of constituents but then some of those constituents die and they're removed   
1:21:24   
from the pool. So the birth death process governs the size of a set of   
1:21:29   
constituents in a population of cells or individuals or molecules. And so we that   
1:21:36   
process can be bursty meaning It's not linear. So you you know you think about births and deaths like of humans in the   
1:21:44   
hospital where there's certain periods where a lot of babies are born at other periods where there's being born and   
1:21:52   
that's normal. You also have that cells where there are a lot of cells being born at some points and no cells being   
1:21:57   
born at other points. If you look at you know you do a census at any given time you can see that birth rates are bursty   
1:22:04   
but also death rates are bursty. Sometimes there's a lot of aptosis in a cell. Sometimes there's very little   
1:22:11   
aptosis in a cell. Sometimes or in a in an embryo. Sometimes the same thing   
1:22:16   
happens with necrosis. These can be natural fluctuations or they could be due to some stimulus on the outside.   
1:22:24   
We consider the burst bursty rift death process with fixed burst size R. So they fix the burst size and they show that   
1:22:31   
basically this is the model here. Um it is assumed that there there's a continuous production of a particular   
1:22:38   
species P in bursts of fixite R according to a posone process a constant   
1:22:44   
rate K. So a plusone process is modeling arrivals and it's nonlinear it's a it's   
1:22:51   
an exponential distribution and then there's this rate K of the process which   
1:22:56   
is constant. The degradation process captures both with active degradation as well as dilution due to cell division   
1:23:03   
and is modeled as a first order bone process with with a rate.   
1:23:08   
Throughout the remainder of the paper, we scale time is towel equals your variation times time and assume all   
1:23:16   
model parameters of scale but the degradation rate. So that that equals one and both time and rate parameters   
1:23:23   
are not dimensional. Bursty birth death process serves as a prud model for bursty expression either   
1:23:30   
RNA or protein due to transcription or translation. We do not here account for   
1:23:35   
the fact that birth size in these cases is not fixed but is distributed according to a geometric distribution.   
1:23:42   
So they go from a simple birth death process to explain that then to this bursty uh case   
1:23:51   
and then thinking about different burst sizes. So burst size   
1:23:56   
is this parameter R and it can be zero or it can be greater than one.   
1:24:03   
And so then that determines sort of variation in the process and you know   
1:24:11   
how long it takes us to reach that first passage. And so to model bursty gene expression   
1:24:17   
we can use something called a telegraph model. Um this is where we have   
1:24:23   
uh we want to know whether or not the same transition in the ratio of the deterministic to stochastic waiting time   
1:24:30   
from below to above one is possible. And so there are these effective reactions here in the model. The gene   
1:24:37   
switches probabilistically between inactive state G star and active state G   
1:24:43   
in which mRNA molecules are M are synthesized. degradation of mRNA occurs   
1:24:49   
independently of the promoter state. So you have this switch between G star G   
1:24:54   
and inactive to active state and that determines how mRNA molecules are   
1:25:00   
synthesized and then the degradation rate may be constant and that's different from the production of mRNA.   
1:25:07   
So your pool is you know it fluctuates stochastically because you have a   
1:25:13   
parameter that's controlled by the synthesis rate but then you have a different process the death process   
1:25:18   
which is a different set of dynamics and that leads to this sort of burstiness in   
1:25:24   
the population. Then there's this auto regulatory feedback loop in which a protein   
1:25:29   
produced by a gene either enhances or suppresses its own expression. Uh we see this commonly in biology. We   
1:25:36   
see this in E.coli where we have negative autoregulation uh for around 40% of known transcription   
1:25:43   
factors. The number of studies that use stochastic models to investigate optimal feedback mechanisms minimize the   
1:25:50   
variability. Uh the reaction scheme for the feedback loop we employ is given here. This shows this where G star and G   
1:25:58   
represent the bound and unbound states respectively and P represents protein.   
1:26:04   
So you have these parameters that involve the unbinding and binding rates   
1:26:10   
and then the variation is the protein degradation rate.   
1:26:16   
And so this is just a feedback loop described by these parameters   
1:26:22   
and it describes sort of fluctuations in in this process where we have earth's   
1:26:30   
deaths and then how does it break and so we measure this by looking at the steady   
1:26:35   
state mean. So again the steady state mean helps us characterize this sort of process but it by no means describes the   
1:26:42   
full reach of the process. So they talk a little bit about the difference between deterministic and   
1:26:49   
stochastic model prediction. Um so for simple birth death processes the   
1:26:54   
deterministic model prediction provides a strict upper bound predicted by the stochastic model. In other words,   
1:27:01   
intrinsic noise leads to a reduction of the trigger time. This effect becomes more appreciable as the mean steady   
1:27:07   
state molecule number decreases. For the bursty birth death process, there is a   
1:27:12   
switch from the regime intrinsically reduces the mean trigger time one where it lengthens it as the target molecular   
1:27:20   
value is decreased. So there's this uh relationship between trigger time   
1:27:26   
between passage time and then between between those and then the size of the   
1:27:32   
population through the storm since the expression of many genes is bursty of transcriptional translational   
1:27:40   
origin. The simple model suggests that the transition may exist in more realistic models gene expression.   
1:27:46   
Indeed, we show that the same transition occurs in the holographic.   
1:28:00   
And so this this brings us to autoregulatory feedback. So for feedback loops we observe significantly higher   
1:28:06   
ratios of deterministic to stochastic FPTs compared to previous models. This   
1:28:13   
has implications for timing of processes governed by molecular clocks where   
1:28:19   
dynamics are often modeled deterministically. So these are things like circadian clocks   
1:28:24   
andospora or segmentation clocks and vertebrates. Additionally, our results provide   
1:28:31   
insights into the non-monotonic dependence observed in both the deterministic FPT and MFPT on the   
1:28:37   
feedback stream. And this is non-monotonic behavior that has been shown in stoastic thresholds.   
1:28:45   
The effect of feedback on the differences between the predictions of the deterministic and stochastic frameworks are well known to increase   
1:28:52   
with decreasing system size or volume of the system and with increasing proximity to the point in parameter space where a   
1:28:59   
system switches from stable to unstable dynamically. So in a lot of stochastic systems uh   
1:29:07   
predictability is hard because when the stoeometry is small the number of   
1:29:13   
molecules is small uh there's a large amount of variation variation will be   
1:29:18   
amplified but small and that's uh so when you have small numbers of molecules   
1:29:25   
it's the variation much higher uh when you're dealing with larger numbers of   
1:29:30   
molecules or larger pools of molecules those differences, you know, stochastic fluctuations of less of an effect and so   
1:29:38   
it's easier to predict. So at the end here they say our study has shown that intrinsic noise has   
1:29:44   
non-trivial effects on the timing of cellular events that simple models could provide intuitive insights into the   
1:29:50   
microscopic origins of the hope whether implicit or explicit that deterministic   
1:29:57   
models predict accurate mean quantities appears misplaced for the MFPTS which   
1:30:02   
have important implications for modeling based studies biochemical type. So as a   
1:30:08   
note here, FPT is first passage time. MFPT is mean first passage time. So we   
1:30:14   
use FPT and MFPT. That's what we're referring to.   
