    
Transcript    
0:02    
Hello. How's it going? All right.    
0:07    
I uh I am new. Okay. Yeah, I see that.    
0:13    
So, how are you? Uh why don't you introduce yourself? Sure. Um my name is Paul Mandel. Uh dual    
0:21    
degree in computer engineering, applied math. Okay. Uh have a masters in cyber, masters in    
0:26    
computer science. Most of my research is in AI and machine learning. Um, but I    
0:32    
right now I'm looking at moving over and doing an MD PhD. Uh, and I want to do a    
0:38    
lot of um, I guess a lot of uh, computational biology and neurology    
0:45    
research. That's okay. Why attending the meeting? Yeah, that's great.    
0:51    
Well, thank you for uh, attending. Of course. Yeah.    
0:57    
I'm not sure who's going to be here today, but um usually we have a pretty    
1:03    
good uh set of things that we do here. So, yeah,    
1:08    
let's see.    
1:17    
Usually, yeah, we usually have more people, but like for now, we'll just start. And here we go. There's Leith.    
1:25    
Hello, Leith.    
1:41    
Hi Lilith. Uh, how are you?    
1:55    
I don't know if you wanted to give an update or not.    
2:12    
Okay, why don't we start? Um, yeah. Hi,    
2:17    
there's a tick. Hello. Good morning. Morning. Oh, good. Leth here.    
2:23    
Yeah. Yeah. Leth is a hard or soft day a leith.    
2:32    
Okay. Yeah, he's    
2:37    
Hello, Paul. Where'd you come from? Uh, I'm I'm new. Uh, my background is in    
2:44    
in CS. Uh, but I want to do computational biology stuff. So,    
2:50    
I didn't have my sound up. Okay. What's your background in? Uh computer science prim primarily AI    
2:56    
and ML. Uh but looking at moving over to computational neuroscience.    
3:02    
Computational. Okay. Okay. Good. We'll have a model for the    
3:07    
brain. So yeah, welcome to the meeting. Um and    
3:14    
today we'll go over a couple things. I'm going to go over some papers that I've been sent and then uh an article on    
3:22    
openworm which would be interesting kind of talk we talked about a little bit in the slack.    
3:29    
So um this is uh I wanted to go over    
3:34    
this real quick. This is a recent meeting that has held in France um at the beginning of June. This is called    
3:41    
shaping life three and um it's a very splashy um    
3:48    
uh program and it's basically about developmental biology. So they have um    
3:55    
a number of different talks here. Um I mean you know you're not going to get much more than that I don't think    
4:01    
because I don't think it was on online uh or recorded. Um so you know there    
4:08    
keynote lectures or other lectures. So the keynote lecture here the first one was shaping the maternal inheritance by    
4:15    
the cytokeleton. Then they had uh this session one new tools and technologies.    
4:22    
Uh they had this invited speaker visualizing morphogenesis across modalities.    
4:29    
Um another talk compressive stress drives morphagenetic aptosis    
4:34    
cell lateral tension and paso1 activity. So you know a good amount of things on    
4:41    
mechanics of morphagenesis and different ways to deal with that. Uh    
4:48    
this other talk chromatin mechanics at the onset of mouse zygotic genome activation.    
4:55    
So some genomic stuff in there. Uh this one highly dynamic mechanical transitions and embryionic cell    
5:02    
populations during Drosophila gasast gastrolation. So another mechanics paper    
5:09    
or another mechanics talk. Bradley, can you put the URL in the chat for that?    
5:14    
Yeah, I don't know if I have URL or at least the title of the conference. Yeah, it was Yeah, shaping life three.    
5:22    
Okay. This was held in France. Uh Cass France    
5:31    
the first two. Yeah. Um so then there was morphagenesis one    
5:38    
which is the first or the second session. Um there was neuronal dendritic    
5:44    
morphogenesis. Um mechano sensing and epithelial tissue in vivo.    
5:50    
uh endopplasmic reticulum networks uh and uh retinoic acid signaling to the    
5:57    
myioardium drives morphogenesis. So a lot of you know uh molecular stuff    
6:04    
uh mechanics and um other things.    
6:11    
Uh then they had this session three Evo DVO and emerging models one. This was um    
6:19    
let's see if you're familiar with Evo DVO you know kind of the topics that they like to to hit. Um they had this    
6:26    
one talk here embryionic development of a distributed nervous system in octopus vulgaras.    
6:33    
So that was kind of talking about the nerve net of the of the octopus.    
6:38    
Uh how a single cell shapes a chute. This I believe is plant morphagenesis.    
6:46    
Evolution of temporal patterning of neural stem cells. Again you know the sort of differentiation in the brain.    
6:54    
uh squaring the hexagon cellular and molecular mechanisms mediating square cell packing in the crustation.    
7:02    
Oh, that's fun. Yeah. So, yeah, we talked about that last week about the shape of cells and    
7:08    
well, actually, uh we talked about why people use hexagons in modeling, but um    
7:14    
you know, there's there is an issue of like why are cells a certain shape as well?    
7:20    
Sinnot has a uh an illustration I think from the 1930s a cross-section of a    
7:26    
plant in his book plant morphagenesis uh which uh uh is square the cells are    
7:34    
arranged in a square array okay okay    
7:41    
I'll put the [Music]    
7:58    
there. All right. Oh, yeah. For Genesis. Yeah. It's It's a very good book.    
8:06    
Okay.    
8:12    
Good morning, Hussein.    
8:17    
Hi. Hi. Hello. Hi. Hi. Yeah. Hey, how are you? Nice to meet you. Nice    
8:24    
to meet you. All right. Um, and then yeah, so you    
8:30    
know, of course, they have methods interspersed in a lot of these conferences. So, this was uh sponsored    
8:38    
by Nikon. This one, intelligent and automated microscopy to extract sample    
8:43    
behavior in one click. So always a lot of things on like how to make microscopy    
8:50    
better. Um you know so uh then there was morphogenesis 2 which was the session    
8:57    
four um and again from embryo shape to the onset of zygotic gene transcription.    
9:03    
So your head uh molecular um studies    
9:08    
this one fluid or elastic direct measurements of m material properties unveiled the early embryionic mechanical    
9:15    
response to morphagenetic forces. So this sort of soft materials    
9:20    
um and material science applications to developmental biology.    
9:26    
Uh and of course you had morphogenesis and drosophila wing disk and this one    
9:33    
osmotic control of mamalian embryo size. So again more things to do with uh size    
9:39    
and shape regulation and development and then of course they had the session    
9:45    
gene regulation one uh which involved a number a lot of    
9:52    
molecular work um as you might imagine.    
9:57    
uh involving embryionic patterning and epithelium morphogenesis    
10:03    
and doing things like spatial transcripttoics and other types of molecular studies.    
10:11    
Then there is physiology and metabolism which is always nice divergence from molecular methods. Um    
10:19    
and this one had a couple of talks here. One was adaptive morphagenesis of a    
10:25    
cellular trap in the developing vertebrate heart. This one was a multisellular actin star    
10:32    
network underpins epithelial organization and connectivity. So a lot of things with respect to the    
10:40    
um sort of you know the the sort of components of the cell and components of    
10:48    
the tissue here. Uh then there's Evo DVO and emerging models too. Uh more of the sort    
10:56    
of Evo DVO approach which is of course the evolution of development. So you're talking about um you know it's not    
11:05    
always strictly about evolution. A lot of times it's about diversity and how    
11:10    
that arises in evolution. So this one here divergent evolutionary strategies    
11:16    
preempt collision between expanding tissues during fly gastrolation. So you have different evolutionary    
11:23    
strategies. This is multisellular developmental diversity at the root of animals    
11:29    
and um a model for bilateral symmetry    
11:35    
and then uh exploring the links between tissue morphology and healing in a    
11:40    
spatial model of sulfate choice. Yeah. And then of course there was a    
11:45    
session on stem cell lineage and regeneration that's sort of regenerative medicine and a lot of things to do with    
11:53    
like cancer research and thinking about how do you model sulfate transitions    
12:01    
and using things like organoids and gastroloids. Now, so this is um    
12:07    
true definitely this actually this one here uh invited speaker why some worms    
12:14    
regenerate and others cannot. So if you're familiar with some of the worm models that we have we have of course    
12:20    
nematodes and then we have flat or round or flatworms instead of round worms.    
12:26    
nematodeses or round worms and then their flatworms and flatworms in contrast to nematodeses are famous for    
12:33    
regeneration and you know and this is the the system that Michael Leven works with for    
12:39    
example this is the one where uh the system where you can take a single cell    
12:44    
from a flatworm and regenerate an entire flatworm from that single cell it's an    
12:50    
amazing sort of question about the nematodes I came across a paper recently    
12:56    
where somebody cut a developing neatode in half. Yeah.    
13:01    
And then each half developed mosaic. So it was diff each half was    
13:07    
different. Do you know if one can stick those halves together again?    
13:12    
I don't know about that. Well, I mean, you know, that would be interesting, but I don't know. Yeah.    
13:20    
Yeah. Yeah. I shouldn't say that all nematodes    
13:26    
can't regenerate. It's just like in general the flatworm is the the model system for    
13:32    
that. Yeah. I don't know. It's that's interesting. Okay.    
13:37    
Yeah. If you could send me that paper, it'd be interesting. If I come across it again, yeah.    
13:44    
Yeah. So that you know and then there's this question of course about regeneration. Why does some organisms    
13:51    
why do they have this regenerative capacity and why others don't? And of course if we could figure out what    
13:58    
causes regeneration we can replicate that say in humans or in other organisms    
14:03    
to see if we can figure out how to leverage regenerate regenerative    
14:08    
capacity. Why are we like mimods? Yeah. Well, not I mean, you know, again,    
14:15    
we we have some regenerative capacity, but not nearly as much as people would like, right? It's uh    
14:22    
yeah, you can regrow your fingernails, but not your spleen or your heart or    
14:29    
even you can grow re regenerate some of your heart, but not enough to    
14:35    
save you in a lot of cases. Okay. I'm sorry. I'm sorry to interrupt. Did this conference have any any published    
14:41    
proceedings or was it just um I think this was just talks so they don't Okay, I was trying to pull up the    
14:48    
article. Hence hence why I asked and I I couldn't Oh, I put the URL in the uh chat.    
14:57    
Yeah. You can check it out, see if they published anything.    
15:02    
Oh. Oh, yeah. No, I I'm I'm looking by the author now, but I tried to type it.    
15:08    
Okay. Yeah, they usually don't not in the biology conferences usually don't    
15:13    
publish a proceedings with it as much. Yeah, I try uh    
15:32    
you can try try these reference sources uh to look up. Yeah, I'm on Google Scholar right now.    
15:42    
So yeah. So uh then there was gene regulation 2 which of course is more    
15:48    
it's uh there's an intersection between the gene regulation topics and evo    
15:53    
because evo is really a lot of it is about how do you    
15:59    
you know how does gene regulation evolve over across fogyny. So you know you'll    
16:05    
you'll see a lot of kind of overlap there. Um this is conserved eye muscle program through development and    
16:11    
evolution for example. Um and this one comprehensive analysis    
16:17    
of three prime UTR changes is an active process during zebra fish early    
16:22    
developments. This is about DNA regulation and then um there a couple    
16:28    
other talks as well. And so that was the last session. Um there was a final    
16:34    
keynote understanding fruit flies in context toxins AI and behavior. Again    
16:41    
there's this focus on merging biology with artificial intelligence methods and    
16:48    
you know getting gaining understanding from that. So it's a very hot area now    
16:53    
every time you uh has anyone done a paper on how old Evo    
17:01    
Divo is now? Well, um, yeah, I don't know. Seems like    
17:07    
an interesting topic. I guess it like kind of came together in like I don't know the 80s, maybe    
17:15    
maybe the 90s. Well, there were studies that were published and you had some landmark studies. I don't know where you    
17:20    
demarcate the beginning of a field, but like the first mention of it maybe would be the place to go, but I I don't know.    
17:28    
Maybe the first mention of it might be as late as the 90s. I'm not really sure, but it might be an interesting thing.    
17:33    
Yeah. Yeah. It seems to me that its promise has not been realized.    
17:38    
Yeah. Yeah. Some people argue that it's not a theory, that it doesn't really    
17:44    
function like a traditional theory. Um, you know, and you think about like a theory, it's supposed to, you know, give    
17:51    
some new set of explanations and it it's kind of like, oh yeah, development    
17:57    
happens in evolution, too. So, but in any case, yeah, I mean it'll be    
18:02    
interesting to see and that's something that, you know, I know that like is pretty um    
18:09    
special 1970s. Okay. Yeah. I'm I'm sure I'm sure I'm sure those    
18:15    
questions uh can be evaluated in terms of the power of the theory looking at potentially explanatory, predictive and    
18:22    
other forms of uh theoretical advantage that it might give. you know, like one    
18:28    
one common bottleneck for many potential scientific theories is they may be descriptive, but they're not so    
18:34    
predictive, you know. Yeah. But it's it's interesting. It's interesting. Yeah. Um like the the    
18:42    
predictive power of those theories would give it potential to to argue that it it it is it is a theory. It's not, you    
18:49    
know, just some random explanation that we don't need. So yeah.    
18:55    
Okay. Um, yeah. So, thank you. Um, so before    
19:00    
we go on, I don't know if Hussein wanted to give an update on his work. I mean, I    
19:05    
know he gave one last week, but Oh, um, sure. So, once again, we're    
19:11    
working on stuff.    
19:16    
Cool. Yeah, I put the I put the, uh, Gro Gro's view on it in the chat.    
19:23    
Don't Don't trust Grog. Don't trust Gro. Oh, I know that    
19:28    
it's far less reliable than chat GPT and the others. It's Oh, really?    
19:34    
In my opinion, you're even even the better models of of Chad GBT, like the ones that use Rag, they    
19:40    
still make references. It's It's horrible. Oh, blatantly    
19:46    
blame blatantly wise Brock Gro just has his own problem with that. So,    
19:53    
um anyways, okay, excuse me. Uh yeah, so working on a couple different projects    
19:59    
uh alongside Zick right now. most mostly the CT project, the RI computer    
20:04    
demography project, but also the peptide project on the origin of peptides    
20:09    
through simulation, how they've been formed over over    
20:14    
whether or not we can we can describe the statistical mechanical model for the lipids first hypothesis and simulate the    
20:20    
early peptide formation division using these 2D hexagonal grids which we had    
20:25    
talked about earlier. Okay. Yeah. So, I'm still working on that, getting the figures and the images    
20:30    
together and consolidating a lot of the writing that Dick had mentioned earlier. One important point that Dick had    
20:36    
pointed out was that I think that the um excuse me, it seemed like Dick had    
20:44    
raised this argument that the what was it that the alpha helys the one    
20:49    
particular hypothesis you you mentioned doesn't hold so much water. Is that correct? Was it about the alpha helyses?    
20:54    
Alpha alpha alpha alpha helyses is that one. No, it was the the problems with the    
20:59    
RNAs. Oh, right. It was the It was the RNA RNA precursors have been detected in    
21:05    
meteorites, but nobody's figured out whether they have kirality. Okay. Yeah, that was the that was the    
21:12    
and we prop we proposed looking at the Mmic. Uh yeah, it looks like the bas the empirical basis for going after them uh    
21:20    
may not be there. Okay. Okay. Well, that's that's important to point out and I appreciate you mentioning that. Yeah. So, just just    
21:26    
to um just just to back up for a bit once again, this is a project that Dick and I are working on with the colleagues    
21:33    
over in Italy with Sabino and his colleagues uh Gala and Gracia. Gracia. Gracia.    
21:39    
Yeah. And um well the basic idea is that the alpha    
21:46    
helix is hydrophobic and uh it can be fairly short. So it can    
21:55    
anchor a peptide in the membrane which is hydrophobic. Yeah. Yeah. Okay. Okay.    
22:02    
And then there's a positive feedback uh uh uh which we think should occur. We    
22:09    
have to simulate it carefully and that is hydrophobic mismatch where the uh    
22:14    
length of the alpha helix is not the same as the length of the uh of the    
22:22    
amphopils that make up the uh the meteorite vesicles.    
22:28    
The meteorite vesicles have been demonstrated but nobody's actually analyzed them as far as I know.    
22:35    
uh uh however uh the components of them are fairly wellnown and they uh I just    
22:41    
came across something interesting. They seem to be daroxyic acids which means    
22:48    
that both ends are charged for the the molecules in the membranes.    
22:54    
uh and that's the same structure as you find uh ina.    
23:02    
Okay. Both ends charged and you don't get a billayer, you get a monollayer but    
23:07    
with both ends charged of the same thickness. Okay. Okay. Yeah. So that's um    
23:16    
Okay. That's that's interesting. And um    
23:21    
um Okay. Okay. And that's how they have the denovo fatty acid synthes right.    
23:27    
Yeah. Deletion of the N oil uh caroxyic acid    
23:33    
reductase gene. Okay. Okay. Well, there no we're assuming there no    
23:38    
gene there no enzymes around or genes or anything like that.    
23:45    
So the the process we're trying to simulate is strictly physical chemical.    
23:50    
Okay. Okay. And the energy source is one that's generally ignored by people who are    
23:57    
worried about energy and that is just the rotation of the earth.    
24:03    
Okay. Okay. Okay. Uh which presumably can drive    
24:09    
temperature or wet dry cycles and therefore lead to polymerization of peptides.    
24:24    
Okay. Yeah, I'm glad you're being thorough with the which hypotheses have been explored and which ones haven't.    
24:30    
So, yeah. Yeah. Exciting stuff. Exciting stuff. Yeah. I mean, some some of these are    
24:36    
experimentally approachable, but uh I'm excited.    
24:43    
Cool. So if you want to trade in your computer for a test tube, go ahead.    
24:50    
Yeah. Speaking of which, um, did you guys appreciate that article I'd sent about the the worm in wired?    
24:56    
Oh, the one on in the in the Slack that you posted. Yeah. Yeah, we'll talk about that in a little    
25:02    
bit. Yeah. The open worm project saying that the open worm project despite being a decade    
25:07    
plus has been an utter utterly an absolute failure.    
25:15    
failure of what? Yeah. Well, I mean, I I I read it a    
25:21    
little bit differently, but I mean, I know that like they Yeah, I don't I don't think the article was quite that vindictive. Uh I do    
25:28    
remember when the open worm project came out though. It's like, oh, this is a small worm. We can gain a full    
25:34    
understanding over the next five to 10 years. And, you know, things went very differently. Um    
25:42    
yeah, they they literally start off by saying the project for 13 years a project has    
25:48    
tried and utterly failed to simulate it. You you can say they're trying to    
25:53    
present the challenges about it, but it sounds like they're just they're just like bashing it. Didn't Tom Portugus do a make a robot    
26:00    
that acted like a worm? Yeah. Yeah. So, I mean, yeah, a couple words about that. Well, maybe we'll get    
26:07    
into the article now, but um yeah, a couple words on that. The first one is is that we did a we talked    
26:13    
about an article uh several months ago maybe where they had kind of the same    
26:18    
claim that like you know any kind of simulation is of uh and then they talked    
26:26    
about open worm in there that any kind of simulation has failed and then like kind of talking about how to how they    
26:33    
could do it better and how you know it's kind of uh this is a way to do it but I    
26:38    
don't think that their way tons of roadblocks, tons of bottlenecks, tons of challenges,    
26:44    
but I I say it's very very far from looking failure. You want to fail, you're looking at human brain project or    
26:50    
something like that. Like, yeah, that's been that's been that's been a train wreck for years.    
26:57    
Well, we we talked about that. I've been trying to get the number of    
27:02    
different kinds of cells in brains and there's almost no literature on    
27:09    
Yeah. It's Yeah. That's the thing about like a lot of these projects and and we talked    
27:15    
about the human brain project as being like like if you're aiming for like    
27:21    
simulating or emulating a biological system. I mean there are a lot of ways    
27:26    
you can fail at that goal but you produce a lot of things underneath it. Even if it's simulation    
27:32    
you produce a lot of potential useful things. And like at the human brain project, they produced a lot of things    
27:38    
that help people um you know doing brain science or simulation or whatever. And    
27:45    
so it's not that sense it's not a failure. And of course you know you can do these simulations and you still learn    
27:51    
things you still build sort of these mo you test to see if models    
27:57    
work to simulate some aspect of the process. So like you know yeah Tom    
28:02    
Portiges has been working on like building uh C elegance inspired robots. Uh he's    
28:10    
been working on other things. He's kind of working on this old school artificial intelligence where he's you know he that    
28:17    
was I think what he did his PhD dissertation on. And you know it's just like this kind of cottage industry of    
28:24    
his where he's simulating um you know different types of intelligence and    
28:29    
things like that. So you know you're learning things about how to do simulations.    
28:35    
You're learning things about the phenomena itself. So it's it's interesting in that way.    
28:42    
Oh, I mean I also need some perspective. Uh uh those of us who were born before the    
28:49    
year 2000 uh are quite aware that for decades artificial intelligence was regarded as    
28:56    
a total failure. Right. Yeah. Honestly up until people have said    
29:02    
it's an absolute failure, you know. Yeah. like like no no nobody cared about AI    
29:08    
making pictures of people if it's like if they look like trash up until this point. So    
29:15    
yeah. Well, I mean yeah. So why don't we go to this article? Um I I app I appreciate that it's once    
29:22    
again just changing the tone here rather than some like straight up cynical dumerism that it's all a fatalistic    
29:27    
project that hasn't worked out. It's more so like a holy grail type thing. Yeah.    
29:32    
Yeah. Yeah. Yeah, I think I don't think a lot of outsiders who don't do research understand that    
29:38    
science is incremental and sometimes you're you're brutally fighting to just gain an inch    
29:44    
and uh in your research area, right? And it's clear that the author doesn't understand that because they even say at    
29:51    
the at the very end that like author the author she's she's a she's a tech writer. She's written about tech.    
29:57    
She knows about computers. She knows how computers work. She's not just saying like like how come we can't just tell the computer like hey program how a worm    
30:05    
wiggles and then it's just right but but the question is are they a researcher like keep in mind    
30:12    
how different regard regardless I think I think the uh you know and talking about who wrote it specifically I think    
30:18    
that the um you know um excuse me um    
30:25    
like it's it's it's a it's a great it's a great article in presenting the challenges.    
30:32    
It's a great article in presenting the challenges of the open world project for the past 10, you know, the past decade    
30:38    
or so as they mentioned. Um, but it's u like like how how incremental    
30:46    
everything has to be for years and years and over decades and stuff like that's what they're trying trying to get out,    
30:52    
right? So that's why I say it's more like a holy grail analogy rather than just like like like an absolute like    
30:59    
like dumpster fire of a project. That's what I mean. Yeah. Oh yeah.    
31:07    
So yeah, let's go to the article here if I can pull it up here.    
31:13    
Right. Yeah. This is uh the worm that no computer scientist can crack. And you    
31:19    
know it's it's there's this history actually of different sea elegance-based projects. And of course we've talked    
31:25    
before about why C elegance is this thing that people want to simulate. Um    
31:31    
you know it has um a fixed number of cells in the adult. We know what the    
31:38    
sort of the lineage we know the full lineage tree. We know what every cell will become. So it's both deterministic    
31:45    
in terms of cell differentiation and it's what we call utellic or has the same number of cells in the adult like    
31:52    
you know every hermaphrodite can has the same number of cells every male has the same number of cells and we know what    
32:00    
developmental cells will become what adult cells so we can simulate it cell by cell and so that and then you know    
32:07    
part of the problem here is that that's just the cells and then you have    
32:12    
simulate the connections in the conneto you have to simulate the molecular millu    
32:18    
inside the cell and we know we have pretty good genetics of C elegance we    
32:24    
know how you know we have the full genome sequence we have uh a lot of    
32:30    
defined mutants so we know what genes actually do in a lot of cases but still    
32:35    
there's a lot of biology there you know and so it's like you can simulate at    
32:42    
different levels of understanding. You know, you can produce like an output    
32:48    
which is like a worm moving around in a simulation that looks a lot like a worm    
32:55    
that is in real life. But then you still have this problem of like is it producing it through exactly the same    
33:01    
means or is it just kind of a similar uh looking process but it's producing it    
33:07    
differently because you're not modeling everything that goes into a war movement    
33:12    
you know. So there are questions like that and and you know it's it's it's fine but there have been because of that    
33:19    
sort of simplicity there have been a lot of projects that have come and gone um    
33:25    
going back to the 1990s. And so that that there's this history here of like you know people kind of    
33:33    
taking that bite and then you know kind of not uh producing anything. I mean    
33:40    
maybe they produce paper or something. So in that sense, openworm hasn't failed at all because openworm has kind of    
33:46    
started off and you know there have been a number of different software projects open source software projects that have    
33:53    
kind of resulted and then what they refer to in the article about running the worm. It's a basically a program    
34:01    
that runs um in a virtual machine that you can run all the programs at once and get an    
34:08    
output. So you can run you know cybernetic which is the bioysics    
34:14    
um the neuron simulations and other things and then run those and    
34:19    
get an output. So it's interesting that you know and and then that's kind of where we've been kind of    
34:26    
we've kind of been for a while you know the the goal of course would be to to merge all that together um into a into a    
34:35    
unified simulation that works. So, um, that's that's kind of the the issue    
34:41    
here. I mean, you know, I guess it's not necessarily an issue. It's just that like if you're looking for an easy to    
34:48    
run simulation of the worm, you can see that like that may or, you know, that    
34:53    
may or may not be like a viable uh goal at this point or maybe any other    
34:59    
point. We have to kind of take steps to get there. We can't just do that in like 10 years to get to the full sort of an    
35:07    
emulation of one and and if you wanted to do an emulation of an organism I mean    
35:13    
you could do single cell simulations or single cell organisms. So we talked    
35:18    
about in this group about some of the initiatives people have done with bacteria like modeling like maybe like a    
35:26    
bacterial cell with a minimal very small genome and then you know how how much    
35:33    
that takes. So like um there are a number of groups modeling whole cells    
35:38    
and you know and these are like bacterial cells that where one cell is the entire organism    
35:45    
and they they to do that they do a lot of uh modeling of the genome, modeling    
35:51    
of the proteome, modeling of the cell morphology and just with that one cell    
35:57    
it takes massive amounts of computational power to generate like molecular dynamic simulations.    
36:05    
maybe at the scale of a couple of seconds. So there's a lot of you know    
36:11    
if you're doing they mentioned they mentioned you know they they repeat that quite a bit 10 hours of compute time    
36:17    
five five seconds of an animation. Yeah they should I mean if you ever if you ever use    
36:23    
Blender um that's how long it can take at times too. I think I I I made like a    
36:28    
20 second animation of a donut spinning and I think it took my laptop like like    
36:33    
two hours to render. Yeah. Yeah. Yeah. It does take a long, you know,    
36:38    
especially when you get into donut tutorial from Blender. Yeah. Yeah. The progress on computers is    
36:44    
incredible. Uh when I first uh dealt with image processing,    
36:51    
the only digital images I could find were pictures of clouds.    
36:56    
uh taken bite oh which were very hard to work with. No    
37:01    
sharp edges, nothing. Okay. Uh and uh then uh somebody came    
37:09    
out with a movie of a uh showing a face citing the poem how brown cow.    
37:18    
Yeah. Yeah. And they pointed out it took them 45 minutes per frame to generate the movie.    
37:25    
Yeah. Yeah. Okay. I remember that one that there's a the Gibly scene of this one Studio Gibli    
37:32    
movie that is like three seconds of a crowded crowded like shop mall type place. So a lot of people walking in    
37:39    
many different directions in an out like market type shop. I think that that entire three seconds    
37:45    
of animation took multiple weeks for their studio to make.    
37:53    
Yeah. I mean, computers are so much faster now. That was all that was all hand drawn. That's the one I always think of. But    
38:02    
yeah, okay. The art the article does does do a lot of good merit for challenges and struggles for Okay. I really appreciate    
38:10    
that. I think it's really solid with pointing out, you know, from a more objective manner the the good sides, the    
38:15    
bad sides of these things. Um, and really putting in perspective how    
38:22    
just how difficult it is when you say I'm simulating a worm on my computer and how difficult that really is from from    
38:28    
biologically and scientifically realistic perspective. So, yeah.    
38:34    
Yeah, I should prepare. Oh, go ahead. Naive question about worms. Do the nerve    
38:40    
cells generate uh spike trains? um not in the way that human neurons do    
38:48    
or mamalian neurons. So they generate like a sort of a semi-action potential.    
38:53    
I guess they generate something like spike trains, but yeah, it's a little bit different than what we know in    
39:01    
Okay. Even that has to be modeled. You can't you can't just    
39:06    
Well, the openor has actually made some pretty good progress on doing stuff like that. Yeah. I mean like    
39:12    
Oh, okay. Yeah. because it's it's really a large component of it is the neuroinformatics    
39:19    
uh aspect because you know we have this connection with INCF. So a lot of the neuronal stuff is that's    
39:25    
that's like a core kind of thing that is of interest you know. So like one of the    
39:32    
reasons why divaorm started was because they didn't really have any good account of development. They had a lot of stuff    
39:38    
on uh connetoics and and the the ner    
39:44    
nervous system but not necessarily on development. So Okay. So it's really been 13 years since    
39:49    
we started this. Uh about 10 and a Oh, open worm is about    
39:54    
been about 13 years. Yeah. Yeah. Okay. Yeah.    
39:59    
Okay. So yeah. Well, like I said though, I should prepare like a a little chart to show like the molecular dynamic    
40:06    
simulations and their computational power requirements and they are time    
40:11    
simulated and then compare that with like some of the openworm stuff that you can produce in a virtual machine on your    
40:20    
laptop which is interesting because you know uh I don't know if there are a lot of containerized    
40:26    
molecular dynamic simulations. I imagine there probably are, but like just showing that you need to have and    
40:33    
especially depending on what you're Yeah. trying to simulate a lot of power for    
40:39    
that. I just need you to make make the jump into uh virtual research as well too.    
40:46    
So, you know, the these these articles in the pop site pop science communications popsite community. They're nice. They're nice like    
40:52    
reminders of what we're doing like to communicate those things to the general audience so they can understand this    
40:57    
stuff too. So um    
41:03    
yeah it is it's uh yeah it is what it is. Cool stuff. So    
41:10    
like I think I think the the biggest um you know remember this is all completely like open sourced right?    
41:16    
Yeah. I feel like the purpose for this idea that if we just collectively gather the power of anybody on every anybody on    
41:23    
the planet who wants to work on this can do it right to get it done as quickly as possible and they're vastly    
41:28    
underestimating how difficult that is too in many ways you know conducting open source can be pretty pretty pretty    
41:35    
confusing at times so um like like an open source research project or just like a simulation type thing it's um it    
41:42    
takes a lot of coordinated up effort yeah    
41:47    
it is cool though it is cool though and I appreciate that like um at least compared to many other open source efforts in science and research. Like    
41:54    
for my background in bioinformatics and computational biology, a lot of the open source products they would make and do    
41:59    
in there still felt pretty like closed off and guarded by the academic community. And I felt like they were    
42:05    
never nearly as responsive responsive to new data, new results, and new power as    
42:10    
quickly as they could be because people just gay kept the hell out of there. And it was it was just it would just be like    
42:16    
pretty simple like like like gene splicing and annotation software, right?    
42:22    
That would never update itself as quickly as it could with you know computational power as a whole for for    
42:29    
technology and software. So yeah. Yeah. Anyways, this um this is this is um I do    
42:36    
I do overall appreciate the article. I do. So yeah. All right. Yeah. So this was published    
42:43    
in Wired magazine. This was uh I don't I mean it's online so it's the worm that    
42:50    
no computer scientist can crack. Yeah, it had a lot it was a lot of interesting things here. Um then they talk about    
42:57    
Steven Larson and Cory Gleason and you know these are two of the people from    
43:04    
the project two of the senior contributors. They also talk about Neta Cohen who's on the scientific advisory    
43:10    
board um and she does a lot of interesting work in like uh doing you    
43:16    
know C elegance uh wet lab work but also computational work    
43:22    
interesting um some of a lot of the things here um yeah so this yeah it has    
43:30    
a lot of the points here that we talked about um someone said here who's Haspel    
43:36    
uh Gal Haspel a computational neuroscientist at the New Jersey Institute of Technology and the lead author in the    
43:44    
reverse engineering paper. This is that paper that was published recently where they proposed taking this you know kind    
43:52    
of new approach to C elegance where you go back and you do experiments and you    
43:57    
bootstrap some sort of uh total simulation from that. So another attempt    
44:02    
at this sort of thing. Um and uh he says estimates that pulling it off it being    
44:09    
this simulation of total simulation or emulation of sea elegance may take up to    
44:15    
10 years cost tens of millions of dollars and require something in the neighborhood of 100,000 to 200,000 real    
44:23    
life worms. So this is like where to do simulations you need to do uh wet lab    
44:30    
biology and seg or in openworm has relied upon    
44:35    
um you know secondary data sets different segans labs and sea elegance    
44:41    
is nice because you have a lot of labs that are devoted to doing things in C    
44:46    
elegance and there's this nice community where they uh share their data and they    
44:51    
share their reagents and they share their defined mut students. So you have access to a lot of these things across    
44:58    
labs. But there what they're proposing in this reverse engineering paper is to    
45:03    
to sort of build the infrastructure up kind of from scratch and then do these    
45:09    
mass experiments where you're doing like you're collecting lots and lots of data    
45:15    
and then training the model and you can actually do very controlled experiments    
45:21    
relevant to the simulation. So it's very much connected to that uh instead of    
45:26    
just kind of you know getting as much data about the worm and then kind of running a general simulation.    
45:32    
But of course that also means that that costs a lot of money and takes a lot of time.    
45:38    
So in the process of this it would generate more data about segans than has been collected in all of science to    
45:45    
date. And what in the end will the reverse engineers have to show for it?    
45:50    
all these people and all these computers, Haspel said, and wind up doing what one little animal can do    
45:56    
right now. Um, he compared that project to a NASA moonshot. It's the kind of    
46:01    
undertaking that drives technology forward, pushing engineers to build better tools and scientists to work    
46:08    
together. And so this is where if you combine enough automation, big data,    
46:14    
machine learning, you get this simulation or emulation of an organism. So this is kind of going through a lot    
46:21    
of this sort of the this this whole area of how do you simulate an organism and    
46:30    
um again you know see elegance the reason people are interested in that and and again we've done like these single    
46:36    
cell simulations that are very detailed at the molecular scale and our single    
46:41    
cell organisms so they're not particularly interesting in terms of behavior or even physiology, a lot of    
46:49    
it. Um, and that's why we want to do sea elegance because sea elegance, aside from being simple, which it's not, but    
46:56    
it's simple enough to get a handle on, you still get this interest these interesting behaviors and these    
47:03    
interesting sort of things going on in development and and in um, you know, in    
47:10    
in the life history of physiology.    
47:16    
Yeah. So that's that's uh I think we'll move on from that paper now. I think that was a good um kind of discussion.    
47:23    
Thank you for to everyone who had some thoughts on that. We'll continue to discuss it in the Slack. It's in the    
47:30    
D.VARMM channel of the openworm slack. So if you want to have more conversation on that,    
47:37    
that's the place to go. I did want to kind of go over before we    
47:42    
go a couple of papers that I've kind of run across that were sent to me etc.    
47:49    
This one was um we we talked I think last week and the week before about mechanical    
47:56    
uh aspects of development. This is a nice paper on control of    
48:02    
epibblast sulfate by mechanical cues. So this might be of interest to people like Susan. Um, and this is where they're    
48:11    
looking at embryionic tissues originating or multi- potent epibblast cells arranged in a mosaic or    
48:18    
presumptive territory. So they're looking at like how um tissues differentiate but they differentiate by    
48:26    
spatially into different different locations in the or in the emerging organism. Uh this is a mamalian. This    
48:34    
actually this is an amnotes. So this is not in sea elegance or in drosophila but    
48:40    
this is an amnotes. So it that's a a portion of um a verte of the vertebrate    
48:47    
fogyny. Um how these domains faded to specific lineages become segregated    
48:54    
during body formation remains poorly understood. Uh using single cell RNA    
48:59    
sequencing analysis and lineage tracing in the chicken embryo. So they use chicken as their model organism. We    
49:06    
identify epibblast cells contributing descendants to the neural tube somites    
49:11    
and lateral plate after completion of gastrolation. We show that intercolation after cell    
49:18    
division generates important movements of epibblast cells which lead to their relocation of different presumptive    
49:24    
territories explaining this broad spectrum of fates. So this is combining    
49:29    
sort of cell differentiation and migration and then the mechanics of    
49:35    
these cell populations the sheets moving around folding doing things like that.    
49:43    
This tissue remodeling phase is transient being soon followed by the establishment of boundaries restricting    
49:49    
cell movements therefore defining the presumptive territories of the epibblast. Finally, we find that    
49:55    
epibblast faces distinct mechanical constraints along the anterior posterior    
50:01    
axis or the AP axis leading to sulfate alterations when challenged. Together we    
50:07    
demonstrate the critical role of mechanical cues and epiblast fate determination. So this is um you know    
50:14    
this is again is in amnotes. So development is a bit different than    
50:19    
how it uh proceeds in C elegance or even in Drosophila. But the the take-home    
50:25    
message of this paper is that you have this sort of spatial aspect to    
50:31    
development. You have this mechanical aspect to development and you have this    
50:36    
aspect of organizing cells in sheets and in sort of these um the shape or these    
50:45    
geometric structures. that then determine a lot of what happens. And so,    
50:50    
you know, we we see these things in development, but we can manipulate them as well and get interesting outcomes.    
50:58    
So, that's, you know, if we're interested in cell mechanics and in the organization of cells into these larger    
51:04    
structures, that's an interesting paper. This is a paper on active pneumatics. We    
51:11    
talked about pneumatics um back I think maybe two meetings ago.    
51:18    
This is has to do with how cells are oriented in development. So are they    
51:23    
oriented all in different directions in terms of the sort of the um the way in    
51:29    
which they move the way in which maybe their oblong shape is oriented to one    
51:34    
another. Um and so if it's uh semeic versus pneumatic and so like you know    
51:42    
you have these different in pneumatic tissues everything is oriented in the same direction and that has consequences    
51:49    
for how development proceeds. And you know, you can think about like, you    
51:55    
know, cells is engaging in this collective behavior where maybe they all move in the same direction during    
52:01    
migration or they're all oriented in the same direction so that they go on to form things like muscle fibers or you    
52:08    
know or things related to um anatomical    
52:13    
orientation. So um this is kind of thinking about this uh maybe larger scale physics than    
52:22    
single cell physics and how that plays a role in development and shaping    
52:27    
phenotype. And so this has this author here, Matias Sarah.    
52:33    
Um this person I think has um done a lot of stuff on uh Lebronian    
52:41    
uh coherent structures which is an interesting approach to physics where    
52:48    
you look at flows and you try to predict these meta structures that exist in    
52:53    
flows and you know you can find lrangeian and coherent structures in    
52:59    
flows where they form these ridges and you can calculate from certain an origin point to the edges of these ridges. And    
53:06    
you can you know say things about maybe the collective migration of cells or in    
53:12    
a lot of physics applications you can say things about the flow itself. What is the flow? You know the flow it has    
53:19    
maybe a certain degree of turbulence um or laminarity by the way in which the    
53:26    
things that are moving in that flow and where they end up. So if you have this    
53:31    
uh very comp you know a ridge with a very complex shape you can say that that flow is turbulent and you know there are    
53:39    
a lot of things you can do with that tool. In this case they're looking at material    
53:45    
transport and fluid flows and they're looking at these objective oilerian coherent structures.    
53:52    
Um and so you know they were talking about this um in terms of like active    
53:58    
materials. So it's not necessarily related directly to development, but    
54:04    
they say that there's an application domain in terms of synthetic active matter and morphogenesis. So if you're    
54:10    
doing things like modeling uh you know cellular structures and we've talked a lot about organoids, you know, there may    
54:17    
be applications to organoids or to morphogenesis in actual organisms.    
54:24    
Um and so they talk about um how objective oil and coherent structures    
54:31    
and instantaneous leaponov exponents govern short-term material transport and fluid flows. And then they talk about    
54:38    
these labran coherent structures and then the way to measure that is through    
54:43    
this finite time liipov exponent. And so you know you use that exponent to    
54:49    
describe these coherent structures and that describes something about the flow    
54:54    
the flow that governs say like migration or movement of things over time.    
55:00    
So um and then of course you have this oilerian rate of strain tensor    
55:07    
which is something we can also use to understand these flows. So here we    
55:12    
devised for the first time an optimal control strategy to create short-time attractors and compressible viscosity    
55:20    
dominated active pneumatic flows. So this is again having to do with treating    
55:26    
development and and cell migration and and things like that as these compressive viscosity dominated flows.    
55:34    
And so you can actually do this to model how things are going to behave in development or in some sort of cellular    
55:42    
system. By modulating the active stress intensity, our framework achieves a    
55:48    
target profile of the minimum value of the rate of strain tensor controlling    
55:53    
the location and shape of these short-time attractors. So they're using dynamical systems uh theory to predict    
56:00    
the location and shape of these sources of maybe like these ridges that I discussed. Those are the short-time    
56:07    
attractors. So the ridges kind of move over time. You're measuring from an origin point to some the edge of a ridge    
56:14    
at some point in time and that ridge changes shape and moves away from the origin point or maybe even towards the    
56:20    
origin point over time and you're looking at the dynamics of that system. So you want to have some account of um    
56:28    
you know order in the things that are migrating the things that are moving around and so that's how you can    
56:34    
describe them through these ridges or these short-time attractors. We show that our optimal control    
56:40    
strategy effectively achieves desired short-time attractors while rejecting disturbances meaning that you're getting    
56:47    
the actual signal instead of artifacts. Combining optimal control and coherent    
56:52    
structures, our work offers a new perspective to stereomaterial transport    
56:58    
and compressible active pneumatics with applications to morphogenesis and    
57:03    
synthetic active matter. So this is a pretty hardcore physics paper um but it    
57:09    
does have some applications to um biology and development. So uh they're    
57:17    
interested in these large scale coherent dynamics where global collective behaviors arise from local interactions    
57:24    
as well as describing individual and isotropies and different types of ubiquitous    
57:30    
activity in such systems. So what kinds of systems are we thinking of? Well,    
57:35    
like I said, we have developmental systems, but we can also use this type of methodology to describe bird flocks.    
57:41    
So collective behavior of agents bird flocks or cells you know in these kind    
57:48    
of computational models sometimes we can describe the movement of bird flocks or    
57:54    
the movement of cells and we don't have to distinguish between the things that drive them or like the brains and birds    
57:59    
and the uh sort of the uh biochemical or    
58:05    
kinetic pathways and cells because they produce very similar behaviors and we just simulate the things that lead to    
58:11    
those behaviors or analyze the things that come out the outcomes of those behaviors.    
58:17    
We can also m model bacterial swarms or ensembles of cells that exhibit    
58:22    
macroscopic patterns whose length scales orders of magnitude larger than the individual size. So you know this this    
58:30    
kind of goes into some of the uh biological domains that we can use we    
58:36    
can use this method with and um and the paper gets into some of the uh    
58:41    
theoretical methods that exist now and how this improves upon.    
58:48    
Okay. Um this last paper here I wanted to highlight is this PNAS paper and this    
58:55    
is um dynamic forces drive cell and organ morphology changes during    
59:01    
embryionic development. And again this is more again about um mechanics and    
59:07    
about how we can um you know develop models to understand these mechanical    
59:13    
forces and some of these other aspects of cell migration. So the abstract here    
59:21    
reads cells, tissues and organs must change shape in precise ways during    
59:27    
embryionic development to execute their functions. So we're dealing with shape    
59:32    
and shape changes both in cells and then in tissues which are collections of    
59:38    
cells and then organs which are collections of tissues. So we're dealing with these three different scales. And    
59:44    
so we need to understand not only that those different scales but then the the changes in shape and how to to analyze    
59:53    
that and understand and of course those shape changes involve multiple mechanisms. So biochemical signaling    
59:59    
pathways and biohysical forces play a large role in that. So if we model those    
1:00:05    
things, we can understand we can have a working model of how those shape changes work and how to predict them in    
1:00:12    
different systems. It's been difficult however to tease apart the contributions of each    
1:00:19    
especially from tissue scale dynamic forces that are typically ignored. So these are you know this is where you    
1:00:25    
have things happening at multiple scales. You have dynamic forces at the tissue scale that affect the cellular    
1:00:32    
scale. But how that works is hard to understand because tissue scale dynamic    
1:00:37    
forces are collective phenomena depends on the cells contributing to the tissue.    
1:00:43    
So it's hard to get to tease all that out and and build an effective predictive model.    
1:00:50    
We use a combination of mathematical models and invivo experiments to study a    
1:00:55    
simple organ in the zebra fish embryo called cuper's vesicle or KV. Modeling    
1:01:01    
indicates that dynamic forces generated by tissue movements in the embryo produce shape changes in KV that are    
1:01:08    
observed during development. laser ablations in the zebra fish embryo that    
1:01:13    
alter these forces result to al altered organ shapes matching model predictions.    
1:01:19    
These results demonstrate that dynamic forces sculpt cell and organ shape during embryo development. So this is    
1:01:27    
again this a very key question in developmental biology. How do organisms    
1:01:32    
robustly control the morphology of cells, tissues and organs given that    
1:01:38    
information for such structures is ultimately encoded at the scale of molecules. So you have these things    
1:01:44    
happening at the molecular level are being expressed in cells. Then that becomes part of the the collective    
1:01:51    
behavior of cells becomes part of the tissue and then those tissues interact to form organs.    
1:01:58    
So if we go to the significance uh you know they talk about how we aim to understand the mechanisms that    
1:02:05    
control precise cell and tissue shape changes occurred for organ function. So    
1:02:11    
it's not enough just to predict shape changes generically. You need to have this sort of model of precision where    
1:02:18    
this change in shape results in this change in function and to be able to predict different changes in shape and    
1:02:25    
different changes in function of which ones are the most predictive. So that's a challenge. You know you can predict a    
1:02:31    
lot of times you can predict shape changes or what leads to shape changes but then you have this additional aspect    
1:02:37    
of function and so that's the the thing that they're looking for. Many studies    
1:02:43    
focus on cell shapes have ignored the role of dynamic forces self-generated by    
1:02:48    
slow tissue flows. So again we go back to the flows of tissues flows that you    
1:02:54    
know we can model using techniques from physics that we often use to model soft    
1:03:00    
active matter and other types of physical phenomena. But recent work showing that tissues are    
1:03:06    
near a jamming transition with diverging relaxation time scales suggests that    
1:03:11    
slow motion could give rise to large forces. So this jamming transition we know it happens in development happens    
1:03:19    
in soft active materials and that is where if you get enough cells or enough    
1:03:25    
particles in a volume of liquid it can induce this jamming transition which is    
1:03:30    
a phase transition where those that population of cells or particles can go    
1:03:36    
from moving sort of in a laminer flow to jamming up and stopping and then nothing    
1:03:42    
moves. So you could have a a a laminar flow where you have a certain density of    
1:03:48    
cells or particles and then as that density increases say through cell    
1:03:53    
division you can end up with this jamming transition where everything freezes up and nothing migrates and you    
1:03:59    
see this in development. We've talked about a number of papers where you see this jamming transition and you can have    
1:04:06    
unjamming transitions as well where things open back up. And so one of the aspects of development uh or one of the    
1:04:13    
open problems is how do these jamming transitions play a role in cell migration and and how does it govern    
1:04:20    
development. So they do this mathematical modeling imaging and use these mechanical    
1:04:27    
perturbations to look at the tissue scale dynamic forces sculpting the shape    
1:04:32    
of an epithelial organ in the zebra fish because many processes during development occur at similarly slow    
1:04:39    
rates. This suggests that self-generated dynamic forces should be investigated    
1:04:44    
more broadly. And so this is an important point here is that in development, you know, we have uh events    
1:04:51    
sometimes in the same type of system that we're working in that happen at many different time scales. So we just    
1:04:57    
talked about how you know it takes a lot of computational power to model a certain time scale. So if you have more    
1:05:04    
details in your model, the shorter time scale you can really afford to have to model something precisely. So if we have    
1:05:12    
these slow time scales, it's going to be hard to incorporate everything from the    
1:05:19    
molecular scale up to the to the organ scale uh and get an effective or even    
1:05:25    
any sort of uh coherent simulation out of it. So I thought that was another    
1:05:31    
interesting point that they made. So I wanted to go through those three papers. I think you know those maybe kind of    
1:05:37    
highlight some of the things we've been talking about um in the recent in recent    
1:05:42    
times. So we have yeah we have some things in the chat. Let me go over those. Um let's    
1:05:49    
see let's go up to the top here. Uh so yeah we talked about saut platogenesis.    
1:05:57    
We have a new paper on shape. This is uh Dick's paper. Is there a relationship between embryionic electric currents and    
1:06:04    
differentiation waves that's in bios? Um and then Paul said I think some of    
1:06:11    
the top polls from this paper this is um    
1:06:18    
relation to the worm regeneration stuff. Okay. Yeah. The author who gave the talk was the    
1:06:24    
senior author of that paper. Oh good. Yeah. Great. Thank you.    
1:06:30    
This is evolutionary dynamics of whole body regeneration across plenary and flatworms. Yeah. Oh, that's great. Well,    
1:06:36    
thank you for that. Um, I'm useful sometime.    
1:06:42    
Oh, good. Thank you. Uh, yeah, Dick had this question here. When did Evo Divo    
1:06:48    
start? And, you know, that's something that maybe is interesting from a biblometric standpoint and might revisit    
1:06:55    
that. Um then uh Hussein said everyone's just so obsessed with numbers.    
1:07:03    
Um and then you were talking about you were talking about like all the scales, you know, like oh this takes 10 hours to do    
1:07:10    
something. It takes a run to do 5 seconds. I was like cool. Well, congratulations. What does that what does that tell you, you know?    
1:07:17    
Yeah. Yeah, definitely. Um yeah,    
1:07:23    
this is also from Hussein. I have the option of taking CSE67.    
1:07:29    
Yeah, a few a few courses in my M's program that I I can now take electives for in the next semester, the next year    
1:07:35    
or so. And um I think one of them even used pineet logo at one point to net lo    
1:07:42    
net logo. Net logo. Remember we used that for the GOC project. Oh yeah, Net Logo. Pinet logo. But um our M's    
1:07:50    
program capstone at the end involves like a like a practicum which is basically more or less like a like a    
1:07:55    
self-run student project that uh we get a full project course course grade and    
1:08:01    
stuff and things for. So yeah    
1:08:08    
yeah um so this is uh what program are you in by the way?    
1:08:16    
MSAMSA the online all masters of science and analytics through Georgia Tech Georgia Technology.    
1:08:22    
Okay. Yeah. Yeah. That's great. Well, thank    
1:08:27    
you. Um, yeah, I had I had I had a class this past semester. It was really annoying. I had    
1:08:33    
to take four proctored exams in this past summer. That just that was it.    
1:08:38    
Yeah, it's it's just it's just annoying. It's just annoying. I feel like they all know that the intro class is just    
1:08:43    
hurdles you have to jump through. So yeah.    
1:08:49    
Yeah. After reading about the experience with graduate algorithms, I was like screw this. I'll be the University of    
1:08:55    
Texas. Yeah. I mean there there are different programs like like uh Jesse did uh UCSD,    
1:09:04    
right? For masters in data science. Okay. U Michigan's really good too. UCSD. U    
1:09:09    
Michigan. U Texas. Maybe maybe Purdue has one. I'm not so sure. probably do probably as too. So,    
1:09:17    
Georgia Tech's uh AMSA is really great because it's completely online and you just have to take about 10 courses at    
1:09:23    
your own pace. So, that's basically so.    
1:09:31    
All right. All right. Well, I think that's all for today. Um, thank you for attending and u    
1:09:38    
see you next week. Thank you. Bye.    
