0:00     
hello   
0:05     
how are you I'm showered Oh yeah That's good All     
0:11     
right Hello I Hello [Music]     
0:17     
So yeah Uh with Did you do anything Do you have anything to follow up on from last time     
0:26     
or Susan Hello How are you     
0:32     
Um pretty good I managed to get the first draft of my     
0:39     
paper written about my thesis So Oh good     
0:45     
Congratulations What was it to a journal     
0:50     
Um I haven't decided Okay It's just in the in the uh loaded up and     
0:58     
ready to go Well I just I'll get it to a journal I     
1:04     
don't know which ones uh Dr Zang would like it to go to but um I wondered if it     
1:12     
would be [Music] um good for the usual journal bio     
1:21     
systems Um maybe Yeah     
1:26     
I I don't really care Just I promised to do this properly and I had to publish it     
1:35     
So yeah Anyways um and I would like to do     
1:41     
some more work on it actually It's just that I need a less expensive program or     
1:47     
more readily available one So maybe I get to learn answers Yeah     
1:54     
I don't know Unless     
2:00     
they give it to me for less than uh $3,000 US I'm like     
2:08     
"No that's kind of expensive don't you think?" Yeah     
2:14     
Yeah That's good Good luck with that whole process     
2:20     
[Music] Uh way you were gonna say something So uh I like uh we discussed about uh you     
2:30     
know the hyperraph structures and you know message passing and we discussed about there are like few things that we     
2:37     
discussed in the last week uh that uh you know we have to see what could be     
2:42     
good data set uh you know to understand the uh the interactions and the     
2:48     
mechanisms and the biological interactions that they would do the causal mechanism     
2:55     
that uh that uh you know each and every cell you know divides during the     
3:01     
morphogenesis and there are various biological interaction patterns You wanted to look at to some of the data     
3:07     
sets and in last previous meeting uh we discussed that you know uh there were     
3:13     
some of the works where they also included phase angle uh as one of the     
3:18     
component to you know they incorporated into the structural features So and uh     
3:26     
yeah I was going through the papers like the know the big list of papers that     
3:31     
Dick has attached like last last week and uh I was trying to read this uh uh     
3:38     
the work uh so which is uh like I can share PDF so it's more of     
3:46     
fusion morphogenesis uh that's one of the paper he suggested so I was looking into that you know how     
3:52     
this can be you know the cential cells how we can group them using a hypographs     
3:57     
and all So I kind of have a know thing that I can show you Okay     
4:05     
So I tried of using that you know incorporating the phase angle uh into     
4:10     
the you know our temporal linear lineage graph tree and you know to find out the     
4:17     
you know uh how is that phase angle you know changing when we divide the you     
4:23     
know cell from the parent to like two child So this is what I have seen for uh the     
4:30     
first uh first division it has around like you know uh 9     
4:37     
198.9° and later the next division it has 195 which is the next subsequent     
4:44     
divisions next subsequent divisions and so on So I was able to capture this uh     
4:49     
exact phase angle So this like a simple formulation that we use for like how do     
4:55     
we calculate phase angle So we can calculate the phase angle using this     
5:00     
function This is for the xy plane So so given uh so the theta is the phase angle     
5:06     
So this is like tan inverse tan inverse of 2 * the the y-coordinate change and     
5:12     
the x coordinate change And uh so this we get in 0 to 360° So this is the map that we get So x     
5:21     
come x1 and y1 are like the coordinates of two daughter centroids So based on     
5:26     
that centroids we are able to plot this So this was for for three time points I     
5:32     
have I mean like shown So this we can incorporate into like the all the time points     
5:39     
So yeah so this is one thing which was you know kind of     
5:45     
little took time and also I was looking into one of the paper that uh you know     
5:50     
for the data sets so what they have done in this paper is uh they have a lot of     
5:56     
data set so they have C elegance volutric microscopy data set consisting     
6:02     
of like standardized power they have pre-process and they have a human in loop machine learning based analysis     
6:08     
tools also So they have a diverse collection of one minute whole brain neural activity imaging data sets they have acquired     
6:15     
from the five labs and they store it in warm id.org So this is their uh warm I mean     
6:23     
like their community where they have like data sets open sourced and uh yeah     
6:29     
how do we extract them and analyze like what are the things that they have provided is uh so they have this     
6:36     
automated cell identification algorithms and uh they have recovered all the cases     
6:41     
and they have a repository to identify factors that influence the developmental positioning of neurons So they wanted to     
6:49     
identify this So this this is their task So for this they have like 118 whole brain neural activity images of that So     
6:57     
this has like an app properly developed and the pipeline looks like this I haven't gone to the paper entirely but I     
7:03     
have looked into this This is like one of the important uh work that I've seen So this is how they have So they have     
7:09     
designated into color spaces and uh they have plotted the angle and at each of     
7:15     
the activity they have this uh traces they have two different types this like calcium based and acquisition and uh     
7:23     
this neuropal acquisition So this is like raw neural     
7:28     
activity and this is neuropal structural volume So this was like what they have provided I wanted to look into this uh     
7:36     
more probably if uh you know if we are I mean like trying to understand the     
7:41     
biological insights and all so probably later in time so you can look into more     
7:47     
details about that this data sets is available in this like for open source so we can we need to spend little time     
7:54     
into this to understand more so I can share this paper in the group so that it     
7:59     
will be useful people so this is two things and I've been reading about     
8:05     
fusium morphogenesis and how that work could be you know useful for our you     
8:10     
know developmental uh embryogenesis and all how can we form this indivial cells together and wire     
8:17     
them and uh like the fusion the cell membrane divides and you know like     
8:24     
certain components are are like you know uh this cell membrane is like a big thing so when it does this fusion and it     
8:32     
I mean like divides into like two different things What are the key changes that actually happen so we can     
8:37     
also model that Yeah that would be good to have Uh yeah thank you Dick for all     
8:43     
the papers that you sent You probably about 20 papers Yeah I was able     
8:50     
to find only one thing that could No I can understand a bit more They were all     
8:58     
more of biology and all Yeah Yeah So no that that paper you just     
9:04     
showed looks really interesting It's like uh Yeah Is this a new one like just recent I guess it's very new one that uh     
9:11     
it was recently published in like cell reports methods like 2020 January 2025     
9:19     
Yeah Okay Yeah I think that's the two things when I calculated the phase angle understanding the you know that uh it     
9:28     
can also give us so some more structural insights that can be embedded into all     
9:34     
the time points so that's one thing and second is uh I was able to understand     
9:39     
the fusomogenesis and read about little more into the data sets that uh recently     
9:46     
have you know can be directly used for like this hypographs so that     
9:52     
the biology that we discussed I mean like the     
10:02     
interactions that's it from us Okay How's it going Thank you     
10:09     
So yeah thank you Lewis Um so looks like uh Jadratha and Priyanka are here Um     
10:17     
would you like to introduce yourself Priyanka Sure Yeah sure Uh my name uh my     
10:23     
background because I'm at home So hi I am Priyanka I'm doing my PhD at Kansas     
10:28     
State University right now I'm in my third year of my PhD So I got to know     
10:34     
about this project from uh GOC and I found it a little bit interesting to     
10:40     
give you a little bit background I did my masters from IIT Gandhiagar in India and I worked as a data scientist for     
10:46     
three years and then in fall 2022 I started my PhD Right now I'm working on     
10:53     
um influence maximization as well as craft neural networks where the idea is to build     
11:00     
scalable and uh transferable system where we not only     
11:06     
looking at how do we represent the system but also uh looking at how do we     
11:11     
identify that uh how these uh uh like units in this uh     
11:17     
particular network or what we are interested in interact with each other and then uh I'm looking for uh     
11:25     
homogeneous as well as heterogeneous graph representation as well and some of the hyperraph work here I'm looking at     
11:31     
some of the causal analysis where we are looking at why certain things uh affect     
11:38     
other part of the networks uh looking at the features representation of those data as well as uh what cause certain     
11:46     
things more important than other so Right now the mostly data I was I work     
11:53     
on is social network as well as infrastructures model where I am representing different infrastructures     
12:00     
model as part of graphs So that is uh like kind of my grant work but mostly     
12:06     
I'm looking at influential node in the network So some of this work like looking at the hyper graphs or causal     
12:14     
analysis excite me to work on this project Right now I got some idea by looking at the GitHub repo and I played     
12:21     
around some of the codes but I did not had anything to directly represent now     
12:26     
but definitely I'll have a look I just want to get into touch with what's going on this meeting and what problems     
12:33     
specifically you guys are looking at and where it is required for us to you know     
12:39     
give some contribution That sounds good Thank you Yeah we're uh     
12:45     
you know kind of working on the or we're we're working on the G-SK project and     
12:51     
and sort of the ideas kind of coming out of that So that's what we've been talking the last few weeks So that     
12:59     
that's and that's it kind of in line with what we were talking about last week analysis So     
13:06     
Yeah Yeah Right now we I am looking at the causal analysis looking at the features     
13:12     
or some of the nodes in the graph like for example I have uh explored some of     
13:17     
the uh notebooks as well where uh these uh worms has certain kind of features     
13:24     
which uh help them uh like maybe directly related to uh their growth or     
13:31     
uh how do they interconnect with each other So that I found little bit interesting and maybe in hyperraph     
13:38     
representation as well like what how do we say that something is come into a     
13:44     
common cluster or not So maybe we can look at some of those features as well which is more important than other Thank     
13:52     
you Yeah Right So this is the we'll come     
13:57     
back to this paper that Lewis uh posted So this looks like a really     
14:03     
interesting paper I think it's from January so it's it's pretty new Looks     
14:08     
like there a number of people on here who uh only one I I familiar with is Ev     
14:17     
Yamini Uh but yeah this is on uh wholeb brain imaging So it's you know there's a     
14:23     
whole community of people in the sea elegance world who do like uh conneto     
14:30     
stuff looking at neurons and then there other people who are doing you know the rest of the organism     
14:37     
along with the brain So there's it's it's kind of uh you know you're going to     
14:43     
get I think with this paper you're going to get a lot of neuron information about     
14:48     
the neurons and of course the neurons are roughly about maybe onethird of the     
14:54     
worm So it's 302 neurons 959 cells in the adult hermaphrodite So     
15:02     
it's like about a third and we know the conneto it's it's     
15:07     
been um estimated and we know all the connections So we have a pretty good     
15:13     
handle on the sea elegance conneto That's what one of the reasons why seeligans is such a a a great model     
15:20     
organism is because a lot of these features of the individual cells are known and we have community data sets     
15:28     
So you know the same holds true for the ner for the conneto We know like every     
15:34     
neuron we know their position We actually know the position of all the     
15:40     
adult cells because it's very regular from worm to worm And so this paper goes     
15:47     
into characterizing these neuronal cells uh the determinance of their position So     
15:54     
they're usually positioned relative to one another Some are anterior uh some are posterior They're     
16:01     
neurons in the tail they're neurons in the head and they're neurons down the body uh length of the body So you have     
16:09     
this nervous system that emerges in development And so the question is is how do the neurons know where to go or     
16:17     
how do they end up where they end up Because it's important for say like tail     
16:23     
neurons to end up in the tail head neurons to end up in the head And then you know neurons to be     
16:31     
near one another because when neurons connect that process of connection is     
16:37     
where axons are sent out uh using an axon cone to find another cell or find     
16:44     
another axon to connect with So this is a process that you know you can't have     
16:50     
cells just at random locations and the worm they have to be you know within a     
16:56     
reasonable distance or with you know uh in a place where those cells can be     
17:01     
signaled in some way So um the highlights of this paper so this     
17:07     
is again cell reports methods uh the highlights of the paper uh they     
17:15     
produced this NWB file um this is a specific uh informatics file format     
17:23     
uh that they've modified to extend supporting C elegance wholeb brain imaging data so NWB is something they     
17:31     
use in neuro imaging It's a data format that involves a     
17:36     
number of different attributes of the neuron the electrphysiological data and     
17:41     
so forth Uh production of a diverse data corpus using harmonization tools at worm     
17:48     
id.org So this is their worm ID site See if we can go through to that     
17:57     
Okay it's not I don't think it's a hyperlink but but anyways they have a     
18:02     
data repository there and some tools neuron ID algorithms improven     
18:09     
performance and do not need lab specific retraining So this is of course where     
18:14     
they're developing an algorithm for doing this prediction and then neuronal     
18:20     
atlas reveals biological determinance of neuron positioning So they are able to sort of get a handle     
18:28     
on why you know why a neuron is in the place that it is and that's a an     
18:33     
interesting question segans but actually also for this notion of um centium is     
18:42     
like why does cells end up in the place that they do I mean we know that they end up in a certain place and you know     
18:48     
it's obvious that some cells need to be near one another certainly if you have a centishum all the sort of the muscle     
18:55     
cells or the epithelial cells or the germ cells that are involved in some     
19:02     
centumium or another need to be next to one another I guess it's it's most     
19:07     
efficient if they're next to one another Like we discussed you can have centum that are you know uh you know all over     
19:14     
scattered and then you can have cell bodies that sort of meander throughout the worm But uh you know if you think     
19:21     
about muscle you want to build a muscle body Uh you have all these uh cells that     
19:28     
divide and they differentiate into the these muscle precursors They have their     
19:34     
own uh nuclei and then the cells fuse into a structure that resembles some     
19:40     
sort of large uh you know solid body that can contract     
19:46     
and expand like a you know a rope or something like that     
19:54     
So you have this uh form higher order form that you want to achieve and in the     
20:01     
ner in the nervous system of course neurons uh form circuits that produce     
20:08     
behaviors So you have to have those connections in place Maybe you know you     
20:14     
have sensory relays that need to be uh that can't be too distant from one     
20:19     
another C elegance the distances aren't very great but you know there are these neural conduction issues if the cells     
20:26     
are just everywhere in the body so this is uh useful     
20:32     
information so they go through the data here their conceptual abstract is here I     
20:38     
don't think I don't know if you can open it up okay see okay here we go uh so     
20:46     
basically they have orain imaging of elegant from five labs using new     
20:52     
methodology They merge these into a standardized format     
20:57     
NWB and uh then they work on software for harmonization and data curation So     
21:04     
they have neuropal which is a longstanding software and then eatm I     
21:09     
don't know what that is Um and then they retrain uh     
21:15     
so a lot of these groups will do things where they're exploring ways to build     
21:20     
algorithms to train uh you know these you know train these predictive models that can give you if     
21:27     
you're in a lab you know information about uh neuronal ID or behavior     
21:33     
classification or something like that So you know you can imagine if you have some high throughput system where you're     
21:39     
imaging a bunch of worms you want to identify the cells their location and     
21:45     
then a lot of different attributes about those cells Say you're doing a a cell tracking experiment where you're getting     
21:52     
microscopy data from a bunch of worms You're doing this high throughput acquisition of the images you're getting     
21:58     
the images and you're just kind of uh trying to ID stuff from maybe like uh     
22:06     
you know a microscopy image or a series of microscopy images and so that's what their aim is here So they retrain three     
22:14     
neuron ID algorithms CPD stat atlas and CRF ID So these are just different     
22:20     
algorithms for this sort of task and they want to see which one is best And then this is worm ID.org which this data     
22:28     
corpus in neuron analysis tool Okay so that's uh the sort of the     
22:36     
summary and then they kind of talk about these whole brain imaging experiments     
22:41     
with single neuron resolution which they call whole brain imaging Uh this is     
22:46     
something that they've developed in C elegance because it's small and we know all the neurons We have a nomenclature     
22:53     
for them We know all the connections So the conneto is is known in sea elegance     
22:59     
but in a lot of other organisms they're not So you know we have for example     
23:04     
ruffleophila which is the fruitfly We know some of it's connectto People have produced high resolution maps of     
23:12     
different parts of the ruffleophila brain such as as the mushroom bodies or     
23:17     
other areas where they have a characterization of the cells in the     
23:22     
connectto but you know it's incomplete and there is a lot more prediction there because there you know there's a lot     
23:28     
more of a less a lot less deterministic process Plus there's just a lot more     
23:34     
stuff in that brain Um and then of course we also have uh     
23:40     
the demonstration whole brain imaging in in the zebra fish and that happened in     
23:45     
2013 And of course you know we've adopted these methods for     
23:51     
uh drosophila for zebra fish and for sea elegans and worked on them since     
23:59     
Um and so you know it's going to be harder to do in mammals because mammals     
24:05     
first of all they have this sort of regulative development where the cells     
24:10     
differentiate according to chemical signals and gene expression signals     
24:16     
rather than like uh you know a deterministic signal So you can have a     
24:22     
bunch of precursor cells that you know uh could be one cell specific cell or     
24:29     
another You know they the the cell type is more restricted but the ID of a certain cell in a certain location is     
24:36     
less deterministic And so that's something that you know uh that's why it's harder     
24:41     
to work out lineage trees in those organisms but also use this method So     
24:48     
but they they're trying and they they've tried in mice to do this apparently Um yeah So this is really     
24:55     
they they make this point here Manual cell identification from fluorescent microscopy images is a notoriously     
25:02     
difficult skill requiring substantial expertise and labor This task is     
25:08     
particularly difficult for neurons labeled the nuclear localization fuoroores which are these fluorescent     
25:14     
signals that you see in the microscopy images You know if you've looked at some of the microscopy data you'll notice     
25:21     
this uh image that's black with a black background with all these lit areas     
25:27     
these like little dots Those are the uh you know we localize the fluorescent     
25:34     
signals to the nucleus sometimes to the membrane but mostly to the nucleus and you pick those up in the mic     
25:41     
you know in in the microscopy you segment those and you can segment the     
25:46     
position of different cells and you can also do this with membranes but you know     
25:51     
the localization of these flororahores the ability for you to     
25:56     
segment them out from the background that's all very tough work it's not the easiest thing to do and so you need to     
26:03     
have these kind of algorithms that can help you uh you know not just segment things but identify     
26:10     
things so knowing the position and knowing Like if a cell divides and its daughter     
26:16     
cells are going to be in a certain position those are that's an important piece of information because then you     
26:23     
can predict where a cell will should end up right like if you have AB like uh uh     
26:32     
like we were showing before and it divides into two cells those cells     
26:38     
migrate from the position of AB and they migrate to new position uh they could be on opposite sides of     
26:45     
the worm They could be um in different positions along the AP axis You know we     
26:53     
we just have these positions that are estimated from uh track cell tracking     
26:59     
data And you know it generally it's we know from the lineage tree that there's     
27:04     
a relative relationship in their AP position their anterior posterior     
27:09     
position So we can have those kinds of estimates So that's what they're trying to work out     
27:16     
here Okay And so they introduced this tool worm ID This resource con consists     
27:22     
of data harmonization tools So like creating these standardized file     
27:27     
formats NWB stands for neurodata without borders and and they're actually some     
27:32     
GOC projects through INCF working on neurodata without borders I'm not sure     
27:38     
what the projects are this year But they're always working on this format and working on how to sort of expand     
27:45     
that format and make it useful to people across neuroiming and and and in in uh     
27:54     
neuroscience So that's uh what they're trying to do     
27:59     
here They do a lot of pre-processing work They have this open- source     
28:04     
software to analyze whole brain activity images And they also have tutorials and     
28:10     
documentation for using these kind of tools in people's pipelines So if you're again doing if you're a wet lab doing a     
28:19     
high throughput analysis of microscopy data you have these tools right at your     
28:25     
fingertips and you don't have to worry about training them and working on the     
28:31     
computational aspects of data wrangling This is one of the things that openworm     
28:36     
has been uh trying to stress um over its time is you know not just     
28:43     
the simulation part but sort of the data analysis the data normalization part So     
28:50     
uh you know the idea is that you want to have people in wet labs which is where     
28:56     
you know you have some sort of worm culture facility and you're interested in basic questions in the biology and     
29:04     
you don't really want to become an informatics lab So you know there are a lot of informatics tools you need to to     
29:10     
do your work And so there needs to be a set of tools and and in the clans     
29:16     
community there are a lot of great tools but you need to have a set of tools that are reliable that can allow you to just     
29:22     
plug and play and find these things and not spend a lot of time on that so you can go to those basic questions So     
29:29     
that's what they're trying to do here Uh so finally we provide a large online     
29:36     
corpus of harmonized C eleganceful brain activity imaging and structural data     
29:41     
that can be used for large scale experimental analysis neurobiological modeling and algorithmic development     
29:48     
This corpus is stored in a popular community archive called the distributed archive for neurohys physiology data Of     
29:55     
course this is Dandy And I don't know if     
30:01     
uh is that here Uh uh Morgan actually is a big fan of Dandy     
30:07     
and he knows a lot about it So if you're interested get in touch with Morgan We     
30:12     
can tell you about these open access repositories Um yeah let's see Uh this is their let's     
30:21     
see if this is the one where they show some of this So this is where we have the angle And so we     
30:30     
model the uh the worm as this uh this volume     
30:36     
uh and it's it's basically an XY surface It's wrapped So this is something that     
30:42     
we've tried to do a lot in the group is establish like a fourdimensional     
30:48     
position So what you have here is you have an XY axis and what happens in microscopy when     
30:56     
you acquire images from a C elegance embryo from a worm I guess too is you     
31:02     
have the spatiality where you have you know the worm or the embryo on a cover     
31:09     
on a on a slide with a cover slip and it's squished down a little bit and you     
31:14     
have this XY position So you have left right anterior posterior which is     
31:20     
forward and backward and then you have the Z uh the Z position and the Z position is     
31:28     
um usually depth So it's uh you know where you go from the top to the bottom     
31:35     
and so that position oftent times in a lot of the cell tracking data is estimated from different uh images in an     
31:44     
image stack So you'll image a stack through the focal plane You'll go from like the top down to the bottom and     
31:51     
it'll just focus in on different uh layers Sort of You you take take a a     
31:59     
sphere you take this um this shape and you cut it across     
32:05     
um and you get these different slices of the image And so you know because it's     
32:12     
squished with the cover slip it's a little bit um distorted and there isn't a lot of     
32:18     
information on the Z-axis At least in my experience I've never seen a lot of information revealed on that Z-axis Not     
32:25     
as much as on the XY uh axes So what they're doing here is     
32:31     
they're taking that XY plane and they're wrapping it around They have this tooidal surface and they're actually     
32:39     
trying to establish the angle So you have the angle you know you could have like an XY angle So you could see like     
32:47     
if something divides um at the lower right and it you know the cells migrate upwards and and     
32:55     
leftwards then you can estimate that angle and you can use a two-dimensional     
33:02     
um estimate of that no problem like we saw with the theta value that Lea showed     
33:08     
Uh but then you have the three-dimensional aspect and there's a way to estimate threedimensional phase     
33:14     
angle as well which is where you have this uh you know you can have like     
33:19     
three-dimensional position where you add in the z-axis and you figure it out or     
33:24     
you have this sort of tooidal space where it's like this continuous space     
33:31     
continuous xy space around the edge of this uh space and And then you of the     
33:38     
and then you have this estimate of position or the estimate of angle So you have this butterfly plot which just     
33:44     
shows the theta value from negative 180 to 180 So that's 180 degrees and     
33:51     
negative 180° and then showing the uh distances     
33:56     
there And you see that there's a lot of uh a lot of phase angles that are center     
34:02     
around zero which should be expected because that's the center or or you know     
34:07     
close to the where the cell I don't know if they're taking it from the origin point of the cell of each cell or if     
34:15     
they're taking it just as an absolute position but you know you should expect     
34:20     
either way that cells won't necessarily diverge as much Some cells won't     
34:25     
necessarily diverge as much from their mother cell in terms of position and other cells will But it's also the case     
34:33     
that you know you don't have that much divergence in position Maybe it's like     
34:39     
you know cells begin at the center line of the worm and move to the edges     
34:45     
Anyways um this this is an interesting plot It's interesting data And then of     
34:51     
course we have activity traces uh for the different neurons uh with a     
34:56     
time course there We have images of neural raw neural activity So this is     
35:03     
where they're imaging I guess it's some sort of electrphysiological like a a voltage die     
35:11     
or something where they're looking at these different things going on in the neurons This is the neuropal structural     
35:18     
volume which I think is a good measure of volume for the cells Um they just     
35:25     
stain I guess the whole cell for this So this is available in the adult Maybe     
35:32     
they have this for development I'm not sure but this is where they have the structural volume I guess for the adult     
35:37     
cells So this would be where we have uh some estimate of that We also have these     
35:45     
uh uh we actually we do have in this case the z-axis and then the y- axis So     
35:51     
the dorsal vententral view which is the z-axis and the lateral view and then we     
35:56     
have um these color space views which um get     
36:02     
into So um why don't we talk about so yeah so there's a lot of interesting     
36:08     
data in here Why don't we talk about this data source worm ID interested in that So this is worm ID This is uh where     
36:17     
you know this is very user friendly I guess Um this is where we're IDing your worm neurons This is where we have just     
36:25     
the neurons of course So we don't have every cell and the adult We have the the adult cells uh which is the they're the     
36:33     
termally differentiated cells in the lineage tree at the bottom of the lineage tree when you have a bunch of     
36:40     
developmental cells that divide and then they end up at a state where they don't     
36:45     
divide anymore They don't differentiate anymore Um and this is what they're what     
36:51     
they have in this So ID or worm neurons worm ID.org is a resource for the CLA's     
36:57     
neuroscience research community to do all the things we talked about So we have data sets     
37:04     
here which is on this tab So we have     
37:09     
uh let's see we have their dandy ID the data set u initials and then the     
37:17     
description and then they they have a source lab So these are I think seven     
37:22     
right Yeah And so uh the first one is from the Hbert lab Uh this is uh     
37:28     
neuropel micrfluidic chip images and GCMP activity That's the one where we     
37:34     
saw bottom right of that last figure This is where we have you know we     
37:39     
have like 21 worms that they sampled We have a number of segmented neurons So     
37:45     
they're not complete you know uh connetos but we get like a pretty good     
37:51     
representation of neurons in each data set So uh this is 166 to 188 neurons and     
38:01     
then this is you know their notes over here where they kind of take data from other sources So you can go back to     
38:07     
those sources and see if you can so there's just a lot of data that you can use for this Um some of these are     
38:14     
unpublished some of these are published So but the point is is that they're available and we can get access to them     
38:23     
Um yeah so this is uh good stuff There's a lot of whole brain neurodynamics whole     
38:30     
brain imaging these neuropal structural volumes So this last one is from the KO     
38:35     
lab at UCSF 10 worms 166 to 180 neurons     
38:43     
um and you know this is unpublished data sets from the KO lab So this is I I     
38:49     
don't know actually if these are available but in any case those are the data sets this is the     
38:57     
code So this is the narrow data without borders uh repository for C     
39:04     
elegance This is uh you know where they're they have the neurodata without     
39:10     
borders formatting files So this is just for people to convert their optophys     
39:16     
physiology data So this is where they're doing like things like gel or voltage     
39:22     
dies and things like that and they're able to convert that to this formula Okay     
39:28     
Um so then they have apps So they have eats worm is a nepari     
39:34     
plugin So npari is a uh imaging tool     
39:40     
Um and uh yeah and then Neuropal of course is something you can use on your     
39:47     
uh you can it's a standalone application you can download uh yeah VMI has been     
39:54     
developing this for a long time So and the reason that's important is because     
39:59     
he's been involved sort of peripherilally with open world process is good And then you have how to use and     
40:06     
you can go through and basically set up     
40:11     
uh narrow data without borders and then extract the activity using each one So that's uh     
40:18     
what that paper's about I think that was a very good paper and Morgan is here now So he's usually travel in transit at     
40:26     
this hour so he probably wouldn't be able to speak to it but you know it's just nice uh data sets available on     
40:34     
Dandy to whatever extent they're available and they're worth checking out     
40:43     
So that's a that's a good presentation So thank you for the detailed uh understanding of the paper So I was     
40:50     
trying to understand the components more thoroughly I thought of you know reading that and bringing out next week but it     
40:57     
was fortunate enough to get it this week Yeah So yeah that's from my side     
41:06     
Great All right So I think we'll finish up here with uh see which paper do we     
41:12     
want to do here [Music] Um we'll do this paper here Um this is I     
41:21     
can't remember if this is something that Susan sent This is kind of shifting to a different topic here but uh this is the     
41:29     
flow analysis of the low Reynolds number So this is physical review fluids     
41:36     
is the journal and uh this is the uh abstract So and when we talk     
41:44     
about flow analysis of course we're talking about when something is floating     
41:50     
in water it's moving through water we want to get a sense of its Reynolds     
41:56     
number which is the sort of the dimensionless parameter of its     
42:03     
uh its ease of motion through that flow So you could have a laminer flow you could have a turbulent flow and then     
42:11     
different things that move through those flows of a Reynolds number The Reynolds number tells you something about the     
42:17     
hydrodnamics So that's what we mean by like a low Reynolds number swimmer It's     
42:23     
moving through this water column The water column has like a certain level of turbulence It could be laminer which     
42:30     
means it doesn't have any sort of turbulence meaning that it's you know there's no sort of there no cross     
42:36     
currents or anything and we're talking about very small amounts of water So it's you know um not like we're talking     
42:46     
about huge volumes of water So but but that's we we want to get this Reynolds     
42:52     
number because we want to see you know what the shape of the body is relative to the environment that the organism is     
43:00     
in or if it's a cell you know what kinds of uh you know what does the cell need     
43:06     
to move through that water in terms of a motility mechanism or something like     
43:12     
that So this is an interesting question for like uh     
43:18     
biopysics for mechanics and things like that Okay So the abstract readming cells     
43:25     
and microorganisms are a critical component of many biological processes     
43:30     
In order to better interpret experimental studies of low Reynolds number swimming which is where the Reynolds     
43:37     
numbers are really low the flows are laminer and you have this uh you know     
43:43     
the swimming where they have to move through the water column We combine experimental and numerical methods     
43:50     
perform an analysis of the flow field around the swimming neatode segans So     
43:55     
sea elegance is usually in some sort of liquid It's usually some suspension It could have it could be like you know in     
44:02     
nature usually sea elegans are in these environments where they're in the soil but they're also in a very wet soil So     
44:10     
there's liquid there there's colloidal material around them and they     
44:16     
have to find their way through So that's why this is re relevant And then if     
44:21     
they're in culture they're usually on a gel or they're usually swimming through     
44:26     
like maybe a bacterial soup So that's another reason why this is     
44:32     
relevant So and they do have these swimming modes where they they are in water and they they kind of wiggle back     
44:38     
and forth and they have these various swimming behaviors So this is also why this is important because we want to     
44:44     
know kind of what kinds of flow fields C elegance experiences in movement So this has re     
44:52     
uh relevance to the muscles This has relevance to the output behaviors and     
44:58     
how to keep their behavior stable so that they can swim and get places Uh we     
45:04     
first used image processing and particle tracking uh velocymmetry uh which we've t Uh yeah     
45:11     
we've talked about that method before We haven't really talked about in depth but there's this method where you could it's     
45:17     
like cell tracking where we track the nucleus of the cells except in this case     
45:23     
we're tracking different we're treating things as different particles and we're tracking their velocity So we're not     
45:29     
just taking the position we're taking the derivative of the position So this is in a way kind of like these phase     
45:35     
angles because you're taking um particles     
45:40     
and you're calculating out their velocity their movement their change in     
45:46     
position over time And a couple weeks ago I did a paper this was after the main meeting     
45:54     
um on these lrangeian coherent structures that people have used and developed So lrangeian coherent     
46:01     
structures are where you have um like you know you use particles to track     
46:08     
uh movement So like you'll have a bunch of particles at an origin point and     
46:13     
those particles will disperse and you have this technique you you construct this lrangeian to track the cells from     
46:21     
their origin point and how they diverge from one another and then you can calculate out these ridges where you     
46:28     
have uh you can make statements about their uh movement together and you can     
46:34     
make statements about the flow you can make statements about the the agency of the the things that are moving etc     
46:43     
etc So there are a lot of lrangei methods we can bring to bear to this this type of thing So     
46:52     
uh we first use okay we did the particle tracking so they use particle tracking     
46:58     
valymmetry to extract the body shape kinematics and flow fields around the neatode So again we got the body shape     
47:06     
we have the kinematics the movement um and then where uh yeah the movement and     
47:13     
then the flow fields around the neatode We then construct a three-dimensional model using the     
47:18     
experimental swimming uh kinematics and employ a boundary element method to     
47:24     
simulate flow fields So we need to also simulate the flow field So we can get the Reynolds number obtaining very good     
47:31     
quantitative agreement with experiment So uh these this they     
47:37     
construct a model and then they compare model with experiment So they measure things and then they build the model and     
47:43     
then measure the model and they see how close they get We use this numerical model to show that the calculation of     
47:49     
flow shear rates using purely planer data results in significant     
47:55     
underestimates of the true true threedimensional value So this is where uh again we have     
48:02     
this issue between the two dimension in the three-dimensional model So we can     
48:08     
represent things in an XY plane and you know a lot of our microscopy is acquired in XY planes and     
48:17     
then we do you know the modeling in XY planes and that's you know one way to do     
48:24     
that Um but then you also have to sort of extrapolate that out to three     
48:30     
dimensions And so when you use planer data which is like the two-dimensional plane you it's easier computationally     
48:39     
but you suffer from not being able to make a true estimate especially in 3D because the anything when it moves     
48:46     
through a water column will move in 3D usually Um like you know you'll have all     
48:51     
sorts of things that will move in a water column They might move up the water column or across and they'll have     
48:58     
variations in terms of you know in XYZ space and how they     
49:04     
move So there are significant underestimates of the true three-dimensional value And then this     
49:11     
leads us to the fourthdimensional model because the four-dimensional model is time So we want to understand this stuff     
49:17     
over time to the variations in time There's acceleration and deceleration in     
49:22     
time and so forth Applying symmetry arguments validated against numeric we demonstrate     
49:29     
that the out of plane contribution can be accounted for by incompressibility and therefore simply     
49:36     
calculated from particle tracking palsometry Um our results show how fundamental     
49:43     
fluid mechanics considerations may be used to improve the accuracy of measurements in bofluid     
49:51     
dynamics Okay so this is uh good stuff Um so they'll talk a little bit about     
49:58     
why this is relevant to sea elegance So they talk about swimmers There's of     
50:04     
course mamleian spermatzoa which are you know very small cells that have uh fleella and they     
50:12     
travel using these undulations So they fluctuate back and forth and they move through the water     
50:18     
colum So there are a whole class of cells and organisms that are these undulatory swimmers One such unilatory     
50:26     
swimmer is a biological model segance Um the genetics and physiology     
50:31     
of the nebode are well studied Its genome has been completely sequenced and a complete cell lineage has been     
50:38     
established The neuromuscular system of sea elegance controls body angulations which allow it to swim dig     
50:45     
and crawl Um and then uh C elegance has     
50:51     
also been used as a model system for experimental studies of propulsion particularly at low Reynolds due to its     
50:59     
simple planer swimming gate and size So this has actually been used in uh     
51:04     
propulsion studies which are see 9 through 15 these citations down here     
51:10     
they don't have the name of the papers but a lot of soft matter and fluid     
51:15     
physics and stuff like that Okay so um that's what they're doing there So we     
51:22     
have this uh uh sort of model that we can build um     
51:28     
the nematode generates planer bending waves uh through contractions of its vententral and dorsal muscles This is     
51:34     
why we're interested in the planer representation So uh when you have this     
51:40     
wiggling back and forth it generates this planer set of waves as you could     
51:45     
imagine um and it contracts vententral and dorsal muscles and it produces this     
51:50     
quai two-dimensional traveling sine wave along its body So uh anterior to     
51:56     
posterior a sine wave at around 1 millimeter in length and 75 microns in     
52:01     
diameter So it's very small Sea elegance is significantly larger than the majority of low Reynolds number     
52:08     
undulatory swimmers So a lot of the unilatory swimmers are cell you know     
52:13     
cell sizes of several microns You're talking about like spermatzoa You're talking about     
52:21     
ecoli other bacteria bacterial cells So we have this large multisellular system     
52:29     
that's much larger than the other types of unilatory swimmers And so we can study these sine waves They're bigger     
52:35     
They have larger dimensions And uh Reynolds numbers and flows behave     
52:42     
differently at different size scales So the closer you get to the nano scale the     
52:49     
more different it is And so these smaller undulatory sores are kind of closer to the nano regime than sea     
52:55     
elegance So what we know about these systems is you know uh kind of limited     
53:00     
down to that time that that size scale So we have a little bit different size scale to deal with     
53:07     
Um the resulting flow fields can be used to probe properties of both the swimmer and the fluid providing new insights     
53:14     
into the physics of undulatory propulsion So and then of course you know just     
53:20     
because we have this planer representation we can build and this swimming stroke that's planer uh the     
53:27     
flow around sea elegance has a complex three-dimensional structure which is difficult to capture experimentally many     
53:34     
use flow properties such as the shear rate and are dependent on out of plane flow contributions not measured by plane     
53:42     
velocity data in essence one measures a two-dimensional slice of a three-dimensional field So we have that     
53:49     
limitation Uh this limits for example our ability to accurately determine drag and propulsion forces experienced and     
53:56     
exerted by the swim swimming neatode In this work we show that the numerical and     
54:01     
theoretical fluid mechanics techniques can be used to improve the processing and analyses of these experimental flow     
54:08     
fields So let's see if we can find some images So this is figure one This is the     
54:14     
instantaneous flow set of instantaneous flow streamlines generated by C elegance     
54:20     
So they've compared computed this from experimental waveform data where they've calculated this in the medium So this is     
54:27     
uh where you see this complex three-dimensional flow field You see things happening along the you know     
54:33     
laterally you see things happening uh in this sort of sea space and yeah you see     
54:42     
the strength of the flow places So the closer to the body the stronger it is So     
54:48     
you can see that this is not a simple sort of uh uh sort of wake that it     
54:54     
produces It produces a lot more Then of course if the fluid is turbulent then     
55:00     
the it complicates things even more is I think this is assuming a quailey laminer flow I don't know what what you know     
55:07     
this is just kind of like um I think this is a very simple     
55:13     
case this is uh where they have the skeletonized body which is here the left     
55:20     
you have the re reason they do that is because they just want to capture the kinematics They're not interested in the     
55:26     
cells All they want is the the shape of the body and then this kinematic     
55:32     
skeleton So you can see where the degrees of freedom of movement are and you can model those appropriately So you     
55:38     
see the degree there many degrees of freedom along this axis and some you     
55:44     
know when you have an undulation some of the worm is moving right some of it's moving left and it keeps switching     
55:50     
position as it undulates So it's going back and forth So you have these experimental settings or you observe     
55:56     
this you observe the kinematics and the direction of the movement and then you     
56:02     
have this uh skeleton that you can build and simulate it So you simulate it under the flow field which is here and you see     
56:09     
this uh so this is a particle tracking velry     
56:15     
um and this shows the flow field produced using the reconstructed 3D mesh     
56:20     
and a numerical model So this just shows uh the speed in terms of millimeters per     
56:26     
second Um so you see at the head where there's a change in direction it's     
56:31     
faster at the middle of the mid uh you know middle of the midline right here     
56:38     
it's also faster and so actually the places where it's switching direct immediately where it's switching     
56:44     
direction you see it slower So these are just basic mechanical characterizations     
56:53     
So this is the spatial distribution of the experimentally measured x component     
56:59     
uh which is uxy simulated x component which is in b So we have the     
57:06     
experimental experimental measurement in A the simulation in B and see we have     
57:11     
the experimentally measured Y component uh VXY and then the simulated Y     
57:17     
component which is the XY simulated So you have experimental simulation for X     
57:24     
and Y components And so this is where again we have our color bars which are these     
57:30     
component velocity magnitudes which are millimeters per second Again we see     
57:35     
differences So the X component you see it towards the head and the tail For uh the Y component you have it     
57:43     
around where it's actually switch it changing position or changing     
57:49     
direction and also at sort of the end of the head and the end of the tail So     
57:55     
they're different components here where this so the spatial distribution is different for the x component and the y     
58:05     
component Okay Uh then this shows the comparison of flow structure between an experiment and     
58:11     
simulation This is uh a is the distribution of u b is the distribution of v which goes back up to the the x and     
58:20     
the y components Uh then there's distribution of speed which is absolute     
58:25     
value of u and the decay rate So you have the distribution here of u v the     
58:32     
distribution speed So this is speed instead of these spatial components and     
58:38     
then you have this decay rate comparing experiment and simulation So you know we try to get the simulation as close as     
58:46     
possible to the experiment and we have some loss in distribution speed at the high end of the speed and then the     
58:52     
middle here of this decay rate and you know some differences in terms of the     
58:58     
spatial distribution as well So this is kind of what they were talking about where the experiment and the model are     
59:05     
not exactly aligned Um and then this is the sheer rate error calculation and     
59:12     
correction So this just shows like this error and they're dealing with it     
59:18     
here Okay So then the conclusion is is that uh there they use theoretical and     
59:25     
numerical techniques to improve the processing of experimentally obtained particle tracking data reducing smooth     
59:32     
velocity fields quantifying the flow around the soying neatodes elegance in a     
59:37     
Newtonian solution So this is Newtonian Newtonian fluid Newtonian uh paradigm So     
59:46     
you could use I guess you could use non-Newtonian fluids and get a different result It's much more complicated to     
59:52     
compute etc We compared our results with a 3D boundary element model of the nematode     
59:59     
generated directly from experimentally obtained nematode kinematics finding good agreement between numeric and     
1:00:05     
experiment We argued that when calculating derivative flow quantities the only non-trivial auto comp out of     
1:00:13     
plane component was wz the uh zder derivative of the zflow So this is uh     
1:00:20     
where we have this uh you know I guess this is out I don't know what an out of     
1:00:26     
plane component is but or this I guess this is the thing that was throwing off the m So it's the z derivative of the     
1:00:33     
zflow the opposite So it's just basically you know higher order uh     
1:00:39     
kinematics This observation was validated by simulation showing that excluding uh wz and calculating the flow     
1:00:46     
shear rate results in a significant underestimate Accounting for a wz     
1:00:51     
component via incompressibility from 2D data eliminated this error Our work     
1:00:58     
illustrated how theory can be used to improve experimental measurement in biological fluid mechanics will directly     
1:01:04     
be directly applicable to the investigations of biological motion in complex fluids like in in nature where     
1:01:10     
they find themselves in flu you know in fluid flows maybe with colloidal things     
1:01:17     
around them uh maybe in a complex three-dimensional plane that they have to travel etc     
1:01:24     
etc furthermore we anticipate ate that as technology for the acquisition and processing of experimental flow fields     
1:01:32     
continues to improve these out of plane effects will represent a hard barrier to     
1:01:37     
increasing the accuracy of results making such techniques increasingly     
1:01:42     
important So this is where you know we're getting more data on experimental flow fields Um and then we have this out     
1:01:50     
of plane effect that people are going to run into when they try to do a plug-andplay type model and they're     
1:01:57     
going to have to figure out why that is So as you as we gain data as we uh     
1:02:04     
improve our ability to measure those things we have to be able to model them and then you know hand maybe hand them     
1:02:10     
off to someone who doesn't have the expertise in this technique to make sense of So that's what they're     
1:02:20     
talking Okay So that's all uh for that I'll see if we have anything in the chat     
1:02:30     
Okay Uh do we have anything else we wanted to mention Well I uh I have a     
1:02:35     
quick question So uh like first of all like uh thanks for the presentation but     
1:02:40     
what kind of tools are we using to simulate some of these uh bombs data     
1:02:46     
like which which one uh for the paper you have shown in the     
1:02:53     
paper like what what kind of tool they usually use     
1:02:58     
to use let's see their methods look like here they might mention it okay usually     
1:03:04     
they some sort numeric simulation tool Um I don't know     
1:03:10     
if they actually mention what they're using here Okay Actually I was wondering like uh since it's growing it might be     
1:03:17     
growing So are we interested in looking at uh different snapshots of the time     
1:03:22     
like what would be the stages of a particular uh worm at a given time or we     
1:03:30     
are interested in how it's growing just like with the help of simulation because     
1:03:35     
uh if we are interested in the specific features or like looking at the details     
1:03:41     
about that then uh like if we are looking at hyperraph representation     
1:03:47     
there's then I think the snapshot kind of model work better So I I I'm not sure     
1:03:54     
about that Yeah Yeah Well you know and Susan knows this like with simulation     
1:04:00     
you have to sort of have a lot of computational power because it's like if you Yeah simulate things down to like     
1:04:07     
milliseconds or you know micros secondsonds It it's really hard like when they do a lot of     
1:04:14     
the protein folding uh work where there are a lot of there a number of groups     
1:04:19     
doing the sort of simulating cells and they're getting data like molecular data     
1:04:26     
and they're simulating the molecular data you know molecular processes are like at like     
1:04:32     
milliseconds and microsconds and they're able to simulate those at very great     
1:04:38     
detail and so you know you can get that kind of detail Then when you move up to the next scale which is some cell     
1:04:44     
process you know you can't simulate um in a multiple like you can't simulate     
1:04:51     
that process for like seconds because it's just too computationally expensive     
1:04:57     
So they have they do screening which is like they have to exclude a lot of detail to get a good approximation of     
1:05:05     
that process Sometimes you have to use them in modeling and so sometimes it     
1:05:10     
makes sense to model everything like precisely Sometimes you have to just kind of you know uh coarse grain things     
1:05:18     
and you know just have a good enough representation for like a chunk of time or a chunk of space and then um that's     
1:05:26     
you know then you can have these larger scale simulations that have biological relevance     
1:05:32     
So um console might be able to model that     
1:05:38     
and ANCIS So those are commercially available     
1:05:44     
um systems um that are     
1:05:52     
expensive Um I don't know you can often people make in-house ones uh usually     
1:05:59     
using Python     
1:06:05     
Um I don't know or a mat lab I don't know if people have moved away from that     
1:06:10     
Yeah mat lab you can use mat lab Yeah     
1:06:16     
Yeah Yeah Okay Thank you I I'll just check uh check with some of those work     
1:06:22     
Yeah I was curious to know because like some of the graph I would be also working like when we like look at     
1:06:29     
different details like the graph sometimes goes will have uh you know lacks of ages or millions of notes So     
1:06:38     
then it would be extremely intensive to even visualize this graph sometime Yeah     
1:06:46     
Yeah That's a whole different thing where you have like the graph layouts you can actually make sense of the graphs but then again you know you might     
1:06:53     
have like uh you might not look at every node right so you just take a subset of     
1:06:59     
the nodes that you know have like something to do with the structure     
1:07:05     
yeah that's like more towards an open streaming graph or something like that where we just interested in what expect     
1:07:13     
we are looking at Yeah thank you You're welcome Yeah     
1:07:21     
I'm going to be trying um mat lab for the 10seg structures Okay I have a a     
1:07:29     
program called steady here Um and hopefully trying warfo and maybe like I     
1:07:36     
said answers We'll see I can get for a student you can get answers for free     
1:07:43     
like beginner answers Oh yeah So     
1:07:48     
yeah I can probably swing that     
1:07:54     
Well anyway and I have um rhyosterus     
1:08:01     
Oh that's a tool or a model It's a sort of a graphics     
1:08:08     
um interface um modeling They use it for architecture So all     
1:08:15     
right Should be able to use it for engineering although I'm not sure     
1:08:23     
Yeah Sometimes they don't give you like the details Yeah Well um it needs to stand up If you     
1:08:31     
if you make it it has to stand up Um if it says it will on the model That's     
1:08:38     
engineering Yeah It must not fall down No matter what Yeah Yeah Well ideally     
1:08:46     
that's good for architecture too but Yes But you wonder Yeah     
1:08:54     
It just has to be good for engineering too is what I'm saying     
1:09:02     
Oh well Yeah All right Well uh thanks for attending     
1:09:08     
everyone Did anyone else have anything you wanted to add There we go     
1:09:15     
Uh so do you think that uh know the angle the angular deviation or the phase     
1:09:21     
angle that we included you know if we include that in a you know while acquiring the structural features for     
1:09:28     
the embryoenesis at each and every time time point you know this uh you know it     
1:09:34     
divides into these two daughter cells and this is the phase that we have seen So including that phase information into     
1:09:41     
that phase angle uh differences into the you know the network modeling do you think that could     
1:09:47     
you know give us a edge to understand the dynamics more better     
1:09:52     
Uh well we'd have I don't know about the edge data but it would give you like some spatiality to the graph So it would     
1:09:59     
be like you'd have like the edges would have like information about their divergence     
1:10:06     
Like if you wanted to say like uh something maybe something belongs to two     
1:10:12     
different modules like you know you have like a mother cell then there's a division event and it moves out and you     
1:10:20     
have things that are se maybe in separate clusters Uh you could say that you know you could say okay uh let's do     
1:10:26     
a clustering algorithm on the points in space their position and then let's do the phase angle and see if they have a     
1:10:33     
correlation You can do all sorts of things like that And it I mean you could I guess you could also use it as like     
1:10:39     
edge information Um you could say well this is a divergence between these two     
1:10:45     
angles So if these these two nodes are in the same hyper node and they share     
1:10:50     
hyper edges those hyper edges are divergent space but they're part of the same hyper node Right Yeah So I mean     
1:10:59     
that that's another piece of information you could use Yes     
1:11:07     
thank you Yeah no problem Okay so let's call it a meeting Um thank you for attending and uh if     
1:11:15     
you're uh wondering about GOC uh projects proposals um those will be the results for that We     
1:11:23     
think we've released on like the 8th of May or something So we're not really sure what the process is going to be on     
1:11:29     
that So but anyways good luck to everyone on that and um we'll be in     
1:11:35     
touch soon about um you know if if you want to if you don't get selected and you want to continue to work on what you     
1:11:42     
were working on we're open to that But we'll we'll be sharing details and all of that soon So um have a good week     
1:11:50     
Thank you
