## Transcript      
0:00   
Hello. Well, how are you? I'm doing all right.   
0:05   
I haven't managed to get back into my research at all. So, I found a good paper. Well,   
0:12   
something you'd be interested in was about they were doing embryogenesis and commod.   
0:20   
Okay. So, we'll see how that looks. that looks like and then I don't know   
0:26   
if if it isn't a 10seg object it won't it'll just be an approximation.   
0:33   
Yeah. Well, we'll see how they do. Then you make a model and it is   
0:41   
um lasts over a longer um I guess strain   
0:49   
than other models and it kind of matches what tissue does. It makes you kind of   
0:54   
skeptical o over other models that um they don't last over the full range   
1:03   
right of strain. So, oh well, we'll see. Does it do stress drain curves? Well,   
1:10   
I don't know. I didn't really look that deeply into the paper to or are they just   
1:15   
Yeah, they they could be curve matching. That's console does curve matching really well.   
1:22   
Okay, we'll start with this paper here. Organogenesis and console. So this is   
1:30   
simulating organogenesis and console multifysics tissue patterning with directed cell   
1:36   
migration trademark multifysics or console multif physics. So you can't use that unless   
1:44   
you're using console. Um so this is the abstract   
1:50   
and this is a recent preprint from the archive was put up last week I guess.   
1:57   
Um the abstract reads we present a com cell multifysics implementation   
2:02   
of a continuum model for directed cell migration a key mechanism underlying   
2:08   
tissue self-organization morphogenesis yes it's a continuum model yeah   
2:14   
not not a tense integrity model no it isn't   
2:20   
the model is formulated as a partial integral differential equation or p   
2:26   
combining random motility with non-local density dependent guidance cubes to   
2:31   
capture phenomena such as cell sorting and aggregation. Our framework supports simulations in   
2:37   
one, two, and three dimensions with both zero flux and periodic boundary   
2:43   
conditions. We reformulated in aian setting to efficiently handle tissue growth and   
2:50   
domain information. We demonstrate that com multifysics enables a flexible and accessible implementation of pes   
2:58   
providing a generalizable platform for studying collective cell behavior and pattern formation in complex biological   
3:06   
contexts. So we have is this uh what we have a continuum model okay and they're   
3:13   
looking at directed cell migration and then they're looking at this pal   
3:19   
integral differential And so this allows us to combine the   
3:25   
sort of uh random motility maybe a grown motion type uh movement with non-local   
3:33   
density dependence guidance cues. So these are you know ways that you have random movement but you also have   
3:39   
guidance cues and it's sort of a mixture of two and you can handle this kind of differential equation.   
3:46   
So then they have you can you know model things in up to three dimensions.   
3:52   
Um, and then you can reformulate that mathematically into so you can deal with deformation and   
3:58   
tissue growth. Okay, so that's kind of the idea here.   
4:04   
Um, so in this article, we present a continuum model for directed cell   
4:09   
migration or DCM. This is where cells self-organize into complex tissues during development.   
4:17   
And so um, it seems like they they have to do this trademark every 10 dimension   
4:23   
physics which is really kind of distracting but um so they they they've   
4:30   
actually did another set of papers on this where they used com multifysics to   
4:35   
simulate biological self-organization reaction diffusion mechanisms these are   
4:41   
citations 2 through eight the first one is simulating organogenesis and com so   
4:46   
the conference proceeding Simulating organogenesis and com forming   
4:53   
and interacting domains. That's another conference proceeding. This is all console.   
4:59   
Simulating organogenesis and com multifysics PTE models. Um simulating organogenesis   
5:07   
and coms uh cellbased signaling models.   
5:14   
Uh six is simulating organogenesis and console multifysics image based   
5:19   
modeling. Seven is simulating organogenesis with console multifysics software based   
5:26   
fieldbased simulations embriionic lung lung branching. So that's an interesting   
5:31   
paper and then eight is simulating organogenesis and com multifysics   
5:38   
comparison of methods for simulating branchingis. So they uh and then they have some other   
5:44   
ones down here as well. So that's kind of where they start off from. So they   
5:50   
have this body of work that they've been building and this is one of those kind of   
5:57   
uh I guess this is an extension of that. So uh you know they've been able to   
6:06   
simulate reaction diffusion mechanisms, tissue mechanics and fluid flow. So all   
6:11   
of those things actually nine and 10 that are tissue mechanics and   
6:18   
uh we implemented these models on growing deforming or interacting domains using arbitrary lrangeian   
6:25   
which is a or phase field methods and represented individual cells with   
6:33   
compartmentalized diffusion signal. So this is where you have these kind of   
6:40   
Of course development is a growing deforming and interacting domain 03 and   
6:45   
they use these methods arbitrarily which is citation three and then phase   
6:52   
field method separately. So we talked about those references already   
6:57   
and so they represent the individual cells um but also with compartmentalized   
7:04   
diffusion and signaling. So you have you know a number of cells in a compartment   
7:09   
and then you link those compartments together. Moreover, we implemented efficient   
7:15   
methods for parameter estimation and use image derived geometries.   
7:20   
Okay. So then they want to talk about more about uh directed cell migration.   
7:26   
that enables the formation of spatial patterns from initially homogeneous conditions aligned solely on   
7:32   
interactions at the cellular scale. So that's uh just like the baseline   
7:39   
condition to pattern tissue on the much larger organ scale which is a sort of   
7:45   
elaborated condition. So you can actually   
7:50   
uh you know they can model things at multiple scales uh or this process occurs at multiple   
7:57   
scales. Uh DCM arises from the interplay between random cellular motility and guided   
8:04   
movement ultimately leading to phenomena such as cell sorting and aggregation.   
8:09   
In biological systems, guidance cues for directed migration include gradients and   
8:15   
chemical signals which are chemotaxis, substrate stiffness, duroaxis, surface   
8:20   
topography, topo taxis and differences in cell cell adhesion or differential   
8:26   
adhesion. So you have all these different things in the biological systems that serve as these guidance   
8:31   
cues and this happens at the tissue level where you have substrate, you have   
8:37   
chemical gradients, you have some sort of topography and you have you know uh heterogeneity   
8:44   
and cell cell adhesion across like a population of cells or a sheet of cells.   
8:51   
And so that's what they're trying to get at with this uh biological phenomenon.   
8:57   
They be able to model all of these things in a multifysics setting.   
9:03   
And so they're using numerical modeling. And so in their approach, DCM is   
9:08   
described using partial integral differential equations which are capable of incorporating both local diffusion   
9:15   
and action terms. And this gives us a more sophisticated dynamics.   
9:22   
And so of course solving PIDs is pretty difficult. Um it usually requires   
9:29   
customuilt solvers based on finite volume or finite element methods. So   
9:34   
this is a lot of computational time. Um and so you know you have to have   
9:40   
something that's gives you the ability to handle those computations.   
9:46   
So that's sort of they kind of plug console multifysics and they have a toolkit for this. Um   
9:57   
furthermore and tissue growth is an inherent aspect of developmental biology or genesis.   
10:03   
So you know we want to be able to model that accurately to handle domain expansion efficiently. We demonstrate   
10:10   
how DCM models can be reformulated in a lrronian framework enhancing   
10:15   
computational performance while maintaining model fidelity. Okay, so they get into the sort of   
10:22   
mathematics here. So we introduce the governing equations of the PD based   
10:27   
continuum model. The model describes the temporal evolution of a cell density   
10:33   
field which is CXT X being the vector and being time. So this is our cell   
10:42   
density field here and then this is our differential equation here where we have   
10:48   
a gradient a directed gradient and a directed uh J term and it's the   
10:54   
multiplication and this J term is the cellular flux. So   
11:00   
this J term is directional and it's the cellular flux in one direction   
11:06   
or it's it's directional I guess it doesn't have to be direction in this case it's consistent with the   
11:13   
with the gradient this flux can arise from a combination of random or pressurdriven motility and   
11:20   
directed density dependent migration so basically uh J uh vector is the   
11:26   
direction of travel of the cells So they're all traveling kind of in a in   
11:32   
the same direction or a similar direction and you know it's kind of measuring maybe the mean heading of that   
11:37   
and then it's traveling down this gradient. Maybe it's the gradient of a stiffness gradient or a chemical   
11:48   
for a single population model. This leads to the following equation in n spatial dimensions. And so this is our J   
11:55   
term here J vector and this is our term   
12:00   
here and this is where this is true. So   
12:06   
here uh capital D denotes the cellular diffusion coefficient and small a   
12:13   
represents strength of the directive motility forces. So you have this diffusion coefficient just describing   
12:19   
the cellular diffusion. Then you have the strength of directed   
12:24   
motility forces. How strong is that force? Okay. The model is designed to capture   
12:31   
four key aspects of collective cell behavior. So the first is that cells exhibit   
12:37   
motility which can be diffusion-like or random or pressure driven thereby   
12:43   
contributing to tissue fluid. So you have these cells that are migrating. It's nominally random. It can be   
12:49   
directed by certain processes, subprocesses, and it contributes to the   
12:54   
the fluidity of tissue. So tissue is not solid. It's always moving around.   
13:00   
Second, cells are able to sense their surroundings. They have a sensory sensory receptors and they can sense   
13:07   
these uh stimuli that uh lead to these toxic behaviors. So you have duro taxis,   
13:14   
chemotaxis, even photoaxis although I don't know how relevant that is here.   
13:21   
Uh this is represented by the integral i direct uh i vector c i x which   
13:30   
integrates local densities within a sensing radius r corresponding to a line   
13:36   
in one dimension, a disc in two dimensions or a ball in three dimensions.   
13:41   
So the sensing radius is basically how far out from the cell it can sense things and it basically integrates all   
13:49   
of the density around it. So if there's like if the gradient is going from weak   
13:56   
to strong then it computes maybe like a line in front of it or behind it. And if it's   
14:03   
going moving towards or the line in front of it is moving towards the source of that gradient then It's able to   
14:11   
determine that part of the gradient or that part of its motion or that part of   
14:16   
its sensory surroundings is the stronger the place   
14:23   
where it needs to go and so moves in that direction. Um if you have a two-dimensional disc,   
14:29   
the same principle is true. you can go you go in the direction of the stronger signal and that gives you a directional   
14:35   
movement or you have a ball in 3D. So it just basically works the same way except you   
14:41   
have movement in this in these three dimensions.   
14:46   
So that's how that works. So each cell has a sensing radius R can sense you   
14:52   
know this gradient that it's looking for and it senses sort of a greater   
14:57   
concentration of something and moves towards that and remember all the cells are doing   
15:03   
this simultaneously. Third cells interact and attract one another with a strength a within that   
15:09   
radius. So they interact with each other as they're all they all have the sensing   
15:15   
radius. They can not only sense uh some gradient but also other cells   
15:21   
and so this is where they can uh you know attract they can exhibit other   
15:28   
behaviors like that. So they can coordinate their behavior and uh in this   
15:34   
uh simulation the attraction is to be a spatially uniform   
15:40   
phenomena although in biology it would be spatially heterogeneous. it just makes it too hard to deal with when it's   
15:46   
spatially heterogeneous. So, um, in in the spatial uniform case,   
15:52   
we're dealing with something that's independent of distance within the sensing radius.   
15:58   
So, that's how you define spatial uniform. Finally, the domain is a finite carrying   
16:04   
capacity such that cells locally compete through space. So that means that they're all kind of competing for one   
16:11   
space and they're jostling for a position and you know they can only one   
16:18   
cell can only occupy one place in space at a time. So if it's you know if   
16:25   
there's an advantage to being in a certain point in space or a certain advantage of migrating in a certain   
16:31   
direction the cells will crowd each other and they'll undergo a jamming phase transition where then they stop   
16:39   
moving and they can't move anymore because they're too densely packed. And so that suggests that you have this   
16:46   
mechanism for packing this crowding effect. um and you know it allows um but it also   
16:54   
allows cells to not aggregate kind of just at random. So kind of counteracts   
17:02   
aggregation. There's this population pressure that moves the cells around and   
17:07   
it keeps them kind of moving around uh and not kind of getting stuck   
17:13   
although they can get stuck in certain cases.   
17:18   
So um in this model the population pressure can be incorporated either into the random motility term DC the directed   
17:26   
motility term HC or the sensing kernel GC. So the sensing kernel is the volume   
17:32   
and and what they're sensing there. The motility term is the random movement and the directed motility term is finding   
17:40   
gradients to follow. And so in this in this paper we focus on   
17:45   
the special case where BC = 1, HC= C 1 minus C and GC= so they set them at   
17:54   
these constants. Um, and then so that's that's kind of explained kind of in another paper.   
18:03   
And so uh then they do this non-dimensionalization.   
18:08   
They done dimensionalize the model by setting where LC is the characteristic   
18:14   
length scale. Here we choose LC equals R. So this is just kind of a constant   
18:20   
value. The non-dimensionalized equation then reads and this is the partial   
18:25   
differential equation here where and then this is the the other part here where they they give   
18:32   
you they factoriize it I guess. Um then there's domain growth. So to avoid   
18:39   
computationally expensive remeshing during domain growth. So that means when the sphere or when the volume is   
18:46   
expanding because it's growing and they don't want to remsh the simulation because remember this is um something   
18:53   
that's approximating a finite element model. So they have to you know kind of reset the mesh as it grows. You really   
19:00   
have this kind of predefined grid and a finite element model and you know you   
19:08   
decompose some surface using that method. But of course if if the thing   
19:13   
grows then you have to reme mesh. So that's that's what's happening here. So   
19:20   
to avoid that we transform the coordinate system from an oilian   
19:25   
to a lrangeian frame. So that's kind of where they come up with this idea of   
19:31   
making an end. This enables a simulation of domain growth on a static geometry.   
19:39   
We begin this transformation with the general form equations five and six which we saw by incorporating a vection   
19:46   
by a velocity field u vector induced by the expansion of the tissue. This is the equation for that.   
19:55   
To transform equation seven which is this into from the uian into the   
20:01   
lrangeian frame we define the rescaled coordinate which is here where we have a   
20:08   
stretching factor. Uh we consider here uniform linear domain growth for this.   
20:15   
The velocity field u vector can be expressed as this. The gradient operator   
20:20   
transforms is this. So we have these three different things a gradient operator a velocity field and this   
20:29   
rescale coordinate system where we have a stretching effect   
20:35   
and so then we have a grad directional gradient L which denotes the gradient   
20:40   
lrangeian coordinates and that's with respect to this uh this   
20:45   
growth factor here this this rescaling factor um since U vector vector times the   
20:54   
gradient directional gradient vanishes in the lronian frame and the divergence of the velocity becomes a velocity   
21:00   
vector for uniform growth and end dimensions equation seven which is up here   
21:09   
transforms into this number 12   
21:16   
the non-local interaction term I vector lengian then reads   
21:23   
and walk through that. Okay, so that's the mathematics. Um, and then this, so   
21:29   
this is the implementation of console. Um, I just wanted to say I thought   
21:35   
Millerian math was used because it was less computationally intensive   
21:43   
and you could actually do the same thing with with Lrangeian equations.   
21:50   
Sort of So you don't necessarily have to mix and match them, but I suppose it's more   
21:58   
computationally intensive if you use just liberian map.   
22:04   
Yeah, I think Yeah. Yeah. I think oileran is like the default, but they said that they needed   
22:11   
to be able to build a model where the system grew or stretched out. So they're using the lronian for that and they   
22:18   
don't want to use that like as the default because it's too expensive but they have no choice because the oileran   
22:24   
uh representation doesn't okay yeah I always thought the uleran   
22:31   
math was a bit less accurate too   
22:38   
okay all right um I may have to study it Yeah. Oh, yeah.   
22:46   
Happy about that. But anyway, it takes a couple passes to get like what they're doing like, you know, in   
22:52   
terms of each symbol and everything, but we're going to get in more into the   
22:58   
uh trademarked console multifysics here in the implementation. So, um this all   
23:05   
simulations were performed in console multifysics version 6.3.   
23:11   
um equations five and 12 from the last section were implemented as a coefficient form PTE. So they're using   
23:17   
partial differential equations here. So this is figure one. Uh this is an illustration of boundary conditions for   
23:24   
two dimensional domains. So remember we talked about how you can have one, two and threedimensional   
23:30   
uh layers or one, two and three dimensional representations.   
23:36   
This is the two-dimensional domain. Um this is where we have   
23:46   
uh geometric coordinate mapping for periodic boundary conditions. So um this   
23:52   
simulation domain is omega it's in the middle here and then we have these   
23:58   
things on the edge. So for the periodic boundary condition we have general   
24:03   
extrusion. we have uh then cmap which is outside of   
24:09   
general extrusion then it's just expanding that simulation domain and   
24:14   
then for c equals z we have an expansion a little bit of an expansion of the   
24:19   
initial domain but it's bounded by these langian terms here   
24:26   
so that's that's how that works   
24:32   
so in the above implementation is necessary to ensure that the integral remains well defined near the boundary   
24:39   
domain where the integration region extends beyond the domain. This was achieved with the coordinate mapping   
24:44   
which we saw here. These uh things on the outside uh edge of this these   
24:51   
squares. This was achieved coordinate mapping. We illustrated the procedure here in two   
24:57   
dimensions for quadratic domain centered at 0 0 two coordinates. The primary   
25:03   
domain of side length L which we saw is embedded into a concentric auxiliary   
25:09   
quadratic domain s of side length L + 2 P uh for periodic   
25:17   
boundary conditions the position for the outer square are then mapped to the inner square with the general extrusion   
25:23   
operator using the following rules in 2D. So these are their rules here. So it's just kind of a a stack where you   
25:31   
have these different conditions under which the model operates. So you have   
25:36   
this transformation from X to X plus L and then it's this if if this is true   
25:41   
then we return something. We keep doing this for different combinations of X and   
25:47   
Y plus and minus L. For example, to implement the mapping Y   
25:53   
to Y + L0 Z for this condition, which is the orange domain up in the figure. So,   
26:00   
it's this area here. General extrusion up at the top and C map down at the bottom.   
26:09   
That's our orange domain. One defines a new general extrusion option. In this case, the three upper domains are   
26:16   
selected with the source region ranging from the XY coordinates of our source   
26:21   
region. Then the destination map is then specified by setting the X expression to   
26:26   
X. As we only want to shift the values in Y direction and the Y expression of Y   
26:32   
plus L, which maps the selected region onto the orange domain labeled CMAP,   
26:38   
ranging from these coordinates in X. This procedure is then repeated for all   
26:44   
four edges and all four corners totaling in eight mappings in 2D.   
26:49   
For the 3D case, the squares are replaced by cubes and centered at 0 0.   
26:55   
And then we have the Z component which adds to the mapping in in this form. So   
27:00   
you're just doing the same thing with a new dimension and you're getting this   
27:06   
result. Okay. Um let's go to this figure here. Figure two. This is where we have   
27:15   
summary of the patterning space. So cells are migrating around for a reason. We want to see what the output   
27:23   
is. So we can get these emergent cell density patterns in one, two, and   
27:28   
threedimensional domains with periodic boundary conditions. So up here in a at   
27:34   
the top this is the one-dimensional case where we have this these bariate relationships between X and C.   
27:44   
And so we can see that there is a oscillation here for their different functions   
27:51   
um that they they they're plotting out here. Uh if you go to 2D you can understand   
27:56   
what those oscillations are. They're basically these these patterns. So you have these blobs that form in 2D and   
28:04   
they're aperiodic as far as I can tell. Uh so they kind of cover they have the   
28:10   
same coverage as the 1D case. So if you think of these humps these these uh   
28:16   
modes of the of the time sphere of the distribution here as these blobs the top   
28:23   
of these blobs. And that's kind of the same thing. It's just in two dimensions. And then in three dimensions you get   
28:29   
these blobs that are aperiodic with some u some variation along the edges. And   
28:36   
you can see that that's kind of in three all three dimensions. Okay. And then B you have a variation of   
28:44   
the initial concentration C 0 and migratory strength A. So you have   
28:50   
variation of those two things. The migratory strength is on this axis on the y- axis the C 0 which is the initial   
28:57   
concentration is on the x axis. So they're varying concentration and they're varying the strength of   
29:02   
migration. So you have a higher concentration and a higher uh migration   
29:08   
strength. you can have a lot of uh motility. If you have a low migration   
29:14   
strength and a a low um   
29:20   
migratory strength, then you have very little very little motility and all the   
29:25   
the variations within. So we have two graphs here. We have uh where we have   
29:32   
T20 and T1000. So they're generating this for different times and you can see   
29:38   
that when there is a low migratory strength, you know, it's just a smear of   
29:43   
a certain color. Um, and then, you know, as you start to get higher migratory strength, you get pattern formation.   
29:52   
At uh low initial concentrations, there are these blobs. At higher   
29:57   
concentrations, there are these kind of stripes. And then at even higher concentrations, you have these inverse   
30:02   
blobs. But the background in the lower concentration uh case becomes kind of a blobs here. So   
30:09   
it's kind of interest it's kind of uh recapitulating what we might think of as   
30:14   
chemical morphogenesis uh under those conditions. You see the same thing for for running it for   
30:21   
longer. You see actually for the uh low migratory states you actually see some   
30:27   
pattern formation where you get striping and then blob formation that's very   
30:32   
illdefined. And then for these other cases uh going down in terms of   
30:38   
migratory strength. So greater migratory strength you start getting these blobs but they're bigger. Um and then you get   
30:45   
these even more well- definfined blobs with a greater migratory strength. So you can see how those two parameters   
30:52   
interplay and and contribute to pattern formation. What this suggests is that of course   
30:59   
migration strength is key to pattern formation but also concentrations and we knew that   
31:06   
from chemical morphagenesis that concentrations are uh more you know as   
31:11   
important but maybe not as important as we thought. It's interesting uh interesting set of results.   
31:19   
And so then they talk about zeroflux boundary conditions. So zeroflux boundary conditions presents a similar   
31:25   
challenge to periodic boundaries as the non-local integral must also be defined at the domain values. Unlike periodic   
31:33   
boundaries however this issue can easily be resolved by setting the solution in the added outer rim to zero which   
31:40   
ensures that the integral is evaluated consistently across the entire physical domain including its edges.   
31:48   
The flux is simply constrained by a zero flex nomen condition which is here.   
31:55   
So that's how they kind of fix this. It's the J vector. It's fixed to zero.   
32:01   
Then they have an initial condition which is a spatial uniform concentration C 0 perturbed by random biological   
32:08   
density fluctuations of amplitude of a certain amplitude to break the   
32:14   
symmetry. For simplicity, these fluctuations are sampled from a uniform   
32:19   
distribution with a vanishing mean. So this is the term. So they basically have   
32:25   
this spatial uniform concentration that gets perturbed by these uh fluctuations   
32:32   
that come from a dens uh probability density function that's defined here. So   
32:38   
you have different amplitudes. it's roughly this uh variance of that   
32:44   
distribution and then you know that that's what you have in the simulation   
32:49   
that breaks up that breaks up this uniform. So you know   
32:55   
again you know we're trying to sort of describe this these sorts of things mathematically   
33:00   
and you know I I guess we get a reasonable result. Okay so   
33:06   
go ahead. Um, I still don't see the stress strain curves.   
33:12   
Oh, well, we're I don't know if we're going to get We're just getting into results. Probably not going to get that.   
33:17   
Well, yeah, we might not. Who knows? Um, but yeah, we'll see. Let's see. So, they   
33:23   
have their uh default simulation parameters here. They kind of summarize them so we know what they're using. This   
33:30   
is an exemplary simulation on a large domain. Okay, that looks interesting. Um yeah,   
33:38   
they don't have very much in their uh results here. And then the conclusions are here and I don't let's see if we   
33:46   
have any stress string curves because it's upon request.   
33:52   
No, I don't think we do. It looks like we have uh some other figures here. Uh   
33:59   
figure four is the impact of mesh element size and number of integration points. So that's here. Uh this is where   
34:05   
we have segmented spots for a mesh with maximum element size of 0.1P.   
34:11   
Scaling of spot area with mesh resolution which is B which is where we   
34:16   
have these different element types. So we have quad types, triangular types and   
34:23   
uh with the max mesh element size uh divided by P and the spot area on the   
34:32   
on the y ais. So the spot area goes up and then down. So it fluctuates as the   
34:37   
mesh element size increases. Um and then C is impacted the number of integration points for the integral of   
34:44   
the pattern. So again we number of integration points. Uh you know by the   
34:49   
time we get to five we get uh you know area settles down. So the area kind of   
34:56   
fluctuates as the number of integration points sort of gets established and then   
35:02   
the number of spots remains constant at nine. So our average spot area is two,   
35:09   
the number of spots is nine and this is like you know after we get to five integration points we can get higher and   
35:16   
the model converges upon. So that's kind of what they're doing here and then the   
35:21   
impact of growth speed on patterning. Uh so the lesson here is that slower growth   
35:27   
which is represented in A leads to fewer more spaced out spots than faster growth   
35:33   
which is B on domains of the same size. So if we have slower growth we get spots   
35:41   
that are fewer in number. They're more spaced out. faster growth leads to   
35:48   
uh you know more I guess a greater number of spots that are less spaced   
35:53   
out. So it's like it's growing fast, it's producing more spots and it's more   
36:00   
dense. That's kind of lessen it. Okay, so that's uh I think that's it for this   
36:06   
paper. Um I'm sorry I didn't give you a   
36:12   
stress drain curve. um they do go through a lot of the stuff that they you   
36:18   
know their methods and I think the point of this paper was to kind of um you know they have this cottage industry of   
36:25   
console multifysics papers where they're modeling different aspects of   
36:30   
morphogenesis and embryogenesis so they might have one some in one of those other papers I   
36:37   
don't know they are kind of going off on a tangent in the   
36:42   
wrong direction is my opinion. You think it's interesting though? It's   
36:47   
interesting research and might apply to other things like um   
36:55   
batteries or something. Um but as to uh   
37:01   
physical cell growth um not as much. Although if they included   
37:08   
the physical um hardness of something um in that model,   
37:18   
they might might get somewhere with that. Not sure. Yeah. Yeah. We need to improve   
37:26   
the elasticity of cells. Yeah. And then they kind of mentioned   
37:31   
that and they said it's a possibility but we we're going to show kind of like pattern formation just as a function of   
37:38   
some of these parameters which is you know fine but it's like pattern formation.   
37:44   
Yeah. Uh you know it's yeah putting the cell in context or if you have pattern   
37:49   
formation like if if the stiffness is different of the substrate what is that?   
37:56   
Yeah. And stiffness is um instead of chemical grad gradient, if you go into   
38:03   
stiffness gradient, and they did have they did have some of that in there, I   
38:09   
believe. Yeah, the stiffness gradient, it um   
38:15   
moves faster than dusion. like the stiffness   
38:22   
can change rapidly whereas chemical dusion is a slower process.   
38:29   
Yeah. Yeah. I think that's important to point out is that the not all gradients   
38:34   
are equal um in terms of how the response of what's going on. response response time.   
38:42   
They're all important, mind you. Diffusion is important, of course, but um it has a different time scale, which   
38:48   
is so you have to watch watch the time   
38:53   
scale of some of these things. Yeah. Anyway,   
39:00   
I don't know. Well, it was Yeah, it was a lot of math and I I'm really kind of I was like   
39:06   
always fascinated by the different mathematical models and the assumptions they make and sort of the ways that they   
39:12   
they employ different techniques. So like the whole we learn lrangeian   
39:17   
mapping, you know, we're using lrangeian methods for growth, but we're you don't   
39:22   
want to use that all the time because it's computationally expensive.   
39:27   
Yeah. No. So, it's interesting. But yeah, now I've not used console that much for   
39:34   
like, you know, embryogenesis. Um, so I don't really know if, you know,   
39:39   
what how well this is done. Um, you know, I I I should go through one of the   
39:47   
um copycell 3D papers and go through maybe as a comparison   
39:52   
point. Yeah, Compell 3D does does what they were doing there. It's the um does   
40:00   
um chemical gradients and physical gradients. Um yeah, Compuell 3D should   
40:06   
be you should be able to do that with it because yeah.   
40:12   
Yeah. I mean, yeah, I guess the point there would just be to compare the methods of   
40:19   
how they arrive at their results, you know. Yeah. See if they're using the same type   
40:26   
of equations. If you can replicate the results with   
40:32   
each platform, be interesting,   
40:38   
right? I think I'm going to try to stick with uh 10segrity   
40:44   
um for the paper I'm going to write and I don't know if I want to get into the   
40:50   
continuum mechanics too much although I supposed to I'm supposed to do a literature review but   
40:58   
that's how a leopard get it gets its spots and I don't know um   
41:05   
how that relates to stress drain curve of a physical object.   
41:13   
Yeah, it's too far out there. So, it can it can look like it it should.   
41:20   
Now, what does the literature look like on the continuum mechanics of tensities?   
41:28   
Yeah, continuum mechanics of tenses. That's an idea. I'll look that up as a   
41:34   
um a broad term to find some more papers.   
41:40   
I don't know if there is a a continum mechanics   
41:46   
of tensity. Okay. So, that's not necessarily a thing. It's just kind of   
41:52   
Well, people have to do that to get um some of their models to work in some of   
41:58   
the commercial programs. Okay. So   
42:04   
they have to work around do a work around and see how it performs.   
42:09   
Yeah. Because it's not um like 10 segreities will all all of a   
42:16   
sudden change shape and then the whatever commercial package you've got   
42:21   
says ah the building just fell down. Yeah. Well it didn't. It was a tense segrity   
42:28   
and it just changed shape. Yeah, that's all didn't fall down.   
42:36   
The microtubules just gave way in part of the cell and it just changed shape and it's still alive and still   
42:43   
functioning the way it should. Yeah.   
42:49   
Very good. I don't know. Yeah.   
42:55   
All right. That sounds great. Thanks. Uh so that that's that paper. Uh, hope that   
43:01   
that was uh useful in some way. Um,   
43:06   
see Morgan's here. Hello, Morgan.   
43:12   
Yes, I just uh still just dropping kids here.   
43:17   
All right. Well, good to see you here. Uh, let's see. I wanted to do something else here. um like get into this paper   
43:23   
and this is going to be I'm not going to get very deeply into it but I wanted to talk a little bit about it just as a um   
43:32   
you know this is a new um archive paper from 2025 from I think uh couple weeks   
43:38   
ago July 31st so this is from biological physics uh that that um that heading   
43:48   
subject category   
43:53   
So this is uh Arvin Muragan and and colleagues and this is called could   
43:59   
living cells use phase transitions to process information. So this is you know   
44:05   
heavyduty bioysics and considering phase transitions as sort of a source of   
44:11   
information processing. So we talked about jamming phase transitions in the last paper. That possibility that cells   
44:17   
can as they're migrating if they get to a certain density can stop migrating because they get stuck or jammed   
44:24   
together. And so then that can be like a way to sort of limit diffusion or it can   
44:31   
initiate fusion if there is a reverse um   
44:36   
jamming phase transition. There are all sorts of other phase transitions in bioysics and you know these are things   
44:44   
where they're changes in the state of it could   
44:49   
be changes in the state of the cell but in what they might be talking about is   
44:55   
changes in the state of a cell population. So   
45:01   
uh the abstract reads to maintain homeostasis living cells process information with   
45:08   
networks of interacting molecules. Traditional models for cellular information processing that focused on   
45:15   
networks of chemical reactions between molecules. So this is where you know the   
45:20   
the cells are producing some um you know some sort of diffusible substance or   
45:28   
secretreting some sort of protein or you know there's something that they're using to communicate with um like   
45:35   
peptides to communicate with other cells. And so that's kind of you know what we think about when we think about   
45:41   
information processing you know taking those signals and decoding them like we talked about with in the last paper with   
45:49   
uh chemical and hardness cues for you know   
45:55   
determining which direction for the taxis. Um so that's kind of what we're   
46:03   
we've been limited to. What they're proposing here is that uh the networks   
46:08   
of physical interactions might also contribute to the processing of information inside cells. So in   
46:16   
particular we focus on the impact of biomolecular condensation a structural phase transition found   
46:23   
inside cells. So we've talked about um cell condensates before in the group.   
46:30   
This is where you have different condensates that occur within I think within the   
46:35   
cytoplasm and it's this phase transition in   
46:40   
different aspects of cellular growth. So they'll get they'll talk about bi they'll focus on biomolecular   
46:46   
condensation as a way to sort of get this idea that phase transitions that   
46:51   
contribute to this process allow for information to be processed.   
46:58   
uh biomolecular condensation has recently been implicated in diverse cellular processes. Some of these are   
47:05   
essentially computational including classification and control tasks. We place these findings in the   
47:12   
broader context of physical computing an emerging framework for describing how   
47:17   
the native dynamics of nonlinear physical systems can be leveraged to perform complex computations.   
47:25   
The synthesis of these ideas raises questions about expressivity   
47:30   
and learning. So expressivity of course we talk about that a lot in machine   
47:36   
learning and in graph neural networks and that is where the range of problems   
47:41   
that cellular phase transitions might be able to solve is either greater or lessened by the level of expressivity.   
47:49   
So in other words, if I have um a combination of things, uh how how much   
47:57   
can that combination of things lead to new things or how much can that combination of things be informative. So   
48:04   
you know you can have algorithms that are highly expressive or less expressive meanings that meaning that they can   
48:11   
solve different types of problems. And in this case you basically have uh with   
48:17   
higher degrees of expressivity a larger range of problems that you can solve with cellular phase transition. So you   
48:24   
know if you think about a system with low expressivity a phase transition might only allow you to solve maybe like   
48:31   
differentiation of a cell with high expressivity it might allow you to do that plus something at the level of the   
48:38   
cell population or maybe the tissue level. And so this is where you know   
48:44   
it's kind of an interesting way to phrase um you know a way to you know   
48:49   
information processing as a problem you know a problem not only a problem be solved but sort of how many problems can   
48:58   
this phenomena potentially solve and then learning of course is how these   
49:04   
systems could adapt and evolve to solve different problems. So this is basically   
49:09   
how can the cell learn you know what to solve and how to adapt to a certain   
49:15   
situation. This emerging area of research presents diverse opportunities across molecular   
49:22   
biophysics soft matter and physical computing.   
49:27   
So this is where they get into talking about how living systems are often far from thermodynamic equilibrium. A lot of   
49:35   
mechanical systems exist in this thermodynamic equilibrium also maybe a   
49:40   
structural equilibrium as well and living systems don't necessarily do that. Um and so you know we don't really   
49:47   
have a good way to model far from equilibrium systems. We usually think of   
49:53   
modeling as an exercise and placing things in equilibrium and then testing.   
49:59   
Same thing with a lot of experimental physics where you assume equilibrium and   
50:04   
then you deviate from equilibrium in some way and you know you might see how   
50:10   
it recovers but it always has this equilibrium point that you can uh you   
50:15   
know fix the equation to zero and of course if something is behaving far from equilibrium it's often going to be away   
50:22   
from that that zero point and so it may not recover for a long time or it may   
50:28   
not recover at all. Um and you know if we think about homeostasis that's   
50:35   
largely an assumption of normality, right? So homeostasis or a set point is   
50:40   
a set point where the system is operating under normal conditions and   
50:46   
it's healthy and that's how we kind of define a healthy state. But there's no reason to think that that's you know   
50:53   
that the set point is either healthy or   
50:58   
is something that the system operates. Um so this is kind of a you know talking   
51:05   
about kind of our assumptions about homeostasis and about how you know energetics is   
51:13   
efficient by sort of uh queuing to this homeostatic standard.   
51:21   
And so they go on um each even an individual cell can sense and compute demount an appropriate response to   
51:28   
external stimula or internal changes of state shown schematically. They show this in figure one. Um and then they   
51:36   
talk about different signals environmental cues and internal signals and these can be of many different   
51:43   
types. Um and so you know we have nutrients and stress coming in from the   
51:48   
environment. And then stochastic expression of genes and the depletion of metabolites coming from internal.   
51:55   
Um and so then from an information perspective, homeostasis is a control problem. So they're going to apply a   
52:02   
control closed loop control system to this. And so again, this is like a model, right? I mean, you know, it's   
52:09   
going to describe the system in a way that they're hypothesizing it operates.   
52:15   
So you know this is again you know just this weight you know so they they kind   
52:22   
of point out here in some cases cells respond to a relatively simple one-dimensional stimula such as the   
52:29   
concentration of a single spec chemical species for example Ecoli and neutrfils   
52:35   
move themselves up a concentration gradient in a process called chemotaxis   
52:41   
and so this is where you you know these different types of uh you know you have   
52:46   
ecoli which is a single cell organism and neutrfils which are in a in a   
52:52   
nervous system they move themselves according to these chemotaxis   
52:58   
more generally cells possess a vast array of physical eg matrix stiffness   
53:03   
and chemical signals eg soluble and matrix bound growth factors so you can   
53:09   
have you know not just chemotaxis but you can have a variety of these different taxes   
53:14   
working indep or semi-independently. There's some joint distribution of them but mainly you know uh they're different   
53:21   
processes that the system is responding to differently and it adds to this heterogeneity.   
53:29   
Processing of these highdimensional stimuli can include classification the assignment of a label to a broad   
53:35   
range of input values. Classificationals allow cells to mount distinct responses   
53:40   
to complex stimuli. uh indeed classification has been recognized as an essential step in stem   
53:47   
cell differentiation. The patterns triggering particular differentiation pathways require   
53:54   
discrimination between many lian combinations temporal patterns. So cells   
53:59   
need to be able to classify things and the classifications be correct can't have a lot of false positives   
54:06   
and this is important in a lot of processes that are sort of high tolerance pro or low tolerance processes   
54:13   
where you have to have high precision and you can't have a lot of error although you can have error in some   
54:19   
processes and there's there's sort of an error built in but error correction mechanisms built in but you know this is   
54:27   
a little bit beyond this In this perspective, we discuss how two   
54:32   
emerging topics in physics, biomolecular condensates and physical computing will complement each other and provide fresh   
54:39   
insights and new tools for information processing, biomolecular networks,   
54:45   
particularly those inside of cells. And so this is where they talk about   
54:51   
biomolecular condensates. So biological cells comprise thousands of different   
54:56   
molecular components. which are organized into many spatial compartments. And so while some of these   
55:02   
compartments are enclosed by lipid membranes like the nucleus, biomolecular   
55:08   
condensates are these domains that are sort of enriched in proteins and nucleic   
55:14   
acids. So you know you have these like basically clumps of material that that   
55:20   
have information in them. They're not enclosed like we see in a lot of organels or in the nucleus but they're   
55:27   
there floating around cytoplasm. This figure one where we have uh a which is   
55:34   
where they highlight that the stimula are multi-dimensional. So you have nutrients, growth factors,   
55:42   
temperature and elastic moduli. Those are all inputs to the cell. They're all   
55:48   
stimulating the cell sometimes in different directions. So you have these stimuli coming in and   
55:53   
then you have this response differentiation division metabolism. So these inputs   
56:00   
lead to these outputs these responses and the idea is that it's processing   
56:05   
this information to get to this response. Um B is where we have u some detail.   
56:13   
This is a condensate here. Just these green blobs which are these balls or   
56:18   
these masses of different you know proteins and nucleic acids and things like that. And so   
56:26   
it's uh you know they're enriched with these things and they float around and they're sort of these storehouses of   
56:32   
these materials and it's very important in a lot of uh developmental processes. So you have   
56:40   
this uh this is sort of internal to this. So this is information processing between input and out.   
56:47   
Okay. So that's figure one. Then figure two is where we have these phase diagrams of biomolelecular condensates.   
56:54   
So we have two different concentrations of condensates. The low concentration   
57:01   
and the high concentration case. And you can see in the low concentration case, you have these small blobs of things or   
57:08   
maybe they're independent proteins and nucleic acids floating around   
57:13   
cytoplasm and then high concentrations lead to this clumping of these different   
57:20   
biomolecules together. These clump these large clusters that you can see in the   
57:26   
green. So you have to have high concentrations to have these large condensates.   
57:32   
they form um sort of as maybe a phase transition of concentration. So when you   
57:39   
have low degrees of concentration, you know, things are independent and then when you hit a certain threshold of   
57:46   
these things of cytoplasm, then you get these clusters and it's spatial, you know, it's spatially heterogeneous. So   
57:53   
you might get clusters in one region but not another. But anyways, this this phase transition   
57:59   
allows you to have or allows the cell to have these condensates available.   
58:04   
So this is system concentration here. This diagram above this these two   
58:10   
figures u there's this tie line and then there's this sort of phase transition   
58:17   
from individual biomolelecules to these condensates. This sharp transition that occurs uh   
58:25   
that's the canonical system. And then the grand canonical system is where we have some attractive surface forming a   
58:31   
subco compartment. So these uh you know this is where this is free floating   
58:37   
stuff in the nucleus and forms this bio condensate. This is in B where they're   
58:44   
attaching to a substrate and you know again they're just kind of attaching at random and there's this sort of   
58:50   
scattering of these biomolecules on the substrate. Whereas in high concentration conditions, the entire   
58:58   
uh surface of the sub compartment is packed with these biomolecules and their   
59:03   
biomolelecules competing to get down to the bottom. And you know, there's this   
59:09   
phase transition that occurs. You get this surface that's filled with biomolecules versus the surface that's   
59:16   
sporadically covered with biomu. And again, up in this figure here, you have   
59:21   
where you have a reservoir concentration of biomolelecules as that increases   
59:28   
um then you get a switch from phase A which is this sporadic case to phase B   
59:34   
which is the packed case. So again you have these different behaviors and it   
59:40   
leads to maybe differences in information processing because you have different modes of these biomolecules   
59:47   
available. So yeah, the composition in the sub   
59:53   
compartment in B uh under goes a discontinuous jump when the   
59:59   
concentration is increased across the phase volume. Okay, so we're modeling the   
1:00:05   
concentration of this. We're looking at the behavior. we're noticing is phase transition which can be described by a   
1:00:11   
nonlinear physics models especially this kind of this is kind of a this isn't the   
1:00:17   
formal model but this is a diagram showing the power law distributions   
1:00:25   
okay so uh typical biomolecular condensates include many distinct compartments or components stress   
1:00:32   
granules uh are thought to have hundreds of components these components are enriched selective ly based on physical   
1:00:39   
interactions which are dictated by specific sequences and chemical modifications.   
1:00:44   
Um while there have been no systematic no systematic census of condensates   
1:00:49   
merging the results of independent studies suggests that there are at least tens of distinct coexisting condensates.   
1:00:56   
The compositional variation across different types of biomolecular condensate yields compartments with   
1:01:01   
distinct physical and chemical properties. They can localize biochemistry through   
1:01:07   
enrichment or depletion of enzymes and diverse small molecules including ions   
1:01:12   
and metabolites. And so macroolelecules could adopt novel confirmations condensates that could   
1:01:20   
impact their reactivity. So being in a condensate allows you to have not only a   
1:01:25   
higher concentration and more you know more sort of strength when there's a   
1:01:31   
site of action available but then there's also this increased reactivity because you can have these novel   
1:01:36   
confirmational states that then map to function.   
1:01:41   
Uh physically condensates have diverse realological properties like viscosity and elasticity interfacial tensions and   
1:01:49   
electrostatic potential. Um and then the you know how do you get   
1:01:55   
these phase transitions? So condensation of these biological molecules can be   
1:02:00   
conceptualized as a phase separation where homogeneous mixture sorts itself into at least two different macroscopic   
1:02:07   
domains. For systems near equilibrium, phase separation is understood as a   
1:02:12   
competition of translational entropy favoring homogeneous mixtape and   
1:02:17   
cohesive interactions favoring segregation into different phases.   
1:02:23   
Okay. Um and so while we consider an entire cell on time   
1:02:29   
scales of reproduction and degradation are negligible, the total amount of material in the system is conserved. And   
1:02:36   
then you know we work in the canonical example and we have this canonical ensemble   
1:02:42   
which is where you have multiple phases and they can differ in concentration by   
1:02:49   
multiple orders of magnitude. Um okay so then this second figure or this   
1:02:57   
third figure figure three shows the phase boundaries of   
1:03:03   
multicompartment systems is decision boundaries so this is where we have um   
1:03:09   
we have uh so a is this space of images and this is just a conceptual model of   
1:03:16   
how decision boundaries work. So like you have um the space of images that are inputs   
1:03:25   
to this neural network and the neural network has to decide whether a certain image belongs in the category of cats or   
1:03:32   
dogs. And so you know this is where you have these black points and each of   
1:03:38   
these points is some variation on cats and variation on dogs. And so you know   
1:03:44   
it has to classify them as either a cat or a dog given the features the feature space of the images. And so it can do   
1:03:51   
this through these synaptic weights and the synaptic weight matrix WJ.   
1:03:57   
So it's through the connectivity of the system that it's it's allowing it to process the information so that it can   
1:04:04   
make a determination. So the idea here is that you have some molecular   
1:04:10   
network that where you have a bunch of information that's being transferred and   
1:04:15   
it's giving information about different features of a of a biomolecular tendency   
1:04:21   
or some confirmational state and it's trying to make a decision that will put it in one category or another. So the   
1:04:28   
output will be one category or another. The input is all this information and   
1:04:34   
you know Hopefully the thing does it correct in order for the cell to survive and to return.   
1:04:41   
Now in a molecular soup, this is not a neural network. They do have the synaptic weights which is wig. So they   
1:04:48   
use that in this model of the molecular soup which is a network of I guess um   
1:04:56   
uh we have these different molecular   
1:05:03   
actors and their interactions between them and you have these binding affinities basically which is also W.   
1:05:11   
The idea is that these uh biomolelecules will bind according to affinities widget   
1:05:18   
and they act very much like a set of synaptic weights in a neural network which aren't actually synapses but you   
1:05:24   
know um that's the way they refer to them. And then this gives the model   
1:05:30   
another sort of set of information which allows it to sort of make a decision   
1:05:36   
whether it should switch from context one to context two. So it has this ensemble of information available to it   
1:05:44   
and from that ensemble much like the feature space of these images it can go   
1:05:50   
from one sort of decision making uh context to another and it can switch   
1:05:55   
across this phase boundary and there's a phase transition. So if we think back to our molecular condensates when we don't   
1:06:02   
have our condensates when the biomolelecules are scattered all over the place then you you maybe you're in   
1:06:08   
context one when that ensemble changes due to there being a biomolelecular condensate available then the behavior   
1:06:16   
output behavior changes in context two.   
1:06:21   
So drawing from this analogy, they're able to kind of make this case or at least partially make this case for um   
1:06:29   
decision making in the cell being driven by these phase transitions.   
1:06:35   
And so comparing figures 3A and 3 C uh we reinterpret phase transitions in a   
1:06:41   
multicomp multicomponent system is classification of highdimensional chemical compositions. Intriguingly,   
1:06:49   
meanfield models of multicomponent phase separation are analogous to artificial neural networks. They are built upon a   
1:06:55   
network representation of molecular interactions where a set of parameters w   
1:07:01   
describe the physical coupling of different molecular species shown in figure 3D. This is here. Um just as WIJ   
1:07:10   
adjusts decision boundaries in an artificial neural network, WIJ moves phase boundaries in a multiol system.   
1:07:18   
And so then they talk about closed loop control and condensation. Um they talk about control systems using   
1:07:25   
feedback and how that's very important. And they talk about a cheostat as an   
1:07:31   
example of this closed loop feedback. So where a cheostat can maintain a fixed pH   
1:07:36   
value within a reactor is even as non-stationary chemical reactions consume and produce pH active species.   
1:07:45   
And so phase separation can greatly simplify control into a single physical   
1:07:50   
process without separate entities to sense, compute and actuate. And so uh   
1:07:57   
phase coexistence can suppress concentration fluctuations. And that's in this paper number 28.   
1:08:04   
And that is this paper here in nature reviews   
1:08:11   
molecular cell biology 2017. So they they kind of go through a   
1:08:16   
little bit more about temporal information processing and how phase separation can kind of   
1:08:23   
model a lot of this and provide a model for   
1:08:29   
for you know these different types of decision making.   
1:08:36   
Okay. So I think that's enough about that paper. I think it's, you know, a nice hypothesis driven paper where they   
1:08:42   
kind of have this idea that this is what's happening in in in a cell and there's this whole field of cellular   
1:08:49   
decision making where we have all these inputs that lead to these kind of outputs that are discrete decisions that   
1:08:55   
the cell can make and there's this effect on collective behavior of tissues and things like that. But it's a nice   
1:09:03   
way to kind of incorporate biomolecular condensates.   
1:09:09   
I like that paper better than the previous one. It has It seems more real,   
1:09:15   
more realistic. Yeah. Anyway, right. Yeah. Well, anyways. Yeah. So,   
1:09:22   
next week maybe we'll do like a copy saw 3D paper. I'll have to go through see if   
1:09:28   
I can find something good. See if they have a Kanan concent concentrates.   
1:09:34   
Yeah. Yeah. Or one that's similar to the spots and   
1:09:40   
stripes one that people have done that for quite a while making um chemical spots and   
1:09:49   
stripes. Um you could ask Dick about it. Yeah. Yeah. Oh yeah, sure.   
1:09:57   
Yeah, that's that's easy. That's an easy trick. The question is is like how do you attach that to the rest of the   
1:10:04   
biology? Yeah. Well, you've got Bz reactions that do something similar   
1:10:10   
with spots and stripes. Um sort of a controlled pattern,   
1:10:16   
right? All right. Well, that's all for today.   
1:10:22   
Thanks for joining. Okay. Thank you. Thanks. Bye.   
   
