## TRANSCRIPT  
0:00     
hello how are you i'm pretty good     
0:06     
great yeah what did you have any updates or     
0:13     
not really i'm trying to investigate um how other people have um studied     
0:22     
tense um yeah I've got one paper     
0:28     
about it um how to calculate their mechanics oh     
0:35     
okay um because they're basically unstable     
0:42     
okay like they because they're in tension so I have one paper on it     
0:51     
and a couple of math papers so I'm just looking into it a bit cuz I'm     
1:00     
like other people do this but they use things like Mathematica     
1:06     
programming so that's something you don't want to     
1:11     
learn well I've I've used Mathematica it's     
1:17     
okay yeah I mean yeah it's good luck     
1:23     
I I just turned off my video feed because it seems to be     
1:30     
stalling things okay     
1:36     
yeah so I'll I I'm good i haven't done much so Oh no okay well that's good yeah     
1:44     
thanks for giving the update you have uh of course our two new     
1:50     
GOC students um Lewis and Giadratha and they've been     
1:55     
to the meetings before i don't know if they're going to be here today uh we're in our community period which means that     
2:01     
they're getting up to speed on the community uh we have the openworm uh     
2:09     
group and then we have the orthogonal research and education lab group and they've been kind of interacting more     
2:16     
with our orthogonal lab group uh so far uh we've we have our open source     
2:22     
meetings and you know we've been talking about different topics in open source really kind of getting a baseline     
2:29     
appreciation for the organizations and some aspects of open source so um I think in about two weeks     
2:37     
they're going to start coding period and that's where they're going to start producing demos and things like that     
2:44     
based on their projects so congratulations to Giadratha and     
2:51     
Lewith and their projects are going to be DVO graph projects so DVO graph um     
2:59     
projects like basically we got like about seven or eight     
3:04     
applications for that project and so uh we'll have to maybe towards the end of     
3:11     
the community period I'll have them present on their project proposals and make sure that we know where they're     
3:16     
going with that looks like Andre is here and is     
3:21     
here um I just uh I'm just software development try to better understand     
3:28     
what's in the openorm group going in and if I can help yeah that um welcome uh     
3:34     
yeah so we're the D.VARM Evaworm group we're part of open worm uh we're kind of     
3:40     
independent of open worm in a lot of ways and what we're interested in is developmental or computational     
3:46     
developmental biology so we're interested in developmental biology of the worm of sea     
3:54     
elegance uh we're interested like the embryo and modeling the embryo modeling     
4:00     
some of the developmental stages of the worm uh there's a lot of uh embryoysics     
4:06     
involved in that there's a lot of uh data collection like imaging     
4:11     
technologies that we'll talk about there's a lot of stuff like what Susan's doing which is     
4:19     
biotensgrity which is looking at the mechanics of the of the worm looking at     
4:24     
how cells kind of fit together in into a tissue or into a structure how those uh     
4:34     
mechanical changes impact the phenotype the way it behaves etc     
4:40     
etc and then you know we have um other groups we do things like uh we work with     
4:47     
networks we work with dynamical systems to describe things like development and     
4:53     
behavior and things like that and so um you know the the goal of open worm is to     
4:59     
give this u modeling of the worm uh modeling of the uh model organism C     
5:06     
elegance from different perspectives so there are different projects some that focus on neurons some     
5:14     
that focus on the electrical activity of neurons some     
5:20     
that focus on whole body modeling and bioysics and then we're interested in development and we're also interested in     
5:27     
uh connetos and developmental conneto so one of the things you have with C elegance is you have that u that full     
5:35     
conto that we know and we understand the connections and we understand the cells     
5:40     
and so that that's of great advantage when we're looking at any sort of question regarding like a nervous system     
5:47     
or connect we can understand that nervous system on a cellby- cell basis     
5:53     
and then we can understand some of the circuits that are involved in behavior but we can also approach that from a     
6:00     
developmental perspective where we have you know where those cells get put into     
6:05     
place where the nerve fibers get put into place and all of that so that's     
6:10     
what we're doing Andre and you know we do have all sorts of different projects     
6:16     
um you know we we're always in our meetings that's kind of the glue of our group we're always talking about     
6:22     
different things that people are interested in and it's really you know it's kind of up to anyone individual to     
6:29     
like take the initiative on something if they find something really interesting they can u drive it forward you know     
6:36     
present in the meetings do a demo or just kind of you know see what what you     
6:42     
know see something works see if there's some computational problem that you're     
6:48     
interested in or a developmental problem or something in in like uh openorm you     
6:54     
know like one of the models that they use in open world as well it would be interesting so welcome and um you know     
7:03     
hopefully you find this useful so we have Sohham and Pri     
7:10     
Priyanka here as well we have Jesse and Morgan and Susan and Andre and     
7:16     
myself okay welcome so like I said we're interested in a lot of different things having to     
7:22     
do with biology i'm going to talk about something now that's a computational     
7:28     
model that's I guess it's related to biology but we'll see so this is a paper that I think is     
7:36     
coming out at SIGRA this year     
7:43     
uh this is uh inverse geometric locomotion so this is a paper that     
7:52     
um so in computer graphics and computer vision people have been interested in     
7:59     
simulating movement of things so like when you want to model say like clouds     
8:05     
in the sky you want to be able to find a hyperrealistic model of those clouds     
8:11     
like the depth the shape the the texture and then you want to be able to animate     
8:17     
that so you want to be able to find an efficient way to animate all that and you don't necessarily care what's inside     
8:23     
of the thing you just want to show a visual representation of it sometimes     
8:29     
you want to be able to simulate like you want to break things down into segments and simulate those segments you want to     
8:36     
have something that's lifelike or something you would see in the real world but you also want to have     
8:42     
something that's computationally efficient meaning that you have to have you have to be able to simulate it     
8:48     
without any sort of lag so it becomes a problem of kind of how to do that and so     
8:54     
people have proposed a lot of different techniques over the years there's been a     
8:59     
lot of math spilled a lot of like computing power and cool animation     
9:05     
spilled yeah so this is inverse geometric     
9:10     
locomotion and this is uh and so this is     
9:16     
the animation here kind of the shorthand of what they're trying to do so they have this sort of worm and I don't want     
9:24     
to call it a sea elegance or anything but basically it's a worm that moves you     
9:29     
know in a in a way that it kind of moves laterally it slithers sort of like sea     
9:35     
elegance but the idea is that this worm is moving through this environment around these     
9:40     
obstacles and so they want to be able to capture the kinematics or the movement     
9:46     
dynamics of this movement so if you had like an actual organism say there would     
9:52     
be like this kinetics of movement which is the energetics of movement the energetics of the muscles the forces     
10:00     
being produced by the muscles and you'd model the muscles you'd model maybe molecular components of the muscles and     
10:07     
other things like that if you want the kinematics however     
10:12     
you want to model the movement so you're a lot it's a lot more free in terms of what you can how you can produce that     
10:19     
you don't need a hyper realistic model of the what's going on inside the phenotype here you just need to     
10:25     
reproduce the movement relative to this body so this rigid body or the soft body     
10:32     
is going to move and you want to be able to have it move around obstacles you want it to be able to move around the     
10:38     
space and you but you want to understand how it's producing those movements right so it's like a dynamical systems problem     
10:45     
where you have this system with a a certain number of degrees of freedom or the number of ways in which     
10:52     
it can kind of change its shape or change its orientation as it's moving     
10:58     
and you know it that sort of enables the kinematics so there's some path that it     
11:03     
needs to follow but it only has a certain number of degrees of freedom and so in in the course of doing that you     
11:11     
know you're able to produce a more or less lifelike looking movement and the only thing you're constrained by is sort     
11:17     
of the energy requirements and this case you know how much you can render in the graphics engine these movements so     
11:25     
that's kind of what this is um so this is uh in in inverse means that they're     
11:31     
solving the problem using an inverse solution meaning they're solving it backwards from the equations and they're     
11:38     
mapping it to this thing this uh these animations so I'll just read the     
11:44     
abstract here um the abstract reads "Numerous tasks in robotics and character animation involves solving     
11:51     
combinations of inverse kinematics emotion planning problems that require     
11:57     
the precise design of pose sequences to achieve desired motion objectives so     
12:03     
these are the pose sequences here where you have this object that's at these different orientations and the degrees     
12:10     
of freedom are in a certain configuration and they have to move like you know based on what those look like     
12:17     
so um however accounting for the complex interplay between body deformations and     
12:24     
resulting motion especially through interactions with the environment this poses significant challenges for the     
12:30     
design of such pose sequences uh so you know if we think about like     
12:36     
what they're doing in cybernetic in open room what they're doing in cybernetic is     
12:41     
they're modeling the biohysical aspects of the worm body and the simulations     
12:47     
that they run are usually embedded in this medium that they they present so     
12:53     
they have to model the medium in which the worm is moving through it could be like a gel it could be like a colloid it     
13:00     
could be something that has a certain density a certain viscosity etc we     
13:08     
talked about low Reynolds number regimes a couple weeks ago where you know we have the Reynolds number which is a     
13:14     
hydrodnamic parameter and it basically describes the flow that uh something     
13:20     
encounters whether it be a cell or an organism and so that Reynolds number if     
13:26     
it's low means it's a laminer flow and there isn't a lot of turbulence and so     
13:31     
we can do this with hydrodnamics we can do this with solids we can do this with     
13:37     
other types of materials and the idea is that this worm is going to be moving     
13:42     
through that material and the the properties of that surface or that     
13:50     
medium are going to affect the body and its dynamics and how it can move and so     
13:56     
that's what they do in in cybernetic is they model that process and they want to understand you know that the worm being     
14:04     
a uh being a softbodied organism and not a rigid-bodied robot or something like     
14:11     
that um is you know can sort of conform     
14:16     
to some of the demands of that medium but you know there's a limit and it     
14:21     
requires a certain amount of energy expenditure requires a certain amount of uh movement compensation so you know you     
14:29     
can run simulations in in those in that kind of an environment where you model different types of medium media and um     
14:38     
you can get information about you know what the worm might experience in that     
14:43     
in that um environment so it tells you something about muscle uh physiology it     
14:50     
tells you something about the way that the uh body is configured and so forth     
14:57     
so that's a very high computational thing like high cost computational thing     
15:03     
these uh biohysical simulations and this is by pretty high computational     
15:08     
uh pretty high computational thing as well because you have this um you have to model all this movement uh as a     
15:16     
dynamical system and so you know this is kind of what we're trying to do here um     
15:22     
that they're trying to do is they're trying to make things look really compelling and what they're doing in the     
15:28     
biohysical simulations is they're trying to get um precise measurements so um so anyways back to     
15:36     
the abstract we propose a computational framework to address these challenges in     
15:42     
scenarios where the motion of a deformable body was entirely determined by dynamic changes of its shape     
15:50     
complementing recent methods on the forward problem which is mapping shape sequences to global motion     
15:57     
trajectories we addressed the inverse problem of optimizing shape sequences to     
16:03     
achieve easier defined motion objectives we demonstrate the effectiveness of our method through a     
16:09     
diverse set of examples producing realistic shape sequences that result in     
16:15     
desired motion trajectories so they have these shape sequences which are the changes in this phenotype it has to go     
16:22     
through this obstacle course and then that res results in these desired motion trajectories meaning it has to conform     
16:29     
to what we might see from like say a worm in nature instead of like just something that you've never seen before     
16:37     
because it could produce those kind of movements but we're not really interested in those so there is this um video here     
16:45     
this is um just kind of their sigraph video so this is inverse     
16:52     
geometric locomotion this is snake locomotion where you see this snake     
17:00     
moving around these obstacles and it's producing something     
17:07     
that's like a snake- like movement right i mean it doesn't look like it's trying to walk or it doesn't look like it's     
17:14     
uh just kind of moving straight forward it's it's making these kind of movements and again this is um I wouldn't say it's     
17:21     
necessarily biologically realistic in the sense of like biohysical simulation     
17:26     
it's just kind of realistic uh looking so this is a single scallop     
17:32     
so this is a go back here all right so that's the     
17:39     
worm and then there's the single scallop versus two scallops on a stick so these are marine invertebrates     
17:46     
these are their shells and they open this shell up um and you know there's     
17:53     
one and then there are two that are going in opposite directions they're connected by a stick so they're just     
18:00     
showing these movements and how realistic they can be this is a turning     
18:06     
snake where this is where the snake is trying to find a a path forward     
18:12     
uh you know using its uh degrees of freedom to achieve     
18:18     
this okay if we go further we see that we     
18:23     
could do that snake some more we have a falling cat using matching userdefined     
18:29     
semantics so we have this cat this this uh point mass model of a cat that's     
18:37     
falling and it's falling onto a surface this is hinged kinematic locom motor so     
18:42     
this is just kind of a I don't know some sort of geometric shape that's like exhibiting some     
18:49     
behavior and it's kind of using its degrees of freedom to produce movements this is the kinematic     
18:56     
locomotive plane so it's just kind of moving around this is a spherical locomotive so this is where you have a     
19:03     
sphere that's operating under the same concept and again they're just showing off the method on these different models     
19:10     
this is nothing well I guess you could maybe model a virus like that or something this is like maybe a cell     
19:17     
surface where you have undulations on the surface but the idea is that you have this this triangular mesh that's     
19:24     
kind of moving around this is uh 12 uh leloi and iigen modes so they're     
19:32     
using different techniques so again this is the it's a spherical     
19:40     
mesh there were 12 points on this this     
19:45     
example and so we could go through this all day i guess seven minutes this is a     
19:51     
ray that's moving so this is a you know looks like a ray this is percel swimmer     
19:58     
so this is a point mass model that's swimming around producing this trajectory     
20:04     
from this middle middle segment and then it's being graphed down here so you can see it's following this     
20:11     
attractor with its movements and it's producing these rhythmic movements that     
20:16     
have this sort of chaotic behavior semiotic behavior okay so I think that's enough     
20:22     
of that um I just wanted to point that out and then you know of course they're using     
20:28     
inverse methods here which means that they're going translating back from a mathematical model to these movements     
20:34     
they're trying to infer what these movements are from the mathematical     
20:40     
model so that's um that was that u paper i thought that was very interesting in     
20:47     
light of some of the things we talk about in the group uh do we have any questions about that     
20:57     
can you share the link as well for the paper yeah yeah let me do that     
21:06     
thank you for the detail description no problem     
21:15     
okay here we go that's the link uh Andre in response to um what I     
21:23     
had talked about before I presented the paper he says "Thank you it was definitely helpful i know that I have a     
21:30     
lot of things to learn so this looks like a good start i definitely lack biology." A lot of people who join the     
21:36     
group either lack like computational knowledge or biological knowledge that's one of the things we try to do in the     
21:42     
group is to merge those together okay     
21:51     
um I should ask if does anyone want to give any more updates you know that     
21:58     
Soham and Priyanka maybe have something they want to share or maybe Jesse or     
22:07     
Morgan uh I just started and actually I was in transition to an internship as     
22:13     
well so I think this week would be a little hectic for me but yeah I would     
22:19     
definitely catch up from next week moving onward okay sounds good     
22:27     
right     
22:37     
okay so let's see uh Andre had another question since this group is relatively     
22:42     
separated from openworm is there a place on GitHub where I can take a look at what's there okay so yeah we have a     
22:49     
couple of places on GitHub let me share the links     
22:57     
here so we have two uh locations on GitHub and I might as well go over those     
23:04     
um as I think they'd be useful to people so the first one is uh our DVORM uh     
23:12     
repository that's the first link and I'm sharing my screen now so you should be able to see     
23:19     
this and so this is our kind of legacy repository where we do a lot of our     
23:24     
projects we have a lot of repositories some of which are you know uh kind of     
23:30     
not operative anymore and some of them are and we revisit them from time to time     
23:36     
so in this repository we have our weekly meeting archive which just basically     
23:42     
goes over our different meetings that we've had we have weekly meetings going back to like 2020 so if you're     
23:48     
interested in different topics you can look through that list and see if there's something that's you want to     
23:54     
follow up on there are different projects people have done ranging from proteomics and     
24:02     
informatics to different projects uh having to do with u modeling biology     
24:09     
uh different packages uh for uh image segmentation uh different packages for     
24:17     
network reconstruction um I'd have to point you to a specific     
24:22     
um repository if you're interested in more I can answer questions about what's     
24:28     
in here because there are a lot of uh repositories and forks and it's kind of hard to follow um you know exactly     
24:35     
what's going on here but that's okay that's the way it usually goes when you have a GitHub     
24:42     
uh account for you know years and years some of these things are associated with     
24:48     
papers like this raising the worm brain um I'm looking for uh a link to our data     
24:57     
sets this is where it is dvorm so in GitHub uh     
25:04     
DVORMDOWROM we have a bunch of data sets and these are data sets that we've collected over the years these are kind     
25:11     
of like our baseline data sets so if we're looking to train a model or we're     
25:17     
looking to build um you know some kind of data set we might use some of these     
25:24     
resources so we have of course the C elegance data from     
25:29     
development we have information about cell birth and death timing um I don't     
25:36     
think we have the conneto data in this repository it's in another place but we     
25:43     
have you know we have data from acidians or or an organism called sea squirt we     
25:49     
have um gene expression data lineage tree data and so forth so there's there     
25:55     
are a lot of data sets in this repository divorm then if you go back     
26:04     
to there should be another one for connetos here um not really putting my     
26:11     
fingers on it right now but we have another set of data for connect     
26:16     
developmental connetos and that is not quite sure what     
26:21     
that is but if you go through this you'll find     
26:27     
um you know a lot of repositories and if it looks like like something you're interested in my my suggestion for this     
26:34     
would be to go through uh maybe the meetings first this is it and think     
26:41     
about what you're interested in and then look for something in the GitHub repository this is what I was looking     
26:46     
for with the embryo genetic conneto this has data uh for different stages in     
26:54     
development and the cells that are involved in the conneto this is linked to some work we've done on this topic um     
27:01     
so there there are CSV files in here which are you know they should give you the comma delimited format um there are     
27:09     
other types of graph ML structures in here so there you know we we have all     
27:14     
sorts of different um we we use all sorts of different programs to do these     
27:20     
things you know you have like graph theoretic programs uh you have uh     
27:26     
biohysical modeling programs susan does a lot of stuff with like cell mechanics     
27:33     
and then we we build and I'm going to show you that in a minute we build our own software for image processing and uh     
27:41     
building uh biological networks so this is a data set for connetos so there all     
27:47     
sorts of data sets in here uh the next one is dvo learn and that is     
27:53     
this uh repository here's this organization so uh in about 2020 we've     
28:01     
uh kind of got to the point where we wanted to build a software     
28:06     
platform for you know segmenting uh images so a lot of the biology that we     
28:14     
do is like taking microscopy images which you can get uh freely available     
28:19     
microscopy data from different repositories um and you can get those data but you can't necessarily say     
28:26     
anything about the biology until you segment out things from the images so if     
28:32     
you have an image of like an embryo you can't say much about that embryo until you've segmented the image into into     
28:38     
cells into membranes into nuclei and like you know have a map basically of     
28:44     
what those structures are in different images so if we have a time lapse of an     
28:51     
embryo you know we have cells we have nuclei sometimes cells are dividing so     
28:58     
we might have more cells in in when we compare to two different images we might have more cells in that new image we     
29:06     
might have changes in the membrane and those are the things we're really interested in so we developed some     
29:11     
software for this it you know it it's ranged from like using uh you know sort     
29:18     
of RNN models and pre-trained models to the segment anything model that Meta put     
29:24     
out a couple years ago and so we have a number of models for image processing     
29:32     
those are located both on GitHub and in hugging face if you go through DVLearn     
29:38     
which is the platform you'll see that we have um a number of different models that uh     
29:47     
we use here that are sort of at the heart of of what we're doing with image     
29:53     
processing um so we have a nucleus cell uh nucleus segment a cell segment and I     
30:02     
think a semantic model so this is really interesting stuff and again you know I     
30:09     
can kind of help you navigate this if you're interested in something specific     
30:14     
and then of course we have D.VO graph and this is the thing we've been working more recently this is uh you know     
30:20     
applying graph neural networks to these kind of microscopy images so it's not enough just to segment the cells     
30:27     
although it's necessary to segment things into membranes and     
30:32     
um nuclei and other things like that but then you want to know what how they're     
30:37     
related to one another so that's where DVO graph comes into play so we have these graph structures that we can     
30:44     
reconstruct from images we segment the images and then we take those uh     
30:49     
components that we've segmented out and we try to find the relationships between them using graph theory and then we have     
30:56     
graph neural networks which process those graph structures and turn them into embeddings and then we can say     
31:03     
things about the biology and we can make you know hypotheses but we have everything saved     
31:10     
in in a computational data structure and so this is what this is what our current     
31:16     
Google summer code project is for this year u we've we've had a number of     
31:21     
contributors to this over the years as well so it's a pretty robust attempt at this um and again like you know that's     
31:29     
maybe something you're interested in or not but there always these opportunities that come up to do things so that's     
31:37     
that's a little tour of our and this is also for our GC students because I want     
31:42     
to make sure that they know what we have in terms of repositories and what we     
31:47     
have in terms of resources available um I know that like the G-S students are     
31:54     
very familiar with the DVO graph repository but maybe not so much with D.VO Learn that's been a focus of uh     
32:03     
Google Summer code projects in years past so you know that's but we haven't talked about it this year so if you want     
32:10     
to do something like cell segmentation if you want to incorporate that into a project I would prefer we draw from like     
32:17     
what we've done with D.Learn to learn instead of like picking something else up off another shelf and applying that     
32:24     
to the problem uh but you know this is all stuff we need to talk about though um     
32:32     
and this is always a thing in open source where people will     
32:37     
uh you know do something in a repository and it's kind of halfbaked     
32:42     
because they don't really know what's going on in the repository and um if you do that enough times your     
32:48     
project becomes really misaligned and um it's hard to deal with so yeah that so     
32:55     
that's all we have in there um whatever people find interesting you know please     
33:00     
let us know so Susan uh is really interested of course like I said in cellular     
33:07     
mechanics and we've been talking about this tricellular junctions topic and so     
33:12     
tricellular junctions are where you have a sheet of like epithelial cells and     
33:18     
then there's a place where maybe three cell membranes will meet and that's     
33:24     
considered a junction between different cells a tricellular junction of course     
33:30     
is three of those cells coming together at a point you can have you know I guess     
33:36     
four four junctions more you know six junctions     
33:41     
depending on the geometry and the packing of those cells so that's um     
33:47     
those are you know ncellular junctions I guess or n junctions and she sent a     
33:53     
couple papers along so this is a paper tricellular junctions and I think we     
33:58     
talked about this in a previous meeting but it's worth bringing up again     
34:04     
because I think it's you know an interesting paper especially given the topic so I'm     
34:11     
going to talk about this paper and then another paper she sent me so um this is     
34:16     
tricellular junctions how to build junctions at the trickiest points of     
34:22     
epithelial cells and tr is the acronym here so it's very cute um and so the the     
34:29     
idea is that there you know these places that cells come together and obviously     
34:36     
when you have a packing of cells it's important to have places with great     
34:41     
structural integrity because that sheet of cells and bends and you know has to     
34:47     
bear loads and has to do things in the body that require this sort of structure     
34:53     
and and of course Susan's interested in biotensergrity which is how to achieve     
34:59     
ultra stability in these structures so the the end junctions are the places where you get this kind of you could get     
35:06     
this kind of ultra stability um and so then the abstract of this paper     
35:12     
uh reads trico or contacts are the places where three cells meet all right that's where you have this geometry     
35:20     
where they kind of come together they get packed together into a point in vertebrate epithelial cells which are     
35:28     
these kind of cells that are kind of like skin cells or other types of uh     
35:34     
cells uh they're usually kind of uh cubicle you know they have a certain     
35:42     
shape typical shape they are found across vertebrates um and they form these     
35:49     
sheets that that are involved in like organogenesis and you know things like     
35:55     
skin um in vertebrate epithelial cells specialized structures called tricellular tight junctions or small     
36:04     
tjs and tricellular adherence junctions or small t ajs so we have tight     
36:10     
junctions and junctions and those are just different types of uh junctions     
36:19     
um in in in the anatomy they've been identified ttjs are important for the     
36:26     
maintenance of barrier function and disruption of TTJ proteins contributes to familial     
36:33     
deafness um TAJS have recently been attracting the attention of mechanobiologists because these sites     
36:39     
are hotspots of epithelial tension so when they say epithelial tension they     
36:45     
mean that the sheet of epithelial cells as a sheet not just as the individual cells but the collective behavior of     
36:52     
those cells form this sheet and the sheet has its own sort of physics so you     
36:58     
have tension you have uh compression you have all these different properties of     
37:03     
the sheet um and then so they're basically saying that these are hot     
37:09     
spots of tension so again ultra stability or tenseity relies on these     
37:14     
kind of emergent uh features of of these larger scale     
37:20     
structures um and then that's of course one of them although the molecular     
37:25     
components regulation and function of TTJs and TJs as well as of invertebrate     
37:31     
triccellular junctions are beginning to be characterized many questions remain     
37:37     
here where we broadly cover what is known about tricolular junctions propose a new model for tension     
37:43     
transmission at TAJS and discuss key open questions so this is where they were     
37:50     
going in this paper um they talk about these different types of junctions     
37:57     
um and you know so that we have the sort of the microscopy which describes the     
38:03     
morphology and then we have this molecular aspect which is where we observe these different     
38:10     
proteins and you know the function is kind of observe the function almost     
38:16     
kind of semi-independently from the morphological structure so we look at the morphological structure at the     
38:23     
microscopy and then we have the molecular aspects of it the molecular     
38:28     
components you know sometimes you measure them sort of in SYU and you get a good characterization of how they they     
38:37     
operate but more often than not we're doing a lot of annotation we're doing a lot of inference of function and so that     
38:45     
you know this is a problem sort of with biioinformatics is that you have     
38:51     
uh molecular data and we know kind of what things do but it's very general     
38:56     
characterization a lot of times you need to do studies where you can actually observe like proteins or other types of     
39:03     
things like gene expression at the site and you know our technologies to do that     
39:09     
are kind of limited so it's hard sometimes to match what's going on in     
39:15     
the molecular millu or the molecular world of like trijunctions with the     
39:23     
morphology and so that's you know so this is actually where we can kind of     
39:30     
get a connection between the two here we have expression of angulin one we have     
39:36     
the sort of membranes here and then the merge where the angulin one is expressed     
39:42     
at the junctions for the membranes so these membranes show this uh these junctions so here we     
39:50     
have a trijunction an example here another example of a tri junction where     
39:57     
three cells come together and as you'll notice they have this kind of uh     
40:03     
shape you know this is I I guess looking at the top of the sheet so they have     
40:09     
this shape that you know they kind of squeeze into this confirmation and then you get these     
40:15     
different trijunctions that form so this is basically wherever you have three cells coming together and as you can see     
40:22     
you have a lot of uh edges to this you have one two three four five six it's     
40:28     
kind of a is it six or seven seven so     
40:33     
it's like seven a sevensided uh object it's kind of squeezed together     
40:41     
this is cell seven tri junctions in that for this     
40:48     
space and so this is where the cells pack together very tightly and form     
40:54     
these junctions now that being said the junctions are always changing so the cells are migrating past one another     
41:00     
they're kind of you know uh changing their shape and so this these these     
41:06     
trijunctions sometimes they're stable sometimes they move around and I guess it's this expression of um angulin one     
41:14     
that really kind of defines where they are and so that's an example of a study     
41:20     
where you can get this expression of these angulin proteins and you know how     
41:26     
they play a role in function or how you can infer that role so I remember going through this paper we talked about the     
41:33     
molecular aspects we talked about the anatomy they have some nice figures in here showing kind of the connection     
41:40     
between the two so you I remember this where you have the different tight     
41:45     
junctions and adherence junctions i guess if you go down so this is looking     
41:51     
at the top of the sheet and then of course there's the dimension where     
41:57     
you're going down uh looking at it from that perspective you see that they're sort of cubicle you know they kind of go     
42:05     
down straight down like this and that's an artifact of this model but um     
42:11     
basically um that they have a depth as well and so in that depth we have along these     
42:18     
junctions we have all sorts of stuff going on along them so those junctions aren't just points they're actually like     
42:26     
axes that go down the sides of these cells so you get all this expression of     
42:32     
these different junctions here and so that's kind of what we're     
42:37     
looking at here so that's I remember this paper is pretty good it had a lot of nice illustrations it talked about these     
42:44     
different the importance of these junctions and how they play a role     
42:49     
in the morphology of these cell sheets this is a paper I think that Susan sent     
42:56     
me more recently and this is uh tricell uh     
43:01     
tricellin secures the epithelial barrier at triccellular junctions by interacting     
43:07     
with ectomyioin so the first paper was in molecular biology of the cell I believe     
43:14     
and this is from journal of cell biology that it necessarily matters but this is     
43:19     
more sort of I think a cell biology paper um so this paper um says or the the     
43:29     
abstract reads the epithelial cell sheet functions as a barrier to prevent     
43:35     
invasion of pathogens so this is why we have epithelial cell sheets maybe one     
43:40     
reason um and so as a barrier it needs to have some sort of stability it needs     
43:47     
to be uh you know strong and you know in the in the face of cell migration in the     
43:55     
face of maybe all sorts of invas invading things that come into the     
44:01     
cell or into the into the tissue I guess is necessary to eliminate intercellular     
44:07     
gaps not only a bicellular junctions but also tricellular contacts so we have     
44:13     
these uh bisicellular junctions and these tricellular contacts     
44:19     
um where three cells meet to maintain epithelial barrier function so you have     
44:25     
to think in terms of the junctions which are these like you know proteins that connect and then the tricellular     
44:32     
contacts which are where the physics the physical aspects of the cells connect um     
44:38     
to that end tight junctions between adjacent cells so you have these tight junctions in each cell and they kind of     
44:46     
you know join together uh between adjacent cells these tight     
44:51     
junctions must associate as closely as possible particularly at tricellular     
44:56     
contacts tricellularin uh is an integral component of tricellular tight junctions or TTJs     
45:04     
which we talked about in the last paper but that but the molecular mechanism of its contribution to the epithelial     
45:11     
barrier function remains unclear so you know what is we again we     
45:18     
don't necessarily know what the molecular mechanism is we know that they exist we know that these tight junctions     
45:24     
exist we know they need to be in a certain configuration but it's hard you know it's harder to infer mechanism     
45:31     
especially in terms of causality and what what it's doing in this study we     
45:36     
revealed that tricellin contributes to barrier formation by regulating     
45:42     
ectomyioin organization at tricellular junctions so this is where this     
45:50     
tricellin is contributing to this barrier formation function of the sheet     
45:56     
by regulating ectomyioin organization so ectomyiosin is     
46:02     
in in the cell it's it's kind of enabling the movement of cells the     
46:08     
migration of cells against one another and it allows to these tricolial     
46:14     
junctions to remain stable despite a lot of changes in this sheet a lot of uh you     
46:22     
know stresses on the sheet a lot of cell migration and things like that     
46:28     
furthermore we identify alphainin which is thought to function not only only at adherence junctions as a novel binding     
46:35     
partner of tricellin alphaanin bridges tricellin     
46:40     
attachment to the bicellular actin cables that are anchored end on at     
46:46     
tricellular junctions we have these acting cables that sort of you know we have actin within cell membranes     
46:53     
they sometimes have to span cell membranes actin of course is kind of     
46:59     
like serves the role of like a stretchy     
47:04     
filament and they're necessary for the sort of flexibility and and they need     
47:11     
but they need to be coordinated because you need to remember you don't need to maintain just the stability of a cell     
47:18     
but all these cells sort of in the array and you want to maintain the stability of that array     
47:26     
um and so they they're all anchored at this tricellular junction and they're all coordinated at that point and you     
47:34     
don't want them to be important right thus tricellin mobilizes     
47:39     
ectomyioin contractility to close the lateral gap between the TJ strands of the three     
47:46     
proximate cells that converge on tricellular junctions so it's this     
47:51     
they're proposing this tricellin which is the thing that's sort     
47:57     
of mobilizing acttomyiosin contractility so acttomyiosin acts a lot like muscle     
48:05     
or skeletal muscle where it contracts and that you know allows you     
48:12     
to move and it allows a cell to move as well so this is why it's important     
48:17     
because you get a lot of stresses a lot of force loading at on these sheets and     
48:24     
you want to maintain these sort of configurations as much as possible you     
48:30     
can imagine if all the cells were sort of br you know they were sort of rigid     
48:36     
bodies and they weren't coordinated like this that if you know something were to     
48:41     
there were a stress put on the sheet the sheet would pretty uh you know pretty     
48:46     
easily collapse and it would have to reassemble itself all the time so this is one of the benefits of what what     
48:52     
we're talking about here so again they talk about tight junctions they talk about these other uh     
49:02     
types of junctions let me see if we have anything interesting here so they talk about     
49:08     
tricellular junctions being tension hotspots or mechanical forces applied to     
49:14     
cell adhesion molecules tricellular junctions experience high tension due to outward     
49:20     
forces generated by acttomyiosin dependent line tension acting along each     
49:26     
bicellular junction homalologues of neither tricellin nor LSR have been identified     
49:32     
in invertebrate epithelial cells so this is a vertebrate specific thing so in     
49:38     
invertebrate epithelial cells we have different means of maintaining these sheets     
49:45     
we talked last week I think or last meeting about you know these transitions     
49:52     
in evolution where you get certain things that have a whole bunch of things     
49:58     
that happen at once that enable new structures and so we talked about     
50:03     
multisellularity in ukareotic cells and you see a lot of these things with like     
50:09     
differences between invertebrates and vertebrates and so you know that and and     
50:14     
of course there's a lot of diversity in both uh taxonomic groups but you can see     
50:21     
that there differences in how invertebrates have solved problems versus avertebrates have solved problems     
50:26     
and sometimes you have things that are enabled in one group or another that are     
50:32     
you know important for the way their tissues form the way they live their life     
50:38     
so it's you know a lot of times we'll point to these differences between vertebrates and invertebrates and it's     
50:45     
interesting sometimes it's just that there's a difference in how maybe their common ancestor did     
50:52     
something you know or maybe it's an innovation that is unique to vertebrates so it's always     
51:00     
interesting to see these differences here so of course when we're interested in     
51:05     
sea elegans we're interested in invertebrates the model organism Drosophila as well is an     
51:11     
invertebrate um so that they kind of talk about invertebrates here for a     
51:17     
while um and their differences so uh one of the well some of them     
51:23     
involve homalologues in identifying those homalologues and these are just     
51:29     
homalologues of the different proteins so maybe it's it's a different protein but it's basically doing the same     
51:36     
thing um so you have but in even in invertebrates you have several cell     
51:42     
adhesion molecules being enriched at these tricellular junctions among them is uh protein     
51:49     
called sidekick which belongs to the amoglobulan super family that those are     
51:55     
concentrated at tricellular adherence junctions in Drosophila so in Drosophila you have a system where     
52:01     
it's a similar sort of morphology but you have different molecules mediating     
52:06     
those junctions in a departure from the vertebrate system sidekick itself     
52:12     
mediates homophilic adhesions between the three cells that comprise the tha     
52:19     
additionally sidekick anchors the acttomyiosin cytokeleton through through     
52:25     
polytoid which is the mamalian homalogue of Z01 and canoe which is the mamalian     
52:31     
homalogue of aphiden at tricellular junction so when they talk about these homalologues they're just like     
52:38     
basically very similar proteins they just have different names and but they do basically the same thing have I think     
52:44     
the same um evolutionary     
52:50     
u origin however in the case of vertebrate epithelia the functional     
52:55     
relationship between either tricelli or lsr and the underlying acttomyosiosin     
53:01     
cytokeleton at tricellular junctions is not known so in the present study we reveal a     
53:07     
tricolon that anchors factin connected to tricellular junctions in an endon manner via     
53:16     
alpha so we anchor factin these tricellular junctions using     
53:23     
tricellin and this is you know done in an endon manner and alphain is mediating     
53:30     
this and alphainin is the principal scaffold protein of     
53:36     
AJS tricellin plays an essential role from the formation of     
53:41     
TTJs by coupling close association of two central sealing elements with     
53:46     
ectomyioin contractility by tricellular junctions so this is not just about maintaining     
53:54     
that junction maintaining the sort of the cornering aspect of this which maintains     
54:01     
the stability of the sheet but also it allows for this contractility of     
54:07     
acttomyiosin and it allows for the stability of the sheet in the face of that contractility because remember you     
54:13     
have three cells and they're all contracting and um you know they may or may not be coordinated so that's that's     
54:21     
why we have all of this molecular machinery in     
54:26     
place okay so and then they they give some results here from these results we propose that the formation of the actin     
54:33     
meshwork at tricellular junctions so there's this meshwork of actin is a     
54:38     
distinct morphological stage during the maturation of epithelial junctions     
54:44     
um in the following experiments we examine the functional roles of the tricellular actin meshwork in the     
54:50     
formation of epithelial barriers so that's basically what they're trying to do they're trying to     
54:55     
understand these barriers they're trying to understand how we can have maintain     
55:00     
these barriers through this uh sheet of epithelial cells how to maintain     
55:06     
stability of that sheet both during like loads placed on it or stress is placed     
55:12     
on it and during like the movement of cells and the adjustment of cells and     
55:17     
the contractility of actin in response to different stimul so there are a lot of things going on     
55:23     
here but I think it's an interesting set of papers because it does describe kind of you know what happens in in     
55:31     
organogenesis right you get these sheets of um epithelial     
55:36     
cells and then they have to be sort of functionally robust     
55:43     
they serve different functions and you know and there are a lot of things that are going on in the     
55:49     
cells that are keeping them sort of in this association and then that leads to you     
55:55     
know further organogenesis so Susan I know these are     
56:00     
uh this is your kind of your area so would you like to speak to some of this     
56:08     
it's interesting that uh cells are     
56:13     
attached to that junction um mainly through that junction the Bjunction is     
56:20     
more um a surface junction and then the deeper junction is a tri junction and uh     
56:28     
it matters how wide that is in my modeling as how the tissue behaves so it     
56:38     
it's um cells are very reliant on their     
56:43     
attachments and the trijunction is a main one and then the other one is the     
56:50     
basement membrane attachments so that's all important and     
56:57     
it gives um a tissue tissue is stronger     
57:03     
than its individual cells by a by a lot i have a paper here that I think it's at     
57:11     
least 100fold stronger mechanically than a single     
57:18     
cell anyway it has different characteristics when when they're attached to each other     
57:25     
so studying individual cells doesn't give you a a complete overview of what's     
57:31     
going on in the body you need to look at at     
57:36     
the tissue and and the attachments yeah that's just the sort of general     
57:43     
thing and then Yeah what do you think the role     
57:50     
is the broader role is for like uh tissue morphagenesis i mean what kinds     
57:55     
of things I mean you know in your thesis did you did you think about or did you     
58:02     
kind of consider I just I just said it it's it's a total     
58:08     
thing it's in general physiology     
58:14     
um homeostasis it's in cancer development     
58:20     
and it's in um development um embryo     
58:25     
development uh epithelial tissue um and their     
58:31     
attachments play important roles in all of those     
58:38     
um yeah I don't know and I have another another paper i think Dick sent it to me     
58:44     
why do you talk it talks about that so um I I'll send you that okay um paper     
58:55     
i was reading it and I put it somewhere yeah total organization here     
59:01     
um yeah I don't know where I I put it at     
59:07     
the moment sorry it's okay no I think that's great yeah I like that     
59:14     
um you know that again we I don't I can't remember what meeting it was but it was a while back and and we did a     
59:20     
couple papers and then you know we keep coming up with new papers on this topic i like the topic it's it's a interesting     
59:29     
you know and this is a hot area because people are publishing in it and discovering new things all the time     
59:36     
yeah it's neat that they've managed to figure out what a triunction is made of what proteins um and that's new like     
59:44     
that's a new development because of new     
59:51     
microscopy new analysis yeah this a new area for sure     
1:00:02     
oh I think that's all for today uh it looks like Priyanka had to leave but thank you for attending and so thank you     
1:00:10     
and then Andrea thank you for attending hope you found this useful and um all     
1:00:16     
right well see you next week thanks for attending have a good     
1:00:23     
Okay bye uh I'll send you that paper all right thanks sashing up
