## Transcript     
0:00     
Hello Bradley. How are you? Good. Good.     
0:07     
Uh last week uh actually Maple was there in the meeting. So I didn't know that     
0:15     
was somehow we were we actually pretty much discussed and also I have given the same     
0:21     
update to him. So that you know we discussed but I didn't have the result at the time but but when he was there I     
0:27     
gave up. So probably I can give tweak updates on what I have done. Yeah. And     
0:35     
uh there were pretty much large changes what happened last week and all. So     
0:43     
in the last meeting what uh I have discussed is uh so     
0:49     
uh last I mean like from the beginning of uh I mean like what I have done is uh     
0:56     
initially I started off with uh understanding the the data set where we     
1:01     
have like the XYZ coordinates of the C elegance uh you know each time point uh     
1:07     
for the you know during the empirises and also we have the lineage change information how it's evolving over the     
1:13     
of time. So what I wanted to understand in this is more specifically apply a hyperraph on top of it and see how this     
1:21     
uh hyper edges uh the edge that actually contributes to     
1:26     
bring a certain set of hyperonodes together and uh what is the underneath     
1:33     
information it resides and which which certain type of cells lead to anosis. So     
1:40     
that's my major understanding I mean like uh that I wanted to understand practically using edits. So where I have     
1:46     
uh in the first couple of weeks I started with uh you know implementing uh the basics of uh you know implementing     
1:55     
uh you know the I mean like the first I decided the workflow and uh the dependencies and the libraries. So I     
2:02     
have figured it out and later I implemented a few of the neural networks     
2:07     
and I thought you know there is a pipeline that that actually can help in order to integrate it but uh in the week     
2:14     
uh five the last previous week I was able to understand that the pipeline that I thought was not helping. So I had     
2:22     
to change and tweak a bit in the networks. So what I have tweaked is probably I'll share it now. So yeah uh     
2:30     
so these are basic dependencies and uh networks uh I have uh taken this uh self birth     
2:37     
and position data set because the temporal ones is quite long so this could be short and I will I will be able     
2:43     
to acquire the you know uh the information from this and uh yeah uh     
2:50     
today what I've decided is I'll probably run the results and show it like how this network is able to understand. So     
2:56     
we have this uh the cell uh parent information and the XYZ coordinates and     
3:02     
also the you know the daughter one and how the parent has been split into and the given time point and from this I was     
3:10     
able to uh get some you know node level features for uh you know for a hyper     
3:16     
graph. So I say at each node which means in a graph we have a node and in this     
3:21     
node what are the information it has. So we have the time and we have the XYZ     
3:27     
coordinates. This is what it actually has and the directed lineage information     
3:33     
uh the parents uh I mean like the lineage information is also resides into the edge I mean saying that you know     
3:39     
this uh the lineage how it traverses exactly. So we have the information that     
3:45     
actually resides into the you know the edge saying that the hyper edge says that uh uh the what what kind of     
3:53     
grouping that we are doing. So it's basic basic basically based on the you know the relationship between the     
3:59     
lineage tree that it has traveled from like you know parent to uh the daughter     
4:05     
and that specific daughter splits into like multiple daughters. So it's like that. So first generation, second     
4:11     
generation and further generations. So I was able to acquire uh um uh 12 node le     
4:18     
nodes and 1212 lineage edges that we we were able to find. So we have built a     
4:25     
hypograph where uh we have uh different uh you know uh all these features that     
4:32     
have been acquired. So in order to have like an hyper edge so we were using K andN with a certain radius so that you     
4:39     
know we will be able to acquire the neighborhood ending so that you know spatial information is already sorted     
4:44     
because in the edge we are anyway having the we are acquiring the information of the lineage. So here we are trying to     
4:50     
acquire the information for the uh I mean like uh the spatial I mean like XYZ     
4:57     
locations and what uh in the next step what we have done is uh so we will be     
5:02     
able to acquire all these uh matrices uh where I'll say in uh incidence matrix     
5:08     
is basically like uh uh adjacency matrix for an hyperqu and uh so we have this     
5:16     
information of lineage information and we have the information of the spatial locations and we have the birth time     
5:23     
positions and uh then we will make a hypograph here. I have specifically made     
5:29     
it alive because I want I also wanted to add a further next next set features     
5:35     
which we discussed in the previous meetings that we also have the cell depth or birth information.     
5:42     
So the fate can be predicted. So I thought you know okay if you're going to predict the fade then probably this     
5:49     
information could be helpful in the future round. So I just added in that and     
5:54     
yeah after that we are able to acquire I changed this hyperraph neural network.     
5:59     
So where I have changed and why do I have to change is the major concern. So last time what I have done is I tried uh     
6:07     
using you know lineage information and also the spatial information. uh based on the lineage and the spatial     
6:13     
information we were able to design an uh hybrid type of model. So what it is not     
6:18     
able to do is so we do not have a you know uh it has a basically a temporal     
6:23     
line where it says that you know at one time point it has a certain characteristics at time point two it has certain characteristics. So the network     
6:30     
that I have designed previously was not able to understand this you know uh the     
6:35     
temporal relationship in order to make it understand the temporal relationship. So I added one more uh component here.     
6:42     
So which is the greater recurren unit. We can use LSTM or you can use     
6:47     
transformers but as of now I started with gated recurren units. So which actually helps to understand it how how     
6:54     
we going to progress over the period of time and uh so yeah so once we are done     
7:01     
with the neural network we have like graph convolution layers and we have like a temporal connections and these     
7:07     
embeddings will be helpful for us to you know build the time snapshots. So what     
7:12     
uh I have decided in the previously in order to train it in a certain way but uh this time I had a change in     
7:19     
perspective. So when I discussed with Mel he said that you know unsupervised fashion is not required as of now you     
7:25     
have a proper label data so probably you can work on this label data. So I was like yes I think yeah I mean like uh I     
7:34     
thought it's a better way to do it. So so I didn't uh do the you know uh I     
7:40     
applied self supervision we were able to not get any results because it's not able to understand well from the data.     
7:47     
So I have tried like proper uh supervised approach where what I have done is here the critical modification I     
7:54     
have done is like uh I have two one actually tries to give us uh uh you know     
8:01     
the uh the loss I mean like the change from     
8:06     
the prediction and the ground truth is basically means per error. So what is     
8:11     
the next it's trying to predict and what it's what it actually has it tries to bring them together and also it tries to     
8:19     
do the uh you know binary cross entropy between the indistance indices uh     
8:24     
incidence matrix. So using uh when you when we try to uh compress this     
8:30     
incidence matrix it actually you know uh the network topology does not uh the     
8:36     
network does not lose its topology. So the topology remains well using this as     
8:41     
I mean like this loss because we just try to minimize the you know uh the distance between two points is not     
8:46     
sufficient. We have to make sure that you know the structure is not lost. So     
8:52     
we added extra type of loss which is like uh incidence BC. It tries to minimize the cost entropy loss between     
8:58     
the incidence matrices predicted and like uh you know uh the the the actual     
9:06     
ones or the ground truth. So when I trained this I was able to get uh you     
9:11     
know it's converged to a certain point and once it's converged I wanted to understand that what this hyper edges I     
9:19     
mean like the main motive of this is what does the information in our hyper carry so how are these embeddings so     
9:27     
when I plot the DSNA uh I was able to see this uh plot so     
9:32     
I'll probably run this so that you know can have a quick intuition     
9:42     
function. This is times information. This is our     
9:47     
neural network and this is our one information.     
9:55     
Yeah. So snapshot. This is our last function. This will take a couple of time. Not much time but     
10:04     
little time. So once it's done we can probably plot them. So I saved the plot     
10:09     
here so that it looks similar to this. I mean like you know the embeddings of     
10:15     
during the uh the division and the hyperraph whatever hyperraph that we     
10:22     
built was able to bring certain set of cells together. So these cells I haven't     
10:28     
annotated them yet but uh so this is AB. So we'll have the all the annotations together. So once we will able to     
10:34     
annotate together then we will be able to see that what is our hyperraph learning so is it learning that you know     
10:41     
brings all these cells together what does it mean so that interpretation probably I'll try to do it next week and     
10:46     
I'll bring that report or kind of PPT so that it's more elicidated so what what     
10:52     
I'm trying to understand here is you know during the training of this model using a hyperraph we were able to see     
10:58     
that the hyper tries to learn certain set of cells being together. There are     
11:03     
certain linear structures that are coming together. Do they really form uh a certain type of characteristics? I     
11:09     
mean like let's say you know uh uh you know the inter neuron come together,     
11:15     
motor neuron come together or you know uh you know I mean like you know locomot     
11:22     
I'm not really sure about what what comes together but I have to you know experimentally observe what's what are     
11:29     
these getting together. So these cell embeddings will actually help us to more understand you know what is the topology     
11:36     
the underneath topology of like you know this growth happens. So this is tsna. We     
11:44     
can also use um map. Um actually gives us more intuition about like the network topology. Uh I actually added the     
11:50     
function here too. So yeah so this is the you know there are     
11:58     
certain cells are like coming together yeah so     
12:04     
so yeah so that's so that's the main thing     
12:10     
so DSNA we observe it's different way and this is different way but uh at the end of the day what we try to interpret     
12:17     
from like uh TSNA is the distances here are like uh uh does not mean anything in     
12:23     
TSNA and in TSNA uh you know uh the the groups that have been formed there when     
12:29     
we really annotate and give them proper colors then we will be able to inspect more in better fashion so I mean so so     
12:38     
this is the update that I have done for this week and probably in the next couple of week I'll try to understand     
12:44     
more in detail interpret them and apply you know a different type of neural network where we can get more     
12:51     
interpretable features or say that you know how well uh it's trying to uh bring     
12:58     
new insights or probably we can say that you know does the network topology     
13:04     
really matters or if you change the neural network are we seeing the same cells forming them together so is it     
13:10     
what the neural network wants or what it actually happens so and I have few questions and probably I'll formulate     
13:17     
them in a more better manner with some experiments and and how you can discuss.     
13:22     
So I think this is what I have done for the past weekend. I have I have just shown the code wise update but I think     
13:30     
uh it would be more better if I give a I mean like a formal type of presentation     
13:36     
which I'll probably do next. Okay. In this Friday. So     
13:45     
sounds good. So any suggestions? Yeah. Well, I like the uh sort of     
13:54     
bringing this all together in a visualization and then showing, you know, kind of how     
13:59     
things are coming together in alternate structures, the tiss and and so forth. Um I mean, I     
14:06     
don't know how those would be. I I don't know if they're also related sort of as a lineage tree or like a reorganization     
14:13     
tree, but you could say like that they're grouped in some way.     
14:19     
Um, I don't know if Well, do you know for a fact or do you know that if they're basically the same as a lineage     
14:26     
tree or are they bringing together cells from different parts of the lineage tree?     
14:32     
Right. Right. That that I need to understand by labeling them and then extract the information from. Oh, okay.     
14:38     
You haven't labeled them yet? Yes. Okay. Yeah, I have labeled them. But while plotting what it happens, it it overlaps     
14:45     
a lot. So when it overlaps the visualization does not look clean. So so     
14:51     
I I had to think of like a more cleaner visualization where you know it prints when I point it out on top of that then     
14:58     
I should be able to see that okay it started from this or no this this belongs to some something.     
15:05     
So I I can I can I can so so what do you     
15:13     
Yeah. So you you talked about kind of the whole point of this was to show how the edges inform different parts     
15:19     
different like hyperraph constituents. So a hyper node would like have certain     
15:25     
number of cells depending on the edges they have different relationships.     
15:32     
So right yeah uh so that's that's how you kind of got to the tene maps or     
15:38     
I mean yeah so what I have done is uh to extract the tsna so we have hyper edges     
15:45     
and hyper nodes and this hyper edges basically form across the nodes so what     
15:51     
I thought is uh okay I made this hyper edges learnable learnable in the sense     
15:56     
they can change across uh you know time trying to learn that you know which     
16:02     
hyper node should come inside the hyper edge. So they are not like static per se that you know only these have to come to     
16:09     
the hyper I mean like suppose I say that you know uh there might be a cell which     
16:16     
is you know proximately closer you know but they are different     
16:22     
but the lineage information is different. It tries to bring you know both the lineage and uh balances out     
16:28     
between lineage and also the spatial proximity and it tries to learn that you know it brings together the uh I mean a     
16:36     
certain type of cells which it it thinks that you know it is closer to the the     
16:42     
the the ground what we have. So during that process the hyper edge tries to     
16:47     
learn to uh bring certain type of cells together. What I wanted to uh I plotted is the     
16:53     
weights which it uh the hyper edge has acquired hyper edge in the sense the     
16:59     
edge which is actually responsible to bring out these cells cell wise relationship I plotted the weights of     
17:05     
that hyper edge then I thought we able to you know bring out the cells which     
17:11     
are actually more relevant or it's more of you know which actually bring the     
17:19     
should I So, so it tries to brings out the uh     
17:26     
the more uh you know uh so cells which are actually more relevant and you know     
17:33     
it tries to understand which cells actually formed together and this hyperagings     
17:42     
of the hyper. Okay. Yeah, that looks great. Um so you'll     
17:48     
just keep working on this kind of you said that your original pipeline didn't really work very well. So uh so original     
17:56     
pipeline what I thought was to apply the unsupervised manner. Unsupervised manner     
18:01     
in the sense I thought okay when without giving the labels I should understand     
18:06     
just by the data. So which was not really working. I actually needed you know what is the actual ground truth of     
18:13     
the next time time next time step prediction without the next time step prediction I wasn't unable to get any     
18:19     
results out of it so it and there was no temporal relationship to find out that     
18:25     
you know it starts from like a zero the time step to like 100 time step there     
18:30     
was no relationship I mean like there was a no network which actually tries to understand that you know the sp uh we     
18:37     
call the temporal relationship yeah So in order to understand that I have to embed one more the grtm transformer     
18:44     
block where it actually has the information that you know at time point zero it it is here and time point.1 it     
18:51     
is here and and the incidence matrix is also like an very good approach. So when     
18:58     
I discussed with it he was like okay this actually preserves the network topology then you're not uh breaking out     
19:04     
what actually the graphic actually has. So the relationships are not lost you     
19:10     
know while trying to model this. So that was the key insights which I have observed this way.     
19:17     
So so that's from yeah that's great     
19:23     
probably more visualizations and intuitive this week. Yeah. Well that's     
19:31     
great. Yeah. Thanks for the update. Okay, Bradley. Yeah. Yeah. I'd like to     
19:38     
uh uh show you a confusion I'm getting out of the literature. Okay. And see if     
19:43     
see if you can resolve it. Uh let me share my screen. Right.     
19:49     
Okay. Start screen sharing.     
19:59     
Okay. Now     
20:07     
it sec     
20:17     
I'm having trouble here cuz I the jity covers up my dock. Oh, you can     
20:22     
hide it. Yeah, I can hide. Oh, hide. Hide there. Hide it. Oh, okay. There you     
20:29     
go. Thank you. Okay. Now, and this is uh a table from this paper.     
20:39     
Okay. Uh but Pod Bowitz worked with uh uh White.     
20:51     
Okay. So this is uh goes back this     
20:57     
his pedigree goes back to the original work here I guess. Okay. Now this is a table     
21:03     
and you'll notice that it's it's a table that claims that there are 44     
21:11     
sensitia. Okay. Yeah. Okay. And this is the number     
21:16     
of nuclei in each sensitium. Okay. Okay. The confusion I'm getting is     
21:23     
the cells are named here, but I can't tell if that's the name of the cell from     
21:28     
which the sensitium came or the name of the of the sensitium cell.     
21:34     
Right. Right. Okay. And I've I've gone through the paper. I can't figure it out. Uh it gives a bunch of references     
21:41     
here. I just started getting those references. Uh, and I'm not sure whether or not they'll help resolve that issue,     
21:48     
but the the paper just doesn't tell me that question. Are these the cell from     
21:54     
which the sensation came or is it the name of the sensation cell?     
22:01     
Is there a simple way to figure that out? Given the name of a Yeah. If if these cells are on the     
22:08     
lineage tree, how do you go from a name to locating the point on the lineage     
22:14     
tree that corresponds to this cell? Uh well, yeah. So these names are the cells     
22:21     
from which the centium came. So how do you know that? Uh because you can see it     
22:26     
in the lineage tree if you we have the interactive lineage tree that you can     
22:32     
look at. Yeah. And that goes post embriionically as well as embryionically. Yeah. Post embryionic.     
22:39     
Yeah. Okay. Uh can you put that in the chat?     
22:44     
Uh yeah. I don't have the link right now, but I can get it later. Okay. Okay.     
22:50     
So, in other words, the interactive one would allow you to put in a name of a cell and locate it on the lineage tree.     
22:56     
Yes. Yes. Okay. Great. And are these do you recognize these as     
23:03     
being on the lineage tree. Yeah. Yeah. I'm going to get to this in in a little bit actually. Um Okay. I have I have     
23:10     
some visualizations to show. Now the next question then is are these the     
23:15     
progenitor? I guess I guess here's an example here. Hype six plus hype seven.     
23:20     
We have two progenitor cells for this huge sensitium.     
23:26     
Yeah. Okay. So that's uh I that's an example where we're     
23:32     
getting a mix, but it looks like the rest are are uh straightforward.     
23:38     
Yeah. So this is the only paper I've been able to locate which explicitly says there     
23:45     
are 44 sensitial souls, right? Okay. And uh I think they're     
23:51     
distinguishing the male from the uh uh     
23:58     
heraphrodite. Yeah. The uh number of snook     
24:04     
reported differences were males. Okay. So there's a small group of those.     
24:12     
Okay. Now what I'm suggesting is very simple, a simple form of hyperraph.     
24:20     
those that come from a particular cell could just be uh shown off the lineage     
24:27     
tree is a circle with the number of nuclei in it.     
24:32     
Yeah. Okay. Uh this would have if this one     
24:40     
would could be represented the same way but it would have two connections to the     
24:45     
lineage tree. Okay. So I think we can I think this     
24:53     
gives us all the data from which to construct a hyperraph where we don't the     
25:00     
hyperraph is is just lineage connected to a circle with a number of nucleotide     
25:06     
and then connected back to the cell from which it came. Yeah. The circle is     
25:13     
connected to the cell to which it came. Uh Leila, do do you do you agree with     
25:19     
this approach? Uh yes. uh I mean like I believe that     
25:27     
you know if uh these cells actually you know they come from the centium or     
25:33     
centium actually forms these cells I I'm like kind of I'm unable to understand     
25:39     
that but once I able to understand that then probably it means that you know uh     
25:44     
cell which actually I'll get these papers where uh they     
25:51     
reference let's see oh here's the superscripts are over here. Okay. Uh so     
25:59     
I'll get these papers and I think we should go through them carefully to make sure we understand this. This graph is     
26:06     
hard to understand because I am not familiar with the names of individual cells. And I didn't know there was a a     
26:15     
uh an interactive lineage tree which is great. Yeah.     
26:20     
So we could modify the lineage tree easily. It seems to me simply by     
26:26     
representing the sensitial cells by a circle with the number in it in the     
26:32     
circle which is the number of nuclei and attached to the cell from which it came     
26:38     
or the cells from which it came. Yeah. Okay. Good. Good. You've clarified this     
26:44     
tremendously for me. Thank you. You're welcome. Okay. Uh Okay. Okay. So, we got a lot of     
26:52     
work to do to get it get these references and make sure and get that get that uh interactive lineage tree so     
26:59     
we can then make make the hyperraph.     
27:04     
And what's nice about this is we get to make assuming this data is correct, we get to make the hyperraph for all of the     
27:11     
sensitious cells, not just one. Yeah.     
27:17     
Okay. So we can we can really make a modified lineage tree which includes     
27:22     
sensitial cells which I haven't seen anywhere. Have you? No. No. Oh okay. So I I think it's it's     
27:30     
it's a weird thing that's missing from the literature because there's lots of     
27:35     
work on sensitial cells and see elegance but nobody seems to have made it to     
27:42     
modified the lineage tree to account for that. Yeah. Okay. Yeah.     
27:49     
So, so it's it's a a large project, but uh well, actually it isn't that large     
27:56     
because by reducing the uh sensitia to a circle with a number of nuclei in it, I     
28:03     
think it's easy to add to a to a lineage tree. Right.     
28:09     
Okay. Okay. That's great. Clarify what work lies ahead. Yeah.     
28:16     
Yes. Okay. Please, please do get us the uh uh     
28:21     
URL for the interactive lineage tree. Okay. Yeah. Okay. Right. Okay. That's     
28:28     
it. Thanks. All right. Okay. Did you follow that?     
28:36     
Yes. I was follow. Okay. Yeah. I think this paper is an excellent     
28:43     
choice as long as one understands the table. Yes. I I mean like I was able to     
28:50     
understand only that paper. Okay. Yeah. I didn't understand it when     
28:55     
I got it and I were read the paper and it didn't explain.     
29:03     
Okay. Problem with people assuming you know their previous work.     
29:12     
I noticed a lot of the tissue is epithelial uh tissue. Oh yeah,     
29:20     
that's interesting. We could lab either color label them uh the little circle.     
29:27     
Probably the best thing to be color label the circles uh representing the sensitia     
29:34     
uh according to soul type. Okay, that that might be a useful     
29:40     
addition. Thanks, Susan. And so I'm going to now uh show some     
29:47     
work that uh I've been doing on the lineage tree and kind of visualizing     
29:53     
some of these relationships going through some of the specific     
29:58     
uh centum that um are in the sea elements lineage tree. So this is the uh     
30:04     
interactive lineage tree here. This is wormweb.org and you often use this to find cells. So     
30:12     
you can uh search for cells here. So I can search. It starts at P 0 and it goes     
30:18     
down and it has the terminal fates labeled. So you have these uh rectangles     
30:24     
with names on them. Those are the cell nomenclature. You have a time scale at     
30:29     
the left. You can go down uh through like the lineage and you can     
30:36     
keep going down and you have these developmental cells which are the cells that kind of divide     
30:42     
from P 0 that divide into the sublineages. You have the founder cells.     
30:47     
I'm following down the AB lineage here following down to some of these cells     
30:53     
that start to change their nomenclature. So these AB cells with the suffixes are     
31:00     
all part of this AB sublineage. But then we get to something called P10 which is a different type of cell. And if we     
31:07     
click on it, it's a postembriionic blast cell vententral cord neuron. So you     
31:12     
start to get these differentiated cells in the lineage tree. you get things um     
31:20     
other like QR cells and this is hypoderm 7 or hypodermal 7 which is actually part     
31:27     
of one of the centishum so you get that cell I couldn't search for it but it     
31:33     
shows up uh hypoderm 7 is has a lineal name V5     
31:39     
it's hypodermal or it's hypoderm was it's born at 1100 minutes which is     
31:45     
postembriionic And then it's terminally differentiated into this hypoderm.     
31:52     
And it's actually then has this additional uh lineage um trajectory where it fuses     
32:00     
with other cells in hype six and hype seven in that class. So that's how they     
32:07     
kind of represent this. It kind of stops at the individual cells     
32:13     
um for the centum. Then of course you can get these cells to fuse and then     
32:19     
there are additional rounds of uh nuclear division that happen. So that's     
32:25     
why you have that discrepancy between the number of input cells and then the     
32:31     
number of nuclei because they're rounds of nuclear division that happen after again it's doesn't show up in the     
32:37     
lineage tree. So that's and then I'm not going to go through was that Bradley     
32:43     
this is where we would add the sensitium then yeah down below this     
32:49     
okay and so this is during the L1 stage of laral development. So that means that     
32:56     
you know we have these four stages of larval development L1 through L4     
33:02     
a lot of the the precursors to the centium are happening in L1     
33:07     
and then the structure itself I think that this u uh contributes to it forms     
33:15     
in L4 and I'll show a graphic of that in a minute but just to give you an idea of     
33:21     
what the cell entry looks like and how you can you know there there's this nice     
33:26     
tool where we can get that information and play around with the visualization     
33:32     
here. So this was made in um not sure what this was made in but um     
33:38     
this is this has been around for quite a while. So um yeah is the original code     
33:45     
for generating this available? Yeah, I think uh Nikio Botla who runs this, I     
33:52     
think he has a GitHub uh repository for it, but I'm not really     
33:57     
sure. Okay, so we might be able to modify his code to show the uh sensation. Yeah, wormweb.org is where     
34:04     
you can find more apps. I don't know if he has a link to to a GitHub repository, but yeah, this     
34:12     
is uh yeah, I don't I don't exactly know what this is made in, but anyways. Um yeah,     
34:19     
so that's one that's that's the interactive lineage tree, but that's of course branching. So you're going from     
34:26     
like the question on the as you said the time scales on the left. Is there time     
34:32     
data available for the formation of the census?     
34:38     
Yes, in the literature there's uh information on this. Okay, so we need to     
34:43     
do some digging then. Yeah. Well, I'll get to this in a minute. Um, so yeah,     
34:49     
this is and this this is just the branching relationship. So this is how you go from sort of the developmental     
34:56     
lineage tree which is defined by these letters AB you know P1 that whole     
35:03     
nomenclature. Then at some point it switches over to these post-embriionic cells which have a different     
35:09     
nomenclature. And so there's there's a lot of cell division happening in embryionic     
35:16     
development when the worm is in its egg. And then it hatches with maybe about     
35:21     
twothirds of the cells uh that it needs that are kind of there. Some of them terminally differentiate in     
35:28     
the in the egg but then some have to wait until after in this laral     
35:33     
development to differentiate and that's what this postembriionic lineage tree is. So Solston published two papers John     
35:41     
Solston in 1977 they published a paper on the lineage tree. Then 1983 they     
35:48     
posted or they published a paper on the postembriionic lineage tree. So they worked out the lineage tree in the egg     
35:56     
and the embryo and then they published a paper on this postmbbriionic lineage tree which involves things like you know     
36:03     
centishia different structures being finalized um and you know seam cells and things     
36:10     
like that that contribute to uh morphological changes     
36:16     
within laral development. So you end up, you know, you start at L1 with this     
36:22     
smaller worm that's moving around. It's missing some things. Um, and then those     
36:28     
things kind of form by L4 right before adulthood. Usually it's things to do     
36:34     
with reproduction and other types of uh structures.     
36:40     
uh and there also there's also a significant amount of environmental plasticity that can happen in the laral     
36:46     
developmental stages which uh involve remodeling of the phenotype. So you get     
36:52     
dower worms, you get uh starvation uh like you you can induce starvation and     
37:00     
the worm will basically freeze itself in development things like that. So there's     
37:06     
a lot of stuff going on in this postembriionic phase. What is the relationship between White     
37:13     
and Soulston? Um I'm not really sure. I think they     
37:18     
were at both at Oxford, but other than that I'm not sure. Okay. uh because uh     
37:24     
you know people have done a lineage tree for certain academics     
37:30     
and it might help sort things out by knowing the relationship between this uh     
37:37     
uh guy pe the guy who generated this table and uh other people he's worked     
37:44     
with. Yeah. I think they were colleagues. Yeah. But I don't know.     
37:50     
Okay. Yeah. Yeah. Yeah. Okay. Uh     
37:57     
okay. The other thing uh here let me put this uh I use a uh     
38:04     
uh for family purposes I use a program called uh family echo.     
38:11     
Okay. And uh what this is is it's a it's     
38:16     
a tree of families. Uh but what it has uh which the lineage tree does not have     
38:24     
is that it allows for marriages. Yeah. Okay. Which are in a sense are     
38:29     
fusions. Yeah. Okay. Yeah. Uh so it it might be     
38:36     
worthwhile looking at family echo to redo the tree     
38:41     
uh uh if uh if we feel that it's worthwhile doing.     
38:48     
Yeah, you might want to you might want to play with that for your own uh family.     
38:58     
See what see how it works. Yeah. But it allows for each for for each person. It     
39:05     
allows you to put a picture in and all sorts of data about them. Oh, okay.     
39:12     
Including contact information. Okay. Yeah. So, so as your life goes on, it     
39:18     
can be a rather dynamic thing. Yeah.     
39:24     
Okay. Yeah. I I suggest you at least look at see if family echo what uh it's     
39:33     
attractive compared to standard lineage trees. Yeah.     
39:40     
Okay. Oh, yeah. I did it for our family and I actually got individuals back to     
39:46     
the 1700s. And you do that by looking up     
39:51     
obituaries. Yeah.     
39:58     
Yeah. So by the time I was done, I think I had over 400 people in my family.     
40:08     
So yeah, let me show these. Uh so this is something I've been working on for this. If you go into the literature,     
40:15     
there are different types of uh structures that you can look at and you know so this is actually let me actually     
40:22     
grab I don't know if I can do that. Okay. So this is one such structure. Uh     
40:28     
this is a structure where there's a centian being formed down here. And you     
40:33     
can see that the way that this is kind of laid out is that we're going and a lot of this is taken from both the     
40:39     
literature and from the interactive lineage tree. So you start off at in uh     
40:45     
in with our developmental cells very early on establishing which     
40:50     
developmental lineages they're coming from. So we're starting with this AB sublineage. We're starting with the     
40:58     
posterior end of that. We have these left and right divisions     
41:03     
and then we have these three categories P5P, P6P and P7P.     
41:11     
So the developmental lineage tree tracing this from 135 minutes with these     
41:17     
different uh developmental cells here. Um and then we go down and there's a     
41:25     
break in the timeline to 1340 minutes. That's L1. And during L1 which is this     
41:31     
remember this postembriionic period you get these three cells P5.P P6.P     
41:39     
P7.p and those then go on to form these other     
41:44     
precursor cells. Uh we have A B C D. We have this forward and reverse and then     
41:50     
we have EF forward and reverse. And these these go on to form other cells.     
41:57     
those cells proliferate and then they start to fuse in L4. So this proliferation takes place in L3 and then     
42:05     
L4 we have this 2780 minutes we have the structure form and so this is um     
42:15     
this is the structure and it's kind of like there is contributions from each of these cell types and there that's how     
42:21     
they're kind of structured. So that's one way I kind of visualized this structure.     
42:29     
Then there's this other way of doing this where you have this lineage tree. We have this other uh this is where we     
42:36     
have our hypodermal cells. Um and they're kind of forming these uh they're     
42:41     
coming from the developmental lineage tree and they end up as these type of     
42:46     
precursor cells for a centum. So you have your sublineage C and you go down     
42:54     
you have the anterior posterior direction and the anterior direction you     
42:59     
have CAA and then you get these hypodermal cells hypodermal 7 more hypodermal 7 over here     
43:08     
and you get these at different time points so that's what these uh numbers in parentheses are you know we get four     
43:15     
here from CA aa one from CAP,     
43:21     
uh, four from CPAA, and four from CPAP at slightly different times. And so     
43:29     
those are all kind of in place. Then for B, this is kind of working out     
43:35     
more of these hypodermal 7 uh, cells again from the AB sublineage. So you     
43:43     
have a sketch of this, how this sublineage breaks down. You get cell divisions in developmental cells and you     
43:50     
get these hypodermal seven cells appear at different stages of development. Some     
43:56     
appearing embryionically, some appearing post embryionically. And then finally this C which shows kind     
44:04     
of the placement of these cells in time. So you have hypodermal 7 cells forming     
44:10     
in the embryo and then you get the larae. So you get these postembriionic hypodermal 7 cells and then they fuse     
44:19     
and then contribute to structures in the adult. So those are kind of some of the things     
44:24     
and I have it written up a little bit better than that. Um I don't I'm not going to pull up that source document     
44:30     
but uh just to show some examples of what kind of how that that proceeds that     
44:37     
process proceeds. There's also of course um you know there there are these     
44:43     
nuclear divisions which weren't represented in there where you get like you once the cells fuse you get nuclear     
44:51     
divisions and they contribute to this number of n the several rounds of nuclear division depending on the     
44:58     
structure and that's why we get those larger numbers of nuclei.     
45:16     
You have any questions or comments?     
45:26     
Just wonder how it all fits in with the mechanics. Yeah. Well, especially when you get     
45:32     
these structures where you get fusion of cells and uh I'm gonna talk about     
45:38     
mechanics next actually. Yeah, I've got some interesting things     
45:43     
on that. Uh but yeah, it's like you know when the cells are fusing and they're forming these structures, you know,     
45:49     
they're kind of coming from different places. So there's some migration and you know they're coming together in a     
45:56     
certain way. So the migration aspect is something that you know that's where you'd have the spatial dimension, right?     
46:02     
You have like these lineages. I I showed them kind of in     
46:08     
um sort of a lineage tree mode, but like they're not spatially correct. They're     
46:15     
not kind of in space and then plugging in. So if you had like a spatially correct lineage tree, and I don't mean     
46:21     
like in terms of anterior to posterior. I mean like a threedimensional     
46:26     
spatial representation. They would come together in certain ways. There'd be a me, you know, mechanical um there'd be     
46:34     
mechanical aspects to that sort of to migration and then fusion. Which cells are touching another cell and which ones     
46:43     
are deformed, right? Yeah. Like sort of the forces that come to bear. like if if     
46:51     
you're getting a fold, you'll have a different pressure than if it's if it's     
46:57     
flat. Lewith said, "These visualizations are quite informative. I was able to comprehend both the lineage and     
47:04     
sublineage level information, both vision and fusion directions.     
47:10     
Yeah. Well, there there's well let's I guess it is fusion when you're talking about cells dividing and then fusion     
47:17     
when they're coming or there I don't know if they're coming back together but there's this uh cell fusion process     
47:23     
going. So it's um but it's you know yeah     
47:28     
definitely like that sort of reticulation lineage tree. Something we haven't talked about a lot in this is     
47:35     
that you know there is this reticulation of the cell lineage tree. um where you know it's it's not just     
47:43     
that it's branching constantly. There's that's the whole point. There's some coming back together. Okay. Bradley were     
47:51     
those figures ones you generate? Yeah, I drew those out. Uh and that was just based on like the lineage tree and then     
47:59     
some information from the literature. Okay. Uh     
48:05     
are they fully compatible with that table I showed you? Um yeah, they should be except for the     
48:11     
uh number of nuclei which isn't represented there. It's just like you know the cells which have one nucleus     
48:19     
per cell until you get to the fuse cells which then isn't specified. Okay. So maybe we could continue that     
48:26     
effort and get to a point where uh we got a satisfactory articulated lineage     
48:32     
tree. Yeah. Okay. uh which which shows all the     
48:38     
information we have. Yeah. Okay. That's that's terrific. Uh thank you. Welcome.     
48:43     
Yeah. So yeah, it's just uh interesting what you can do with like the da available     
48:50     
data out there. See elegance is a good model organism for that.     
49:00     
So in promising Susan some mechanics here     
49:06     
Um I have a paper and then some so um I want to talk about this paper here which     
49:13     
is this new paper um emergence of cellular pneumatic order     
49:19     
is a conserved feature of gastrolation in animal embryos. So this is about     
49:25     
gastrolation but it's also about pneumatic or cellular pneumatic order.     
49:30     
And so what does that mean pneumatic order? And so that's where we have to talk a little bit about     
49:38     
physics simulations and about soft active materials and about things like     
49:43     
liquid crystals. So you know this is where we have     
49:49     
um you know cells being like these oblong things or these circular things     
49:56     
that have this collective organization in terms of their orientation to one. So     
50:02     
in tissues a lot of times cells will be oriented in different ways that might be aligned with each other that might     
50:08     
overlap with each other. There might be some like orthogonality to their arrangement. So they're you know um     
50:16     
organized maybe parallel organized perpendicular in a perpendicular manner     
50:21     
and all that. This is a software package called Freud which is has nothing to do     
50:28     
with psychology or anything. Um this is a Python library     
50:34     
to analyze trajectories obtained from molecular dynamics or Monte Carlo     
50:39     
simulations. So a lot of times when people do physics simulations they'll do molecular dynamics simulations and those     
50:46     
give them the tools to do like uh things like cell you know simulating the physics of cells. So there are a lot of     
50:52     
molecular dynamics functions that are available. You know people do a lot of things with like you     
50:59     
know um particle physics and with you know uh biochemistry where they have to     
51:06     
work out the molecular dynamics of something. And so and then of course you use Monte Carlo simulations to simulate     
51:14     
sort of the stochastic nature of those events. And so this uh library supports many     
51:21     
input formats and outputs numpy arrays enabling integration with scientific     
51:26     
Python ecosystems for many material science workflows. So     
51:31     
these is a lot of material science and what we're talking about here is material science in a sense because     
51:37     
we're talking about these kind of soft soft active materials.     
51:43     
Um, and so, you know, what we're talking about are these liquid crystals where we     
51:48     
have these rod-shaped or, you know, oblong type objects that are arrayed in     
51:56     
some sort of matrix. Um, there's this thing called pneumatic     
52:02     
order, which is where they're aligned versus a smctic order or some other type     
52:08     
of order where they're not aligned in the same way. And so what you want to do is you want to be able to compute the     
52:15     
sort of pneumatic order parameter which basically allows you to take the orientations of these um objects which     
52:23     
in our case are cells. They could be molecules, they could be particles and determine their order in terms of the     
52:30     
orientation. And so like most complex systems where we want to know um you     
52:37     
know in a group of objects what their collective behavior is we have an order parameter. And so this is the equation     
52:44     
we use to calculate the pneumatic order parameter. So I just took this kind of     
52:49     
this is an AI overview but it's largely from the Freud documentation.     
52:54     
Read the docs and it kind of goes through um some of this. So this is um     
53:01     
basically the elements of a 3x3 matrix where they're computed and we take the     
53:06     
particle orientation vectors which have some heading and then we multiply the components together. We sum all the over     
53:14     
all the particles in the system. So we see it has an integral there and we basically get this order parameter. So     
53:20     
it's basically um it's not really an average. It's kind of like um you know a collective     
53:27     
measurement of all of the orientations. And so this gives us this degree of     
53:33     
alignment in this order parameter S. So basically when we get this order parameter S, we can tell whether things     
53:40     
are perfectly aligned or whether they're random. So if they're random, S is near zero and if they're perfectly aligned, S     
53:46     
is near one, I believe. So that's what we're dealing. Um and so that it kind of     
53:53     
gives you some uh breakdown of the formula. Again, um the the     
53:58     
interpretation here is that of S sub1 is where we have perfectly aligned molecules where all the angular     
54:05     
measurements equal 0 degrees and S equals Z are randomly oriented     
54:11     
molecules where the average theta is 54.7 degrees. Again, you know, there was     
54:17     
no pattern there. And then an S of negative half or negative0.5     
54:24     
is where molecules are aligned perpendicular to the director. So there's like one set of molecules that's     
54:30     
that are oriented one direction and another set that are oriented in another direction. And that's actually not     
54:36     
random. That is uh there's an alignment there. It's just not pneumatic. And so     
54:43     
we can actually get a sense of how these uh molecules or in this case cells are     
54:49     
aligned using this simple uh parameter.     
54:54     
And so they say in real pneumatic liquid crystals the order parameter typically falls between 0.3 and 0.9     
55:02     
with the exact value depending on factors like temperature. So these are in physical systems. Now in biological     
55:08     
systems I don't know if temperature plays a role here but you know this is     
55:13     
something that um we're going to talk about in the paper. So this is where you have a     
55:19     
pneumatic order parameter of 0.03. So this you can see that there is no     
55:24     
order here. These trajectories are at all different angles in this space and     
55:30     
so it's very much they're all kind of in different or orientations.     
55:35     
an order parameter of 0.5 where there's a director and then there     
55:40     
are some other uh orientations. You can see that there's a little bit     
55:45     
more order. They're kind of in the same direction, but they're a little bit, you know, off. You see that you're starting     
55:51     
to get the sort of pneumatic order, but it's not quite there. And then finally, an order parameter of 1.0.     
55:58     
Almost looks deterministic. It's perfect pneumatic order where they're everything is oriented from left     
56:05     
to right. You can see every orientation is the same pretty much. And Bradley,     
56:10     
has this been applied to uh swarms like uh birds and fish? Yes, I believe so. Um     
56:19     
the math is the same. Um, and I don't know the papers off the top of my head,     
56:25     
but I've seen some where they've talked about the sort of of, you know, where     
56:30     
you're you're looking at the trajectory heading of the individual agents and,     
56:36     
you know, basically asking the question, is this ordered or disordered?     
56:42     
Okay. Okay. So then that brings us to the paper of the day here. This is the     
56:49     
nature communications. It's actually I guess it was published online uh this this week.     
56:56     
So this is uh John Walling for on this Mark Per they're um developmental     
57:03     
biologist Robert Huner and so they're basically trying to     
57:08     
figure out whether we can look at cells in development and say whether or not     
57:13     
these have some sort of pneumatic order to them and how does that happen?     
57:18     
So the abstract reads, cells undergo dramatic morphological changes during     
57:24     
embryogenesis and how these changes affect the formation of order tissues remains     
57:29     
elusive. Remember in this case we're talking about individual cells sort of forming a tissue but they're orienting     
57:35     
themselves in different ways. So you might be forming like skin cells or you might be forming uh uh heart muscle     
57:43     
cells and things like that in order uh sort of the orientation of cells is important in that sense. Uh here we show     
57:51     
that a phase transition leading to the formation of a pneumatic liquid crystal state during gastrolation     
57:59     
and the development of embryos of fish, frogs and fruit flies occurs by a common     
58:04     
mechanism. despite substantial differences between these evolutionarily distant animals. So     
58:10     
basically the embryos and all these model systems they looked at them and they found that there was this pneumatic     
58:15     
order that was playing a role here. And remember this is not this doesn't necessarily have anything to say about     
58:22     
mechanics explicitly. there's an implicit um sort of discussion about     
58:28     
what kinds of mechanics lead up to this pneumatic order and what kinds of mechanics does pneumatic order uh imply     
58:36     
and so that's that's I think maybe the question here with respect to mechanics but um you know this is something that's     
58:43     
happening kind of as a way to sort of um maybe it's to maintain the integrity of     
58:50     
the tissue maybe it's uh for signaling you know the cells need to signal one     
58:55     
another and kind of be aligned. Maybe they're migrating and they need that pneumatic order to migrate properly.     
59:03     
You know, um importantly, pneumatic order forms early before any discernable     
59:08     
changes in the shapes of cells. So, this pneumatic order is an early sort of precursor to um shape changes in the     
59:16     
cells and by extension differentiation of the cells. So, you can have pneumatic order before that.     
59:23     
All three species exhibit similar propagation of the pneumatic phase reminiscent of nucleation and growth     
59:30     
mechanisms. So this is where we have um this pneumatic phase grows across the um     
59:37     
the I guess the cells and it propagates. So you know this ordering isn't just     
59:43     
like local it can be it can become larger uh over space I guess. Um     
59:51     
the spatial correlations in the pneumatic phase in the noticord region are long ranged and follow a similar     
59:57     
power law decay which is given here with alpha less than unity indicating a     
1:00:03     
common underlying physical mechanism. So this means that a lot of these processes     
1:00:08     
this process in in these different model organisms happen according to they they happen according to this power law     
1:00:16     
across a certain anatomical region. So you have local pneumatic order and then     
1:00:21     
you can measure you know whether if you you start at one point whether that pneumatic order persists across the     
1:00:29     
region and that gives you some idea of maybe how it starts and how it propagates. And this this is a very     
1:00:36     
similar sort of uh lo local versus global relationship in all these     
1:00:41     
different uh organisms which suggests that there's some sort of physical mechanism. It's very much common to     
1:00:47     
every organism. So that means that there's probably a wolf for mechanics here um setting up this sort of um you     
1:00:57     
know setting up pneumatic work in these different structures. Um to explain the     
1:01:02     
common physical mechanism, we created a theoretical model that not only explains the experimental observations but also     
1:01:10     
predicts that the pneumatic phase should be disrupted upon loss of planer cell     
1:01:15     
polarity which you see in the frog. You also see cell adhesion in the frog and     
1:01:21     
the ntoord boundary formation in zebra fish. So let's go back to this. This is     
1:01:27     
where they're trying to figure out the common physical mechanism. They created a theoretical model and they want to     
1:01:35     
they they're able with this theoretical model to predict that the pneumatic phase should be disrupted in cases where     
1:01:42     
you lose planer cell polarity, cell adhesion and nicort boundary formation. basically disrupting these uh these     
1:01:49     
mechanics and seeing what happens or these physics of the the embryo and     
1:01:55     
seeing what happens. Uh gene knockdown or mutational studies confirm the theoretical prediction. So     
1:02:01     
you can actually disrupt all these processes by knocking down certain genes     
1:02:06     
and disrupting this process. So cell adhesion you know the adhesion molecules     
1:02:12     
cell planer cell polarity you can disrupt that and then notocord boundary formation all those things can be     
1:02:18     
disrupted by some sort of you know using either defined mutants or some sort of     
1:02:23     
knockdown strategy and this confirms what their theoretical model suggests.     
1:02:29     
The combination of experiments and theory provides a unified framework for understanding the potential universal     
1:02:35     
features of metazone embryogenesis. So this is something that you'll see across     
1:02:40     
animal embryos. Um and so this is again, you know, going back to maybe some of     
1:02:46     
the first embryos, what were going on in those first embryos that we have. Um I     
1:02:52     
think we talked about this a while back and I did a blog post on like uh the the     
1:02:57     
origins of embryos. You know, they were finding these embryos in um in uh     
1:03:05     
paleontological assemblages. Some of them looked like, you know, collectives of cells. So they were like, you know,     
1:03:12     
multisellular aggregates. And then some of them, you know, kind of look more like modern     
1:03:18     
embryos. And so, you know, that that's worth thinking about, you know, this sort of pneumatic order. Is this     
1:03:25     
something you'd see very early in um embryo evolution? That's something you'd     
1:03:31     
see maybe in these very early multisellular aggregates. And if you go to multisellular aggregates and say     
1:03:38     
something like yeast would you also see these kind of pneumatic order     
1:03:43     
transitions and did they play a role? So it's kind of interesting that there's     
1:03:48     
this very deep phoggenetic history of pneumatic order in embryos.     
1:03:55     
So um basically you know we want to understand the advent of ordered     
1:04:00     
structures and this is one way to do this.     
1:04:05     
Bradley. Yeah. Uh in looking at the formation of sensitia,     
1:04:12     
I came across a very interesting case which seems to contradict this work.     
1:04:17     
Okay? And that is some cells actually make a loop and fuse     
1:04:24     
with themselves and that's not pneumatic at all. They     
1:04:32     
Okay. So it's zero in pneumatic and yet it's very strange. It forms tubes. The     
1:04:38     
cell creates itself turns itself into a tube.     
1:04:43     
Okay. And uh it doesn't fit their model.     
1:04:53     
Okay. So it's an interesting exception. Yeah. Yeah.     
1:05:00     
I wasn't aware of this until I was looking at various forms of cell fusion     
1:05:07     
which could or could not lead to sensitia. And obviously if if a cell makes itself into a ring still has just     
1:05:14     
one nucleus and so it's not a sensitium. Right.     
1:05:21     
Okay. So I I I discounted that as an example of a     
1:05:27     
proper sensition, but it is an exception to their structures. Yeah.     
1:05:36     
Okay. So while they think they've got something universal, they obviously don't. Well, yeah. I mean, I don't know     
1:05:42     
if it's for every structure in the Yeah, I know. Well, they're they're touting it     
1:05:47     
as Yeah. So yeah, this this kind of talks about u     
1:05:54     
what they see as this evolutionary conserved process of convergent     
1:05:59     
extension where you get a narrowing and elongating of the body during gastrolation     
1:06:06     
uh where you get this a transformation from largely spherical early embryos     
1:06:11     
into the elongated animal body plan with this headtotail axial organization. So     
1:06:17     
you get this sort of convergent extension that and this is you know something you'll see across species and     
1:06:24     
again uh this convergent extension is driven by directed movement of cells in     
1:06:29     
one axis which they call convergence resulting in extension of the perpendicular axis. So you have     
1:06:36     
convergence along one axis and extension along the other. So it's like going from a kind of a sphere down to this     
1:06:43     
elongated form. That's basically what they're talking about. And then so this     
1:06:48     
this is something that that people have sort of studied for, you know, the history of embry embryogenesis,     
1:06:56     
but it remains unclear how cellular scale behaviors translate to larger scale physical transitions. And so this     
1:07:03     
is kind of what they're selling as universal. And you know, if it happens across a lot of species, you could make     
1:07:09     
the argument that it's universal. There always going to be exceptions, however. and different systems as well. So, you     
1:07:17     
know, it's kind of like when you say something is universal, you're kind of g have to specify the mechanism quite uh     
1:07:24     
extensively, especially in biology. Um     
1:07:30     
so, yeah. Uh so, confluent cell populations such as those found in the     
1:07:35     
early embryo and exist in a variety of physical states. For example, a high density tissue consisting of isotropic     
1:07:43     
non-prololiferative cells will be in a jammed or solid-like state. So this is interesting in a lot of the sort of the     
1:07:51     
liquid crystal literature in the soft active materials literature they talked about these different phase transitions     
1:07:57     
of matter. And one of them is that you have maybe like a free flowing flow of     
1:08:04     
particles that are moving in a maybe like cells migrating and they're doing     
1:08:09     
this freely and then at some density as they increase in density they jam or     
1:08:16     
they freeze up and they stop moving together. And so basically the idea is     
1:08:22     
that you know if you think about this as a linear process you have a bunch of particles or cells that are migrating     
1:08:28     
and if this were a linear process they would just simply slow down gradually and then freeze at some density that you     
1:08:37     
know where you might expect it to freeze. There's no um you know kind of     
1:08:43     
uh sharp difference in the behavior of those cells or those particles. In the     
1:08:48     
case of jamming or this jamming phase transition, you get this uh free flowing     
1:08:54     
state, this migratory state and then at some critical parameter value, they jam     
1:09:00     
and they become a solid and it be it's a quite an instantaneous event at this     
1:09:05     
parameter value. So this is an interesting thing about the role of uh     
1:09:11     
order order parameters and phase transitions in uh developmental biology     
1:09:17     
as an extension of uh material science or soft active materials in that this     
1:09:23     
the behavior collective behavior of cells can change quite suddenly and be subject to these phase transitions.     
1:09:31     
So you know this is in contrast to a growing tissue undergoing dramatic morphological changes uh where they must     
1:09:39     
be in a more fluid-like or unjammed state. So you get these unjam states and jam states and they these phase     
1:09:44     
transitions in between and I think we've run into a number of papers where people have proposed that there are these     
1:09:52     
different u you know transitions between jammed and unjammed states. So sometimes     
1:09:58     
you know development will proceed by having like an unjammed state where cells can migrate and then a jammed     
1:10:04     
state where cells are kind of stuck um and in in stasis and then another you     
1:10:10     
know it transitions back to an unjammed state where cells then are free to migrate again and this is all governed     
1:10:17     
by some sort of density. So if you reach a critical value of density, if the cells become dense enough, they freeze     
1:10:24     
up in these states. And so this is what they're kind of getting at in development. And then this kind of leads     
1:10:31     
to the order or the the orientation of cells because the orientation is     
1:10:36     
important in determining this kind of density because if they're all in the same direction, they can pack more     
1:10:43     
densely and if they're not in the same direction, they can't. It also governs you know other types of migratory     
1:10:50     
behaviors or things like that. So this kind of goes through the pneumatic phase     
1:10:56     
and how this kind of works. Um you know they kind of so here are some figures.     
1:11:03     
You can see that these cells here are in these different orientations.     
1:11:08     
Um and you know they kind of they're able to track them and look at their orientation over time.     
1:11:20     
Um and they they have a section here that says the onset of the pneumatic phase occurs prior to major changes in     
1:11:27     
cellular anisotropy. So anizotropy is where you get this uh this difference in     
1:11:33     
orientation and you so you get these major changes     
1:11:38     
in cellular anisotropy and the pneumatic phase occurs prior to those changes.     
1:11:45     
Um so they use the spatiotemporal changes in the alignment of cells to     
1:11:51     
determine the mechanism of formation of the pneumatic order as convergent extension proceeds. At this end we first     
1:11:57     
assessed the correlation between the shape index and the pneumatic order parameter and they looked at the     
1:12:04     
spatiotemporal dependence of um SI and S which are their parameters. At early     
1:12:11     
times, there are no discernable changes in SI along the medial axis. In     
1:12:16     
contrast, a dome-like shape appears at S along the same axis, indicating the     
1:12:21     
presence of a pneumatic phase in the medial region. So, there's this local     
1:12:26     
pneumatic phase. Um, and it's governing this sort of transition suggesting that     
1:12:32     
changes in orientational order precede changes in the cell shape. So again you     
1:12:37     
know the order of the population of cells maybe locally can precede how the     
1:12:43     
cell changes its shape. So basically the cells are becoming ordered and then they're changing their shape and they're     
1:12:49     
undergoing this kind of change in global phenotype or global uh morphology     
1:12:56     
of interest. Microtubules also displayed polarized growth prior to over avert cell shape changes suggesting a role for     
1:13:03     
the microtubial cytokeleton in early pneumatic order formation. So much as we know from mechanics cytokeleton is     
1:13:10     
playing a large role in microtubules especially in in changing these     
1:13:16     
confirmational shapes and changing how cells interact with them.     
1:13:23     
Yeah. question. It they say it the uh     
1:13:30     
the behavior of the cell proceeds as shape. Yeah. But shape determining shape     
1:13:37     
requires uh uh an inaccuracy in measurement.     
1:13:43     
Yeah. Okay. Could it be that they simply don't have enough spatial resolution?     
1:13:50     
That's possible. I mean, you know, and again, like shape is a tough thing to measure in a lot of ways. Yeah. Yeah.     
1:13:58     
So, yeah, I don't know. That's interesting. I mean, I don't know how they I'm not going to get into too much     
1:14:04     
about the measurements, but it's possible. Yeah. And, you know, there are a lot of     
1:14:09     
variation. There's a lot of variation in shape that we don't understand the significance of, too. So, they need to     
1:14:16     
look at actton ring dynamics. Yeah,     
1:14:23     
inance they seem to contradict themselves in the sense that the microtubules     
1:14:29     
elongation may be the primary thing and it's got to     
1:14:34     
start at a uh uh by the addition of single tubulants     
1:14:42     
which they can't detect and yet may have an influence.     
1:14:50     
Well, if you want to get into tensity, it's both the microtubules and the     
1:14:55     
acting ring acting together     
1:15:02     
to change the What I'm saying is     
1:15:09     
there's got to be some sort of feedback between microtubial extension and cell     
1:15:15     
shape, Right? And you could easily miss the beginning of     
1:15:21     
that because it the be beginning of that process occurs at extremely low resol     
1:15:29     
scale. Uh namely the size of individual tubulants.     
1:15:34     
uh so uh uh I don't I can't agree with them that     
1:15:41     
uh change in cell shape comes after uh the process of nomatic formation.     
1:15:49     
I think they just didn't detect it. Yeah.     
1:15:57     
Okay. I mean it it's how should I say it's it's similar to standing a pencil     
1:16:03     
on its point. Yes, it falls and you may not be able to     
1:16:08     
predict the direction it falls but the initial phase of it falling could be     
1:16:14     
detectable detectable uh by having a sufficiently high resolution picture of the pencil.     
1:16:23     
Yeah. Okay. And even though you can't predict it, once it starts, it's determined.     
1:16:31     
Yeah. Okay. So, uh     
1:16:37     
that conclusion of the is not convincing. Yeah.     
1:16:46     
So, the last thing I want to talk about with respect to this paper is a sort of spa uh sort of long range spatial order.     
1:16:54     
So they talk about this here to characterize the nature of pneumatic     
1:16:59     
order in the noticord regions. This is a different area different u thing that they're talking about. We calculated the     
1:17:07     
spatial correlation describing how the alignment of cells varies with distance. So remember we have these local effects     
1:17:13     
of pneumatic order and then we can look at the global effects but we have to understand like how the pneumatic order     
1:17:20     
might decay over space. So you know we have a local area where there's pneumatic high pneumatic order it might     
1:17:27     
decay across a certain structure or region and then you know what is so what is the sort of     
1:17:35     
consequence of that? So an exponential decay of the function would indicate that alignment weakens rapidly as the     
1:17:43     
distance between the cells increase. So as cells become less densely packed or     
1:17:49     
less um you know have less sort of uh density between them you know as the     
1:17:57     
they're sort of packed together um you might have a rapidly weakening     
1:18:03     
alignment. So this is all of course dynamic. So it's not just enough to say we can take a snapshot of the region or     
1:18:11     
of the cells and say okay they're at this orientation. So this is going to happen. There's this active process     
1:18:17     
where the cells are constantly remodeling. The cells are constantly changing their density. They're     
1:18:23     
constantly moving against one another. And so we can only take the averages for any one area local or global. And so     
1:18:31     
this is another problem that you have with these kind of collective systems is they're constantly changing. So you     
1:18:37     
might take a static measurement you get this dynamic effect because there all these things happening kind of     
1:18:43     
continuously. So anyways uh but but there is this relationship between density of of the     
1:18:50     
cells and pneumatic order and decay over space. So if so if there's this     
1:18:57     
exponential decay um that would mean that a line weakens rapidly. However, if     
1:19:04     
this decreases slowly exhibiting a power law pattern and this is different than     
1:19:09     
linear decay where you just have this linear decay over space. Um it suggests     
1:19:14     
that there is significant alignment even between cells at large separation indicating long range order. So in other     
1:19:22     
words, the order is long range in terms of space. There's this, you know, it     
1:19:28     
ranges over a larger area of space and it's not so much the local density, it's     
1:19:35     
the global density that's controlling this um process. So they do this spatial correlation which is this parameter C     
1:19:43     
subsr this function and this is how they define it where SRRI is the pneumatic     
1:19:50     
order um of cell at location RI and the average is over all cells in the notord     
1:19:57     
region. So they're looking at this global average. Strikingly, the decrease in correlation     
1:20:03     
function follows a power law. And so this is how you state that um with a     
1:20:08     
small value for alpha indicating the pres presence of a long range spatial correlation in the noticord region. The     
1:20:16     
value of the exponent alpha changes in a timed dependent manner but remains small     
1:20:21     
throughout which is consistent with the finding that the pneumatic order in the noticord region emerges early in the     
1:20:27     
convergent extension process and remains persistent. So this is where you get     
1:20:32     
this um early emergence and then this persistent     
1:20:37     
um uh effect. Taken together, these data reveal that the formation of the zebra     
1:20:43     
fish noticord is accompanied by the growth and propagation of a pneumatic phase starting from a seed reminiscent     
1:20:50     
of a nucleation and growth process observed in crystals. So this is of course the liquid crystal     
1:20:57     
model. So there's this analogy between the embryo liquid crystal and how these     
1:21:04     
nucleation and growth processes occur. Observation of a pneumatic liquid crystal phase during zebra fish     
1:21:10     
convergent extension prompted us to determine if this liquid crystal state is an evolutionarily conserved feature     
1:21:17     
of convergent extension. And when they say evolutionarily conserved, it could be that there's, you know, um it's     
1:21:25     
probably more of a physical parameter that governs a lot of embryionic systems. I think that's probably a good     
1:21:31     
way to put this because that's you know common processes are happening across     
1:21:37     
different model systems. So there's a lot more in this paper I'm not going to talk about lot a lot of     
1:21:43     
interesting figures and of course their measurement techniques which I'm not going to get into. Um, but I think it it     
1:21:50     
raises a number of interesting questions with respect to sort of collective behavior     
1:21:57     
and the embryo is a complex system as well as sort of these mechanical     
1:22:03     
issues um about how cells behave in a tissue. Do we have any other comments or     
1:22:08     
questions? Yeah, I think I think they didn't simulate this behavior at an     
1:22:14     
intracellular level, did they? I don't think so. No. Okay. Uh, I'd suggest if     
1:22:22     
somebody wants to tackle it, a simulation of microtubules inside cells and their effect on cells shape change     
1:22:29     
and how they affect whole tissue. Isn't the notic Well, is the noticord 2D or 3D     
1:22:35     
tissue? Isn't an epithelia? Everything's 3D.     
1:22:44     
Everything's 3D. Okay. Okay, fine. uh it doesn't matter. What I'm saying is that     
1:22:50     
as the microtubial elongates in the uh in the cell it may one could try to     
1:22:57     
calculate the effect on the next cell and there might be some feedback in     
1:23:03     
orientation there uh let alone communication of I don't know     
1:23:10     
how you get cellto cell communication on tubulant concentration or tubulant     
1:23:16     
condensation rates But uh I think there's a simulation to be done here to     
1:23:22     
see what parameters cause the cells to align and whether you get your power law     
1:23:28     
or not. Okay. So yeah,     
1:23:34     
lovely power law. Okay, the power you know you could     
1:23:41     
uh you know one thing for example uh     
1:23:46     
microtubules have this dynamic instability uh where they elongate rapidly.     
1:23:55     
Well no actually they elongate slowly I guess and then collapse rapidly.     
1:24:00     
Yeah, can and that kind of a thing could definitely influence alignment of     
1:24:08     
neighboring cells because     
1:24:13     
while it's elongating uh it's kind of acting as a stiffening element when a     
1:24:19     
shortening it's less stiffening. Okay.     
1:24:25     
So uh I I think a a simulation uh of the     
1:24:30     
pneumatic properties based on intracellular microtubial might be worth     
1:24:37     
looking at uh in realistic molecular fluctuations.     
1:24:43     
Microtubers for instance can bend uh they can be stiffened by uh lateral     
1:24:51     
interactions uh literally as we showed by a factor 10 to the 4th     
1:24:57     
and uh there could be all kinds of intracellular things which are having a large influence in setting the uh the     
1:25:06     
initial uh drive towards the pneumatic phase of a tissue.     
1:25:13     
Okay. So, it might it might be worth tackling instead of like 11, right?     
1:25:21     
I wonder if I should be taking that workshop again.     
1:25:26     
It's a whole whole weekend. It's Python programming. So,     
1:25:35     
yeah. What's What's the relationship? I don't understand.     
1:25:40     
Um anyway it's it's Compel 3D. It's that modeling program     
1:25:46     
and they're having a workshop and they do interactive you you make you interact     
1:25:52     
with their program and then you do a project at the end.     
1:26:00     
Oh okay. Oh I I see. It's you're not just learning Python.     
1:26:07     
No you're programming in it. So yeah, you're learning it while you go. Okay.     
1:26:14     
Okay. At any rate, uh the uh as I said, the the the start of     
1:26:21     
an instability such as alignment of pneumatic structures     
1:26:27     
uh might have to do with the statistics of molecules inside those structures.     
1:26:37     
How about uh their growth and their collapse and their bending and uh their attachments     
1:26:45     
to other things. Uh and how many of them are are there?     
1:26:53     
Right. Right. How many are there? Their relationship to uh microtubial organ     
1:27:00     
organizing centers. Uh so yeah the initial phases of the matic     
1:27:08     
formation of tissues is approachable.     
1:27:14     
Okay. Well I know where microtubules are in cells and they they tend to to line     
1:27:22     
the sides and crisscross the loose side ones. They're ones with the apical     
1:27:27     
glands that are perpendicular. Yeah. you know that's Beth Burnside's     
1:27:34     
work okay so yes they do all sorts of things     
1:27:39     
but the uh mechanics therefore of an individual cell and its relationship to     
1:27:45     
the tissue might be an approachable problem well I know it depends on its     
1:27:50     
attachments and microtubules are involved in that so its attachments to its base and     
1:27:59     
attachments to the cells to the side. They're very important in its mechan     
1:28:09     
um for the junctions to the what triunctions     
1:28:14     
try or quad junctions. Junctions will be at the ends     
1:28:19     
at the a at the where the cells meet     
1:28:25     
where three cells meet or four cells meet or five cells meet the they     
1:28:30     
actually micro tubes attach to those points. Yeah. Oh, that's curious. Well,     
1:28:36     
I don't know that they go across it, but the microtubules form the base and then     
1:28:42     
uh you get um linking molecules     
1:28:47     
in between. Yeah. And there's all sorts of literature on bundles of microtubules also which might be involved in the form     
1:28:55     
but I haven't seen any on formation of the bundles. Oh okay. Yeah. Well, let's     
1:29:02     
there was something. So uh too much literature, too much literature.     
1:29:09     
It's enormous. But uh     
1:29:17     
yeah, I I think the the these these pneumatic properties of tissues     
1:29:24     
could possibly be explained. Okay. Yeah, there's different types of     
1:29:29     
of um cell sheet movement     
1:29:35     
apparently. Yes. But this is a chicken and egg problem they presented us with and     
1:29:41     
they've claimed that that the chicken comes before the egg. Yeah. Yeah. Okay.     
1:29:49     
Okay. So, I I kind of doubt that.     
1:29:55     
Wow. Anyways, I mean I mean any process which     
1:30:01     
uh explodes in the sense that it gets bigger and bigger     
1:30:07     
requires a starting point. It might be a small fluctuation but but it does     
1:30:13     
require it. Yeah. You need a So what starts that? I     
1:30:19     
mean Yeah. Right. Exactly. What starts it? Are fluctuations sufficient or or     
1:30:24     
you have to add something? What stiff microtubules     
1:30:30     
with lateral connections might do it? Okay. Well, I haven't studied that part of of it     
1:30:37     
because I was just doing stationary to ignore the cell sheet movements. Oh, yeah.     
1:30:44     
Yeah, I know. I know that's hard enough, but but yeah, there's they've said, "Oh, there's three     
1:30:51     
different types of cell sheet movement now."     
1:31:01     
There's a new type of cell sheet movement out there. Yeah. Just recently published really paper. Yeah.     
1:31:10     
Yeah. Send that along here. Yeah. If I could find that. If you could do cell     
1:31:15     
sheet movement and then a new form of it, maybe it could find it. I was     
1:31:23     
surprised this stuff about cell fusions. I thought there were only a couple of them and I ended up with a list of six     
1:31:30     
different kinds of cell fusions. Yeah. Okay.     
1:31:36     
Okay. And that includes this ring one which contradicts this whole business about pneumatic order. Um, I was also     
1:31:44     
reading something that said, "Yeah, cells sort of have a pneumatic order,     
1:31:49     
but it's not quite the same as a crystal."     
1:31:57     
It's critiquing that it's kind of a a metaphor that's applied because it's not     
1:32:02     
exactly the same. Yeah, I know. It's a universal metaphor which is not universal. So, I guess that's all for     
1:32:09     
today. Um, thanks for attending. All right.     
1:32:15     
All right. See you next week. Thanks. Okay. Bye. Bye. Bye.
