     
Transcript     
0:01     
hello Hi Hi How are you i'm good Over here Yeah Did Did you Were     
0:10     
you able to find the paper reference I sent to     
0:15     
you it starts off with geometry Okay It's a good paper about development and     
0:23     
epithelial tissue and how that interplays with development Oh well I've     
0:28     
got re-entrant re rigidity transition in planer epithelial with volume and area     
0:35     
elasticity All right Yeah So I'll go over that     
0:40     
Yeah And then well it's got ordered and disordered and calmer and to     
0:50     
flat tissues and yeah it just looked interesting Okay So why don't we get     
0:57     
started um so yeah thank you Susan for that     
1:02     
reference and we'll look at that reference and we'll talk about some other things     
1:07     
here So uh yeah we'll do a couple things here Um the first is this paper Susan     
1:14     
sent out Um and this is uh I'm just bringing up     
1:19     
the archive stub here uh sakar and crangic uh re-entrant rigidity     
1:26     
transition in planer epithelia with volume and aerial     
1:33     
elasticity So this is the paper um let's do the abstract and then we'll     
1:41     
get into some of the figures So we recover a rigidity transition in     
1:47     
3D epithelial monollayers described by cell volume aerial elasticity So this looks kind of     
1:54     
like the surfaces that we've been talk like to talk about these epithelial     
2:00     
monollayers which uh if you look at sort of a cross-section of it     
2:07     
remember they're these square kind of like this and they're kind of     
2:13     
array like this and then 3D we're just looking at     
2:19     
this part here where we have some depth to this layer So this is the mono layer     
2:26     
here and then this is the depth and then these are the interaction     
2:31     
points and we've talked about these tri junctions So you might have here this     
2:38     
point back here might be a tri junction This point here might be a tri junction if you had a     
2:43     
cell in you know this is no longer a monollayer or you know it it has more     
2:50     
cells in it So this is like kind of the basic unit of what we're talking about here It's a     
2:58     
nice model to look at some of the properties of cell packing cell populations Looking     
3:05     
at the model layer you get this nice these nice mechanical     
3:10     
properties along Okay So um we recovered the a     
3:17     
rigidity transition in 3D epithelial monollayers described by cell volume area elasticity or VAE So this is where     
3:25     
we have our volume and our area So these are just two measurements of each of     
3:31     
these cubes and you you know we get this measure which is     
3:38     
VA has something to do with our volume and our area     
3:46     
And then we're measuring elasticity which is this property of     
3:52     
deformation basically So the deformation is when we     
3:57     
put some pressure on this monowlayer How does it deform how     
4:03     
do these uh you know where the cells meet how do they interact and so it has     
4:09     
a lot to do with these trijunctions And in this case it would be a bjunction     
4:15     
uh would be kind of the normal mode of things and then you might have a tri junction or or an end junction that     
4:21     
would also you know exhibit some interesting properties An inflame isotropic strain     
4:28     
drives a re-entrant columner to squamus rigidity transition And so they talk about     
4:34     
re-entrance um and you know this is I guess kind of this property you see this you hear this     
4:42     
term a lot in neuro anatomy it I think it's basically feedback     
4:48     
um but you know they're talking about this sort of property again of this     
4:54     
entire sheet not just single cells it's describing the sheet plus the     
5:00     
interactions when a strain or stress is applied     
5:06     
uh with a critical point near the unit cell aspect ratio In addition to the vanishing shear     
5:12     
modulus the phase diagram also features floppy states with zero inplane bulk     
5:17     
modulus and a discontinuous columner squamus transition controlled by the lateral     
5:25     
tension Our results provide a 3D context the rigidity transition to the well     
5:31     
studied 2D area and perimeter elasticity or AP model of epithelia So this is of     
5:38     
course this we'll get into this offering a resolution to the counterintuitive     
5:44     
compression induced tissue softening predicted by the 2D model So we'll get into this figure in a     
5:51     
minute I want to mention a couple things from the introduction here During     
5:57     
morphagenesis epithelial tissues must have an ability to transition dynamically between elastic and     
6:04     
fluid-like states So you know there's this elastic state and this fluid-like state So we     
6:12     
have this sort of state transition or this phase transition that's queued up     
6:18     
But we know that soft tissues have these properties that are you know allow them to not be rigid So you know mechanic     
6:26     
purely mechanical systems are rigid and the problem there is that you know if you model these kind of if you use rigid     
6:35     
uh system simulators to model biological tissues sometimes you can run into     
6:40     
problems that they're they don't you know your model doesn't behave like a tissue What they're doing here is     
6:46     
they're pointing out we have these elastic and fluid-like states and we have to have this sort of ability to     
6:52     
transition between these states This allows the sheet to support mechanical     
6:58     
stresses on the one hand but also facilitates cellular rearrangements and     
7:03     
large scale tissue flows on the other So we want to be able to these this cell uh     
7:09     
layer we want it to sort of contract so that they can be packed very tightly So they can like you know be very     
7:17     
narrow You can pack it in very tightly You also might want to have     
7:22     
other types of deformations like this     
7:30     
So the cell sheet is kind of impacted by these different     
7:35     
forces You might have something like this You might have forces like this or you know you might want to have     
7:42     
them break out of this uh this sheet and start to flow or you     
7:48     
want them to be fluid And so that might mean where you know one cell might migrate     
7:55     
upward against the rest of the sheet and you might have like a flow     
8:01     
like this where cells are kind of moving away from one another And of course you have a medium     
8:08     
outside of the sheet of liquid say or some sort of     
8:14     
flow and the cells are embedded in that So they're responding to those forces as well And so you know you have all these     
8:22     
diverse behaviors And so that's what we're kind of trying to account for So there's this dual solid fluid     
8:30     
behavior People have studied this using uh the ape model or the area and     
8:35     
perimeter elasticity model which assumes that each cell's apical area A and     
8:41     
perimeter P tend towards preferred values AL and P respective     
8:47     
So if we go back to the uh ourselves again we have this ape model or AP model     
9:00     
And what they're referring to here is if we have cell a     
9:08     
cell we have the apical aspect here which is a z and then we have the     
9:16     
perimeter which is p 0 And we can do that for each cell We can do that for the cell sheet And we     
9:23     
can get this elasticity I guess that's a coefficient     
9:32     
and tells us something about how these cells are kind of transforming in terms     
9:38     
of shape So it's a shape description and there probably better ways to do it There are a lot of ways to do shape     
9:44     
descriptions Um but this is I think a good     
9:50     
approximation Okay Um so this model assumes that each cell's     
9:56     
apical area and perimeter tend towards preferred values So A Z and P 0 the     
10:02     
preferred values sort of the stable states The the model predicts a rigidity     
10:07     
transition controlled by the preferred cell shape So it means that tissues are rigid when the actual cell shapes are     
10:15     
incompatible with the preferred one and floppy when cells can attain     
10:20     
their preferred area perimeters So basically when it's outside of its stable state the tissue becomes rigid So     
10:28     
you not only have one cell you have a bunch of cells And what we find is that when they're not in their stable state     
10:35     
they become rigid So if they're in like this state they might become rigid less     
10:40     
they kind of come apart as a tissue or if they're in their preferred state they might become flexible So this     
10:46     
might be its preferred state maybe it's relaxed state and then you get these kinds of this ability to be flexible     
10:56     
So that suggests that there are a lot of maybe a lot of things happening at the tissue level as well as at the cell     
11:03     
level to keep this relationship stable because you don't want your tissues to fly apart under conditions that are uh     
11:12     
not preferred But then when you're in a preferred position you want to have that flexibility And again this is just a     
11:20     
property of the mechanical aspects So that's what we're dealing with     
11:26     
Okay Um in the floppy regime the linear shear modulus vanishes and depending on     
11:32     
their structure tissues may also display zero energy barriers for     
11:38     
cellular So this is where we're in this floppy regime where we have our preferred areas and perimeters or     
11:44     
preferred shape In this regime we have uh no linear modul no linear shear     
11:52     
modulus and so we can rearrange ourselves as we well this is not we     
11:58     
don't we don't do that but the biological system can uh rearrange things very     
12:04     
freely these findings extend to fully three-dimensional space filling cell     
12:09     
aggregates analogous to 2D the generalized 3D model employs volume and     
12:15     
area elasticity VAE which minimizes the energy when cell volume is in surface areas not just the     
12:22     
preferred values uh V0 S0 so this is similar to AP which is the volume in     
12:29     
area elasticity as opposed to perimeter elasticity okay so then we so we have     
12:36     
these two different models uh we have the two-dimensional model which is AP and the three-dimensional model which is     
12:43     
VA so we have our AP which is two dimensional and our VAE which is     
12:52     
threedimensional So this perimeter is a description actually of I guess this     
12:59     
area here So okay     
13:08     
Um all right So as in 2D the preferred 3D cell     
13:15     
shape controls rigidity and the linear shear modulus vanishes and tissue transitions from rigid to floppy     
13:22     
states Okay So while 3D uh BAE models naturally extend the concept of rigidity     
13:30     
transition to bulk cell aggregates it remains less clear how 2D AP model     
13:35     
translates into 3D epithelial model So how do we get this two-dimensional model     
13:41     
transitioning into this 3D model a recent model developed by some of us     
13:46     
addressed this open question by coupling an apical cell mechanics uh by coupling apical cell mechanics     
13:54     
described by AP to the basilateral cell region under surface tension This     
14:00     
coupling inherently introduces apical basil asymmetry and cell mechanics     
14:05     
suppressing the rigidity transition such that tissues remain rigid regardless of the preferred apical subshape     
14:13     
So this is uh figure one This is a simulation snapshot Um this just shows ordered and     
14:21     
disordered phases So Susan mentioned that they talk about ordered and disordered phases in the paper This is     
14:29     
um this phase transition So you get the ordered which is here So you see the way they've modeled this as a cell sheet     
14:37     
that has this depth So you know we have this model here     
14:42     
where we have the AP model where we have the perimeter of a cell and then we just would have this kind of depth to this     
14:50     
cell sheet here keep going like this and you know we want to know kind     
14:58     
of the constituents of the monollayer back here So um that's what this     
15:06     
comparison is And we have the ordered phase which is here which is where they're tightly packed and they're kind     
15:12     
of aligned very uh neatly And then the disorder phase Sorry they're hexagons     
15:17     
They're hexagons Yeah And then the disordered phase where they're kind of out of phase in terms of their alignment     
15:25     
They're hexagons but they're disordered So you see the packages Um the     
15:31     
disordered ones are probably more that u some of them are five uh sided and some     
15:37     
of them are sevensided because those two will do a tiling Okay And the five seven     
15:44     
tiling ones are more fluid Oh okay Yeah     
15:50     
So the 57 tiling ones would be in the disordered phase or in the ordered phase     
15:56     
Yes And of course there's other ones like you can do 38 or something as well     
16:03     
just depending Um and that goes this     
16:08     
looks like it goes for the basement membrane as well as the top apical part     
16:15     
of this the tissue And um sometimes you for the     
16:24     
um any of these uh the top tends to move around more than the bottom And you'll     
16:29     
see um different um structures on the top     
16:37     
than than you have on the bottom Okay Yeah Well you can see that that the     
16:45     
top can be can move around a lot Yeah Where they're attached to the bottom     
16:51     
So that changes as well So yeah we have the basement membrane down here which is     
16:58     
and then we have the tops which are the apical end up here and that's where you get this movement So you get like     
17:05     
deformationations kind of along this seam like get some bending here     
17:10     
Yeah and twi twisting too because you get scentoids to twist too So yeah     
17:19     
Okay Um yeah So that's your ordered and disordered This is rigid like columner     
17:25     
This is floppy like cubuidal So this is where we have the same kind of thing where we have ordered and     
17:32     
disorder Then we have rigid like squamus where we have the ordered and     
17:38     
disordered as well So they're doing these simulation snapshots of these sheets and they're just looking     
17:45     
at so it says for clarity around 10% of all cells are shown the fraction of     
17:51     
nonhexagons in disordered sample d equals     
17:57     
0.437 So I don't know if that means that they in the disordered yeah in the     
18:03     
disordered sample they have I guess a parameter value for non-hexagon So they     
18:08     
have a different fraction of non-hexagons in the disordered phase that are here So you can see that like     
18:15     
Susan said they're not just hexagons in the disordered phase but other types of     
18:21     
shapes as well Figure two are these two-dimensional simulation     
18:27     
snapshots of this transition and order tissue for uh     
18:33     
SO5.75 and disordered tissues for SO5.9 So this is where um we can see some of     
18:42     
these We have this uh epsilon value which goes from 0.2 to     
18:49     
1.0 to 1.9 We have the ordered at the top and the disordered at the bottom So you can     
18:57     
see for range this uh parameter sweep what those changes look like you get     
19:03     
this uh I guess the ordered it's sort of     
19:08     
it's a hexagonal lattice a purely hexagonal lattice at a higher value of epsilon     
19:15     
it's a little bit sort of the tiling is aperiodic and then you go back to a     
19:21     
periodic tiling at 1.9 and then for the disordered you I     
19:27     
guess you can also recover this uh it's aperiodic I guess And then it's     
19:33     
less a per it's more a periodic and that it recovers that timing     
19:39     
Um and then so this just shows the phase uh space here where you have this floppy     
19:46     
regime this rigid regime there's a critical point here uh this is between I     
19:52     
guess so and epsilon for these different values you get the ordered regime at the     
19:59     
boundary between rigid and floppy and then this disordered regime within floppy Okay Um and then this just shows     
20:09     
um some other aspects of the phase space So I guess this is the important     
20:16     
uh figure 2D the important thing to point out which is that there's this     
20:21     
floppy phase this rigid phase and then this uh critical point and then this     
20:26     
boundary between order and disorder Okay     
20:33     
Um yeah and then we go down to figure three We see     
20:41     
total energy per cell We actually see this difference here between squamus and     
20:46     
columner So in 3B um and this is just an energy profile in     
20:51     
A but this shows uh this difference here the sort of phase transition between squamus and     
20:58     
columner Squeamus is where it's pushed it's compressed in terms of its height     
21:04     
and then columner is long um you know has a longer Z-axis I guess um we get     
21:11     
this value of epsilon where it's stable and then there's these degenerate states at the phase transition     
21:19     
um and then you get uh for positive     
21:24     
values of gamma I believe this is the columner state so we reach     
21:30     
uh zero value for gamma we get we go from negative gamma to positive gamma     
21:36     
there these degenerate states right around the phase transition and this is a transition from squamus to column and     
21:43     
so that's what they're getting there um I'm sorry the stress drain curves for     
21:49     
squamus were different from the colmer um okay in my measurements oh really     
21:57     
yeah they're They behave differently because of sheep Yeah     
22:03     
this is my model Oh in your model And then what so what's the difference between what they're doing and what     
22:09     
you're doing um you realize I have over 200 graphs     
22:15     
right i know Yeah What's the difference um it's how they     
22:23     
um let's see how they behave     
22:28     
in compression as opposed to tension Okay     
22:34     
Um some I don't know squammer cells are very interesting Yeah     
22:42     
[Music] Uh yeah I I could look it up I could um     
22:49     
once actually once I get this paper written on it I'll I'll present that     
22:55     
Okay that'd be good Yeah that'd be good Okay So yeah then uh they have     
23:02     
their discussion Our results show that the rigidity of epithelia with VAE may     
23:08     
be tuned non-monotonically by inplane isotropic strain describing a re-entrant     
23:14     
CST rigidity transition or by tissue transition from rigid columner to rigid     
23:19     
squamus states through intermediate floppy cuboidal states exists and     
23:26     
additional constant lateral tension distinguishing lateral from apical basil sides introduces a discontinuous this     
23:33     
columner squamus transition which we saw in figure two or actually figure three     
23:40     
uh with energy degenerate states with zero inplane bulk modulus where lateral     
23:46     
tension is absent So u our 2D project model shows that     
23:51     
varying 3D cell aspect ratio which is by enclosing inplane isotropic strain     
23:58     
effectively corresponds to nonlinearly changing the effective inplane preferred cell shape index P 0 which in turn     
24:06     
controls a rigidity transition This provides an alternative physical interpretation of P 0 which has been so     
24:13     
far interpreted through a competition between cortical tension and adhesion     
24:18     
strength Um and then the 2D8 model which inherently does not describe cell height     
24:25     
counterintuitively and in contrast with the standard jamming predicts tissue softening upon inplane compression by     
24:33     
accounting for a third dimension Our model changes this view Indeed compressing and dilating a monollayer     
24:41     
away from a cubuidal state both leads to its rigidification So this is uh     
24:49     
where they kind of differ from some of things in the past They use this uh     
24:55     
third dimension and they get the same interpretation Our analysis shows that     
25:01     
the VA monollayer model is a more direct 3D analog to the 2D ape model than the     
25:07     
model in which apical cell sides describe a are coupled to the     
25:12     
basilateral cell domains under surface tension Indeed while the apical basilateral coupling in the latter model     
25:20     
breaks the apical basil symmetry preventing the rigidity transition altogether our 3D VAE monollayer model     
25:28     
reserves this transition and then the limit of incompressible cells even exactly maps to 2D fixed area     
25:36     
a so yeah they they mention here while the     
25:41     
assumption that epithelial cells tend to preserve all their volumes of total surface areas that this is yet to be     
25:48     
experimentally validated So the models where we kind of make this assumption This     
25:55     
phenomenology this phenomenology may readily apply to studying synthetic planer packings of strongly adhering     
26:02     
lipid vesicles where the membranes bending elasticity may be negligible and     
26:07     
the mechanics may be well described by the VAE model So that is that paper     
26:15     
Um thank you Susan for uh pulling that up for us Um     
26:23     
yeah So I want to mention one more thing and then I'll get to Leila     
26:28     
Um so Dick put out a new paper in bio systems is brand new     
26:36     
So Dick of course Dick Gordon is a member of our group and we I think we've talked about     
26:41     
this in the meeting um but this is something that he has I guess been     
26:46     
asking um is there a relationship between embryionic bioelectric currents and     
26:53     
differentiation waves so if you're familiar with sort of this idea of bio     
27:00     
electricity and development this is something that has a deep history and     
27:05     
developmental biology and most recently Mike Levven has picked up the torch on     
27:10     
that and they have now of course a journal devoted to bio electricity The idea is that bioelect     
27:17     
electricity plays a role in morphagenesis So you have these uh you     
27:22     
know cells can generate bioelectric currents They can generate bioelectric     
27:28     
signals Of course we know about um action potentials and those happen in     
27:35     
neurons but you have all sorts of cells producing different bioelectric signals     
27:42     
Um all cells have ion channels and they produce some sort of measurable trace at     
27:49     
you know with in the case of action potentials they have a spindle and they have all these properties that are     
27:57     
unique to action potentials but you actually get other types of signals as     
28:02     
well and so you know then the question is what are they actually doing in your body and so the answer I guess is that     
28:10     
they're coordinating cells they're creating fields that can be you know uh beyond the single cell and they     
28:18     
can set up different morphagenatic events They also may play a role in uh     
28:24     
wound healing and other types of regenerative capacities and tissues And     
28:29     
so what Nick is asking the question uh of is you know are these bio or     
28:35     
embryionic bioelectric currents that will surely result from um these kinds     
28:41     
of interactions do they have uh uh play a role in setting up these differentiation     
28:47     
waves so the kinds of differentiation waves that we've talked about in a number of papers that Dick has been     
28:55     
talking about for years do they actually have a relationship that's interesting     
29:01     
and so I noticed that Dick has been publishing a bunch of these kind of theory related papers in the last year     
29:07     
or two where he's asking questions and posing them and it's not like we have a     
29:12     
good answer for them but it's that he wants to get those questions out there and people may be working on them but     
29:19     
he's posing the question So the abstract is short It reads     
29:25     
differentiation waves usually spread in embryionic epithelia until they reach boundaries with stopping powers but     
29:32     
unexplained So differentiation waves in short are these uh waves that are     
29:39     
propagated through tissues or or these sheets of cells And so they may     
29:45     
originate in a single cell and they they send this wave across cells and the     
29:51     
cells are getting this basically the stimulus to differentiate and there have     
29:57     
been different mechanisms that Dick and colleagues have explored Um there all sorts of mechanism     
30:05     
molecular mechanisms there are mechanical mechanisms that affect cell division all these different types of     
30:12     
things that are proposed So we don't really know what exactly sets up I mean     
30:17     
we know one of what sets up a differentiation wave It could be a chemical signal So be an electrical     
30:23     
signal could be a mechanical signal But then when you know when an individual     
30:28     
cell gets that signal what's the mechanism by which it's transduced into     
30:33     
the cell itself and then it's able to produce this uh result in the phenotype     
30:40     
which is that the cell differentiates or otherwise changes its morphology changes its relationship to a     
30:48     
tissue So we did a nice analysis of this androsophala imaginal disc looking at     
30:55     
the imaginal disc as there's this wave that goes through this furrow that go     
31:00     
moves through the imaginal disc and it changes this imaginal disc from this     
31:06     
disordered tiling which we saw in the last paper to this uh set of omatidia     
31:12     
which are going to form then the eye structure So there you know we can have     
31:18     
these waves that go through tissues they can enable this kind of differentiation     
31:24     
We want to know if electric bio electricity has a role in this and if so     
31:30     
what is it so um differentiation waves usually     
31:36     
spread in embryionic epithelia until they reach boundaries who stopping power has been unexplained Bio electricity in     
31:44     
embryos there's two ranges cell to cell and long range So you have like in     
31:50     
bioelectric studies usually you have sort of these local and global effects     
31:56     
and we might have a sheet of cells again we might have like this set     
32:01     
of let's just do some hexagons for the sake of uh making this     
32:08     
simple and we have a sheet of cells and we might have a signal that originates     
32:14     
in this cell down here and it goes it sends its signal     
32:21     
outward and these are the cells it's going to affect So you can have the cellto cell relationship So cell A and     
32:27     
cell B might have a relationship It might be a reciprocal relationship where     
32:33     
you have something that is affect A is affecting B B is affecting A and there's     
32:38     
this coupling and that's a local relationship and so that would be A to B     
32:45     
coup coupling and then you might have a more     
32:50     
uh regional or global relationship So in this case you'd have A B and then C and     
32:56     
D and you'd have this relationship here where the signal propagates out There's     
33:02     
this signal that propagates maybe linearly with decay So the farther you get out the less signal you have And so     
33:09     
it affects maybe this global or regional     
33:15     
signal It affects a host of other cells     
33:21     
So A to B and then you might also have C and B     
33:27     
which are a little farther away and they might also be affected the strongly as B But you get the picture that there are     
33:33     
these different levels of effects and so we want to understand maybe you know this is how you get uh you know     
33:41     
different effects at different length scales and so there are a lot of physics uh implications for this Susan Um yeah     
33:49     
it's a a cell cramp I actually have a picture of one on the surface of an axle     
33:55     
Oh yeah And it it definitely is electrical and mechanical for sure All     
34:01     
the chemicals I mean it's calcium You uh cause a cell epithelial     
34:07     
cell to cramp with calcium because it works like a muscle You've got actin and     
34:13     
meosin If you add calcium then it it does its ratcheting motion and causes     
34:20     
the area to cramp to to contract So you     
34:26     
have um these cramps coordinate eventually and become a wave of cramp     
34:33     
that go up the side of the axel salamander egg and and in some of these     
34:40     
insects that you um uh can see Um and you get     
34:49     
um what Dixs is calling a differentiation wave because after a     
34:54     
cell has undergone the cramp it's rearranged its     
34:59     
um cytokeleton into say a more polymer um     
35:07     
configuration and maybe it relaxes back down but it's rearranged its um cellular     
35:15     
components And these waves last long enough that it um rearranges the nucleus     
35:23     
It changes the shape of the nucleus and that changes the gene expression So     
35:29     
that's all the theory there but I have pictures of a cramp Oh wow     
35:35     
Okay Yeah that's good Yeah there uh you know we have pictures uh available I     
35:41     
know Dick has some in his books and I know you probably been playing around with it and uh I I we did a a review of     
35:49     
this in the P bio systems paper we published last year So there's like you     
35:55     
know basically reinterpreting a lot of not just um signals from cells but you     
36:00     
get like calcium waves in the embryo and they you know also can be kind of     
36:06     
interpreted as differentiation waves and so you know you have you can observe     
36:11     
these empirically We can understand that this is a theoretical concept where we     
36:17     
have many different types of activity and then we have you know this is it     
36:22     
it's it's a way to kind of think about how     
36:28     
uh morphagenesis proceeds not just in one cell but in a b     
36:34     
in a tissue where a bunch of cells are linked together Yeah And then yeah go     
36:39     
ahead The thing is is they are slow so people haven't noticed them because it's you do     
36:47     
a time lapse or you slow down the movie they occur occur over     
36:53     
hours So you can't really personally you can't see     
36:58     
this but if you take images and then do a movie of them um you'll see that it     
37:05     
really is a moving wave Actually I was going to take pictures     
37:11     
with one of the um microscopes that I have just of the top of the egg so we     
37:18     
can get so people can see this Like um Dick lost his his     
37:25     
movies There's a miscommunication or something But um I'm I have a microscope     
37:32     
It would just simply if I just put an egg under it that it would take pictures     
37:38     
um with with a proper 24 megapixel camera and um get really decent     
37:47     
resolution of it So I'm just I'm just going to take a picture of the top of it And my flipping microscope decided not     
37:54     
to work So and rather than fix it or anything since I'm I'm busy I'm just going to     
38:02     
take pictures over the top of the the egg and I should be able to see some of     
38:08     
these these waves Okay that sounds good     
38:15     
Yeah Um yeah So yeah there's So there's this     
38:21     
temporal aspect there's a spatial aspect And so the what what he's trying to do     
38:28     
here is um get a sense of what you know how this     
38:33     
problem can be solved basically So it is postulated here that the long range     
38:38     
component somehow alters gap junctions and intermediate filaments creating the     
38:44     
boundaries for the trajectory differentiation So in these cells we have gap     
38:50     
junctions we have intermediate filaments which are all things at the boundary of     
38:56     
two cells or sometimes you know you could have things that are propagated     
39:01     
from A to B to C to D and through different means So there are all sorts     
39:06     
of ways in which these could be waves can be propagated um a combined EIT electro     
39:14     
electrocompetence tomography vvisual microscope is proposed to investigate     
39:19     
this proposed correlation So yeah he lays out kind of what needs to happen for this     
39:28     
um and he talks about sort of an example of mechanical initiation of     
39:33     
differentiation waves uh this is one that is observed to correlate with primary neural     
39:39     
induction The blast seal is known to come under pressure We have shown how this might launch differentiation waves     
39:46     
during primary neural induction The artificial release of pressure in zenipus leads to multiple neural     
39:53     
plates Um and then uh it is possible to organize differentiation waves as     
39:59     
differentiation trees subsuming lineage trees whose evolution has been postulated to explain     
40:06     
heterocchronone macroevolution and metamorphosis A differentiation code has     
40:12     
been postulated to be the genomic basis for these trees Heterocchronone can be explained by variation between species     
40:18     
in the timing of branch points in the differentiation tree So this is where     
40:24     
you know heteroproni is sort of the timing of growth and the timing of     
40:31     
differentiation And so if we go back to our differentiation tree which is derived from the lineage tree and     
40:38     
actually is has more information than the lineage tree and I'm not going to get into drawing out a differentiation     
40:44     
tree now u but you basically have this tree this binary tree and the the length     
40:53     
of the branches were correspond to sort of the length of timing of this process     
40:59     
So this wave might be fast or slow and then it leads to this sort of change     
41:06     
that can be uh you know described using this heterocchroone concept where you     
41:11     
get changes in the you know the the swiftness and degree of     
41:17     
[Music] differentiation Macroevolution may correspond to topological changes in the     
41:22     
differentiation tree by a copy and paste portions of the genome Metamorphosis may     
41:29     
be the result of rare concantenation of two differentiation trees by emittings     
41:34     
of different species And he's talked about this problem with um you know     
41:41     
where well Williamson has this controversial proposal that different     
41:47     
genomes have mixed over evolution So you get like different parts of uh genomes     
41:54     
that kind of lead to uh differences you know that sort of the DNA of different     
42:01     
species have kind of intermingled through um uh transposon uh means or horizontal     
42:10     
gene transfer or something like that And so you have this ability to take     
42:16     
different basically differentiation tree programs and cantonate them into the     
42:22     
same differentiation tree Um there are other you know mechanisms where you're     
42:27     
going to have the evolution of these different programs that lead to differentiation trees and so forth     
42:34     
And so then he gets into bio electricity and then he talks about combining the two approaches to     
42:40     
embryionic development that involve bio electricity and looking at differentiation trees by     
42:48     
suggesting that the long range bioelectric currents alter gap junctions in a way that halts the propagation of     
42:55     
differentiation waves from one tissue to another and thus limits their trajectories     
43:01     
So using these kind of local connections gap junctions things out the membrane     
43:08     
and then getting this propagation from cell to cell instead of like some sort of uh global signal where you know it     
43:15     
might be like a mechanical or chemical signal that kind of gets propagated out     
43:21     
broadcast in this way that actually has an effect on limiting the trajectories     
43:29     
Um and so this is a I guess a finer grained approach to this idea and so it     
43:37     
requires specific types of measurement and you know and so that's     
43:43     
what he's proposing So he gives a number of examples here in embryos kind of goes through the     
43:49     
background alternative models Um and so you know this is just kind of laying out     
43:56     
the problem space and how you might approach     
44:01     
this So that's a nice little uh sort of hypothesis theory paper Um     
44:08     
congratulations to Dick on that All right So     
44:14     
um okay so we have some references by Susan this is the one we talked about     
44:19     
re-entrant rigidity transition and planer epithelia with volume and aerial     
44:25     
elasticity This other paper by lang uh matas and key vertex model is capturing     
44:32     
subcellular scales and epithelial tissues This is a plus paper and then     
44:37     
mahul said I had to leave got an insect bite So hopefully Mahul is okay Um okay     
44:43     
Now Leila if you wanted to present uh that last part of the presentation     
44:51     
[Music] Uh hello Uh hi Uh that was a great work     
44:59     
by and uh and like very insightful presentations previous ones Uh how much     
45:06     
time do we have Bradley please we have as much time as you want I mean maybe you know like not an hour but     
45:15     
sure uh I'll quickly try to uh discuss So I made little changes in the     
45:22     
presentation so that uh little I added little more diagrams Okay I thought the     
45:28     
slides were little emptier last time So so hi I am     
45:33     
Lindharaju from triple Hyderabad brain cognition and computation lab specifically and     
45:41     
uh yeah so I'm going to work on this for the next uh uh couple of     
45:48     
months under the INCF like horizontal research lab and education under the     
45:55     
guidance of uh professor Bradley and uh Jesse Neul     
46:01     
And this so yeah so so to give a fine     
46:07     
introduction what the I mean like background and the motivation So the motivation is like the segance is one of     
46:14     
the smallest organism neimat which has like very limited number of connections uh     
46:22     
limited number of neurons and so if we are able to develop like neural networks     
46:28     
on this small networks I mean like small possible modeling if I can do this then     
46:33     
we can see a bigger picture like more biolog biologically plausible AI and we can design more enhanced and deeper     
46:40     
neural network So that's one of my inspiration to delve into this and the     
46:46     
neural developmental is a very high dynamical and temporally extended     
46:52     
process where uh it's not just extends in a simple division manner but it has a     
46:58     
certain cell division pattern and it evolves in a lineage tree in a very hierarchical fashion and the spatial     
47:05     
locations and the migrations happen in a very complex fashion When we try to model this these hierarch hierarchical     
47:13     
interactions and multisellular interactions that actually evolve over the time then possibly then we believe     
47:20     
that you know we are taking a one step ahead to understand the you know uh the     
47:27     
the how the organism is developing from a single cellular to like a     
47:32     
multisellular organism So that will give us a very much insights into like how the tissues have been divided how the     
47:39     
neurons have been you know segregated into like certain uh I mean like the sensory the sensory     
47:46     
neurons the motor neurons and you know uh how are they divided So we can understand more into details about all     
47:53     
these things and in the next so when what we     
47:59     
believe is because it's a hierarch higher order interactions and     
48:05     
multisellular organisms depend on hierarchical functions uh I mean like     
48:12     
when we try to model these using hyperraphs sorry so sorry so when we try     
48:19     
to to model these using hyperraphs We believe that these high rider interactions can be captured in a very     
48:26     
good manner rather than a d or a static graphs So so this was the main motivation and in the basic introduction     
48:33     
of what is uh the project is divided into three preliminary objective First     
48:39     
we design the foundation that is we design how we want the you know the the     
48:45     
structure of the uh I mean like uh the data that we want So we we lay the     
48:50     
foundations in a way that you know the we structurize data in certain format that so whenever we use a different     
48:57     
hypograph networks or uh we use different neural network or when we use a different uh you know uh uh I mean     
49:06     
like we can use this framework in such a way that it can be a nice plug and play So the when we keep the foundation right     
49:13     
with the data then it could be one of the goods Second we are interested in     
49:19     
the unification because when we have the data then uh the I mean like the     
49:26     
fundamental idea is to you know to this can be applied to a different models I     
49:31     
mean like suppose we try to understand the development of a certain cla so but it should be replicable for various     
49:39     
other limits which have been taken birth So not just one time single organism     
49:45     
data but it should be useful for multiple and whatever the model that we are trying to use I mean like uh we     
49:53     
already knew that you know uh so the the kind of data that we get from C elegance     
49:58     
is more of an invariant but still we need a unification framework where we need not ex uh additionally add certain     
50:07     
layers or certain neurons but it is uh holistic in in itself And the third is     
50:14     
we want to see how this emergence happens when we use this framework as you know when we vary the layers when we     
50:20     
vary the you know the message passing mechanisms when we vary you know this you know I mean like the the number of     
50:27     
uh the neurons that we use or the type of uh you know the hyperage connections     
50:32     
that we use So like how is the emergence happening so this this entire framework helps us to understand this So we divide     
50:39     
I mean like I divided it into three phases based on my understanding and uh you know uh the way I can you know     
50:46     
construct this project in a more better manner So the data unification you know     
50:51     
the dynamic hypograph construction is the first foundational step Second is the unification of hypograph the message     
50:57     
passing the architecture the the layers and the weights and they all can be unified into a single framework where we     
51:03     
can just plug and play for a for the performance and uh and uh the emergence     
51:10     
where we see that how the chronic terms and the functional modeling happens So the so it's basically the outcome So     
51:17     
yeah So and uh the previous we saw that the taxonomy in one of our previous     
51:24     
videos where we see that modeling of higher order interactions can be taken in like you know how we are taking the     
51:30     
input features How are we you know encoding it in an input structure and later we saw that you know we can use     
51:36     
different message passing mechanisms and certain way of learning these message messages So this is the whole taxonomy     
51:43     
of how we going to do So I divided into like three dimensions where the first is how do we construct an hyper edge and a     
51:50     
hyperraph where we include all the lineage spatial function and all the     
51:55     
different other features and also use a different message passing mechanisms where we not     
52:02     
just use a simple local or global attention or like you know summing up     
52:07     
the features or having a mean of all the you know relevant features or can we apply certain attention to the features     
52:14     
which are more relevant or the these type of message passing mechanisms you can include it in and uh the temporal     
52:21     
dynamics So the previous works more like they focus more into the the static     
52:29     
frameworks which cannot be used in real time Whereas in growing topologies you know there can be a cell division which     
52:36     
strongly that can you know happen and that can lead to the fat uh you know uh     
52:41     
which can be fatal uh the which can lead to a cell fatality So these these things     
52:49     
should be captured while you know the modeling So the static nature of the     
52:54     
graphs could actually imped this thing But whereas this hyperraphs allow us to you know model these dynamic topologies     
53:01     
and the growing topologies and uh so I wanted to give a short quick sorry uh     
53:08     
and anyone has any doubt you can just stop me     
53:13     
Okay So the hyperraphs are hyperap just to give a basic introduction So hyperraph     
53:19     
is basically a generalization of regular graph where the edges are certain hyper edges where they do not restrict to a     
53:26     
diic connection that is a one-on-one connection but it can connect to any number of node not just two So so here     
53:34     
we see a simple graph where V is vertex or edge V is vertex or the node or E is     
53:40     
edge where I we represent in this as V small I are basically number of     
53:47     
nodes which in a given whole subset of capital V and E small E subscript I says     
53:53     
that it's edges and whereas coming to hyperraph we denote it as H V the number     
54:01     
of nodes are same but the the edges are different I mean like the connectivity is different where epsilon I is     
54:07     
basically the hyper edge we can see and this hyper edge can carry any     
54:12     
information So I have here given an example where XYZ are the coordinates of     
54:18     
RC elegance and five is the phase angle So just to give an example you know these all information can be fed into     
54:24     
the hyper edge so that the modeling could happen And so when we try to     
54:29     
understand the mathematical intuition you see the hyper node actually connects     
54:35     
multiple nodes instead of a single node So that's a hyper sorry hyper edge So     
54:40     
here red color box sorry red color oval is basically the A the B the C are the     
54:47     
hyper edges which are connected to all these nodes and one can expand it in     
54:53     
like two different fashion One is click expansion So which might not be useful because we cannot retrieve retrieve this     
55:01     
click expansion back into the graph But whereas the star expansion we see in the right third uh the third one which is     
55:08     
you know it more of like an entity and uh so node is considered as a you know     
55:16     
the features I mean like the connections and ABC are basically entities here the     
55:21     
entity ABC are the kind of hyper edge connections that the the nodes consists     
55:28     
of so it's basically the star expansion that's that's one of the way that we can you model the hyperraph when we convert     
55:35     
a hyperraph into star expansion then it is possible to convert the star expansion back into a hyperraph so     
55:42     
that's one advantage so yeah so and uh so we are trying to understand a simple     
55:48     
mathematical intuition where here as we discussed earlier so v is basically the     
55:54     
number of nodes that are present and small e is a edge and uh what This basic     
56:02     
function here tells is here uh the h belongs to the     
56:08     
hyper the layer of a hyperraph where it multiplies with the weight vector and     
56:13     
then it adds so basically this is a kind of and each summation happening at each     
56:20     
and every time means that it's actually aggregating the information by adding a     
56:26     
certain weight to that specific set of uh you know the hyper nodes     
56:31     
So here this expansion this is basically one of I mean like the intuitive example     
56:37     
of how a hyperraph can be captured and here t plus one is a time step So each     
56:44     
and every time step while it's evolving so it captures uh how is it growing and     
56:50     
the growth can be captured using this basic network So this just the mathematical intuition So it's a step     
56:58     
one or like you know it's kind of a hello world to you know kind of you know     
57:04     
hyperraphs So yeah so so here when we represent this as     
57:10     
like this is a dynamic hypograph representation So this actually models complete developmental process where     
57:17     
each sequence it carries a node starting or a node which has like a bunch of     
57:23     
features and it encodes all these lineage depth the spatial coordinates and molecular state and everything into     
57:30     
these features and then it carry forwards into the time So here time t and t plus one reminds us to like each     
57:37     
and every time step how the you know graph evolves So that's why you call it dynamic and uh the interactions that     
57:44     
that the functionality that it has that the summations and uh you know the the     
57:50     
the I mean like the the nonlinearity that is applied on these functions     
57:56     
basically tries to you know capture the developmental events here such as the     
58:02     
way the cell is divided and the type of cential fusions it happening and then     
58:08     
the special neighborhood relationship ship that's actually acquired during the period of time So these all these can be     
58:15     
captured using these hypographs and uh so what     
58:20     
uh I mean like I I proposed during uh the project is so how can we model this     
58:27     
uh growing hyperraphic neural networks So as we have seen in the time it has     
58:32     
grown I mean like it moved from time to time the way the hyperraph evolves So we     
58:37     
call it as like growing hyperraph neural networks So the main advantage is the dynamic edition of the nodes and the     
58:44     
hyper edges because the hyper hyperraphs have an advantage to learn these nodes very dynamically every step And the     
58:51     
second is we are trying to integrate all the information that has like no lineage I mean like spatial the functional cues     
58:58     
and also like the phase angle relationships and various other things And this is more of a learnable and this     
59:05     
is a datadriven simulation That means if we if you have a data of like six to     
59:11     
seven C elegance uh you know developmental process then maybe we can simulate uh a more I mean like if we     
59:19     
have like 10 to 20 examples we can make the model learn that okay this is how it starts from a single cellular to a     
59:25     
multisellular organism and this is how the spread of each and every you know     
59:31     
the cell divides and it and this is how the lineage tree grows and this is how     
59:37     
the spatial coordinates move in order to finalize into a organism I mean like uh     
59:44     
and that's how uh you know the developmental process can be captured with like 20 examples you can simulate     
59:50     
like 100 more so by learning them and yeah just as an example here the AB cell     
59:58     
divides into ABA and ABP and the lineage tree again grows in the second time     
1:00:03     
point division where ABA divides into AB A L and AB A R and then AB divides     
1:00:12     
into AB ln and ABP R and this goes on So as we have seen that you know initially     
1:00:19     
the coordinates of AB have been at a certain location but they have been     
1:00:24     
dispersed during the developmental process and this also can be captured as     
1:00:29     
we can see that this is just a two-dimensional example but we can move deep deeper into things by     
1:00:36     
threedimensional visualizations and even in threedimensional visualization we can see that there are change and the shift     
1:00:41     
in the coordinates and this is the movement happening the way the organism has to you So it swirls around itself     
1:00:48     
and and finally becomes a worm with a head at a certain location and a tail at     
1:00:53     
a location So this all the events can be captured by this network by you know     
1:00:59     
modeling all these time points together So this is just an intuitive example saying that okay the cell has been     
1:01:05     
divided and the red indicates the first division the green indicates the second and the blue indicates the third division So so these uh this is just an     
1:01:13     
intuitive example but this can be you know applied to a graph neural network where it can understand the complete     
1:01:21     
evolution process So yeah so when I include the phase angle part here we can see that     
1:01:27     
you know first the cell was divided with the you know uh 198.9 degrees but later     
1:01:34     
the divisions were like smaller in number So that I mean like we can see the change I mean like why it was     
1:01:40     
divided into know 198 and how it divided into that So we can get the threedimensional coordinate exactly and     
1:01:46     
uh we can get the you know the movement where it starts from and where it ends     
1:01:51     
from that the spatial locations can be I mean like captured more accurately in these     
1:01:58     
scenarios So so how did I design this timeline is in the uh first two weeks     
1:02:06     
I'll see how we can set up the things the data sets and the literature what we're going to work on and the complete     
1:02:12     
design if it's ready and if it's you know agreed with the mentors then then     
1:02:18     
we can move into the next step where we can uh you know design the hyperraph and integrate and see how it can you know     
1:02:26     
work with uh I mean like dynamic growth and later we can add these message passing mechanisms and we can evaluate     
1:02:32     
how these message passing mechanisms are like helping the model to learn I mean     
1:02:37     
like not every message passing mechanism might help but we have to understand which one is actually good for it and     
1:02:44     
and in the in the seven to eight I mean like in the end of the second month we'll probably see that you know how     
1:02:50     
this tempor temporal modeling can be scaled suppose I have a developmental data of like one C eleant     
1:02:57     
am I able to understand the two three four I mean like the other set of     
1:03:03     
developmental process of like other C eleants And later we can see whether the conneto     
1:03:10     
that that is being developed and the fully you know trained network if the network is well trained and when I see     
1:03:16     
that network when I'm trying to understand that okay this when I start from a how how it can you know evolve     
1:03:24     
the one way it can evolve and is this conneto aligned with the you know what     
1:03:29     
is the actual thing so that we can observe and we can have the nice visualizations and check okay The     
1:03:36     
modeling one has been performed like this and you know the original is like this So we can see the correctum     
1:03:42     
alignment and visualizations too And in the final step we can do the if we are     
1:03:47     
you know significant enough from the existing literature and if we are able to make a big difference then we can     
1:03:53     
document it in a very good manner and one we can do a final submission to the     
1:03:58     
know the GOC and also we can you know write a good piece of work where the community actually see that there is a     
1:04:06     
uh I mean like the like good improvement in this space So that's uh the like     
1:04:13     
main moto of you know doing this project So the contribution how could I     
1:04:19     
contribute to INCF is first designing a unified model So this could be very much     
1:04:26     
helpful because you know uh existing works do not capture all the relationships into a single framework     
1:04:33     
And the second is when we use this uh hyperraphs as a developmental uh u I     
1:04:40     
mean like when we apply this this hyperraph into a neural developmental program then this this adds a very high     
1:04:47     
weightage to understand that you know in a further piece of works because uh the existing works focus on just like     
1:04:55     
designing the connections and just visualizing them but this can be leveraged to learn the complete patterns     
1:05:01     
of like starting from from a single cellular I mean like organism to a fully     
1:05:07     
multisellular organism and the pipeline could be reusable for the further     
1:05:12     
research and visualization tools can be used uh to not just to visualize the hyperdrafts but also to visualize     
1:05:18     
various other things like how the uh like an example I have shown before that     
1:05:23     
you know how is the variation that is observed you know when uh the cell     
1:05:29     
division happens from uh ABA to AB B So yeah so in future work what I     
1:05:37     
believe is how can we extend this so once we understand the certain mechanisms of like how this the entire     
1:05:44     
pipeline works and all So maybe we can uh you know take some you know design     
1:05:50     
principles and you know take some motivation from here to design a new line of architectures uh in the space of     
1:05:58     
AI because AI is more of you know scaling these days but uh there is no underneath biological principles     
1:06:06     
uh like they used to be previously because convolution neural network has a little bit of you know biological     
1:06:11     
principles but uh the latest set of models have been you know used to feed the data But if we can understand that     
1:06:18     
okay feeding the data can make the network learn but why so these design principles can be extracted So by     
1:06:25     
understanding uh you know uh the I mean like the you know the living living     
1:06:32     
creatures starting from like C elegance and we can design biologically plausible neural networks because when we once we     
1:06:38     
starts from like C elegance we can we can try see extending this to     
1:06:43     
like you know a bigger brain which is compared to those C elegance is like broiler and if we if it works in broil     
1:06:52     
then we can extend or we can see the limitations and build a better model where it can you know I mean like scale     
1:06:57     
to you know starting from like you know uh I mean like C elegance to that of     
1:07:03     
like human brain human brain is much more complex but yes I believe that one day we can do that and uh yeah so and uh     
1:07:11     
one of the crucial thing I wanted to pinpoint here is the cential relationships can also be you know     
1:07:18     
useful like the way the cell divides that actually makes the centium to form     
1:07:24     
at that certain place I mean like you know uh when uh uh um I mean like uh     
1:07:31     
like in the previous uh discussion we were having about like epithelial cells     
1:07:37     
and epmal cells and all So if there is like an epidermal synthesium that is formed     
1:07:44     
there So is it formed during the developmental process for like all the     
1:07:49     
uh you know the sea elegance and is there any connection that you know it forms in a certain fashion we can     
1:07:55     
observe these emergences during the developmental process I believe that could be a one of the good extinction to     
1:08:00     
understand those regions So so yeah so that's the end of the presentation and     
1:08:07     
these are the some of the references and yeah thank you     
1:08:13     
Well thank you Leot That was good Um yeah     
1:08:19     
Uh yeah we're looking forward to working over the summer on this some of these open questions So I     
1:08:27     
mean you know I know you have your timeline Um so we're starting next week on working on the coding period We've     
1:08:34     
been doing things with the um community period So now is a time sort of when we     
1:08:40     
start doing this and uh yeah I know that uh I got in touch with Gadratha and he's     
1:08:48     
also getting ready for everything So     
1:08:53     
yes sir very much in all right thank you Looks like Ibrahim is here from     
1:09:00     
Neuroch X or um would you like to introduce yourself okay go ahead     
1:09:09     
Hi Hi Yeah Okay cool No just jumping in I'm     
1:09:15     
working with Morgan in San Francisco Uh recently joined Neurochics as a contributor so I'm just a fly on the     
1:09:21     
wall I have uh nothing of value to say at this time That's okay Welcome     
1:09:28     
Thanks All right Well um I think that's it for today Uh did we have anything else i     
1:09:34     
know Morgan and Jesse joined late Um did you have anything you wanted to mention or     
1:09:43     
Oh no Go ahead     
1:09:48     
No I it's um I'd forgotten about the meeting because of because of the     
1:09:54     
holiday So um just juggling juggling kids with this So no problem Uh yeah     
1:10:02     
things I might say more next time or later this week but [Music] um the internship stuff's going really     
1:10:10     
well for other projects Um I I still think a a lot about sort of the     
1:10:18     
simulation agent computational stuff and even some actual developmental     
1:10:25     
develop little bioish tieins of that some of the time for that and then um     
1:10:31     
applied to some interesting stuff at at some other projects which I may uh share next week that have a little bit of tie     
1:10:38     
into some of this as well but mostly good stuff and meeting and excited for     
1:10:43     
the summer code So thanks everybody for that uh more next time Yeah thank you     
1:10:49     
That sounds good Um I one more comment I wanted to make on some of the things that Lewis presented Um he talked about     
1:10:57     
simulating development and generating the hundreds of simulations using the model that he had he's wanting to work     
1:11:04     
on We've tried been you know that been one aim of kind of what we want to do in     
1:11:10     
the group We've done this in different ways We've done like cellular automa We've done other types of uh lineage     
1:11:17     
tree modeling and we've have done this uh I think a few years ago in uh DVO in     
1:11:25     
one of the projects we uh used um uh trying to remember what we used maybe     
1:11:31     
like a GAN model generative adversarial network to model or to simulate um     
1:11:38     
images of the embryo working from a microscopy image you     
1:11:44     
segment out the features and then you simulate that It was interesting but you     
1:11:50     
know again it's not as rigorous as we kind of like it to be So we're kind of have all these pieces in place and     
1:11:57     
there's a lot of value in these kind of simulations when you get a lot of     
1:12:02     
simulations of a single process because we know that development has these properties of phenotypic     
1:12:10     
buffering you know these pro properties where you get a very stable     
1:12:16     
phenotype given um you know this process that unfolds again and again this     
1:12:21     
generative process We know that you know to get viable phenotypes you need to have that kind of     
1:12:28     
stability and we even have like um epigenetic uh landscapes that describe     
1:12:36     
this process as this process of canalization buffering And so you know without     
1:12:42     
simulating the genotype we can simulate the phenotype in these different ways to     
1:12:47     
see what that process might look like to see what happens if you change certain things and see what the viable     
1:12:54     
phenotypes are because that's a lot of development is about producing viable     
1:12:59     
phenotypes So you get a you get variation in developmental phenotypes but generally those are um you know a     
1:13:07     
lot of times those phenotypes are not viable So they don't really count in the sense of like what the end process of     
1:13:14     
development is So just some parting thoughts on     
1:13:20     
simulation and the simulation of development why that's important     
1:13:25     
particularly important I think for advancing some of the things we want to do in addition to the titans to openorm     
1:13:33     
and the simulations that they're doing of different parts of the worm So     
1:13:38     
bioysics the conacto so okay so that's all for today Thank     
1:13:46     
you for attending Oh go ahead Okay Yeah I put the um paper reference that I've     
1:13:53     
been talking about in in the chat Okay sounds good Yeah we saw that I think     
1:13:58     
Okay All right Thanks for attending     
