## Transcript     
0:01     
all right hello welcome to the D.Va meeting for this week uh today I want to talk about uh some     
0:07     
data sets that we have available it was from a paper that was published in 2016     
0:13     
some work on binarizing the lineage tree and turning it into what we call     
0:18     
differentiation code and this was by request of one of our     
0:23     
GOC scholars Girotha um and then I will get into papers that     
0:30     
I have to to go over for the group so they can see what's going on there uh we     
0:36     
had some nice presentations last Friday in the Orthogonal Lab and Orthogonal     
0:42     
Research and Education Laboratory uh open source meeting uh we've gonna     
0:48     
move those over there now because I want to have all of them together so Jiadratha and Lewith uh Buu they they     
0:57     
both uh have talked about their projects so Leith is working on hyperraphs     
1:04     
and trying to figure out how to build hyperraphs more efficiently and     
1:10     
Giadrotha is thinking about time hyperraphs or temporal hyperraphs     
1:18     
so that's what's going on there so if you go to our uh website     
1:23     
dvaworm.weebly.com we have uh publications so if you go to     
1:32     
more academic output publications we have a lot of publications here for     
1:38     
you to look over this spans the over 10 year history of the group so we've done     
1:45     
work on sea elegance and nematodes we've released data sets     
1:53     
we've released software we've done papers on fruit flies one paper on fruit flies exclusively     
2:01     
we've released some data on this we've done some uh papers on datons on the     
2:08     
general basil area but other datons as well and we've done comparative     
2:14     
development so we've done a lot of work on looking at not just sea elegance but     
2:19     
sea elegance in evolutionary context so if you go down the line so if you use     
2:24     
other species as a comparison we can say things about development that you know     
2:30     
basically putting sea elegance in context and sea elegance is its own unique system but also has a common     
2:38     
ancestor to some of these other systems uh we can we can find out some very interesting information about     
2:44     
development and so this paper here from 2016 quantifying mosaic development this     
2:51     
is a paper that Richard Gordon and myself worked on and we have a lot of um     
2:57     
supplemental data in that paper and quite frankly a lot of ideas that um     
3:03     
we've kind of forgotten about because this was almost 10 years ago this paper was sort of a comparison or     
3:11     
at least uh a dual analysis of sea elegance so our nematode and sea     
3:17     
intestinal analysis which is a core which is a marine inverter so this was u published in the journal     
3:24     
biology basel and there's some supplemental data so if you go to the supplemental data tab it gives you this     
3:32     
uh zip file and in the zip file we have these two files uh it's table S5     
3:40     
and table S so if we look at the paper here this is quantifying mosaic development     
3:46     
and you know we we do this analysis of Siona and test analysis and counterpitis     
3:54     
elegance so one is of course our neatode and seion in test analysis a c square     
4:00     
you might think that's a quite a diff you know it's quite a biogenetic distance how do you say anything about     
4:08     
it and one of the things about both of these systems is first of all there's a lot of secondary data that we can     
4:14     
analyze but secondly they have a very similar mode of     
4:20     
development that is they have the sort of deterministic mode of development early     
4:25     
where you can trace the cells through a cell lineage tree and this unfolds in a     
4:32     
stereotypical way across different members of species     
4:37     
and we actually have a good nom so these are two systems that we can     
4:44     
compare not necessarily at least directly but it gives us this framework     
4:49     
for doing what we're going to do many of the things we did in this paper um and     
4:55     
so I want to highlight some of the things that we did in this paper     
5:01     
so let's start with table supplemental 4 this is um a type of analysis that we     
5:08     
did on the lineage tree it's called the differentiation     
5:15     
and so what the differentiation code is it's a binary code     
5:21     
that and we also did this at the lineage tree as well directly what this is is a code that describes     
5:29     
relative position in lineage and the relative position in     
5:35     
sort of the age of the embryo so if we start with zero So we start with a     
5:41     
single cell embry that single cell divides into two cells     
5:47     
the left hand cell and the right hand cell left hand cell being 0 0 right hand     
5:53     
cell being 01 in segans that lineage tree is organized     
6:00     
anteriorly posteriorly in Siona and test analysis it's organized in a different way but it's     
6:07     
it's very similar in that it's connected to the morphology connected to the phenotype     
6:14     
and so as we move through the tree we add in our uh divisions     
6:20     
we end up with these codes for each level of the tree so if we go to the next model we have 00 0 on the left one     
6:30     
on the right and there uh all the different uh variations in the middle so     
6:36     
as you know from uh translating binary to decimal you know that 0000 is zero     
6:44     
and 111 is one or is the highest number and then you have things in between so     
7:02     
so if you're familiar with binary and translating binary to decimal you know that 00 0 is 0 and 11 one is four     
7:13     
and then we have intermediate states 001 and 11 one so basically every time the     
7:20     
cell divides we flip a bit in one direction or another we then count for     
7:26     
that level and we have our four cell state     
7:32     
we also have our eight cell state so if we start from 0000 on the left one one     
7:39     
on the right we get eight cells and we have these intermediate states the point     
7:45     
is is that we can characterize the length any the length of a code and it characterizes a level work division it     
7:53     
also describes differentiation because it describes if a cell differentiates where that is in the tree or where that     
7:59     
is in the lineage and so the differentiation code is varies from the     
8:05     
lineage code by being reorganized by the I think in this case it's the smaller     
8:11     
cell on the left the larger cell on the right and this work on differentiation codes based on Richard Gordon's work on     
8:18     
differentiation trees and differentiation codes basically reorganizing lineage trees so that you     
8:25     
have tissues of different sizes resulting in different parts of the tree     
8:30     
so you're basically reorganizing the tree by some significance criteria and so     
8:37     
that's that's the difference between the differentiation code and the code but this actually provides a means for us to     
8:44     
characterize each entry in the tree and address it in a way that's computational     
8:50     
so one of the advantages of of generating this code is that we can     
8:55     
describe it not in terms of uh well we can describe in terms of sublineage and we can also describe it in terms of its     
9:03     
relative position so if we decide to say knock out a cell if we want to run a simulation and we have our lineage code     
9:10     
or differentiation code and we run a simulation we have an addressable set of     
9:16     
cells we know kind of their location in terms of how we organize a tree so the lineage code if we have 0000 we know     
9:24     
this is at the third level of division we know it's at the left hand side of     
9:31     
the tree and then for the lineage tree and sele we that that's the anterior most end and so we can figure out very     
9:39     
quickly where cells fit in and so we can remove cells and mutate     
9:44     
the lineage tree and run experiments to see if the lineage tree is still intact     
9:50     
or if it falls apart if it results in a viable embryo versus an inviable because     
9:57     
remember all these developmental cells that generate the sunage code go on to differentiate and if we knock out     
10:04     
certain cells we can say because in C elegance and also I think in Siona those     
10:10     
original cells have to be completely there in order for the next stages of     
10:15     
embryogenesis and development to occur so if those cells are missing then you     
10:20     
have no chance to recover those now in mammals that works quite differently you     
10:26     
get plasticity you get cells that divide in different ways and differentiate in     
10:32     
different ways so they can fill in the gaps but in our what we call uh     
10:39     
our mosaic uh development mode this is not the case so these codes     
10:45     
are very useful for this reason so this is uh table S4 is for se test analysis     
10:53     
you can see it has a nomenclature and the nomenclature is um it's based on not     
10:59     
really sure what it's based on can't remember but it's you know it's it's a little bit unwieldy um unless you really     
11:06     
know what you're doing the point is is that we can actually create a code for     
11:12     
this and if I were going to do this again I think we'd probably add some     
11:17     
other geometric cues to this or you could even combine lineage code or     
11:23     
differentiation code with different um in from other information so you'd have     
11:29     
a string which is the differentiation code you might have another string which is a geometric code which describes the     
11:36     
sort of the coordinates of the cell and certainly in C elegance we have those data we have the uh we you know we could     
11:45     
have any sort of coordinate system we want uh a lot of this data that we have     
11:50     
are sort of inherent the ukitian uh distances you could have other projections as well other coordinate     
11:57     
systems and of course you can add this code with the coordinate system so you can represent this computationally as a     
12:04     
binary string and a tpple with the coordinates in the tuple and the binary string as this uh location in a tree and     
12:12     
then the location in a volume and so um that's that's what this is and so this     
12:19     
makes it much more uh much simpler computational sort of processing here um     
12:25     
and then we of course have the hamming distance so the hamming distance is the distance from     
12:32     
um it's sort of the distance from the origin so we have a number of hamming     
12:38     
distances calculated here we also have the lineage depth which basically is the     
12:43     
same as the length of the binary code which is not surprising since I already     
12:49     
um told you that this is every time there's a division event we add in     
12:55     
another bit to the string and it's longer and you get more possibilities     
13:08     
if we go to If we go to table five this is actually     
13:14     
for C elegance so now we can see C elegance how this works we have uh     
13:21     
nomenclature on the left which of course we have we recognize AB and its     
13:27     
sublineage and then we have P1 in its sub lineage and all the cells in the     
13:34     
developmental lineage tree and so we we have a few more measures in here we have     
13:39     
of course a Hamming distance a composite Hamming distance and of course a lineage     
13:44     
depth so here we start at AB and P1 so AB is zero P1 is one um and so we start     
13:52     
from a little bit different starting point than in Siona but we can align these I think pretty well um and so we     
13:59     
you know as as you go deeper into the tree as you get division events lineage depth increases the length of this     
14:06     
binary string increases you can calculate a Hamming distance from the origin pointer from any two points in     
14:14     
this addressable system now one of the things we did and I'm not I'm not going to pull up this table is we calculated     
14:20     
something called a cast code and so this is not um something that I think is     
14:26     
useful for hyperraphs which is originally why I'm showing you this data but uh the cast code is basically like     
14:34     
it's like a blast uh sequence analysis where you take two     
14:39     
sequences and you align them and then you calculate a dis distance between them so the cast uh method is where you     
14:47     
take these two differentiation codes or two lineage tree codes and you align them and they might be of different     
14:54     
lengths so you have to uh you know account for the gaps and then you can use an algorithm to calculate the     
15:01     
distance between and so this is another thing we can do with lineage tree data     
15:06     
once every every cell is addressable then we can do a lot of this other stuff     
15:11     
okay so looks like Morgan is here hello Morgan i wanted to go over a     
15:17     
couple of papers uh that are in our queue and I think those are going to be     
15:22     
interesting um for a variety of reasons i'm not going to go too deeply into them but I'm going to go through uh three     
15:29     
papers and I'm going to go through their abstracts and some of their significance to the group     
15:35     
so the fa first paper is up i think this is a review article or it's it's an overview article in e life and this is     
15:43     
something that Susan sent me so I wanted to go over it a little bit more detail this is uh the overview article for this     
15:52     
paper evaluating the transcriptional regulators of arterial gene expression     
15:57     
by a catalog of characterized arterial enhancers so this is a deeply um     
16:03     
molecular paper so um why is it that we're talking about a memor group and     
16:08     
that's because it helps us understand endothelial cells so these are um     
16:14     
basically what determines whether an endothelial cell becomes part of an artery a vein or a capillary so this is     
16:22     
where you get diff cell differentiation in the heart where you get these endothelial cells and they become part     
16:29     
of the morphology they differentiate so a nice and Of course you know uh Susan's     
16:35     
work on epithelial cells we have these epithelial sheets and we're looking at     
16:40     
the physics and the mechanics of the sheet of course that leads to gene expression and that leads to the     
16:47     
integrity of those sheets and in this case we have a different um     
16:53     
you know we have endothelial cells and we're thinking about how those cells in     
16:58     
a say a malian system become differentiated     
17:04     
and so they they kind of talk about the uh blood vessel system so the blood vessels in our bodies are either     
17:10     
arteries veins or capillaries arteries transport blood away from the heart and     
17:16     
veins carry it back and capillaries connect arteries and veins also facilitating the exchange of gases and     
17:22     
nutrients between the blood and different organs the inner layer of every blood vessel is formed by     
17:28     
endothelial cells but what factors determine whether a new endothelial cell     
17:34     
is part of an artery or a vein or a capillary and so this is about the formation of     
17:41     
blood vessels during embryionic development and this is where it begins with the differentiation of meodermal     
17:48     
cells into endothelial cells so we have germ one of the three germ layers and     
17:55     
those cells from the misodermal cell layer basically differentiating endothelial     
18:01     
cells this allows for new blood vessels to be generated through a process known     
18:06     
as vasculogenesis and so there's a citation show 1998 that     
18:12     
talks about vascular genesis and reviews that system subsequently additional blood vessels     
18:18     
arise from pre-existing ones by angioenesis although the first arterial endothelial     
18:24     
cells are produced during vascular genesis studies performed on humans mice     
18:30     
and zebra fish have shown that most arterial endothelial cells originate     
18:35     
from venus endothelial cells and so the differences between arterial     
18:40     
venus and capillary epithelial cells has become the subject of much research with     
18:46     
the substantial changes in gene expression that are required for venus and theio cells to become arterial     
18:52     
endothelial cells have received less attention so this new paper elife uh     
18:58     
here cited Norns at all um     
19:05     
report that they've identified a list of genetic enhancers and transcription factors that have a crucial role in this     
19:11     
process enhancers are non-coding regions of DNA that are involved in regulating     
19:16     
gene expression while transcription factors are proteins that regulate gene expression by binding     
19:22     
to enhancers and This goes through kind of what they do here in the paper um     
19:31     
and they find 41 putitive arterial enhancers three of which which have been     
19:36     
previously studied in vivo subsequent experiments on genetically engineered zebra fish that use green fluorescent     
19:43     
protein so this is a marker that you can see under a fluorescent microscope and it tells you uh sort of its presence in     
19:52     
in the thing that you're trying to look at uh it's it's basically used to report on     
19:57     
gene expression but also the presence or absence of certain types of cells labeling those cells and their nuclei to     
20:05     
local where it could be you know other parts of the cell to localize this GFP     
20:10     
signal and tell you if something is where you need it to be     
20:15     
uh showed that the 16 of the enhancers drove gene expression in blood vessels with 15 specifically driving gene     
20:22     
expression in arteries moreover the researchers identified at least one enhancer for each of the eight     
20:28     
genes further experiments on embryionic mice showed that these enhancers also drove gene expression in articles     
20:35     
so this these experiments were done in both zebra fish and mouse     
20:41     
and so they were able to successfully identify a cohort of enhancers capable     
20:47     
of directing gene expression of arterial endothelial cells     
20:52     
uh then they also explored potential transcription factors that might bind to these enhancers     
20:58     
these transcription factors such as ETS soof and fox are enriched in arterial     
21:04     
endothelial cells although they can also be found in venus endothelial cells the     
21:10     
researchers also found that the transcription factor MEF2 binds to enhancers in a subset of arterial     
21:17     
endothelial cells associated with angiogenesis and then the binding of another     
21:24     
transcription factor or BPJ suggests that notch signaling pathways play a     
21:30     
role in regulating the expression of arterial related genes so this has this     
21:35     
um complex interplay between general developmental mechanisms and     
21:41     
some of these specific genes for this cell type     
21:48     
so this paper provides insight into regulatory mechanisms that define what     
21:53     
they call arterial identity and the identification of various arterial enhancers and transcription     
22:00     
factor binding sites for these enhancers so you know when we think about cell     
22:05     
differentiation we think about a cell and it's differentiating into some state it could     
22:13     
be a neuron it could be a arterial cell and a lot of times in     
22:20     
uh what we call regulative development we have what we call a cell identities     
22:28     
or a tissue identity and that is that the cells aren't diff like unlike in C     
22:33     
elegance the cells aren't deterministic with respect to their fate their cells stem cells and progenitor cells that can     
22:42     
you know basically be moved to a number of different fates it really depends on     
22:47     
their micro environment and where they're located with who their neighbors     
22:52     
so there are all sorts of experiments where people have transplanted stem cells     
22:58     
uh precursor cells into different tissues and those precursor stem cells     
23:04     
those pur potent cells take on the uh identity or the phenotype and the     
23:10     
molecular properties of the surround mutation so this is something that     
23:16     
people are very much interested in from a regenerative medicine perspective how do we get these cells to sort of better     
23:23     
integrate into tissues can we coax them in some way can we deliver some sort of     
23:30     
molecular cocktail can we you know identify cells that are maybe more     
23:36     
integrated or less integrated into their host tissue and you know basically how does this     
23:43     
process work so you notice I've not mentioned anything about mechanics of the cells     
23:49     
and so this is interesting because this sort of work you know they they talk a lot about molecular mechanisms that they     
23:54     
don't really talk about the mechanical aspects of different types of you know     
24:00     
how these cells are in so going back to our example from epithelial cells we     
24:07     
know that these epithelial sheets exist in this sort of mechanical world where     
24:13     
the mechanics are very specific to keeping the cells in place even though they're sort of migrating and moving     
24:20     
against one another constantly uh and they're also triggering a lot of molecular mechanisms there's a lot of     
24:27     
sensory information each cell is providing to its neighbor and there's all sorts of transduction of these     
24:34     
signals and it triggers a lot of uh transcription factor activity and so it's very interesting how you know they     
24:42     
again they don't talk about the molecular as or the the mechanical aspect of it but it's very interesting     
24:49     
to think about studies like this in terms of the mechanics of the tissue     
24:54     
that we're looking so this looks like I mean I didn't want to get into the full paper because this     
25:01     
is something that you know is probably uh beyond the understanding of the group and maybe my understanding of some of     
25:08     
these techniques this next paper is something from the bioarchchive this is an older paper but     
25:14     
something that I found in kind of researching things and uh this is     
25:22     
talking about the development of olive oils and so this is an idea that goes back um in developmental biology and an     
25:30     
evo uh to this notion of there being a sort of a biotypic stage of development     
25:38     
so this is uh not in segans but this is in this regul regulative developmental     
25:44     
mode where you have a lot of variation before the photypic stage then you have     
25:49     
the phyotypic stage which is very regularized and then you have the post-photypic stage where embryos will     
25:56     
diversify into their different morphologies so basically you have this     
26:02     
stage where the mechanisms of embryogenesis are very stereotyped and very similar and that's kind of in the     
26:10     
middle of the process hence the term hourglass so if we know what an hourglass is we have this top chamber     
26:18     
and this bottom chamber and they're connected by this narrow passage and usually sand goes through the hourglass     
26:24     
and the sand is sifted through this narrow chamber and that's the photypic period is this narrow chamber     
26:31     
so in this paper they're actually thinking about this hourglass which is a metaphor that people use in development     
26:40     
but instead of just thinking about it as a metaphor they actually do a computational model and things like that     
26:47     
so they use dynamical systems theory to get at this point determining the general laws between evolution and     
26:53     
development is a fundamental biological challenge developmental hourglasses have     
26:59     
attracted increased attention as candidates for such laws but the necessity of their emergence remains     
27:06     
elusive we conducted evolutionary simulations of developmental processes to confirm the     
27:12     
emergence of the developmental hourglass and unveiled its establishment     
27:19     
we considered organisms consisting of cells containing identical gene networks     
27:24     
that control morphogenesis and then evolve them under selection pressure to produce more cell types by     
27:32     
computing the similarity between the spatial patterns of gene expression of two species that evolve from a common     
27:38     
ancestor a developmental hourglass was observed so this is where they're looking at the     
27:45     
spatial patterns of gene expression across the embryo they're looking at different cells uh from anterior to     
27:52     
posterior end of the embryo they're looking at these gene expression patterns     
27:58     
and they're looking across fogyny so they're looking at two species separated     
28:04     
by phoggenetic distance but having a common ancestor and so we can see this     
28:11     
developmental hourglass which is defined as a correlation peak in the middle stage of development     
28:18     
so what does that mean that means that in the middle stage of development there's this correlation between the two embryos whereas earlier and later in     
28:27     
development we don't see this correlation we can assume that in the earlier and     
28:32     
later stages of development that there's been divergence biogenetic divergence but in that middle stage things are     
28:39     
essentially conserved and so this is perhaps important for uh in terms of     
28:45     
regulating different physiological processes and so forth the fraction of     
28:51     
pleotropic genes increased whereas the variance in individuals decreased so you     
28:56     
actually start to get these pleotropic genes turn on which means that you have genes that have multiple effects you see     
29:03     
pleotropy in their outputs so you get you get basically you're getting sort of     
29:10     
transitioning to this later stage development where you really get this uh difference in phenotypes     
29:18     
and but you also see a variance in individuals decrease and this is consistent with previous experimental     
29:24     
reports reduction of the unavoidable variance by initial or developmental noise essential     
29:31     
for survival was achieved up to the hourglass bottleneck stage followed by     
29:36     
diversic diversification and developmental processes whose timing is controlled by the slow expression     
29:42     
dynamics conserved among organisms sharing     
29:48     
so the study suggests why developmental hourglasses are observed within a certain phoggenetic range of species so     
29:56     
you see these hourglasses within certain plates within certain parts of the tree     
30:03     
of life and so it's very uh they don't really give you a lot of insight from     
30:08     
the abstract but it basically gives you this uh     
30:14     
I guess in this case we're reviewing sort of what a development hourglass is and what it produces so     
30:21     
the validity of the developmental hourglass model was first noted based on the coincidence between the photypic     
30:27     
stage of vertebrates and the period of temporal and spatial progression of the hawk genes it's in these developmental     
30:35     
systems you have hawk genes which define segments of the organism which show up     
30:41     
in terms of striping pattern you see this in versophila you have very famous     
30:48     
oxygen experiments in versa showing that there are these segments of embryogenesis     
30:54     
where you can define these different sort of stripes of phenotypic differentiation     
31:00     
there's the spatial progression so it's spatially explicit and then there's this photypic stage of     
31:07     
vertebrates so this this stage where there's this sort of typicality     
31:14     
and so there's this connection between the two and that's the idea behind the development     
31:20     
so this has been characterized various taxa by bioinformatics analysis over the     
31:26     
past two decades we see this for cordates for ainoderms arthropods for     
31:32     
nematodes even for mollisks and also suggested beyond the animal kingdom for     
31:38     
flowering plants for fungus so you see this across development you see this across the tree     
31:44     
of life you see this in different uh parts of different uh taxonomic groups     
31:52     
and so there are all these views on this one may state that the observed data     
31:58     
supported developmental hourglass at least in some filing so there's variation on this interpretation of the     
32:05     
data despite the growing evidence for the developmental hourglass model whether it's just a contingency in     
32:11     
evolution of multisellular organisms or a necessity therein is still in question     
32:18     
in other words the possible mechanisms underlying the evolutionary acquisition of mid embryionic conservation remain     
32:25     
elusive recent experimental studies have concluded that the development of autoglass is not sufficiently explained     
32:32     
by fragility in the bottleneck stage but rather developmental constraints or limitations and phenotypic variability     
32:40     
are suggested to be the main driving force in the emergence of the developmental hourglass     
32:47     
relevance of self-organizing processes to the formation of the hourglass was discussed in this citation 26 whereas     
32:55     
there's no consensus on the mechanism for developmental processes so this is a     
33:02     
schematic uh in their simulation so what they did is they took development and they used a simulation an abstraction of     
33:09     
this so uh in this case we have we go from A to B to C     
33:17     
in a which is here we basically have each individual which is a unit of     
33:23     
selection contains cells aligned in a one-dimensional space these contain the     
33:28     
same gene network the state of each cell given by the gene expression pattern     
33:34     
changes according to identical gene networks there are 100 individuals in the evolving population each generation     
33:41     
so they have this sort of populationbased model in A in B starting from the initial states     
33:48     
the gene expression patterns change according to the gene regulation after sufficient time the expression of     
33:55     
each cell reaches the terminal state on or off cell type was determined     
34:00     
according to the onoff patterns of the target genes for the four target genes there were two to the fourth possible     
34:07     
states so here we have this initial state and then development and we have     
34:13     
these different states that are colored and then we have the terminal state so basically we go from the initial state     
34:19     
to the terminal state and there can be this sort of up to the terminal state     
34:24     
and then C which is here so it's this uh you know simulating over generations the     
34:32     
selection procedure of species diverged from the common ancestor and so the left chain represents the     
34:39     
results of the first step of the evolutionary simulation to select one species so we have this first step where     
34:45     
we have evaluation at every 200 generations     
34:51     
and then the second step is where an ancestor at a certain generation was     
34:56     
cloned and additional evolutionary simulations were carried from the ancestor to the same generations of the     
35:02     
original so they took every 200 generations they took actually it starts     
35:08     
at 400 generations here they took a clone and evolved it alongside the     
35:13     
original so they were doing this sort of clonal evolutionary simulation and so     
35:19     
they conducted this evolutionary simulation to investigate the mechanism behind the establishment of this     
35:25     
developmental hourglass they adopted a dynamical system scheme to separate spatial patterns which is     
35:33     
commonly adopted in theoretical studies the model for the simulation can represent cellular states that are     
35:39     
determined by gene expression patterns governed by gene regulatory networks and     
35:45     
spatial cell cell interactions leading to the differentiation of cell types and pattern formation     
35:52     
so this is where we you know we want to model the sort of diversity of things     
35:57     
that allow for differentiation it's basically you can have onestep     
36:02     
differentiation multi-step differentiation but there are a lot of things that contribute to each type each     
36:10     
uh stage of differentiation it's you know it's not just simply flipping a switch although that's often     
36:17     
the way it's sort of portrayed in the literature you have all these things kind of leading to     
36:24     
individuals must be selected according to a biologically relevant fitness function we assume that if there are     
36:31     
more cell types within an individual they attain more functions such as the     
36:36     
fitness is higher the fitness function reflects the idea that different cell types have different cellular functions     
36:43     
so that species with many cell types can deal with multiple tasks hence they are more adaptive than those fewer cell     
36:50     
types each cell type was defined according to the on or off state of the     
36:56     
given in output genes in other words our simulation mimics competition for survival where individuals acquiring as     
37:03     
many cell types as possible are the most adaptive and would leave more offspring     
37:09     
so there this is an interesting kind of developmental simulation that that's driven by this sort of evolutionary     
37:15     
simulation um and it's really interesting the way they set this up now     
37:21     
to study the developmental hourglass the developmental processes of extent species that diverge from a common     
37:28     
ancestor must be compared so we have this sort of comparison     
37:34     
between two different systems thus the model must represent not only a motherto     
37:39     
single chain pedigree but also must be able to examine a phoggenetic tree that contains multiple     
37:46     
pedigrees as branches so this is why they did this evolutionary simulation in     
37:51     
figure 1 C they wanted to take basically from 0 to 400 generations you have this     
37:58     
common ancestral state and then at the second step you start to get these clones that that also evolve in parallel     
38:06     
and so you end up with these things you can compare your system to so they did this two-stage evolutionary simulation     
38:14     
in the first stage uh simulation of one lineage was performed to obtain one     
38:19     
species by selection in the second stage evolutionary simulations were executed     
38:24     
over the same number of generations using an ancestor in the lineage of the first step simulation     
38:30     
using different random numbers for mutation extant species branching from the common ancestor routine     
38:37     
and so then using the above setup we aim to examine the correlation between gene expression patterns and the development     
38:44     
of multiple species that diverged from a common ancestor the results showed a     
38:49     
developmental hourglass in the sense that the peak of such correlations lie in the middle of development we also     
38:55     
confirmed other reported characteristics including accumulation of expressed pleotropic genes and reduction of     
39:02     
intraspecies expression variance at the developmental bottleneck stage the     
39:08     
developmental period of the hourglass bottleneck contained in the simulations is determined by the genes whose     
39:14     
expression change slowly during development acting as genes to schedule them so this     
39:20     
is this scheduling process uh where You get this change in gene     
39:26     
expression over development and there's a sort of schedule to the bottom like     
39:32     
serves as the developmental foundation that sufficiently attenuates the effects of initial noise to prepare for     
39:38     
subsequent development processes based on these results the validity of developmental hourglasses and possible     
39:45     
mechanisms for its emergence were discussed so they use a what they call a GRN or     
39:52     
gene regulatory network model where you have this network of genes and they have this uh signal that you know     
40:01     
kicks off the cascade and you have these different genes that upregulated or downregulated and then affects the next     
40:08     
partner in that network and you continue through the network until you get these products that are the outputs of the     
40:15     
network and then the products of that network diffuse through case     
40:20     
so you have this sort of uh I guess you could call it a spatial GRM that they're using     
40:26     
and so then they run this simulation until they reach a fixed terminal state     
40:32     
after sufficient developmental time where the expression of each gene in the network has reached an on or off state     
40:40     
depending on the combination of onoff expression patterns of the given output genes a cell type is determined     
40:48     
and so in figure 2A they show specific examples of the developmental dynamics of two individuals acquired from     
40:54     
evolutionary simulations after 10,00 generations     
41:00     
and so this is in figure two this shows examples of the evolutionary simulation that show our glass-like developmental     
41:06     
correlation and so these are very colorful graphs you have A1 A2 and then B and C     
41:15     
so in A we have A1 and A2 so these are two examples of gene expression dynamics     
41:21     
of evolved genetic regulatory networks so this is one genetic regulatory network this is the second so this is     
41:28     
developmental time from 10 to 1,00 and then this is the cellular index which     
41:34     
they define in the paper and then this shows sort of a phase portrait of that     
41:40     
this is different for these two different examples this is presuming I guess two different species and how     
41:46     
those behave those tax with foggy distance the idea here is that we have     
41:52     
this these spatial pattern transitions that we need to identify we can see that     
41:58     
they these things uh change uh they're more frequent in the early     
42:03     
developmental time periods this is consistent with experimental observations thereafter the     
42:09     
developmental time was plotted in models following evolving 800 generations the     
42:15     
two GRNs evolved from a common ancestor of the 800th generation and then     
42:20     
simulated further 200 more generations while suffering from different genetic changes such as that the two have     
42:26     
diverged from a common ancestor so that's the first one and then of     
42:33     
course we have so this is actually let's see B is here so this is where we have     
42:39     
developmental time of A2 so this is A2 here and this is developmental time of     
42:44     
A1 on this axis this is A1 here so basically we're comparing A1 and A2 what     
42:50     
we're doing is with a heat map so we can see that there's this sort of distinction between A1 and A2 so this is     
42:58     
uh figure B the heat map of the similarity indicator comparing the developmental dynamics of the two     
43:03     
individuals in A as you know we have so we have developmental time of A1 and     
43:10     
developmental time of A2 both axes are in a logarithmic scale each point indicates the similarity of gene     
43:17     
expression states between the two examples at specific time points as the     
43:22     
time scale of development can be different between the two in general the comparison at different time points is     
43:28     
included and so the highest level of similarity appears to be in the middle stage which     
43:34     
is this area of the heat map here and the green line indicates locations     
43:40     
where the similarity indicator takes its maximum for each developmental time step of individual ones this is here this     
43:47     
green line so I think I said that maybe the green line was something else that's not true     
43:54     
um this is just each point in this simulation so this entire phase portals     
44:02     
is basically the comparison okay now C of course then is the projection of the     
44:07     
similarity indicator of the green line in B against the developmental time of the first species so this is the green     
44:15     
line here and then there's this developmental time for A1 and we have developmental time for A1 here and then     
44:21     
this green line restated as this uh developmental correlation so you can see     
44:27     
that there's a correlation here in the middle uh period the photypic stage and     
44:32     
then down here we have very low similarity later because things are diverging those two uh species are     
44:40     
diverging away and and before the photypic stage we have some a little bit     
44:45     
higher level correlation um but not much so the photypic stage is     
44:51     
where you have this enhanced correlation between two species     
44:58     
okay this looks like you know there are a lot of other results in here and I'm not going to get into them there's a lot     
45:04     
of cool simulation in here and looking at the results sometimes these simulation results are hard to interpret     
45:11     
because you have to live in the world with the simulation and now it's gone so     
45:16     
um and then this just shows like the steps in the evolutionary simulation and     
45:21     
again more information about developmental correlation so basically they're playing around with um you know     
45:29     
different ways to do this um but you see again this developmental correlation in the middle of the simulation last paper     
45:36     
I'm going to talk about here is this paper uh by Michael Leaven and     
45:42     
colleagues um in the Saturday morning neurosim meeting that we had on Saturday     
45:47     
from the orthogonal research and education lab uh we talked about another Michael Leven papers actually from I     
45:55     
think a lot of the same authors on uh these biobots or these anthrobots and     
46:02     
their transcriptional profiles and things like that um is an interesting paper this is a     
46:08     
different paper but it's very similar um is this the same paper no it's not a     
46:14     
different paper um this is the running title is biobots with neurons so this is     
46:20     
where we have we're looking at these biobots instead of anthrobots we're     
46:25     
looking at you know these basically these organoids with neurons and uh     
46:32     
they're using these bio-engineering techniques to generate these real organisms or I guess they're not     
46:39     
real organisms but they're biologically they're biological in the sense that     
46:45     
they're cells but they're also they're constructs I would say is     
46:50     
probably the best way to put it so this is self-organizing neural networks and novel moving bodies     
46:57     
anatomical behavioral transcription characteristics of a living construct with a nervous system     
47:08     
and so if we look at the abstract um they mentioned a great deal is known     
47:14     
about the formation and architecture of biological neural networks and animal models which have arrived at their     
47:20     
current structure function relationship through evolution by natural selection little is known about the development of     
47:27     
such structure function relationships in a scenario where neurons are allowed to grow within evolutionarily novel motile     
47:34     
bodies previously previous work showed that when a piece of ectoermal tissue is     
47:40     
excised from xenopus embryos allowed to develop xvivo will develop into a     
47:45     
three-dimensional eucosiliary organoid and exhibit behaviors different from     
47:50     
those observed in tadpoles of the same age these biological robots or biobots for     
47:58     
short are autonomous self-powered and able to move through aquous environments     
48:05     
here we report a novel type of biobot that is composed of siliary siliated epidermis and additionally incorporates     
48:12     
neural tissues these are neurobots we show that neural precursor cells implanted with within the zenipus skin     
48:19     
constructs this is a frog model embryo they have these skin constructs so what     
48:24     
they're doing is they're putting neural precursor cells into a tissue uh skin     
48:30     
and it's allows these or or they're able to get these uh neural precursors to     
48:35     
develop in the mature neurons and extend processes towards the outer surface of the body as well as among each other so     
48:43     
what they're doing here we talked about this in the first paper uh when you put tiss when you put different precursor     
48:49     
cells in tissue it responds to the cues local cues of that tissue so we could     
48:55     
very well have um you know we might not be able to achieve     
49:01     
um you know full fully mature neurons in a skin construct it takes a lot of     
49:08     
guidance in terms of you know how these cells are implanted and how these cells     
49:13     
are able to develop and so this is where we're trying to develop not just a single     
49:20     
tissue but a compound tissue that resembles an organism so we have neural precursor cells in a skin construct we     
49:29     
are able to get mature neurons we're able to get uh processes extended to the     
49:34     
outer surface so we have the system where we have a a layer of scan of sensory cells     
49:42     
uh we show that neurop precursor cells implanted okay these self-organized     
49:48     
neurobots show distinct external morphology generate more complex patterns of spontaneous movement and are     
49:55     
differentially affected by neuroactive drugs compared to their non-moral     
50:00     
lcm imaging experiments show that neurons within the neurobots are indeed active transcrytoic analysis of the     
50:07     
neurobots reveals increased variability of transcriptional profiles expression     
50:12     
of a plethora of genes relating to nervous system development and function the shift towards more ancient genes and     
50:19     
upregulation of normal genes implicated in visual perception     
50:25     
now we talked about the accompanying paper on Saturday so I refer you to that     
50:31     
discussion but basically they found the same types of things so it was a different experiment but they found     
50:37     
similar things um you know you get with uh with these neurobots you see of     
50:43     
course increased variability of transcriptional profiles so this is where we don't necessarily have a     
50:49     
photypic stage here but we're trying to construct uh different types of complex tissues     
50:57     
from uh basically from scratch we're we're trying to work we're trying to uh     
51:04     
engineer them and so we get this sort of incre variable transcriptional profiles     
51:10     
would suggest that you have this differentiation of cells that are mature and that are stable um although it isn't     
51:19     
always stable if you have a variable transcriptional profile but that's what kind of suggests we get the expression     
51:27     
of genes related to nervous system development function which means that it indeed is that these neurons are indeed     
51:35     
uh differentiating down a neural pathway and not some other pathway and that they     
51:40     
they are mixed in with like skin uh related genes and things like that     
51:46     
a shift towards more ancient genes which is interesting because I think in the last paper on Saturday we mentioned     
51:52     
there was a followup item which is they talked about like sort of phoggenetic origins of some of these programs so     
52:00     
like you know this shift towards more ancient genes is basically uh you know     
52:07     
you get these I don't really know what they mean exactly but there's this connection to     
52:13     
phoggenetics that's really interesting and upregulation of neuronal genes     
52:19     
implicated in visual perception which I'm not really sure why that would be but there may be that there are these uh     
52:26     
genes for perception you know in sensory neurons that are maybe you know relevant     
52:33     
but in in any case it's a very interesting set of experiments um they're they're trying to build these     
52:39     
bots from biological tissues they're trying to construct these complex biological tissues and they're having     
52:47     
some of the same problems that they're you see in development so it's quazi development really but you have a lot of     
52:53     
the same issues okay so this maybe explains more um sens     
52:59     
sensing cues from the environment and translating them into appropriate responses is the fundamental function of     
53:05     
the nerve system in all animals critically nervous systems endow animals     
53:10     
with the ability to generate context and experience dependent changes in their behavior so this is where we have you     
53:18     
know plasticity based on experience dependence or context dependence     
53:23     
basically when you put cells you put sensory systems into a new environment     
53:29     
they change how they're how they work and so this and they they exhibit plasticity so this is uh kind of at the     
53:37     
heart of what we're trying to do with these biobots they're trying to uh figure out what the limits of plasticity     
53:44     
are so nervous systems are also known to be plastic and adapt structurally and     
53:50     
functionally on a much shorter time scale to changes in sensory and our motor factors that might occur in the     
53:56     
lifetime of the organism say for example as a result of injury amputation or     
54:02     
sensory deprivation the degree to which the nervous system of an animal can adapt to a new body     
54:09     
plan is especially impressive in cases where changes in sensory motor architecture are particularly drastic so     
54:17     
the ectopically induced eyes in the tail of xenopus tadpoles were shown to confer light sensitivity to the iOS hosts so     
54:24     
this is an example of where you have sensory uh cells you have these eyes     
54:30     
that are uh induced in the tail of the zenopus this is not a place where you usually see eyes um but they can     
54:38     
actually confer light sensitivity so they basically function as photo     
54:43     
receptors just like photo receptors in the eye but they don't really give any     
54:49     
visual input to the brain they just give this ability to sense light and so you     
54:54     
might have this pathway that's constructed from that where you get light sensitivity in different parts of     
55:00     
the body her pineal gland in humans is like that where you have the pineal     
55:06     
gland is a was at some point in evolution a light sensing organ and then in humans     
55:13     
it's sort of evolved to be this pulsatile uh organ for     
55:22     
hormones so what are the limits of neuroplasticity     
55:28     
of nervous systems how would wild type neurons establish pattern and function in a new embodyment     
55:34     
so basically we're taking neurons we're putting them in a new embodiment in this     
55:40     
case a layer of skin cells and we're getting them to behave in a certain way will they behave in the way we intend     
55:47     
and so we want to create these truly novel configurations of biological materials which allow us to probe the     
55:54     
plasticity of evolutionary hardware to adapt on developmental but not evolutionary time scales to truly novel     
56:02     
circumstances which have applications for regenerative medicine human augmentation and biological engineering     
56:10     
we sought to establish a model system in which we could study the morphology and function of neural networks in novel     
56:16     
scenarios better understand the evolutionary developmental biology of nervous systems and improve the design     
56:23     
of future innervated constructs meaning that you have cells in all your cells     
56:29     
and that they're nerves that come out of that and through the whole construct     
56:36     
so they point to sort of the history of this where you have neuromuscular     
56:41     
assemblids that have been used to model corticospinal motor functions and neural     
56:46     
cultures that have interfaced with computers or robots using microelerode arrays to learn or perform various tasks     
56:54     
and these are of course hybrid robots that were not built exclusively using bio biological tissues not fully     
57:02     
embodied not self mot so there have been people who have done these kind of biohybrid robots and other types of     
57:10     
hybrid robots and they the challenge here is to improve     
57:18     
biohybrid bots that move through neuromuscular actuation have also been built by embedding neural and muscle     
57:25     
cells in silicon polymer scaffolds similarly such bots do not self assemble     
57:31     
require external stimulation to propulsion and only move at very slow speeds have a very limited behavioral     
57:38     
repertoire and still require synthetic scaffolds and so what they have here is they have     
57:45     
tissue ectomal tissue that's been excised from the animal pole the late stage blast joinus     
57:52     
embryo so we have sort of what we would call or early development before the     
57:57     
photypic stage um and we're taking ectoermal tissue from the ectomal layer     
58:02     
using that as a way to model self motile self-cor 3D muscular muscular     
58:12     
and so yeah we basically have these four cell types that are normally present in     
58:18     
attachable skin multisiliated cells mucus secretreting goblet cells ionic     
58:25     
sites that regulate ionic homeostasis of epidermis and small secrettory cells so     
58:32     
these multisiliated cells can act as motor factors in these organoids and can     
58:38     
sense flow and this arises from their polarized beating and this can generate     
58:44     
a suite of stereotype movement trajectories and velocities interestingly uh small secrettoryy cells     
58:51     
were shown to secrete serotonin which stimulates an increase in silarity beating frequency through serotoninergic     
58:58     
receptors expressed on these multisiliated cells these self motile mucioy organoids which     
59:06     
we refer to here in as biobots are capable of navigating aquous environments generating a suite of     
59:12     
stereotype moving trajectories and velocities so this is really interesting they're     
59:18     
using this sort of system they're uh it's sort of a a derivative of xenopus     
59:23     
embryos and they're able to produce these four cell types in the skin but     
59:28     
more importantly they're able to establish the sort of uh sensory motor system very primitive sensory motor     
59:35     
system where you have an embodied sensory motor system and then you can     
59:41     
introduce um you can introduce neurons into the mix     
59:47     
and create neurobots     
59:52     
so they have some really interesting results here um not going to get into     
59:57     
those but yeah I'd like to point people to that discussion on on the research     
1:00:05     
and education laboratory at last Saturday meeting where we talk about the other paper from this group on     
1:00:13     
anthropots     
1:00:18     
are you available I don't think you're available to comment on this but it' be kind of interesting to hear from     
1:00:25     
I'm I'm afraid I have been multitasking but um but you know diving into the the     
1:00:33     
the kind of literature review that we're doing for um uh 3D bioprinting and you     
1:00:41     
know the the work that came up in terms of uh what Dev Gartner is calling 4D     
1:00:48     
bioprinting right so it's it's three 3D plus devel velopment right and um uh     
1:00:57     
it's just made me want to dive back into the the Anthrobot literature and or you     
1:01:05     
know the the papers around that work and the again the kind of um mechanisms and     
1:01:13     
uh um you know the the the     
1:01:22     
kind of like the the the physics that's important in this in in these in these     
1:01:28     
considerations or you know the the what's what's driving     
1:01:34     
um the cells multisellular organization right so     
1:01:41     
around the kind of thing that you posted on um uh the the engineering for for life um     
1:01:50     
what was the Dresden meeting that you posted yeah it's like the mechanics of life or something like that it's     
1:01:56     
something like that right like like Yeah yeah and and again there's there's you     
1:02:04     
know the the kind of stuff that comes up i mean I didn't necessarily see the like     
1:02:12     
the Morpheus group in that but but you know Dresden has that Morpheus project     
1:02:19     
where you know again like like tissue organization um especially uh in in early stages     
1:02:28     
being driven by you know cell contact and     
1:02:35     
you know these these these kinds of things that only make sense when you     
1:02:40     
actually do the simulations of of you know how how you can Yeah how     
1:02:48     
much of that self-organization is driven by cell adhesion differences for     
1:02:54     
instance right um and uh     
1:03:02     
yeah so it's um uh Yeah great great stuff and and you know     
1:03:10     
just wanted to um see if you had a chance to look at that um the video from Aras     
1:03:16     
um is just another nice really um you know     
1:03:23     
a researcher giving a a wonderful overview of of development okay that     
1:03:28     
that Barcelona guy oh yeah yeah anyway but     
1:03:36     
Um great stuff and you know I I'll try and review better when I'm you know but     
1:03:42     
it's not uh uh it's school's school's over at this point but um but there's     
1:03:48     
still plenty of family activities at this time     
1:03:53     
all right thanks for showing up and and listening in um yeah so that's all for     
1:04:01     
today um thanks for attending twitter take care
