## TRANSCRIPT    
0:00     
hello Bradley hello how are you um pretty good     
0:07     
i'm It's warm here now so I'm Feels like     
0:12     
summer oh May 5th 55 yeah people were sending me notices     
0:20     
yesterday may May 4th be with you     
0:28     
yeah yeah quite a few people yeah I know     
0:40     
all right any updates uh things going on     
0:45     
with I'm taking Blender from you Demi um you can actually do optical coherence     
0:52     
tomography with um Blender I think     
0:58     
um I don't know if it's just to illustrate it or or what but I've I've     
1:05     
got a paper anyway so I'm doing     
1:13     
that and then I I'm taking a Python course or started to because it said it     
1:20     
would teach me machine learning and get me into AI and it's just like take our     
1:27     
course we'll show you what you do with it you know with statistics and with machine learning so I went okay     
1:36     
um I need to understand Python better because I     
1:42     
know it it's very similar to mat lab but I might as well like I've never taken     
1:47     
that language so I might as well add that to the     
1:55     
managerie yeah they didn't stick with Pascal they     
2:00     
changed it to python I don't even know if they did     
2:05     
[Music] Anyway I just I just know about mat lab and there's still     
2:13     
um the programming in mat lab that that does tense integrity that I have to     
2:19     
investigate but I have been trying to clean my house yeah literally or figuratively     
2:29     
um it's literally clean my house yeah like my house has u five stories sort of     
2:38     
oh wow five levels cuz we added a small house onto a slightly larger house     
2:46     
so it's all uneven and the part that's up over that way always gets um     
2:54     
neglected shall we say so I've been trying to clear that     
3:04     
right yeah well Morgan     
3:14     
um yeah so I guess let's get started um so yeah all right uh let me start     
3:24     
with a few things uh a couple words about the Google Summer of Code projects     
3:29     
coming up um that's going to be I guess the decisions are going to be made this     
3:35     
week so I don't know how many slots we'll get uh we'll see so I guess it's like the ETH or something um so good     
3:43     
luck to everyone in that um I got a way with put an article in the Slack     
3:52     
in the open room Slack we'll talk about that and then we'll talk about another     
3:58     
article that I found interesting so okay I'll shut off     
4:03     
my camera and stuff because it a bandwidth okay issues sometimes happen yeah no     
4:13     
problem okay     
4:18     
so this is this article connomics 2.0 uh this was from the     
4:26     
transmitter so in the Saturday morning neurosim group we talk about stuff from     
4:31     
the transmitter all the time and this is a nice uh popular     
4:39     
uh source for neuroscience and and computational neuroscience even and they     
4:45     
have a lot of good articles so this article is on contoics 2.0 simulating     
4:50     
the brain and it's on the fly conneto so we've talked about the fly conneto we've     
4:57     
talked about C elegance conneto we know all the cells in C elegance conneto we     
5:03     
know their connections and you know we have a number of different connetos we have the     
5:12     
um the gap junction base conneto we have the synaptic conneto we have the male     
5:19     
and hermaphrodite contos and then we have various laral connecttos which     
5:24     
actually varies quite a bit in terms of the connectivity during the laral stages     
5:30     
that you see in sea elements so even with a very small conto you have a lot of     
5:36     
diversity and um you know it's it's it's complicated     
5:41     
to sort of get your head around but with the fly with neur drosophila we're     
5:47     
dealing with a much larger brain and of course drosophila is much smaller than     
5:52     
say like the mamalian brain you know even other types of nervous systems and     
6:01     
So this is uh where they're trying to basically get a segance level uh     
6:09     
resolution to the Drosophila brain so uh Drosophila conneto     
6:16     
uh which we don't know all the details of this is this article talks about all the articles that have been published in     
6:22     
the last maybe 10 years on elucidating this conneto both you know in terms of what     
6:29     
the connections are and what their linkages to behavior so it's also important to get you know the cells and     
6:37     
and connections right but it's also important to see how that translates into behavior so     
6:43     
um they've done this in parts you'll see that they've done this in different regions of the fly brain and they've     
6:50     
released different regions of the fly brain with high resolution imaging as you can see here mapping that     
6:59     
to where it is in the fl's body and then how that relates to behavior and so this     
7:06     
is an example of some of the imaging they've done where they have you know the the nerves the cells and then the     
7:15     
receptors on the surface and all of that uh so the subheading here is with a     
7:23     
complete fly conneto in hand researchers are taking the next step to model how     
7:29     
the brain circuits fuel function okay so this is     
7:34     
uh uh if you look at the subtitle of this figure forman function using     
7:41     
simulations based on connectivity maps so they're also simulating this uh     
7:46     
connectivity map as they're building it and they're simulating its action to     
7:51     
give them some indication of the behavior because you know you can measure some of the behavior from     
7:58     
uh some of the neurons but it's hard to get good measurements of a of a conneto     
8:05     
um so we also use simulations researchers are exploring how much they can learn about a circuit's functions     
8:12     
from its connections alone okay so they talk about in 2012     
8:18     
neuroscientists Sebastian Sun and J anthony uh Mauvshin squared off at a     
8:23     
Columbia University event over the usefulness of connetos so Sebastian Sun     
8:29     
has been a salesperson for this idea of connecttoics and how that's going to revolutionize I mean it kind of already     
8:35     
has revolutionized neuroscience but at that point you know he he wrote a book     
8:41     
on it and he was really trying to sell it as this indispensable tool um so how     
8:49     
useful are contos uh and so such a map soon argued crack     
8:54     
open the brain's computations and provide insights into processes such as     
8:59     
sensory perception and memory so you know we have a lot of different concepts     
9:05     
and tools you know at one point people thought that you had little part portions of the brain     
9:12     
uh that had different functions and they were compartmentalized uh you know later people had this idea     
9:20     
that you know there were these uh pathways I mean there's we still have pathways but networks have become more     
9:28     
common in the last maybe uh 20 years and of course having the conneto as a     
9:34     
network and understanding that network um is you know that that's really the     
9:40     
key to understanding the brain uh but Mauvshin a professor of neural     
9:46     
science and psychology at New York University countered that the relationship between structure and     
9:51     
function was not so straightforward that even if you knew how all the brain's neurons connect to one another you still     
9:59     
wouldn't understand how the organ turns electrical signals into cognition and behavior so there was a debate there     
10:05     
between people who were at least one person who thought that the conneto     
10:11     
would be sort of the Rosetta stone of neuroscience and then this other person     
10:17     
who thinks that well having a conneto is nice but it's not going to tell you a     
10:23     
lot about cognition and behavior i mean it may tell you something but you're still left with a gap and I think you     
10:30     
know it's I guess you know you could say obviously the latter person here was     
10:36     
correct because we can't understand everything about cognition behavior but     
10:41     
you know we see from the C elegance conneto we've been able to map the conneto we've been able to simulate the     
10:48     
conneto and because it's fairly simple we can grasp sort of the the beh well we     
10:55     
have behavioral circuits like the movement circuit and we have different parts of     
11:01     
the conneto that we know do things and so you know that it gives us a pretty good idea of what kinds of behaviors can     
11:09     
be generated on the other hand when you get to larger connectto homes you start to have this problem where there's a lot     
11:16     
of sort of activity that you can't classify was activity that we don't know     
11:23     
what exactly it does it's not a onetoone relationship so there's a lot to understand about this i guess this     
11:29     
debate is kind of confounded by the fact that you know we're very we're in very     
11:35     
early days of contoics so uh we're not going to understand a lot either way     
11:41     
just by mapping the conneum um the debate in the field continues even     
11:47     
though Sim and his colleague in the FlyWire Consortium completed the first connect of a female Drosophila     
11:53     
melanagaster in 2023 so they actually were able to produce these     
11:59     
connetos uh they they have this consortium called the flywire consortium     
12:05     
where they kind of have uh I think it's been working with Janellia Farm and some other uh collaborators to put these maps     
12:14     
together and then of course build computational models from those maps to     
12:19     
simulate behavior so um in fact doing all of this work doing     
12:26     
the the modeling doing the mapping that actually has revealed a lot about     
12:33     
function and so we're actually learning quite a bit about function not just about the anatomical map uh this quote     
12:41     
here all the work that's been done in the past year or two feels like the beginning of something new     
12:46     
um and so you know they kind of uh talk about some of these papers these     
12:53     
papers are kicking off a beginning to an entirely new field which is contoic     
12:59     
based brain simulation so you know we have people have tried to do brain     
13:05     
simulations where you know they have well we have neural networks which     
13:11     
aren't particularly biologically um precise but they do simulate things     
13:18     
that we think are happening in the brain or brain-like processes they're     
13:23     
producing things that are intelligent like behavior that sort of thing so we have that kind of uh level of simulation     
13:32     
and so what they're doing here is they're actually able to simulate something that we know is an actual     
13:38     
brain we know what you know kind of how that operates then if we can simulate     
13:44     
that network and produce outputs and they look like outputs that we would see     
13:49     
in the actual fly then we can say okay yeah we're actually getting somewhere with this under understanding what these     
13:55     
connetos do why are things connected like this why are things segregated into     
14:01     
functional modules why do you have different structures in the brain instead of just a mass of neurons i mean     
14:08     
we have neuropil which are these masses of neurons but you know why why do we     
14:13     
have it why is it organized like that it's not organized like that in mammals it's not even organized like that sea     
14:19     
elegance so why why do we have this difference and so that I think this provides sort of an a start to     
14:26     
understanding uh a simulated fruitfly optic lobe     
14:32     
detailed in the September 2024 Nature paper i think we talked about that in the Saturday morning neurosin meeting     
14:40     
soon after it came out they basically uh mapped out the fly optic lobe they     
14:47     
were able to simulate it and they published a huge paper on that whole     
14:52     
thing and this has become sort of a cottage industry where they'll map a certain part of the uh Drosophila brain     
15:00     
then they'll produce simulations and they'll write a bunch of papers and they're always published in like nature     
15:06     
and cell and places like that um and so     
15:11     
this simulated fruitfly optic lobe accurately predicts which neurons and living fruit flies respond to different     
15:17     
visual stimuli so you can simulate an actual circuit that we know does     
15:23     
something you can get it to be of a certain level of precision you can't get it to be absolute precision because you     
15:31     
can model the neurons you can model the connections you can model the you know     
15:36     
the details of the surrounding tissue the supporting cells you can model the     
15:42     
ion channels and and you know get get some of those dynamics right um and then     
15:48     
you know we have to then simulate it i mean we're not simulating every single     
15:54     
thing about a neuron and a nervous system but that's probably the level of granularity when we get down to that     
16:01     
level of granularity then the question is can we produce the types of behaviors that we see in the organism and the     
16:08     
answer is to some extent we can do these kind of predictions and it's you know it's     
16:14     
really exciting because you know you're basically simulating that we know kind of we     
16:20     
understand the the parts of the puzzle or the major parts of the     
16:26     
puzzle all the work that's been done in the past two years feels like we're on at the beginning of something new     
16:34     
um but you can see how this will build up to something that is eventually very     
16:39     
insightful so this is not like you know a finished product but it is very     
16:46     
promising um having the connetoshaved years off the time it takes to say     
16:52     
identify a neuron involved in a particular behavior so you know there's     
16:58     
been this approach in neuroscience where they've tried to map out cells and     
17:03     
and there was a time when there were like specific cells in in insect nervous     
17:09     
systems in some insect nervous systems in some parts of the insect brain you have specific cells that do specific     
17:17     
things or specific nuclei that do specific things and so that's part of     
17:24     
what insects do uh in mamalian brains usually cells are     
17:29     
interchangeable in terms of what they can do but you've still had people     
17:34     
propose things like different types of cells that have specific types of     
17:41     
functions of course we have uh visual neurons we have auditory neurons just     
17:47     
neurons that are tuned for these types of things we have our different types of neurons like stellway neurons and other     
17:53     
types of uh neurons like motor neurons and things like that um but you know     
18:01     
it's not a matter necessarily of just identifying a neuron and its function     
18:06     
and that's all compartmentalized a lot of times a lot of things happen not because of a single     
18:12     
cell but because of the network it's connected into and so that's the lesson we're learning from     
18:19     
contoics um so you know basically it can help us     
18:25     
because if we have all the cells that are involved in a network we can do better experiments because we're not     
18:31     
like testing one cell at a time for its function we're not trying to find the     
18:37     
optimal network and that would take years to say test every cell and see how     
18:42     
they're connected and see how you know what cells we need to include in our experiment to get get a certain result     
18:50     
uh we can just go to a connecto map look at it and say okay we'll we'll record     
18:55     
from these cells and that takes you know a just maybe about a week to sort of     
19:02     
plan all that out maybe um so that's that cuts a lot of time off of your experiments it cuts a lot of time off of     
19:09     
what we can do or it it adds time to what we can do so um you don't have to     
19:16     
spend months or years chasing down dead ends simulation is going to improve that even more so simulation can even help     
19:23     
because we don't only have the map the static map but we have dynamics that we can simulate and see are these the     
19:30     
dynamics we think are involved in this so we can then say okay this is we have a pretty good chance of testing our     
19:37     
hypothesis if we set the experiment up this way let's do that and let's test it and see if it actually works in in     
19:44     
biology instead of in silicone okay so they talk a little bit about the     
19:50     
sea elegance conto and how you know that's kind of a very basic conneto and     
19:56     
we're kind of expanding our understanding to uh larger and larger neural systems     
20:04     
uh but even with sea elegans uh you know people were excited about     
20:10     
the sea uglins conneto but they were also very reserved about how much time     
20:15     
and money would be involved in recovering that conneto and there were concerns that the payoff might not be     
20:22     
worth it okay so then there's this uh uh     
20:28     
this paper in nature methods this commentary that lays out some of these concerns um this is by Eve Martyrder and     
20:36     
Cory Bergman uh they wondered at the time what additional information beyond the brain synaptic connections whether     
20:43     
those connections are inhibitory or excitatory for instance would be needed to make truly informative models so     
20:50     
again you know there's this level of detail that we need to understand the conneto and make it useful for sort of     
20:58     
say applying it to an experiment now we can figure that out i guess uh it's you     
21:04     
know we we do have some of that information but it's you know we don't know exactly all the information we need     
21:11     
to have simulations say that are very informative or you know in the ballpark     
21:17     
even so uh so that they point out that more than     
21:22     
a decade later fly connectto data still lacks that basic information they also     
21:28     
don't account for electrical synapses although in in so this is in Drosophila     
21:34     
in seoligans we have you know the electrical and chemical um uh     
21:41     
connecttoones so electrical synapses are gap junctions chemical synapses are     
21:47     
where we have synapses that signal different chemicals across uh the     
21:52     
syninnapse and you know these are different the cells are connected in different ways so those connetos look     
21:59     
different they behave differently they process information different but again we don't have that     
22:06     
information for versophila and that may be playing a     
22:11     
role in the behaviors that are produced so if you don't have an electrical conneto then you're missing something     
22:19     
um and so uh yeah they talk a little bit about     
22:25     
that in the absence of knowing who's electrically coupled to whom you can make some assumptions from a chemical     
22:31     
circuit conneto that are going to be missing a lot of parallel pathways in its most paired down form a     
22:38     
conneto represents the connections between neurons mathematically in the case of the fruitfly each connection     
22:44     
appears in a matrix of 139,000 or so rows and columns each representing one     
22:49     
of the flies 139,000 neurons the cell display numbers that indicate how     
22:55     
strongly the two neurons are connected most contain a zero because most of pairs of neurons do not touch we end up     
23:01     
with these sparse matrices usually uh in a conneto where we we compare each cell     
23:08     
to each other cell so we get these pair-wise matrices and you know there     
23:13     
there's they're pretty sparse because you know only certain cells are connected with other cells so it's not     
23:20     
just an alltoall connection and in fact that's sort of the what the power of the     
23:25     
conneto as it were if it were if everything was connected to everything else and you might think that would be     
23:32     
advantageous there's no structure to the conneto it's just kind of like everything's connected to everything     
23:38     
else everything's interfering with everything else and putting aside this     
23:43     
sort of the the polarity of the connection whether it's excitatory or     
23:48     
inhibitory that can be just you know not really biologically helpful I guess if     
23:54     
you think about like why would one neuron connect to another neuron a lot of times you'll see in     
24:02     
neural circuits you'll see an inhibitory connection connect to another neuron and     
24:09     
then there's an excitatory connection upstream of that or downstream of that and that helps to modulate the final     
24:17     
target so you have these inhibitory and excitatory loops where you get you know     
24:23     
some sort of modulation if there's activity it's both inhibitory and excitatory and those connections of     
24:30     
course if everything is connected to everything else those connections are less powerful because those can't be     
24:37     
structured in a way that's useful to the to the circuit so you see the problem with kind of like the allto connectivity     
24:45     
it's actually that you need a very sparse uh matrix to have effective I mean I     
24:52     
want to say effective connectivity because that's a term of art but something that's going to produce     
24:57     
behaviors and allow you allow an organism to switch from behavior to     
25:03     
behavior okay um and then so simulations have to take this all into account     
25:09     
um     
25:16     
I'm interested in whether you could do that for the the physical network where     
25:23     
you'd have um the stress and strain in the the     
25:30     
cell form a network times because they join in     
25:37     
different configurations in different situations     
25:43     
um you had a I think you had a paper about     
25:50     
that at some point where you have different different types of networks     
25:56     
depending on on the connections in the cell and they were physical i've got the paper around     
26:03     
somewhere yes um anyway I guess they're not doing that     
26:12     
if somebody was I'd be interested     
26:17     
you know as in the 10 segreities 10 segreities I'm working with there are     
26:24     
probably some different ones in different cells and also in in the brain     
26:32     
um there's also physical connections um well that are     
26:41     
um that have to do with uh their um elasticity     
26:49     
that's for sure because the gray matter and the white matter are totally different it's like they were two     
26:55     
different um substances     
27:03     
yeah i think there's actually like a lot of and this is just kind of the basics of contoics there is a lot of that sort     
27:11     
of modeling that people do like with white matter and and gray matter and all     
27:17     
these other things and there's a whole field of uh where where they look at the physics     
27:23     
of the brain as part of the conneto so um you know they're doing like physical     
27:30     
studies of the physical properties of neurons the physical properties of um     
27:36     
axons and and things like that and they're taking that into account so it's     
27:43     
really interesting work that kind of goes beyond this little popular article     
27:48     
as for the tense segreges I think that's really interesting you know again the the network structure     
27:55     
connect uh the connectivity structure is the sort of the most basic level right     
28:01     
so you have you basically want to know from that matrix what it's you know     
28:07     
wig what is connected to what what is influencing what and then you could say     
28:13     
have another uh sort of you know you might have like a a simple connectivity matrix where     
28:20     
it's either zero or one where it's either connected or not or sometimes you'll have like a a covariance or     
28:28     
correlation matrix where you have the strength of a a correlation or the strength of coariance you basically want     
28:35     
to know if two units or neurons are having an effect of you might have a     
28:41     
series of measurements into and then that's just the basic sort of matrix and     
28:46     
then there are other matrices you can pile on top of that or you can have the sort of additional     
28:52     
information you know the properties of a certain connection um and then you can you can build subn     
28:59     
networks that basically address those questions so you might be interested you     
29:04     
might say um you know we know that they're different material properties of     
29:10     
certain connections what are those properties and then those might be like in a     
29:17     
discrete value you know have a set of discrete values where we can characterize them and you say "Okay     
29:23     
let's build subnet networks out of these different categories and see you know what how they're connected how are they     
29:30     
influencing other uh subnet networks and things like that so there are a lot of things you can do with that kind of and     
29:37     
again you know contoics is just with neurons you could do sort of a     
29:43     
contoics where you know or you could use a lot of the tools in contoics and in     
29:48     
network biological networks are going to be very similar because they're asking basically similar questions ultimately     
29:55     
it's about what things are communicating with what to produce some outcome     
30:01     
in you know in tenseity it's that they're different points and they're all     
30:07     
interacting and they're producing this me uh this uh sort of     
30:13     
uh meta outcome or this ultimate outcome of stability so that's and so they're     
30:20     
doing very similar things i think it's yeah it's important to I don't know how you know there are tools that kind of     
30:26     
develop in one field and they get applied to the other and then that's considered an innovation when in fact     
30:32     
they were just kind of like you know just using the same tools and just making re reasonable conclusions     
30:41     
so yeah this you know a lot of this fly contoics work has been happening at     
30:47     
Janellia where they have um you know they've been doing a lot of really     
30:53     
high-end research like this where they build these tools uh imaging tools and     
30:59     
they integrate it with uh you know simulation and and do a lot of the you     
31:05     
know make enable a lot of this work okay um and so they're able to uh identify     
31:14     
neurons in different ways it's not just magic you have to do there are a lot of interesting technical imaging challenges     
31:22     
to get a lot of these neurons and you know kind of figure out where they are     
31:28     
have spatial uh you know high spatial resolution for     
31:33     
us and so then we have to figure out what different elements do so um you know we have to     
31:43     
figure out kind of what say like different classes of neuron do so     
31:49     
um so they say as these models process simple movies they sped out the activity     
31:55     
from the neurons in the optic loes motion pathway and across all 50 that     
32:00     
activity aligned almost entirely with data recorded in 26 studies of living flies so uh the models that they built     
32:08     
from the imaging data identified a set of neurons called elementary motion detectors known as T4 and T5 neurons so     
32:16     
this is an example of where you can build a model of the using sort of a model of the conneto that you you get     
32:22     
through imaging you can simulate things and figure out what neurons are processing what and then go back to the     
32:30     
to the actual flies do some experiments to confirm functional uh or or to sort     
32:38     
of confirm their functional role in these networks so uh T4 and T5 are the     
32:45     
primary drivers of motion detection neurons outside of the T4 and T5 group     
32:51     
also seem to compute motion suggesting other as yet unknown visual system cells     
32:57     
that might exist so we we can identify sort of the cells of greatest effect in     
33:03     
a contoone we can identify like you know if they're cells that deal with motion     
33:08     
detection we can identify those cells and then of course in doing a contoic     
33:14     
study we can also then because we have all the cells interacting in real time     
33:20     
we can simulate that we can then say that maybe there are other unexplained     
33:27     
sources of variation that you know we may need to track down and assign to other cells or other mechanisms within     
33:34     
that network so it's really interesting work um this is an interesting paper     
33:41     
here um there was a in April of this year Juraga's team published in nature a     
33:48     
whole body simulation of a fly that uses machine learning to walk and fly that     
33:53     
model is concurrently constrained by an artificial neural network but it could instead be guided by the conneto as well     
34:00     
as a map of a nerve cord that connects the fly's brain and body in a similar way to Turaga's optic lobe simulation so     
34:08     
this is the same team that did the optic lobe simulation we also did a whole body     
34:13     
simulation that uses like a neural network as the driver and then they were able to use the body and do this other I     
34:20     
think we did this we also covered this paper in Saturday morning neuros where they were able to produce behaviors from     
34:27     
this using this machine learning approach but what they want to do is use the conneto to do that we don't     
34:33     
understand the conneto well enough to do that sort of study but I think it shows     
34:39     
the potential for this as well as kind of using machine learning tools in tandem with these connetos to answer     
34:47     
these kind of functional questions and so we couldn't get there     
34:52     
before the connectto and it's you know it's sort of this thing that is     
34:59     
revolutionary I guess um so this is really there are a lot of challenges in     
35:04     
this work but there are a lot of things we can learn i think that's the bottom line um and you know there's this     
35:13     
persistent question of sort of what level of analysis are we do we need to     
35:20     
be at to to get good answers to some of the questions or to get good models to     
35:26     
answer some of the questions we're interested in um this is uh a section     
35:32     
here where they talk about how accurate a model needs to be     
35:38     
um so uh there's this quote here you wouldn't want to stare at an individual     
35:43     
dot in the painting and infer this is exactly what happens here um and then     
35:50     
the follow-up statement to that is some of the papers have tried to stare at every dot what they're getting at here     
35:56     
is that with connetos we can answer questions not only of function and what     
36:02     
the different contributions to to see like output behaviors are from this network but also this emergence that     
36:10     
happens in nervous systems where you know all these neurons are interacting     
36:16     
they're sending signals to one another and through those interactions and only through those interactions can you     
36:22     
produce the behavior and so this is where if you study a single neuron you're not going to understand the     
36:28     
behavior um and a lot of studies do that where they kind of identify specific     
36:33     
neurons record from specific neurons and then try to make inferences about behavior the alternate approach is to     
36:40     
say that you have to study entire networks and then that behavior emerges from those networks and it's not down to     
36:48     
any single cell although single cells can have uh you know outsized effects on     
36:55     
the behavior if you say ablated those cells got rid of them remove them from the network     
37:02     
um and so you know there's tension between sort of contoics and physiology     
37:09     
or the reductionist physiology and then you need to you know you need to use both of them in tandem that that's what     
37:15     
they're saying in the article um and then of course we don't necessarily want perfectly accurate models of the brain     
37:22     
that's not necessarily the goal um we can build different models that     
37:28     
approximate different mechanisms that say summarize different features of     
37:33     
behavior and in fact you know we can't go down to you know we can't simulate     
37:39     
every aspect of the biology the physiology because some of those aspects are not relevant to say producing a     
37:46     
movement behavior or some sort of sensory input or output um so this is     
37:54     
where we get this um you know someone uh all else is     
38:01     
quoted here saying it seems odd to me to design a model that totally ignores that level of biology so it's kind of like     
38:08     
you know we have these ways of kind of we've talked about this many times in     
38:14     
this meeting we have these ways of sort of abstracting uh things to certain     
38:21     
representations maybe neurons maybe we use you know all sorts of different connections between neurons and you know     
38:28     
which ones are the most informative um yeah so this is a this is a good     
38:35     
paper or this is a good article um you know it's really kind of a back and     
38:40     
forth between a number of different topics that we talked about in the group and using the the the things going on in     
38:49     
Drosophila conneto as sort of a backdrop so thank you Leworth for     
38:54     
putting that in the slack i wanted to go over that paper of that article     
39:02     
thank you sir i thought it was like really interesting read and uh thank you for putting it out very clearly I joined     
39:08     
it late but yeah sir I watch the video     
39:14     
thank you yeah no problem so we've got a bunch of people who joined while I was     
39:19     
presenting that article um yeah welcome Jesse and Hussein and Soham and Leila     
39:27     
um yeah so the people who joined later do     
39:33     
you have anything you'd like to share with the group     
39:42     
hello hi yeah um thanks thanks so much Bradley uh     
39:49     
Dick and I have making a lot of great progress uh over the past several of months this is about the uh the CT the     
39:55     
computer tomography project yeah um I don't know I don't have any particular questions about what you've just presented although we are now going to     
40:01     
be um expanding into a lot more projects pretty soon so this is all going to start becoming relevant like on     
40:08     
embryoenesis elegance uh even like um discussing the next steps and specifics     
40:13     
for hard hardware software from from the uh from the from the CT     
40:20     
from the RBY RCT projects perspective so now is the time for me to just want to     
40:27     
start internalizing everything and like absorbing as much information as I can but um uh okay so in the meantime uh     
40:34     
Dick and I have been very busy alongside Muhammad Al Majid Raf over in Greenwich     
40:42     
over at DK we are getting our grant proposal ready together for funding for the um RBY RCT project so um I can give     
40:50     
just as a brief update we're using the Astro toolbox i can share that i can share that how about I just uh maybe     
40:55     
like like share my screen does that sound good yeah thanks okay yeah because     
41:00     
that that's better um so you're not just like listening to me um excuse me i wonder where he is right     
41:09     
now is Dick like on I I don't know you guys i'm so sure     
41:14     
let's see if this works astro Astro Astromography hi cool cool     
41:25     
so Astra should be able to get the geometry correct for us to uh create the     
41:30     
images for the simulation so once again just to back up for a bit we are comparing different geometries of     
41:37     
computer simography uh approaches um including um to compare our our RBY RCT     
41:44     
approach against the uh excuse me excuse me against the uh     
41:50     
the fan beam and the um the cone beam fan the fan beam and the cone beam which     
41:56     
is what we were doing before excuse me those Those are like the the established methods that we're     
42:02     
trying to prove that to prove against to show that our our technique works better so I'm I'm trying to look for my     
42:08     
manuscript so hopefully I can find this um I don't remember where I saved it let     
42:13     
me bring it up     
42:27     
so our approach ray by ray we want to compare this against the fan     
42:34     
beat the latest version of things but we are still we are still getting our manuscript     
42:40     
together to more or less     
42:46     
show I should be able to work um we We should we should be able to show that our approach with the ray by ray     
42:56     
approach so this is what the geometry looks like     
43:01     
between parallel     
43:07     
beam okay ray by ray being the approach that we're using with ray by raymography that we want to show is much more     
43:14     
efficient and much more effective compared to those other two parallel and infinity and um I think I have the     
43:19     
results here yeah as you can see these images are not accurate they're not accurate at     
43:27     
all with three different rows parallel fan beam ray by ray but they all look completely different because we're still     
43:34     
sorting out the ge but     
43:41     
uh yeah that's that's basically where we're at right now mohammed     
43:55     
all right that's good thank you uh it was a good update um yeah for people who have     
44:02     
joined us uh since you know LA Hussein last presented on this they're doing a     
44:08     
imaging project where they're looking at these different ways of imaging this is medical physics so they're looking at     
44:17     
um you know different ways in which the light scatters different ways to capture     
44:22     
objects especially when embedded in some sort of noise uh usually if you're doing     
44:27     
things like imaging in inside the body uh one of the domains is breast cancer     
44:33     
where they have to try to find a tumor embedded in other tissues you know say     
44:39     
you're trying to pick a target that has a lot of noise around it has a lot of other tissues and you're trying to     
44:45     
define the limits of the tumor so that you know people can do surgery on it or whatever um and then there are different     
44:52     
ways that you have to deal with light scattering in tissues and things like that that you wouldn't necessarily have     
44:58     
to deal with if you had like a a wellprepared slide which is a lot of times when we get data from like C     
45:06     
elegance or drosophila if we have slices of tissue there it's usually been     
45:12     
prepared and it's been put on a slide and you don't have to worry about a lot of these effects although we do have     
45:18     
those effects even in uh you know slide microscopy where you're you know     
45:24     
acquiring things from a well-prepared slide you have uh you know you'll use different stains     
45:31     
to identify things um in what medical imaging you use contrast agents but     
45:38     
those are a little bit diff behave a little bit differently so that's that's what they're doing and and they're     
45:43     
trying to work on these different methods uh for doing this um yeah it's     
45:48     
it's really cool um and then this toolbox is of course uh make makes this     
45:54     
a lot easier to kind of work with because it's it's a hard problem and     
46:00     
then the image processing is hard     
46:05     
yeah yeah so if you guys have ever played Wordle like that's the game where you uh have to guess what word it is but     
46:11     
uh you make guesses and somebody else tells you which letters are correct or which letters are uh in the like the     
46:17     
correct letter but in a different position things like that that's essentially how the MR algorithm works when it's reconstructing an image okay     
46:24     
so um even from a sparse data even when you don't know what the image is based off of how how single singular photons     
46:32     
like singular excited photons interact as they uh hit the image can you can you can you detect and refine and refactor     
46:40     
the way you are producing an image to to produce the image of something like a breast cancer tumor as uh efficiently     
46:47     
and effectively as possible these three uh columns left to right parallel fan     
46:52     
fan beam and then ray by ray ray by ray being the one that we are trying to show is better and um the as you can probably     
47:00     
guess the fan beam is not working like it's just not that's what and I are trying to figure out okay     
47:10     
yeah says looks really awesome the same impressive yeah yeah it's very good so     
47:18     
Yeah you guys uh been working on papers you were talking about a patent you're     
47:24     
talking about a grant so all these things are kind of     
47:31     
cooking yep yep we're we're we're patterning this one for those for those of you who are just jumping in yes we are patenting this work yep yeah     
47:41     
all right very good thank you yeah it's it it always surprises me in some ways too how much of this like is as as as     
47:48     
simple as it seems like someone says you're just making an image and you're just taking like like literally an image file and trying to reproduce it and it's     
47:54     
like why is that so difficult and it's extremely difficult we've been working on this for over a year now and even explaining like the mark the     
48:00     
multiplicative algebraic reconstruction technique algorithm explaining that to Muhammad like that takes that takes time     
48:06     
too you know it's it's so it's so simple that like it seems it seems like it would be trivial but it's not like even     
48:12     
solving this problem and showing that you can patent it is not it's not obvious it's not obvious at all so yeah     
48:20     
yeah like um for for one thing um like like we've been we've been pretty     
48:27     
like like black box about the about the Astro tool the Astro toolbox like like like just treat it as it is and see if     
48:33     
it works then try try it but we can't really do anything about it you know and I think right now we're trying to dig in     
48:40     
a bit deeper to see if we can um like like literally inspect the code to see what's going well with it so yeah     
48:51     
okay it's really good to hear that you're working on that project     
48:57     
yeah yeah yeah i think it's important so so we we are also um I mean like I I     
49:03     
don't mean I don't mean to go on for if other people to talk about things but Dick and I are working on a couple other projects too so I don't know if you want     
49:09     
me to keep going because like we've started work on the peptide project the peptide project did he mention that one     
49:15     
to you like the uh yeah why don't you talk about it alongside Sevino Longa     
49:20     
over in Italy i don't know if you're familiar with him you know Sevino Sino Sevino Sevino Longo in Italy sino Longo     
49:27     
um yeah and he'd mentioned a problem of random random DL DL peptides with     
49:33     
different Okay so sorry guys this is a this is a completely different project i'm shifting gears entirely okay but     
49:39     
basically um yeah after talking about some manuscripts on string processing models     
49:46     
and uh performing some simulations we are basically trying to answer questions     
49:51     
related to um the origin of membrane peptides during the origin of life based off of positive feedback back mechanism     
49:58     
of a hydrophobic mismatching so that's quite a bit and that's quite a bit to uh     
50:03     
overturn um you know but this is this is basically what we're trying to answer with some of our future simulations and     
50:09     
research too so yeah okay that sounds great     
50:16     
what other projects are you working on oh plenty he gave me a list of like 40     
50:22     
oh no did he Did he show you the the project list you I'll put it in the chat how     
50:28     
about I put in the chat because that's that's the only way I can share all this with you um like because I' I've been     
50:34     
keeping track of all them we literally discuss Go ahead     
50:39     
take a look at this enough to do like like 50 or 60 PhDs     
50:45     
yeah these are all kind of like Dick's uh different projects that he's defined     
50:50     
uh yeah there's just a lot of them here yeah yeah and it's not just in biology     
50:57     
or developmental biology i mean these span everything     
51:03     
but some of these you know yeah some of these are have maybe solved them in different     
51:09     
ways others are probably not really haven't been well addressed some you     
51:15     
know been addressed with like old school experimental methods so like if you     
51:20     
added a computational element it might be very helpful so Nick will have any     
51:26     
problems that he's working on or has encountered in like the last 50 or 60 years of his career and it's really     
51:34     
interesting because you know these are you you think well we could solve this problem pretty easily but you know there     
51:41     
are problems in biology that go back like a hundred years or more and it's sometimes it's about bringing technology     
51:49     
or bringing new methods to a problem sometimes it's about like doing the right experiment sometimes it's just a     
51:56     
really hard problem that you can take a stab at you can get a good result but     
52:01     
you're not going to solve it but that's kind of the the great thing about like all these different problems is that you     
52:09     
know um there are so many ways to contribute and even if it's not like you know solving a problem you still     
52:17     
contributed something to knowledge and you know maybe you develop a method that     
52:22     
you can use in other domains so it's it's always think in this way of     
52:28     
answering different longstanding questions yeah yeah i mean it's     
52:34     
um you know um well everything I I appreciate I appreciate the insane you     
52:40     
know the degree of like like polymath type thinking the the the approach     
52:46     
towards stuff that's basically built on you know just purely purely asking questions and answering them you know no matter what the domain is what the     
52:51     
discipline is you know um being able to be driven by like just trusting other people and professionals in those fields     
52:57     
and areas that like the work you do it has merit like in and of itself and it's not like you're doing things     
53:03     
um like I don't know like um like just just just showing showing that that     
53:08     
ability of like you know having having as much impact as you can with your research but still being selective with     
53:14     
which questions you ask     
53:19     
okay yeah yeah so if people want to take a look at that in the chat and if you     
53:25     
identify a problem that you're interested in let us know um you know um     
53:32     
yeah so any other people want to update us on     
53:41     
anything i know Jesse has been ha has had some pretty interesting recent     
53:46     
career moves not necessarily relevant to D.VAR group but he's interested in a lot     
53:52     
of mentoring things so you know there's this mentoring aspect that we don't talk about as much um and you know that's     
54:00     
something that if you're interested in Google Summer of Code uh we'll actually be having and this is     
54:08     
a separate meeting um and I'm not sure how I'm going to work this this year but we're we're going to start our     
54:14     
open-source meetings um that are run by the Orthogonal Research Lab um that's     
54:21     
going to start this coming Friday so we usually do these meetings and they're seasonal we do them in the summer in the     
54:28     
fall late spring and we kind of match this to     
54:35     
Google Summer of Code so like the idea is that people working on Google Summer of Code uh join our group we have other     
54:43     
people in our our lab that are interested in open source it's usually Morgan and Jesse and maybe Sue Hussein     
54:50     
has joined these meetings in the past where you just come in and we talk about different topics and open source     
54:56     
development um and you know maybe open publishing open science people give     
55:02     
updates on their projects um uh first meeting is 9 or 15 it's     
55:10     
uh it would be nine right because that's Friday May 9th yeah     
55:18     
yeah it be May 9th so this f this coming Friday uh and it'll be at uh 12 o'clock     
55:25     
Eastern time in North America so if you're uh in India it     
55:30     
would be in the evening if you're in other parts of North America be in the morning um and so forth and so I'll send     
55:38     
out when we get people accepted into GSOC or if you don't get accepted into     
55:45     
GG Soc um that you're still welcome to join our group but I'll send out that     
55:52     
information then um so yeah and you know this is again     
55:58     
it's a good way for people to kind of develop their projects you know we do     
56:04     
when we do G-So we do the weekly meetings where we kind of go through uh     
56:10     
updates you know what people are doing for that week as you know from GOC we give you this one of the requirements is     
56:17     
to do this schedule where you have to make a weekby- week account of what you're going to do and that's a project     
56:25     
management technique I think is quite useful because it helps you to sort of time yourself properly so you can go     
56:31     
through and hit all the points that you need to and you know it it's not     
56:37     
supposed to be you know like you're not supposed to absolutely stick to the schedule but it gives you this timeline     
56:43     
that you can work from with the open source meetings this gives you another dimension to that where we can like talk     
56:49     
about some of the topics the larger topics the larger bigger picture topics     
56:55     
that we would think about uh we wouldn't think about normally within the course of that schedule so you know we've     
57:02     
talked about different things like you know the different types of uh     
57:08     
research software development opportunities that exist in organizations or you know how to kind of     
57:16     
get things organized for GitHub repository some of the licenses you might use to license content     
57:24     
um you know what are some of the aspects of project management that you should be     
57:29     
working with u and Jesse's had a nice series on kind of like project     
57:35     
management management in the workplace so let's see if what we have in the chat so Jesse said uh I'll finish     
57:43     
my write up of the summer cohort this week so folks who want to do things with us this summer with or without GO proper     
57:51     
know what's up it's a whole community of things and it's pretty cool yeah so be on the lookout this week i think um     
57:59     
we're I think decisions are made on Google Summer code on Thursday so it's going to be a short turnaround but uh     
58:06     
I'll lay out the details then like I said you can Yeah hussein did G-S twice     
58:13     
it's a good experience oh once once with ORL and once with another group not Not with ORL yeah i don't I don't think you     
58:21     
can do it more than once with a specific group there's a limit of two per person so     
58:26     
I think that's what it is i think that's Yeah so um and again if you don't get     
58:33     
selected I don't know how many slots we're going to get this year but if you're not selected because we have you     
58:39     
know a lot of people applying you're still welcome to work on the project if that's something that you want to follow     
58:44     
up on um and you know it's it's up to     
58:50     
you uh but we do have that infrastructure in place so that you can pursue this     
58:55     
um you know and see something through you might use it for another purpose or     
59:01     
you might just be interested in attending your open source meetings because it's usually a pretty good uh     
59:07     
you know pretty good discussion you'll learn a lot okay so I wanted to finish up with another and it's this paper from     
59:14     
PNAS um this is uh it's a complexity paper it's also a biohysics and     
59:22     
computational biology paper um this is uh I think this was published recently     
59:28     
yeah this was published in uh just recently so this is called the emergence     
59:34     
of ukarotes as an evolutionary algorithmic phase transition and so this     
59:40     
paper kind of deals with this early life so we've talked about early life a lot in the group we've also talked about     
59:47     
sort of the computational aspects of biology and you know we've talked about     
59:53     
how you can characterize uh different biological processes using     
59:58     
you know like algorithms and matrices and things like that cell and this is kind of an interesting exercise in     
1:00:05     
looking at this process of the emergence of ukarotes so if you're not familiar     
1:00:12     
with the history of life you had um you know proarotes you had proarotic cells     
1:00:20     
you had archa bacteria uh you well you first started with     
1:00:26     
proaryotes and then you had this common ancestor between proariots archibacteria     
1:00:34     
and ukarotes and ukarotes and archibacteria uh I think form one cate     
1:00:40     
and proarotes form another cate with common ancestry and so     
1:00:46     
ukareotes emerged from this sort of early soup of life um and it emerged     
1:00:52     
from by having a lot of uh the organels that you see in ukareotic cells being     
1:00:59     
independent organisms and so there were these uh symbiotic events that defined     
1:01:06     
ukareotic cells and their emergence uh that don't define uh uh proarotic cells     
1:01:14     
nor archibacterial cells so archibacterial cells and proaryotic cells are look a lot different than     
1:01:21     
ukareotic cells and the question is is what kind of an evolutionary transition     
1:01:27     
had to happen to get these ukareotic cells so famously ukareotic cells have     
1:01:33     
this sort of symbiosis component where a lot of the organels are were     
1:01:38     
incorporated into a single cell so the thinking is is that a lot of the     
1:01:43     
organels you see today in the ukareotic cell form these uh ecological networks     
1:01:49     
where they were these organisms that were dependent on each other and then they just formed this single cell type     
1:01:56     
and it became an integrated hole and then that led to all the ukareotes that we know today some like yeast or some     
1:02:05     
like humans or some like other types of insects invertebrates etc etc and so     
1:02:14     
it's an interesting question as to how ukareots sort of evolved     
1:02:19     
um and you know how that process happened so they proposed this evolutionary algorithmic phase     
1:02:26     
transition so I'll read the abstract and then the significance and then maybe go through a     
1:02:33     
little bit of the details of the paper but not too many because we're already uh over time so um the abstract reads     
1:02:41     
the origin of ukarotes represents one of the most significant events in evolution since it     
1:02:47     
allowed the posterior emergence of multisellular organisms basically means that ukareotic cells enabled     
1:02:55     
multisellularity although that's not entirely true I mean there a lot of     
1:03:02     
multisellular organisms are ukarotes proarotes can form different multisellular collectives and it's you     
1:03:09     
know but the point is is that ukareotic cells enable a lot of complexity on that     
1:03:16     
sort of scaffolding on that sort of template and so it remains remains unclear how existing regulatory     
1:03:22     
mechanisms of gene activity were transformed to allow this increase in complexity here we address this question     
1:03:29     
by analyzing the length distribution of proteins in their corresponding genes     
1:03:35     
for 6519 species across the tree of life so they do this proteomic study where     
1:03:41     
they look at the length distribution of the proteins and genes for these different across a a large number of     
1:03:49     
species in the tree of life so it's like you have the sampling of different uh     
1:03:55     
biological contexts different organisms of different complexities     
1:04:01     
um we find a scale and variant relationship between uh gene mean length     
1:04:06     
and variance maintained across the entire evolutionary history hey Bradley     
1:04:12     
yeah do you know what or what extent uh the size of these organisms are     
1:04:17     
beingated not only just say by the genes involved in their actual like like the     
1:04:22     
size of like say the membranes but the actual content of the of the DNA itself     
1:04:28     
like getting to the point where organisms having too much DNA for them     
1:04:34     
based on their size to even to even handle do you understand what I'm saying like they're being a limit saying Yeah yeah     
1:04:42     
yeah well that so that's a separate question but like people have looked at like uh genome content and like     
1:04:48     
complexity of the organism and it's interesting that there isn't a good correlation between the two so you might     
1:04:55     
have something like puffer fish that have very large genomes and other organisms that are maybe more complex     
1:05:02     
than puffer fish having smaller genomes so the genome size and like     
1:05:07     
the if you want to define like organismal complexity there isn't really a good uh correlation there there's this     
1:05:16     
idea of the C value uh people have mentioned but it's you know it's kind of     
1:05:22     
a this is where maybe this paper becomes a little bit hard to understand     
1:05:27     
because you know it's like you can look at like the length of a genome or the length of a gene and it doesn't     
1:05:33     
necessarily correspond to complexity yeah yeah it it always seems like the     
1:05:39     
people who have the answers to these questions um they themselves uh it's it's it's not it's not even so much the     
1:05:45     
difficulty in like looking for those answers those answers in particular but how how on earth you would frame those     
1:05:51     
hypotheses and go about the uh the simulations and the like the simulations and code to to get it done i     
1:05:59     
just wanted to sorry I wanted to interject something amphibians have     
1:06:05     
larger larger genomes and the axelottles I I sometimes work     
1:06:13     
with can have poly ployy in other words they can have twice the normal genome     
1:06:20     
and still exist and to produce a larger animal     
1:06:26     
yeah so there there there are a lot of things going on with genomes especially polyloy where you have like different     
1:06:33     
gene duplications and gene doses and things like that where um yeah it     
1:06:39     
doesn't necessarily correspond to complexity they're specific it's more     
1:06:44     
about like evolutionary history but what they're doing here is     
1:06:50     
they're using like a uh like they're using complexity theory basically     
1:06:57     
So they're talking about scale and variant relationships so they're probably going to do some like power law     
1:07:02     
analyses or something uh we'll see uh using a simple model we show that the     
1:07:08     
scale and variant relationship naturally originates through a simple multiplicative process of gene growth     
1:07:15     
during the first phase of this process corresponding to proarotes protein length follows gene     
1:07:21     
growth so you get gene growth and then you get protein length i guess what they mean is that there are these     
1:07:26     
duplications of genes and then that adds to the protein length and it adds to the     
1:07:32     
complexity of proteins um at the onset of the ukareotic cell however mean protein     
1:07:39     
length stabilizes around 500 amino acids uh so that's interesting back at the     
1:07:45     
time when ukareotic cells originated mean protein length was around 500 amino     
1:07:50     
acids and then from there even though the cell doesn't you know well you you enable a lot of complexity but the cell     
1:07:59     
itself is uh you know that sort of template is fixed um and you get a wide     
1:08:06     
ranging uh length of different types of proteins while gene continued grow while     
1:08:11     
genes continued growing at the same rate as before this growth primarily involved non-coding sequences that complemented     
1:08:18     
proteins regulating gene activity so that means that you have uh the number     
1:08:24     
of genes would increase in like the precursor to uh ukareotic cells and then     
1:08:31     
it when you have ukareotic cells there is this uh rate of of uh genes growing     
1:08:39     
but this uh growth after the ukarotic transition primarily involved non-coding     
1:08:46     
sequences that uh were involved in regulation and so that's the argument they're     
1:08:52     
making that when you have this ukare ukareotic transition you're you know you     
1:08:58     
get larger genes you get larger sequences but they're involved in regulating the genome so if you look at     
1:09:05     
a a proaryotic genome for example it's much smaller it it's much more what they     
1:09:11     
call optimized which means it has like uh it does have a regulatory region each     
1:09:17     
gene but it's usually something uh you know that's that's somewhat simple and     
1:09:23     
then you have a coding region and you don't have a lot of interstitial DNA it's very what they call optimized     
1:09:30     
um if you uh look at the um at the genome of a of a mitochondria it has     
1:09:38     
this similar structure where it's pretty optim optimized but the reason that's optimized is because a lot of the uh     
1:09:44     
genes that are not involved in energy production have been moved out of that into the nuclear genome of the ukareotic     
1:09:50     
cell that it that it resides in so you know you can see this process of sort of     
1:09:57     
optimization in things that are not like ukareotic nuclear genomes but when you     
1:10:04     
have a uh ukarotic nuclear genome you need these regulatory mechanisms and     
1:10:09     
things to evolve so that's why you have more DNA and thus larger proteins and     
1:10:15     
more variable protein uh our analysis indicates that the shift     
1:10:20     
at the origin of the ukareotic cell was due to an algorithmic phase transition     
1:10:26     
equivalent to that of certain search algorithms triggered by the constraints in finding increasingly larger proteins     
1:10:33     
so this last sentence is like the bomb that they throw into this which is to say that like they find this result in     
1:10:39     
biology across ukareotes and compare with proariots and then say okay uh     
1:10:47     
there's a lot you know there's a lot of regulatory activity that needs to happen when you scale up to these ukareotic     
1:10:53     
cells and then you need you know this is actually similar to things that we see     
1:10:59     
in in the behavior of algorithms where search algorithms need are uh     
1:11:07     
constrained when finding increasing little proteins so let's look at the significance of this paper so we kind of     
1:11:14     
understand why it's important so for almost half the history of life on earth     
1:11:20     
complexity of all organisms was limited to that of simple proarotic cells and     
1:11:25     
this is where we have bacter like I guess you bacteria and     
1:11:31     
archibacteria um and most of the life on earth actually is bacteria the process by     
1:11:38     
which genes are activated which is at the root of the functioning of all living beings was entirely regulated by     
1:11:44     
proteins so this is where um we have you     
1:11:49     
know we don't we have genes but those aren't enough we actually need proteins to build biological complexity and have     
1:11:57     
like some use for the genes that's why we information is encoded in genes but     
1:12:03     
it needs to be activated through proteins um this set up a limit on     
1:12:08     
cellular complexity as finding even larger proteins became computationally unfeasible so basically when you have a     
1:12:17     
geno a genome that's highly optimized like what you see in in bacteria it's harder for     
1:12:25     
um larger proteins to be produced basically that's the argument um and so     
1:12:32     
this limited cellular complexity for a long time the ukareotic cell     
1:12:37     
characterized by membrane nucleus and organels so you have you know these     
1:12:42     
different organels we have the membrane bucleus we have more structure to the     
1:12:47     
ukareotic cell emerged as a compromise between a conserved process of gene     
1:12:53     
growth and a change in genetic regulation which incorporated non-coding     
1:12:58     
sequences so what they're also arguing is that bacterial genomes you can keep if you     
1:13:05     
had sort of a bacterial template you could keep growing genes in terms of their size that circular genome could     
1:13:12     
become larger and larger but it wouldn't add anything to the ability to make more complex proteins because it didn't have     
1:13:20     
the regulatory machinery would just produce maybe larger proteins that weren't as uh complex where it would     
1:13:28     
just produce more proteins and not necessarily more complex     
1:13:33     
proteins uh and so there's this compromise between uh you know having     
1:13:40     
this change in genetic regulation in this process of gene growth and incorporating non-coding sequences this     
1:13:48     
increase in cellular complexity which occurred continuously but in an abrupt manner at a critical point unlocked the     
1:13:55     
path towards multisellular organisms so this happened all at once what they mean by phase transition is that you had the     
1:14:03     
system that was working well but wasn't giving you this level of complexity that     
1:14:09     
you would uh later see in more complex organisms so there was some need     
1:14:14     
obviously for this mechanism for complexity and it probably had to do with you know being more efficient in     
1:14:22     
the survival of all these different organels and cells in in early life and     
1:14:29     
so we had this phase transition where this very short period of time where all     
1:14:34     
these organels came into association within a membrane and then this had an     
1:14:41     
effect on the growth of the genome and how the genome started to produce more     
1:14:47     
complex proteins so that's and so that's basically what the paper is about they     
1:14:54     
do this analysis where they look at these distributions so this is a complexity theory analysis where they're     
1:15:02     
looking at the length of a genome or the length of a gene and that's on the x-     
1:15:07     
axis so this is from 10 to the 1 to 10 the 6 so that's you know this the sort     
1:15:14     
of the length scale that they're looking at and then the probability density function for both proteins and genes so     
1:15:21     
basically you have this distribution of different sizes and you know it's sort of peaks for a specific species and then     
1:15:29     
there's this variation along the edges so you have basically this unimodal     
1:15:34     
curve and that describes the distribution of different gene lengths     
1:15:39     
relative to uh observed genes and then this is for a specific species so this     
1:15:46     
is for zebra fish and the proteins uh     
1:15:51     
have a certain length with a mean length and a standard deviation around that mean length and then genes also have a     
1:15:58     
length so the proteins are generally smaller than the genes you have um this     
1:16:04     
distribution in the so they overlap um and so the length is on a logarithmic     
1:16:11     
scale probability density is on the y axis so gene and protein     
1:16:18     
length distributions are logn normal is for this example for zebra fish we can     
1:16:23     
see that length distributions for genes which is in blue and corresponding     
1:16:30     
proteins in orange uh are given note that the lengths are represented on a     
1:16:35     
logarithmic scale the red curves are fits to logn normal distributions     
1:16:41     
uh mean lengths and standard deviations of genes and proteins are 31,084 base pairs and 538 amino acids     
1:16:49     
respectively so they're measuring that's the way they're measuring this total number of genes and proteins plotted is     
1:16:58     
25,432 and 25,76 respectively so that the numbers     
1:17:03     
are around the same these logn normal distributions are typically found in almost all species across the tree of     
1:17:10     
life and then this is another analysis where they give the mean gene length uh     
1:17:16     
in uh increments of from like 10 to the 2d to 10 the 5th and then gene length     
1:17:23     
variance which is from 10 to the 5th to 10 the 10th and they give this for different uh uh uh different uh parts of     
1:17:33     
the tree of life so archa bacteria proests fungi plants     
1:17:40     
invertebrates and inver invertebrates so we have ukarotes and     
1:17:46     
procariots and so we have uh for all these different they're     
1:17:51     
colorcoded it's kind of hard to see the colors uh but we have this distribution     
1:17:57     
here where we see that uh for bacteria the mean gene length is lower     
1:18:04     
the gene length variance is lower and the bacteria are clustered down here you     
1:18:10     
have uh protests and archa down here still up towards the bottom which we     
1:18:16     
might expect uh given their complexity and given there not being ukarotes and     
1:18:22     
then we have fungi plants invertebrates and then finally vertebrates at the top     
1:18:28     
of this where we have a longer mean gene length and a longer gene length variance     
1:18:34     
so this is where we're getting this effect of regulation in ukarotes so as     
1:18:39     
we move up from uh you know we have plants invertebrates and then vertebrates and we have primates here     
1:18:46     
birds fishes so we have our vertebrates in this area or invertebrates in here     
1:18:52     
and there's a lot of variation in this but basically that's how they're doing this analysis     
1:18:58     
so that's kind of the take-home message here and then they have this model of     
1:19:03     
gene growth where they start with a series of genes with arbitrary initial lengths they have an iterative process     
1:19:10     
by which a gene is randomly chosen and its length is multiplied by a random     
1:19:15     
number generated from a given probability density function this model leads to logn normal gene length     
1:19:23     
distributions and so uh they they use this model to predict two laws     
1:19:29     
describing the growth of genes through time first mean gene length grows     
1:19:37     
exponentially through time according to this equation uh L0 stands for the mean gene     
1:19:43     
length of the last universal common ancestor and then this parameter is the     
1:19:48     
mean of the stochastic multiplicative factor that's in this equation here this     
1:19:53     
predicted exponential growth is compatible with previous empirical findings reporting that genes incre     
1:20:00     
increase in length over revolutionary time with a potential acceleration the most recent ones then they use this mean     
1:20:07     
of the logarithm of gene lengths or log l and this is this equation here which     
1:20:13     
grows linearly through time according to this equation where g0 stands for the     
1:20:19     
mean gene gene log length of the lucas genome so the luca is the last universal     
1:20:24     
common ancestor they calibrate everything to that so all these different taxonomic groups that they     
1:20:31     
showed in the last figure where we had the uh the log uh log linear     
1:20:38     
distribution that's you know everything is calibrated to that and compared with the luca example and so these two these     
1:20:47     
are the two laws of gene growth that go into this graph here okay okay so yeah     
1:20:52     
uh yeah that's that's that paper um so I don't know if we had any     
1:20:58     
questions about that paper     
1:21:08     
my salamanders are vertebraes so yeah they would fit Yeah fit the upper part     
1:21:13     
of the curve yeah yeah it's it's standard complexity     
1:21:19     
theory they're trying to use that to answer some evolutionary question which is why are ukareots ukarotes different     
1:21:28     
or weird or whatever you want to say compared to bacterial cells how did they     
1:21:34     
become like this you know what's this evolutionary transition all about why did it happen I mean you know there's     
1:21:41     
the the symbiosis argument there's the complexity argument     
1:21:46     
but then you might say you know we can achieve yeah we can achieve maybe certain forms of multisellularity with     
1:21:54     
proaryotic cells and we maybe we could have done that but why exactly do we     
1:21:59     
need like you know larger genomes why do we need to have     
1:22:05     
uh the things we see in ukarotes to add to that complexity so they do this analysis of uh genes and proteins and     
1:22:12     
that's I think it's interesting I don't know if it really answers the question     
1:22:17     
of why I know that single cells um have a different mechanics than multiple     
1:22:25     
cellled um tissue the tissue is different from the single     
1:22:32     
cells that they are made of they have a different different stress strain curve     
1:22:38     
yeah yeah different mechanics yeah so yeah that and that's of course     
1:22:46     
you know you you need to have a variety of proteins to do that i mean you can't     
1:22:51     
just do that from simple maybe you need more complex proteins for that well you     
1:22:57     
do for Yeah you do um join them together     
1:23:03     
and just look at the cell linkages and the tri junctions and it's all it's it's     
1:23:09     
different in different species but and it the tri junctions are different     
1:23:15     
from the cellto cell junctions so yeah that's a bunch of different proteins     
1:23:23     
yeah yeah it's also worth saying that like there's a lot of you know there is     
1:23:28     
like multisellularity in things like yeast which are ukarots but that multisellularity looks     
1:23:34     
very different from other types of ukarots so then the question is why is that is that because of the the the     
1:23:42     
types of proteins that are produced or is it something else going on is it just     
1:23:47     
evolutionary history what's going on so it's you know it's a that that may and a     
1:23:54     
lot of that may play a role that there's a lot of going on with adhesive proteins and things like that all right so and     
1:24:02     
then the last point in that paper they talked about uh this behaving like an algorithm so like a search algorithm you     
1:24:09     
know a search algorithm is trying to find like the optimal proteins for the     
1:24:14     
job basically and they say you know in in procarots if you grow your if you get a     
1:24:21     
longer gene length you can have this these proteins that will find the     
1:24:27     
solutions in that protein space so you can produce more proteins you can produce longer proteins they could do     
1:24:33     
more they're functionally uh more diverse in that way but that that hits a     
1:24:38     
limit and so to get to that next level of search of protein space and protein     
1:24:45     
space is this hypothetical space where you have uh various configurations of     
1:24:51     
proteins that do various things so when you get to the point where you have not only longer genes but that they're     
1:24:57     
regulated in different ways then you can explore more of that protein space and it opens up a lot of avenues that were     
1:25:04     
previously closed so that's what they were getting at with that search algorithm uh statement but I I I found     
1:25:11     
it interesting that like there is this parallel between computational systems and biological complexity that I don't     
1:25:18     
know um you know it's an interesting kind of parallel but I don't know if     
1:25:23     
there there's more to it no well if you want a couple of papers about trijunctions and bjunctions I I     
1:25:33     
have them or I they're in my thesis um if if you want to know about the     
1:25:39     
proteins in a tri junction I have a have a paper about it and I think it's the     
1:25:44     
only one um so far that was really helpful to me     
1:25:52     
um yeah it's um I don't know where it is anyway it's in my thesis and I can look     
1:25:58     
it up um I'll send it to you Bradley all right that sounds good thank you okay so     
1:26:06     
Do we have something in the chat here um so Jess made one more comment uh I'll be     
1:26:13     
doing some work on writing up things and may ask some folks Jestor etc     
1:26:19     
experiences so yeah uh Jesse's going to be kind of organizing that experience     
1:26:25     
this summer helping to organize it and so yeah when we get into GOC we're going to have a lot of interesting things     
1:26:32     
going on and again you don't get selected for your project or you can you can still pursue that project or be     
1:26:38     
involved in the lab in different ways and it's always a good experience so     
1:26:43     
good luck to everyone on that okay uh well thanks for attending     
1:26:50     
um have a good week okay bye bye take care take care bye bye     
