## TRANSCRIPT   
0:01   
I see people are here. Hello. Look like Susan's here.   
0:06   
Anyways, I don't think we're going to have a full meeting today. Uh it's pretty early in the year. Uh I have I   
0:15   
see a couple people here who want to uh maybe they want to introduce themselves   
0:20   
and I know maybe some of you are in the uh Slack channel for Diva. So   
0:27   
uh that's good. Uh if you're interested in the group, join the um channel. We   
0:34   
have meetings here usually on Mondays, sometimes we don't. Um and then we have our YouTube channel where we have all   
0:41   
sorts of things related to the project. So um looks like we have Shreas   
0:49   
and uh Marsh. So would you like to introduce yourselves?   
0:54   
myself and me joining for the first time. So I got interested in this and I was looking   
1:00   
up the repo. So yeah like this is my first meeting I'm joining for the first time.   
1:05   
Okay. Yeah. Recently like I've been working in deep learning. I'm not from a neuroscience background per se but I   
1:12   
recently I've been like interested in applications of deep learning uh applications of uh deep learning in   
1:18   
neuroscience. So yeah kind of want to know how it is and interested in this field. Thank you.   
1:25   
Yeah. So, yeah, that's uh something we've done a lot of uh with small   
1:30   
connetos and with, you know, cells and and things like that. So, yeah, that's   
1:36   
that's something um you know, I don't know if you're looking to build a project, we should talk about that   
1:42   
though. Um you know, come up with ideas. We have data sets available. If you look   
1:48   
at the things we were doing in the last Google summer of code, I think you can get an idea of what we are interested in   
1:55   
and that that those are those videos are on YouTube if you go back to like last summer some of those meetings that were   
2:02   
look at the repo um and see what you know kinds of repositories we have. So I know that you   
2:09   
know you can't learn everything from the documentation but if you have questions that should guide you to those questions   
2:16   
and then we can discuss that. Yeah sure. So primarily I got to know   
2:22   
about this about this organization through Google of itself through INSCF   
2:28   
and then I looked up the repo and the project for past year the hyperraph project and it looked pretty   
2:34   
interesting. Yeah. Yeah. you've been doing the hyperraph project for probably five   
2:42   
years now. Uh and you know just various versions of it and building on what we   
2:48   
have. Um so that that's and that that of course is in the DVR organization. So um   
2:56   
and again you know we basically the hyperraph project is where we have data   
3:02   
like image data, secondary data of different types. we use a some sort of   
3:08   
hyperraph algorithm to analyze them and you know we've had different ways of thinking   
3:13   
about this. Um last summer we had a project you know we're really kind of   
3:19   
focused on representation uh the summer before that we worked on   
3:25   
some implementation. So all the code is in there that that has been developed.   
3:31   
The question is, you know, what what can we do with it or how can we make it better?   
3:36   
Um because I don't want people to keep reinventing the wheel, obviously. So, we'll cheer that and see what you   
3:42   
think um what you can contribute or what you find interesting. Yeah. Uh I'll be looking at uh like the   
3:50   
last year's project and and understanding what was the progress done and then I will ask you questions   
3:56   
regarding it. Okay, sounds good. Yeah.   
4:01   
Thanks. Uh I don't know if Versed is available.   
4:07   
Is is that a noteaker app or is that? This one seems to be the noteaker app, I   
4:14   
guess. Okay. Wow. All right. Uh so it's good. Uh Susan, hi.   
4:23   
Hello. You started late today or um No, I'm just checking in to see what   
4:28   
people are up to. Oh, okay.   
4:33   
Um, what am I up to? Collecting papers. I need to start reading.   
4:40   
Yeah, I have to write um a paper about my   
4:46   
thesis. So, I might I could present it to the group   
4:51   
at some point if I if I can get going. Yeah.   
4:58   
Yeah. I don't know. Uh what what well it's a pretty big tall order paper uh based on your thesis like because you   
5:05   
could have a lot it could go in a lot of different directions. Yeah. Um I think I have to compare um   
5:13   
regular mechanics with what I'm doing. Okay. So what have I got here? Yeah. Mechanics   
5:22   
and biomimicking of wet adhesion in nature. There you go.   
5:28   
Okay. Talk about a large paper. How long is that?   
5:34   
44 pages. Okay. Well, that's not bad, actually. Yeah, it could be worse.   
5:41   
Yeah, it's a nice review. Anyway, I I'll look at it. Um I just   
5:49   
have to compare Continum Mechanics to tense. So,   
5:55   
I need to do some searches and um yeah, there's there's been a few a   
6:02   
few articles. I' I've got quite a few. So, including optical clearance   
6:08   
elasttography, a review of that. So,   
6:15   
yeah. And I've got Oh, this one might be interesting. uh constitutive framework   
6:21   
for tension, compression, failure, asymmetry in soft materials.   
6:30   
I don't know. Anyways, whatever. I' I've got things I have to review here. So,   
6:41   
I guess it's better than going for a walk out in 30 below. So, we'll we'll,   
6:47   
you know, That's good. Yeah. So, yeah, keep us   
6:54   
posted on that. That'd be good to see progress on that.   
7:03   
And by the way, the before Christmas, the last um paper you presented, I I did   
7:09   
find it interesting. Uh which one was it? I can't remember. Uh on the neurons, they're growing them   
7:16   
in gels. Oh yeah. Yeah. The paper is from Yeah. the the mind in vitro group   
7:22   
that uh Morgan is always talking about. Yeah. That that's uh you know that's   
7:29   
just like growing that on some sort of substrate and getting it to culture. So   
7:34   
like you know cell culture you always want to find a good substrate for two dimensional threedimensional   
7:41   
arrays of things like you know like for a simple cell culture you have a model layer and you can grow cells out on that   
7:48   
but of course if you want to understand tissues you need to have the three dimensional layer and you can build a   
7:55   
structure there and you know uh try to replicate that complexity   
8:01   
and then that's that's kind of how You know, people think about that a lot. How to do that on different substrates,   
8:08   
how to make sure that they adhere properly, that they grow properly, because sometimes if you get the wrong   
8:15   
substrate, you can kill the cells off or they don't behave in the way that they would behave in the body. And then of   
8:22   
course, they're looking at like things like transplantation and other down the   
8:28   
road things. Yeah, it's uh definitely I think really interesting. um from that   
8:35   
standpoint, but also how cells are organized in different populations and   
8:40   
different functional populations. Yeah, you there's a a method of   
8:48   
stiffening gels just or um manipulating the stiffness of gels too. So, they   
8:55   
might be on to something. Yeah. Yeah. The Is it like an electro?   
9:02   
Yeah, add a bit of well, I don't know, you can put magnetic spheres in it, but   
9:10   
um also um give it electrical stimulation. So, there's a couple of   
9:17   
methods out there. I haven't studied them. I've I collected the papers, so   
9:22   
that's interesting, and moved on. I think I call that thesis fever   
9:29   
or something. Well, I think you just coined a new term.   
9:35   
Yeah. Well, I think that's going to be pretty   
9:40   
popular, actually. Oh, that looks interesting. I can't even   
9:47   
look at it. Yeah.   
9:53   
Oh, well. Um, anything else? I don't know if I I I   
10:01   
found a couple of papers on on networks of cells. Um if I find if I can find   
10:10   
them, uh would you like a couple of them? Yeah. Yeah, that would be good.   
10:17   
Okay. I don't think I buried them too far.   
10:24   
Yeah.   
10:30   
Well, yeah. I look forward to reading that. It was interesting.   
10:38   
Do you have anything else you want to talk about?   
10:43   
No, I'm I've had uh I just   
10:50   
trying to do do Christmas or New Year's season and Yeah.   
10:58   
So um like since you guys are here like   
11:03   
just as a beginner to understand stuff like uh what do you guys exactly research on and like I look forward to   
11:09   
joining the meets and understanding this in more in detail. So since we are having a meet I would uh I would go back   
11:16   
and further look into GitHub repos and everything but just as a starters like uh what is the research going on and   
11:22   
what would you suggest to me like what should I start with what come comes up somehow. Uh well I think probably the   
11:31   
the summer of code stuff um you know the repository that we have for hyperraphs   
11:38   
if you go to our YouTube channel you know we have things from you know if you search that channel for like hyperraphs   
11:45   
there uh presentations that people have given on it we have reviewed papers on   
11:51   
it and that you know going over that and looking at how we talk about it because we have specific ific interest in sort   
11:59   
of you know using biolog developmental biological data for this. So looking at   
12:04   
that re going through that and then thinking about some of the people who   
12:10   
were involved in last year's uh hype hyperraph project in the year before you   
12:16   
might get in touch with those people as well. Oh right   
12:22   
sure. Um who are those people? I would like just get in touch with them. blew up.   
12:27   
Yeah. So like Leith Baru was doing work last year on hyperraphs and uh neural   
12:34   
developmental programs and um Giadroth guy and was doing the work on temporal   
12:41   
graphs and yeah so people like that you'll see them in the uh GitHub repository   
12:50   
you know so if you see some code you're interested in you know look at that person.   
12:56   
Yeah. Sure, sure, sure, sure. So that project is running, right? So that's   
13:02   
active repon that's active project, right? Okay. Nice to hear.   
13:08   
Yeah. Yeah. Looking forward to contributing and helping. Yeah. Learning.   
13:14   
Yeah. Sounds good. Yeah. Thank you. Thank you. Yeah. Anna, happy new year.   
13:20   
All right. Yeah. Happy new year. Get going.   
13:25   
Thank you. Oh, well, happy new year, too.   
13:31   
Yeah, happy new year.   
13:36   
Although, I think my my family have stopped barging into my life here.   
13:42   
They're all back to work today. Yeah.   
13:49   
Oh, well. Um, I'm still interested in what um,   
13:57   
Dick wanted with the 10segrity information, but um, he isn't here, so I   
14:03   
don't know if he's looked at it. Um, oh, did you send him something on it or Yeah, I sent you and him a a dump of   
14:11   
about 10 papers. Yeah, I did see that. Yeah. happens.   
14:18   
Have you had a chance to look at it at all? A little bit, but I haven't really uh   
14:23   
you know, I don't have anything up and ready to look at. Um   
14:28   
Oh, okay. Well, I just I'm like I said, I'm I'm interested in knowing the   
14:34   
background math so I can study that before trying to read these things because   
14:40   
um I've already tried to read them. I know it contains linear algebra.   
14:47   
That's fine. Um but there's something else going on.   
14:52   
Yeah. Yeah. I I'll Yeah, I'll take I'll I'll   
14:58   
make an assessment of it and see what I think. Okay. Yeah.   
15:05   
Um thanks. All right. You're welcome. Okay. Talk to you later. So, I wanted to get into a   
15:11   
few papers here uh today. So the first paper is uh called non- Darwinian   
15:17   
molecular biology and this is by Alexander Palazo and Nevraj Quu and   
15:24   
they're from the department of biochemistry at the University of Toronto and uh their argument is you know   
15:32   
basically that we can't understand molecular biology without thinking about   
15:38   
something called molecular neutral theory. And molecular neutral theory has an interesting history with respect to   
15:46   
uh evolutionary theory and that is it introduces this non-darwinian force of   
15:52   
evolution which is neutral or a neutral drift. So we already think of evolution   
15:59   
as having four forces. We have mutation, we have selection, we have drift and we   
16:06   
have re combination. So drift is already one of those forces. But of course in   
16:12   
neutral theory we develop sort of a theory of drift and something that   
16:17   
operates at the molecular level which allows for things to drift basically out   
16:24   
out of function. So or in and out of function. So if you have a population of organisms,   
16:31   
that population of organisms is sort of a sampling of uh maybe a larger   
16:37   
population of organisms or it's a sampling of the genetic diversity inherent in life or in some fogyny or   
16:45   
some lineage in fogyny. And so you're dealing with a subsampling of these   
16:52   
things and you have because of that subsampling you end up with these frequencies of alals that are maybe not   
17:00   
representative of Hardy Weinberg equilibrium or some sort of static equilibrium that you would find in the   
17:05   
larger population. So that means that you can have all sorts of changes in   
17:10   
evol or evolutionary changes that are not subject to selection but subject to drift. And so in molecular biology   
17:19   
there's been this whole theory of uh neutrality. This comes from the work of   
17:26   
Mu Kamura in the 1960s and they've developed that from there. So it's very molecular and it's   
17:33   
basically um if you go to the biochemistry literature it's almost exclusively a   
17:38   
biochem biochemical sort of phenomenon. they talk about in terms of biochemistry exclusively but it   
17:45   
has an effect on the phenotype because obviously the biochemistry constrains the phenotype in that way. So   
17:53   
this paper is a nice review of neutral theory but it's kind of thinking about the field of biochemistry in general and   
18:01   
how you know basically it's they they think in their words they call it a   
18:06   
drift because they're not thinking about the questions that they're asking in   
18:12   
light of neutral theory. And so this is um this person Alexander Palazo is at   
18:19   
the University of Toronto. And if you go to the Sandwalk blog, which is a good blog if you're interested in finding out   
18:26   
things about um the theory of evolution and molecular evolution in particular   
18:34   
and looking at some sophisticated arguments for it, uh Larry Moran, who is   
18:39   
actually used to be at the University of Toronto at the Missaga campus, he uh you   
18:47   
know covered he's covered this historically over over like the last 20 years. And so this is a post from 2015   
18:56   
on functional RNAs. So in this paper they mention um the distinction between   
19:02   
function and nonfunction. So one of the things about neutral theory is it predicts that most of our genome is not   
19:08   
function. And there have been these recent um studies such as encode and other studies   
19:16   
that have proposed that the genome is largely functional that you can calculate up to 80 or 90% function for   
19:24   
genome or any genome for that matter. And uh what they're arguing is that   
19:29   
that's not the correct way to look at it. that a lot of these estimates of function are picking up spurious   
19:36   
function, spurious associations of transcription factors with binding sites. And so we need to think about   
19:44   
function when we talk about function, we need to think about it in terms of what neutral theory. And so that's been a   
19:51   
thing that Larry Moran has talked about a lot in his blog is this notion that junk DNA is necessary and of course   
20:00   
prevalent in genomes. Most of our genomes are junk. And the reason is is because this is predicted by neutral   
20:06   
theory. It's predicted by the evolution of genomes. the things that have gone   
20:11   
into the evolution of genomes. So our genomes are of course inherited from u   
20:18   
uh you know ancestral forms. Uh we have recombination events. We have different   
20:25   
types of mutational events. We have retroviruses that get incorporated in the genome. And all these things can   
20:32   
render genes nonfunctional. we have gene duplication events which can uh expand   
20:38   
the number of genes uh and some of those can evolve to some sort of non-function.   
20:44   
So there are all sorts of events that lead to sort of nonfunctional components to our genome. So we should expect to   
20:51   
see a lot of non-functional DNA. And so this is a post that he did on   
20:58   
functional RNAs and thinking about this same problem of uh DNA function in terms   
21:05   
of RNAs. And so this is kind of going through and they do site he does site a   
21:10   
paper in this post uh Pazzo and Lee 2015   
21:16   
non-coding art. This is a different paper from this paper. This is non-coding RNA what is functional and   
21:22   
what is junk. And so this is a good paper on um this distinction and   
21:30   
thinking about this in terms of evolutionary theory and um   
21:37   
neutral theory more specifically. So if you know anything about RNAs, you know that there's like this zoo of different   
21:44   
types of RNAs. you have uh you know there are all sorts of classifications   
21:49   
of RNA long non-coding RNAs uh intergenic RNAs   
21:56   
uh of course messenger RNAs mRNAs uh short RNAs you know all sorts of   
22:02   
things that we can think of um and so there's all sorts of uh different   
22:08   
classes of RNAs they're all produced from genes being regulated in different ways and they all you know maybe served   
22:16   
these putitive functions, but of course it's kind of hard to know. You know,   
22:21   
people name an type of RNA, how many of those are functional, if they're even   
22:26   
functional in the way that we would like to think of function. So, it depends on our definition of function. It depends   
22:32   
on what it's doing across the genome, this these classes of RNAs, etc., etc.   
22:39   
So, this paper is on non-coding RNA, what is functional, what is junk. This   
22:45   
paper is um so that paper was from 2015. This paper was from 2022 and this is   
22:51   
called non-doranium molecular biology. So this is a larger review of this idea   
22:57   
of neutral theory. So why don't we go through the abstract and we'll talk about like maybe what what are some of   
23:03   
the implications of this. With the discovery of the double helical   
23:08   
structure of DNA, a shift occurred in how biologists investigated questions   
23:14   
surrounding cellular processes such as protein synthesis. So instead of viewing   
23:19   
biological activity through the lens of chemical reactions, this new field use biological information and gain a new   
23:26   
profound view of how biological systems work. Molecular biologists asked the new   
23:32   
type new types of questions that would be inconceivable to the older generation of researchers   
23:38   
such as how cellular machines converted inherited biological information into functional molecules like proteins. So   
23:45   
this has been a question of course it's not only relevant to evolutionary biology but really goes back to the sort   
23:51   
of the early days of thinking about DNA as a as a helical structure as something   
23:59   
that you know basically is encoding things for proteins to express and to uh   
24:07   
turn into function. So there's always been this link between function and sort   
24:13   
of the the molecular millu of an organism. How does that molecular millu   
24:18   
turn into the the phenotype? How does it do things like metabolism and so forth?   
24:24   
But then there's always been this uh idea of function in biology. Some of that goes back sort of to the   
24:30   
pre-scientific the pre-scientific era where we thought about function very uh   
24:36   
very much in a non-evolutionary sense. So like functions can be things that do   
24:43   
something um that uh confer some sort of advantage or trait. And so we kind of   
24:49   
when we think about function we think and sometimes we think in functional terms like anything that exists has to   
24:56   
have some use. And so functional thinking is very uh seductive and   
25:01   
convenient for thinking about an order. you know a sort of a sort of a natural order or sort of an order of how things   
25:09   
should be. But of course just thinking about it in that way isn't enough. We need to find the mechanism. We also need   
25:15   
to figure out whether something actually is functional or if we're just give ascribing a   
25:21   
function that is spurious. In other words, like um thinking about, you know,   
25:26   
door handles and thinking about, you know, those are obviously have some sort of function because humans design them   
25:33   
for a function on things like, you know, doors or cars or whatever. But if you   
25:39   
see something that looks like a door handle in nature, it doesn't mean that it was put there by some sort of uh   
25:46   
creation process or by some even by some sort of evolutionary process. It just   
25:51   
means that it has this shape and indeed things like that might end up evolving   
25:58   
into something with a function where it might have no function at all. And we can't know until we find the mechanism   
26:05   
until we figure out the sort of the evolutionary history of this thing. And in fact, it's better to assume that   
26:11   
things have no function at all uh than to assume that everything has a function. So this is the world that we   
26:18   
live in with molecular biology but also biology generally evolutionary biology   
26:25   
etc. And so we have this collision of sort of modern techniques with sort of   
26:31   
remnants of sort of pre-scientific philosophy coming to together to think   
26:37   
when we you know influencing how we think about function and how we think about evolution.   
26:44   
So um because we were able to discover the structure of DNA, we were able to   
26:51   
sort of ask new types of questions. Um and some of those questions involve   
26:56   
things like how cellular machineries convert inherited biological information into functional molecules like proteins.   
27:04   
The new focus on biological information also gave molecular biologists a way to link their findings to concepts   
27:10   
developed by genetics and the modern synthesis. Now, when they talk about biological information, they're really   
27:18   
referring to the base pairs on a backbone of of a of a helix and then   
27:25   
converting that into a set of RNAs and then translating that into proteins into   
27:31   
amino acid sequences that assemble into proteins. So, that information isn't,   
27:37   
you know, the kind of information you might have in a book. People have tried to make these kind of analogies, but   
27:43   
it's really that you have specific combinations, base pairs that translate into specific uh amino acids that uh   
27:52   
contribute to specific protein structures and secondary structures as well. And so we have this uh information   
27:59   
that gets transferred. So it's specific sets of you want to call them   
28:05   
instructions but basically there's specific pieces of information you know   
28:10   
binary uh things that get transmitted and define things to look like one thing   
28:16   
instead of another and the reason I'm hedging here is because there's this temptation to again   
28:23   
go into functional mode and say well there's a code here that we can you know uncover and that it gets it's more   
28:29   
profound than it maybe it is otherwise. And so that's what I want to avoid. And that's kind of what we're getting at   
28:36   
with uh using neutral theory because the argument here is that a lot of this kind   
28:41   
of thinking about DNA and when you have a structure to how DNA is organized and   
28:49   
how RNA is organized that we kind of think of this in terms of being   
28:54   
organized in this functional way. And that's not exactly the way we want to think about it. The authors here think   
29:00   
that you know if you don't use something like neutral theory that we go astray and we kind of go off into these fields   
29:08   
of conjecture. So this new focus on biological information also gave molecular   
29:14   
biologists a way to link their findings to concepts developed by genetics and   
29:19   
the evolutionary modern synthesis which of course uh integrated Darwin's   
29:24   
thinking about natural selection with uh population genetics and gave us   
29:33   
mathematical tools to look at them to look at the changing al frequencies over   
29:40   
However, by the late 1960s, this all changed. Elevated rates of mutation,   
29:46   
unsustainable genetic loads, and high levels of variation in populations challenged Darwinian evolution, a   
29:52   
central tenant of the modern synthesis where adaptation was the main driver of evolutionary change. So, this is um   
30:01   
thinking about, you know, they say Darwinian evolution. And I guess what they're getting at is like that things   
30:07   
are all the product of selection. So again, we have this functional aspect   
30:12   
where you know we assume that something exists because it was selected for   
30:18   
because it had a function and that function was optimized through selection. and what began to find in   
30:24   
evolution or in molecular biology was that you had these things that spoke to things that weren't um you know selected   
30:33   
you know optimized through selection. So you have all sorts of especially genetic load and thinking about variation which   
30:41   
doesn't necessarily map to the strict functionalist view.   
30:46   
I'm not crazy about the language. I don't think Darwinian is like kind of the right way to think about it. But in   
30:52   
any case, uh this is a challenge of the modern synthesis   
30:57   
uh where we don't want to you know there's this tendency to think as adaptationists but of course there's   
31:03   
something else there which is of course where Kamura comes in and he developed this neutral theory of molecular   
31:10   
evolution. So instead of thinking of everything as sort of the product of selection and thus being functional, uh   
31:18   
he thought of things as being neutral. Most mutations for example were neutral   
31:23   
with respect to evolution. That you had genetic drift which of course was where   
31:30   
a population was subsampled and whatever alil frequencies were in that subsample   
31:35   
of the population became dominant in that small population. and persisted   
31:42   
in in the absence a lot of times of selection. So these are things that of   
31:48   
course exist in nature and we have to explain them and so uh mutu kura advanced the neutral theory of molecular   
31:54   
evolution which advocates that selection and multisellular ukarotes is weak and   
32:01   
that most genomic changes are neutral and due to random drift. So again,   
32:06   
random drift is acting as a force of evolution, pushing subpopuls away from that   
32:14   
equilibrium, doing the same work as selection might do, but not having the   
32:19   
same sort of functional consequence as selection.   
32:26   
Um, also you know the the kind of the point here is that selection is actually weak   
32:32   
in a lot of multisellular. So we shouldn't assume that selection is even particularly strong when we do see   
32:40   
things like adaptive changes. And so this is again um something that was   
32:46   
proposed in the 1960s. It was uh elaborated on by King and Jukes, Jack King and Thomas Jukes in   
32:53   
1975 in their paper non- Darwinian evolution where they point out that the observed changes seen in proteins and   
33:01   
the types of polymorphisms observed in populations only become understandable   
33:06   
when we take into account biochemistry and this new theory of neutral uh or new   
33:12   
neutral theory. And so we don't we can't necessarily say that uh you know we can   
33:19   
look at like phenotypes and we can assume function as a consequence of   
33:25   
selection and it makes a lot of sense right you have um functions you know you   
33:31   
see a function that's been selected for you see a function that's well adapted to its environment you assume the   
33:37   
processes is that this is all the product of a tight feedback loop of natural selection and function   
33:44   
So everything is kind of in place and it's um I guess it's hermetically sealed   
33:51   
in the sense that it's kind of you know this you we have say like   
33:58   
hands for grasping things and those hands of course evolved to grasp things   
34:04   
and so you you know there's no kind of room for molecular stochasticity there   
34:10   
because you trace it back maybe to genes for building a hand for grasping and you can map those genes to the phenotype and   
34:18   
everything is um you know makes sense. But of course what they're arguing here is at the molecular level at least   
34:25   
there's a lot of stochasticity there is a lot of data that suggests that things   
34:31   
don't evolve in this tight feedback loop that you have changes that are atypical   
34:38   
of a tight feedback between selection and function. And so that's how they're describing   
34:44   
sort of the world of the molecular millu of of neutral theory. So these were two   
34:52   
important works uh the Mu Kamura work in the 1960s and King and Jukes that's sort   
34:58   
of established neutral theory as I wouldn't say an alternative to uh the   
35:04   
synthesis but it kind of got incorporated in the synthesis a little bit and then you know it's it's it's a   
35:10   
biochemistry theory of course uh but it's something that is you know it's not   
35:17   
and it's not something that really challenges evolution. It's just something that we also need to take into   
35:23   
account. So it doesn't it doesn't really fall into the category of the expanded synthesis either. It's just kind of in   
35:30   
between. And so this is um something that we need to kind of incorporate into   
35:35   
our I think better into our theory and how it fits into our concepts of   
35:40   
function. And they talk about the encode study in this paper. They don't know it.   
35:45   
They they kind of bro they kind of broached the topic. But this was a huge thing back maybe about 10 or 12 years   
35:52   
ago where the encode consortium that released a bunch of papers that made the   
35:58   
claim that uh genomes were like 80% functional. And there were a lot of   
36:03   
people pushed back on that in something that was called the function wars where   
36:09   
they said well you know you can find from biioiniratics and uh genome   
36:14   
sequencing uh you can do these association studies with function and you can find very high levels of   
36:21   
function but uh neutral theory doesn't predict that and in any case we can show   
36:27   
that most of the genome is junk. So there's this debate between um you know   
36:32   
certain groups as to how much of the genome is functional and it draws directly back ties directly back to this   
36:38   
neutral theory. So 50 years later most molecular biologists remain unaware of these   
36:45   
fundamental advances. Their adaptationist viewpoint fails to explain data collected from new colorful   
36:50   
technologies which can detect exceedingly rare biochemical events. For   
36:56   
example, high throughput sequencing routinely detects RNA transcripts being produced from almost the entire genome   
37:03   
yet are present in less than one copy per thousand cells and appear to lack any function. So like the encode study   
37:11   
where they did these you know use these new technologies to say that the genome was 80% functional we can also use   
37:19   
technologies to show that the stoeometry of rare events or things like you know   
37:25   
if you transcribe a gene once every 1,000 in one of one you know one in every   
37:32   
1,000 cells that you have processes that produce things at a very low stoeometry   
37:40   
that are nonfunctional and if you add those together that that adds up to a   
37:45   
large proportion of the genome. So there are a lot of ways to look at like   
37:51   
biological processes and how they contribute to the variation that we see   
37:57   
and then trying to sort of classify that into function becomes hard because these   
38:02   
things are very rare. These events are very rare and they have a very low sort   
38:08   
of copy number uh have a very low stochometry in the cell. When I say   
38:13   
stoeometry, I just mean the the amount the degree of which this a molecule is   
38:19   
expressed. So sometimes if it's a variable with stochometry, you have you   
38:24   
know a lot of it can vary by its sort of stochastic nature. So you have one event   
38:31   
that can produce five molecules. You have another event that can produce one molecule and you have a lot of fluctuation there that is uh not in line   
38:41   
with a very refined function or any function at all for that matter. So there are all sorts of things like that   
38:47   
in biochemistry. We can uncover those with newer tools and it sort of supports   
38:53   
this idea that neutral theory explains the data better.   
38:58   
So you know this is something then they talk about how molecular biologists kind   
39:04   
of you know do a lot of experiments but don't have a guiding theory and this is the guiding theory that they propose.   
39:10   
uh molecular biologists must now reinccorporate ideas from classical biochemistry and absorb modern concepts   
39:17   
from molecular evolution to craft a new lens through which they can evaluate the functionality of transcriptional units   
39:25   
and make sense of our messy intricate and complicated genome. So a few highlights from the paper. Um so the   
39:32   
paper kind of lays out a nice history of uh evolutionary theory with respect to   
39:38   
lot of biochemical findings and how a lot of the early biochemical work really   
39:44   
contributed to sort of a challenge to kind of what we thought of as conventional evolutionary theory at   
39:50   
least looking down from the phenotype and thinking about what's going on at the molecular level. So one finding is   
39:58   
of course of the molecular clock. So there's this idea of the molecular clock where you know if you look at mutations   
40:05   
in different lineages um you have you know the assumption of   
40:12   
course uh in descent from common ancestry would be that those uh changes   
40:19   
not only sort of share a common origin but should mutate at a similar rate   
40:24   
because they're from a common ancestor. they should share that evolutionary rate   
40:29   
and they should share um mutations or they should share sort of a   
40:36   
that they should share an underlying um mutational structure. And so 1962   
40:44   
uh it was found that the number of amino acid differences between the hemoglobin proteins isolated from any two species   
40:51   
correlated with how long ago they had diverged from a common ancestor. And this was of course based on pale   
40:57   
weightlogical estimates. So you can think about like how long ago the common   
41:03   
ancestor existed between any two taxa and you can kind of say okay we should   
41:09   
see a certain number of amino acid differences if there's a common uh rate   
41:15   
of mutation and that's of course exactly what they found. And this is this concept of the molecular clock which is   
41:21   
often used to calibrate fogynies. And so this is an important finding that said   
41:28   
that basically um   
41:34   
that different proteins and protein paralogues evolve at a constant rate in different lineages. And so that was the   
41:42   
first kind of finding in molecular biology that related to ev in molecular biology that related to evolution. And   
41:49   
it kind of uh   
41:55   
it kind of suggested that there was this conservation of things um in terms of of   
42:00   
mutation. Uh the second part here is that   
42:08   
the second part here is that Jack Hubby and Lard Richard Lanton uh found that   
42:14   
different individuals within a species had a surprisingly high amount of variability for any given protein. So   
42:20   
not only did we have this molecular clock, but we also had a lot of variation in how proteins were expressed   
42:27   
across the species. So you had this interspecific intrapecific variation and   
42:34   
you have this interspecific regularity. And so it's just really interesting how   
42:39   
these things um deviated from the population genetics models that had been   
42:46   
proposed back in like the 30s and 40s. Uh so others noted that many of these   
42:52   
substitutions led to a mostly neutral change in proteins. uh which means that a lot of these mutations, a lot of this   
42:59   
variability uh didn't really mean anything in terms of evolution. It was just change in the protein sequence and   
43:07   
not necessarily change in corresponding change in function. And so this is when we get to Much   
43:14   
Kamura's 1968 publication evolutionary rate at the molecular level. And so in   
43:20   
this paper uh Kamura finds that the amount of protein variation between and   
43:26   
within species estimated by the new molecular biology techniques inferred   
43:32   
such a high substitution rate that if they all consisted of alals that were under selection, the cost of replacing   
43:39   
these alals would be intolerable. And they he uses the term genetic deaths. So   
43:46   
this is where wheels would have to be um sort of selected out of the population.   
43:53   
And so the idea is that there's this uh   
43:58   
that basically if you have this   
44:09   
that given the amount of protein variation, if we assume that there's some function to it, um that you would   
44:17   
have to keep killing off of wheels and that would make it would sort of undermine   
44:23   
The entire concept of uh   
44:34   
sort of functional improvement by genetic change. Basically, there wouldn't be enough genetic material   
44:40   
around to facilitate evolution as was proposed or natural selection was   
44:47   
proposed. So uh Kamura proposed instead that the majority of these mutations   
44:52   
must be neutral. Furthermore, Kamura demonstrated that slightly dilitterious or beneficial   
44:58   
mutations behave like neutral mutations provided the absolute value of their   
45:03   
selection coefficient was smaller than the inverse of the effective population size which is here. So basically um then   
45:16   
in cases where selection isn't absolute or isn't very high you have these neutral processes that dominate. So in   
45:24   
this equation here we see the absolute value of s is uh is smaller than this   
45:32   
one over the effective population size. The selection coefficient being S is the   
45:37   
average decrease or surplus in offspring in the mutant compared to a wild type individual. And the effect of population   
45:44   
size is the number of breeding individuals that randomly made in a species. So this is where we have we   
45:52   
have to consider population size. we have to consider the decreaser surplus   
45:58   
in offspring in a mutant population or in a population where those um molecular   
46:05   
changes have occurred and then we take the proportion of that. In practice the effective population   
46:13   
size is much lower than the actual population size as it is dependent on a   
46:18   
number of different parameters. In humans, the effective population size is estimated to be about 10,000, which   
46:25   
is a bit lower than the typical number for most other animals. If a mutation falls within the parameters of the above   
46:31   
equation, it does not clear what Michael Lynch calls the drift barrier and its fixation will be dictated by neutral   
46:38   
evolution. So, we have this idea that you know neutral evolution has sort of   
46:44   
can dominate when selection doesn't dominate. So when selection is weak or   
46:49   
when selection is sort of moderate, we can have these neutral processes and   
46:55   
they can dominate what we see as evolutionary change. And a lot of that neutral change is not necessarily   
47:02   
functional. A year after Kimura's paper, Jack King and Thomas Jukes independently   
47:08   
established a similar conclusion. This is King and Jukes in 1969.   
47:13   
However, many of their arguments relied on the biochemistry of proteins and the types of mutations seen in nature.   
47:21   
And so there's this debate in evolutionary biology, the neutralist selectionist debate. Some population   
47:27   
geneticists in the selection camp have advocated that most mutations enrich fixation because they are linked to   
47:34   
nearby alals that are under positive selection. This is evolution by drift.   
47:40   
Um, and then these arguments do not invalidate the idea that most mutations by themselves are nearly neutral. The   
47:47   
main thesis of the neutralist scale. Indeed, as pointed out by several commentators, rampant draft, if   
47:54   
anything, lowers the effect of population size and would thus further erode natural selection's ability to   
47:59   
weed out slightly dilitterious mutations.   
48:06   
So although many details about molecular evolution are still debated, the main tenants of the neutral theory have been   
48:12   
confirmed by numerous studies. Predictions made by the neutral theory match closely what is seen in countless   
48:18   
analyses of single nucleotide polymorphism distributions or snip   
48:23   
distributions and comparative genomic surveys. And then there's some references given there. So, one of the   
48:30   
take-home messages of this paper is that molecular biology is drowning in a sea of data and needs a sort of a unifying   
48:37   
theory or a form of knowledge that uh can unify the field. So, another set of   
48:43   
papers I'm going to review maybe next week or another time is this uh theme of   
48:50   
shape changes in soft materials. So this is a multifysics-based uh concept where they're using   
48:58   
multifysics and of course we talked about multifysics last year talking about different aspects of the physics   
49:04   
contributing to shape some phenomenon. So we're   
49:09   
interested in uh biomultifysics which is how biological soft materials   
49:16   
are formed by these different physical processes and how you can model those processes all at the same time. And so   
49:24   
this is a paper on multifysics based control shape change in soft materials. So I did a literature search on this and   
49:31   
I found that there was a lot of stuff on multifysics and soft materials but not a   
49:37   
lot in terms of how that affects sort of biological tissues and developmental   
49:42   
biology which is of course our interest. So I have a few references here   
49:50   
that kind of go through this idea. Um, one of these is from frontiers of   
49:56   
materials and this is an editorial on tackling the multiscale phenomena of   
50:01   
soft materials for engineering innovations. So this talks about the promises and challenges of soft   
50:08   
materials the role of multiscale understanding which is key of course not   
50:13   
only for just generic soft materials but for biological systems as well.   
50:19   
uh different types of initiatives, applications and engineering and the   
50:25   
need for mechanistic studies and all those of course we know about from our work in the group on tengities.   
50:33   
Um   
50:42   
so there's an u basically in this editorial they talk about the study of   
50:48   
soft materials being a cornerstone of material science. It's advancing what they call the materials 4.0 era which   
50:56   
we're I guess apparently in which is expected to drive the fourth industrial   
51:01   
revolution. There also furthermore initiatives like the materials genome   
51:06   
initiative that are at the forefront of addressing critical material challenges.   
51:12   
Soft materials have diverse and complex properties ranging from amorphous and hierarchical structures   
51:19   
uh ranging from amorphous structures to hierarchical structures and hold immense   
51:25   
innovative potential. Uh now their full exploitation is hindered by an elusive process structure   
51:32   
property relationship which remains a significant barrier to their application   
51:37   
across various engineering fields. Despite this advances in computational capabilities and experimental techniques   
51:44   
are paving the way for breakthroughs. So there is a lot of promise but there   
51:50   
also a lot of challenges. this process structure property relationship   
51:56   
is something I think that they're arguing can be understood through multiscale pro like understanding   
52:02   
uh soft materials and the multifysics of soft materials is this multiscale   
52:08   
endeavor and certainly in biology that's sort of the key aspect of this. So you know if we have say like a tissue the   
52:15   
tissue is made of cells the cells express proteins which are expressed by genes and we need to understand all of   
52:21   
those work together. Um so the structure the process structure   
52:28   
property relationship is central to material science yet soft materials remain poorly understood. The   
52:35   
relationship governs how the processing of a material influences its structure and how that structure in turn   
52:42   
determines its properties.   
52:48   
So for soft materials we have different phenomena that operate across different   
52:53   
scales and it makes it really hard to figure out this relationship. And so   
52:58   
this is similar to the idea that we talked about earlier about function and how function is you know instead of   
53:06   
function here uh we're talking about structure and function. So you know this is again uh a very similar problem. How   
53:14   
do we map what we're doing in our experimental work to this concept? And   
53:21   
maybe it's an outdated concept. Maybe we need to rework our concepts a bit. But that's that's kind of what   
53:28   
um they're getting at here. There's a lot of interplay between different processes. Uh things that happen at the   
53:35   
nano scale, for example, can significantly influence macroscopic properties, but it's hard to sort of map   
53:42   
that in a linear fashion from one scale to another. And so this is why people   
53:48   
are turning to multiscale modeling and simulation, which allows for examination of material behaviors across different   
53:54   
scales. Um and so Of course, the obligatory mention of AI and machine learning   
54:01   
methods and how those can be helpful, but I think also good conceptualization   
54:08   
and good models for transferring between scales is also important.   
54:18   
And then finally this need for mechanistic studies and how you know we can use those mechanistic studies to   
54:25   
sharpen our understanding of these phenomena and how they interact.   
54:34   
This paper which is an abstract um   
54:40   
it talks about multifysics based controlled shape change in soft materials. So here again we have this   
54:48   
idea of multi- um this idea of using multifysics as a way   
54:55   
to understand these hierarchical soft materials. So what they're trying to do is they're   
55:02   
trying to control shape change. So a pre precise controlled shape change   
55:07   
of responsive materials requires a development of theoretical computational   
55:12   
approaches capable of reproducing the involved physical phenomena in their interaction. When polymer based   
55:19   
materials are considered, shape change capabilities can be obtained from external stimula such as temperature,   
55:26   
light, humidity, electrical field, and many other things that can affect this   
55:34   
polymer. And so we have these inputs that can trigger shape change. Um these   
55:41   
stimuli are precisely spatially controlled. In other words, if we can control where the effect polymer, what   
55:48   
location and at what time can deform a structure under the desired configuration at a given time. So we   
55:56   
want to say for we take a developmental process for example, we want to deform a   
56:01   
certain part of an embryo at a certain time. we have to know kind of what that underlying process is and when some   
56:09   
stimulus uh of whatever type needs to be delivered at the precise time. And so   
56:14   
this in development we're also concerned about stimul. So some molecular cascade that's uh an   
56:23   
that's activated or some sort of a bioelectric uh signal that's activated   
56:28   
but the principle is the same. uh this paper investigates the   
56:34   
multifysics based uh driven shape change in highly deformable composite   
56:39   
structures made of a passive elastic structure and an active one capable of   
56:45   
responding to an environmental stimula. So this is where we have a passive   
56:50   
elastic structure and an active structure. Active elastic structure and   
56:56   
we look at these two different uh forms that are both triggered by envir   
57:02   
environmental stimula by harnessing different mechanisms uh which are these uh four mechanisms.   
57:11   
So the first is the swelling of gels. The second is deformation of   
57:16   
electroactive dialect elastimemers or dees. Uh   
57:23   
sorry the first one is the swelling of gels and the deformation of electroactive dialectric elastors or   
57:31   
dees. The secondary controlled mechanical stresses that can be transmitted to the passive portion   
57:37   
leading to its deformation. uh and this is done in the present study   
57:43   
by an elastic tube filled with a gel where the volume deformation is driven by the fluid uptake.   
57:51   
Three is a billayer structure made of an elastic substance and a DE or dialectric   
57:56   
elastimemer or active layer whose deformation is induced by an electrical potential   
58:02   
and four uh where these are considered as morphing structures.   
58:08   
So we go from the swelling of gels, the deformation of some sort of dialectric elastimemer.   
58:14   
They're controlled mechanical stresses that can be transmitted. Um this is done through an elastic tube   
58:21   
filled with the gel. Uh then we have of course the billayer structure   
58:27   
and then four is where they have these morphing structures. So it is shown that by properly   
58:33   
distributing the mechanical properties in this hyperlastic passive element interacting with the responsive active   
58:39   
material precise shape change could be achieved. To this end we numerically   
58:45   
solve the multifysics coupled problems and adopt a machine learning approach for solving the inverse problem given a   
58:52   
prescribed target shape. The proposed approach enables us to obtain the proper   
58:58   
spatial stiffness distribution of the passive electric element enabling the   
59:03   
achievement of the desired def deform configuration.   
59:09   
As you can see that you have a couple of different types of physics here. We have electrical physics, we have of course   
59:15   
mechanics, we have this elasticity and then we have these different sorts   
59:23   
of uh you know the physics of layers and the elasticity we have to account for. So we   
59:31   
have all these things working together to form these morphing structures. So again we're just we're just trying to   
59:37   
change the shape at a certain time in a certain location in in the structure.   
59:43   
And so we could map this to say like soft robots or other types of applications where we're interested in   
59:49   
this sort of morphing. But we can be more daring and think of biological systems and sort of poking   
59:56   
biological systems from the phenotype level. And so what's interesting is when   
1:00:02   
you start to add in real biological systems because of course real biological systems have a lot of things   
1:00:08   
underneath the hood, a lot of molecular pathways and using those molecular   
1:00:14   
pathways as things that can trigger changes as well. uh be maybe very interesting but maybe also make things   
1:00:20   
even more complex.
