Transcript     
0:00     
Hello. Hello. How are you? Um, fine. I I found um a GitHub with uh     
0:11     
tenseity on it. Um tensity programming     
0:17     
and but it's programmed in Rust. Okay. You have to learn.     
0:23     
It's another pardon. You have to learn Rust. Yeah. Yes.     
0:30     
Well, welcome Susan and Shreas and Curel. Hello. Glad you could make it.     
0:36     
So, uh, Carilla wants to present on his things. Uh, he's doing some sea elegance     
0:42     
modeling and um, you know, that's so uh, you would you like to introduce     
0:49     
yourself? So, this is, uh, Susan's, so this is her     
0:54     
link to the 10segrity lab software. Um, this is in Rust.     
1:03     
Yes, Rust. Okay. Like most of it. Okay. I said there was a bit of Python in     
1:09     
there, but Okay. Mostly Rust. Yeah. Yay. Yeah.     
1:14     
It's about the 10th computer language I'm going to have to know. This is ridiculous.     
1:20     
All right. While we wait for Carill and Carill's going to talk about his work on C elegance modeling. So, if you're     
1:26     
familiar with Azimov Press, they do a lot of popular science writing and     
1:31     
things like that. And this is uh what's it like to be a worm? And it's it's a     
1:37     
pretty long read. I don't I'm not going to go through the whole article, but basically it's about sea elegance.     
1:43     
And it's about um basically thinking about sea elegance behavior     
1:49     
and what it what is it you know is it like just kind of mechanical behavior or     
1:55     
deterministic behavior I mean if you look at a lot of the research on sea     
2:00     
elegance one of the reasons people choose C elegance to work on as a model organism is because it's very you know     
2:08     
you can trace it from a neural circuit to behavior that you can predict from     
2:14     
the neural circuit activity and from body movements there everything's very     
2:19     
stereotyped and so that's an advantage of like looking at different types of behaviors     
2:25     
but then they have this uh question of what they call sentience which of course     
2:31     
is not limited to sea elegance it's limited you know it's not limited to organisms biological organisms it's     
2:37     
limited it's it's been applied more broadly to artificial intelligence these     
2:43     
So this is just kind of goes through the history of sea elegance biology and as it turns out Darwin was really     
2:50     
interested in some of the behaviors of sea elegance that were sort of you know     
2:55     
maybe sentient and so he wrote a book on uh or he wrote I guess a text on     
3:02     
bioturbation which is the reworking of soils by organisms. So you know uh sea elegance     
3:09     
moves around in nature moves around through a soil column and collectively     
3:14     
they can basically change the composition of their home soil. So they     
3:20     
change the composition of the soil and they you know basically eat vegetable     
3:25     
molds. they like the bacteria and vegetables, rotten fruit and other things and then they uh basically     
3:32     
excrete things and rework the soil by moving through the column soil column.     
3:38     
So this is something that was quite interesting to Darwin. Um and it kind of     
3:45     
it's a reflection on animal sentience. Um this is the quote. Judging by their     
3:50     
eagerness for certain types of food, they must enjoy the pleasure of eating. Their sexual passion is strong enough to     
3:56     
overcome for a time the dread of light. Uh notably they were their their     
4:02     
hermaphrodites. So you know uh for the most part although worms are so remarkably deficient in the several     
4:08     
sense organs, this does not necessarily preclude intelligence. And we have seen that uh their attention is engaged. They     
4:17     
neglect impressions to which they would otherwise have attended and attention indicates the presence of a mind of some     
4:22     
kind. That's a nice 19th century quote that you know I mean it's just kind of a     
4:28     
reflection. There were no experiments done on that. Um whereas Aristotle wrote     
4:34     
about sentience in in organisms like bees and in other types of organisms     
4:42     
like fish. And what they mean by sentience, interestingly, is just basically those behaviors for which     
4:49     
there's no immediate functional imperative. Is that really sentience or is it something else? And this is     
4:55     
probably the problem with the term sentience throughout history is that you     
5:00     
have this very illdefined term. Um, and it's maybe not quite um, you know, what     
5:10     
we would like in a scientific context. Um, and so this just kind of goes on     
5:17     
thinking about some of the behaviors in humans that you know is related to     
5:22     
sentience. So this is an article on infant sense of pain. Um, and then     
5:28     
there's uh more stuff here on kind of linking sort of awareness to organisms     
5:35     
like sea elegance. So now we can explore the conneto. We can explore the     
5:41     
neurohysiology of an entire brain. As small as the elegance conneto is, it     
5:47     
still can produce coherent behaviors. And perhaps that's a way we can sort of     
5:53     
understand the um neurohysiology of sentience. If if such a thing exists,     
5:59     
then there's this term borderline sentience. While sentience is often defined as the capacity for pain and     
6:04     
pleasure, this definition is incomplete. Sentience in the sense we are using the     
6:10     
term. So they kind of use the term it's a philosophical version of the term basically it's from this book the edge     
6:17     
of sentience by Jonathan Burch and it's about the ability to feel pain     
6:23     
and pleasure and that's kind of where they kind of draw the line for this this     
6:29     
article. Um and then uh sentience is a combination of two things consciousness     
6:35     
and veilance. To say an organism is conscious means that there is something it is like to be that organism. That is     
6:43     
experiences the world from its own subjective point of view. That is how philosopher Thomas Nagel puts it in his     
6:49     
famous paper what's it like to be a bat? which is a philosophical paper on the     
6:54     
sort of you know uh ascribing the the features of batness to someone else and     
7:02     
you know being able to experience them. Veilance is short for hedonic veilance     
7:07     
from hedon the Greek word for pleasure to say a conscious experience is veilance is to say that is pleasant or     
7:13     
unpleasant that it feels good or feels bad and so you know we're looking for     
7:18     
that type of thing. Um and uh     
7:24     
so this this kind of just goes into the study of uh veilanced experience. uh     
7:29     
Ginsburg and Jablanca speculate that certain ancient animals such as jellyfish might experience a meaningless     
7:35     
white noise sensation and inner buzzing and crackling arising as a side effect     
7:41     
of the of their decentralized nervous systems. And so this just kind of goes on about     
7:47     
kind of reflecting on on sentience and this more specific veilanced behavior or     
7:54     
veilanced experience. And so that's and they get into things like persistent vegetative states and     
8:00     
other types of autonomic functions uh sleepwake cycles and thinking about     
8:07     
uh you know the boundary between awareness and non-awareness.     
8:12     
So this is uh you know and then they get into some brain organoid work which is interesting. Uh neurorgonoids offer     
8:19     
great potential for advancing neuroscientific research without the need for invasive experimentation.     
8:26     
or on humans and other animals. At the same time, the pioneer of neuroeththics     
8:32     
expressed concern that in growing something like a portion of a human brain tissue, we might unwittingly be     
8:38     
creating something like a portion of a human mind capable of positive and negative experience. just kind of a     
8:44     
point on on just you know talking about brain organoids and other um other types     
8:51     
of systems um as as an alternative to like brain     
8:56     
actual brains in a dish. Um and so then there's this limited behavioral     
9:02     
evidence. Conscious experiences are private in the sense that they are not directly observable from the outside.     
9:09     
When it comes to healthy adult humans, this does not pose too great a problem.     
9:14     
If a research psychologist wants to know what a patient is experiencing, they can ask. It would be handy if we do the same     
9:20     
with earthworms, but regrettably, they can't speak. The same is true for comeos humans and organoids.     
9:28     
Well, we ask a chatbot or a parrot what it's experiencing. Their answers are of little use as evidence given that     
9:35     
they've been trained to produce humanlike responses without necessarily possessing the underlying subjective     
9:41     
experience those responses typically indicate. This all goes to say that studying     
9:47     
borderline sentience forces researchers to seek out and validate markers of sentience other than subjects verbal     
9:55     
reports. So a lot of sentience research has been reliant on self-reports, verbal     
10:00     
reports and we'd like to have a neurohysiological um set of coralates for that. And so     
10:08     
this is kind of goes through um you know how this might be done and more     
10:14     
parallels with humans and then where neurohysiology could play a role here in     
10:20     
some of the findings. So that's good. Um this is something that you know run     
10:25     
across in reading Azimoff press. It's a good article. Uh there's a lot more     
10:31     
detail of course. So now Carella is here. He's been in the openor slack um asking questions and you     
10:39     
know especially in the divorm slack and he wanted to go over his repository that he's been working on for um simulating C     
10:47     
elegance. So, I'm pleased to welcome Carell and uh he can introduce himself     
10:54     
now and then share his screen and talk about his project.     
11:00     
Uh yeah. Hi guys. Uh hi guys. Uh uh uh my name is Kuriel. Uh I work uh at     
11:06     
Nvidia. Uh and I'm software engineer. Uh my uh specialization is graphics. Uh so I uh     
11:15     
work in a group called AI for graphics. uh but uh in my spare time I work on     
11:21     
this project uh which is a virtual cell essentially. So I'm trying to create a     
11:26     
model of living organisms in the computer. Uh and uh     
11:34     
and uh the cigans is a very good organism for that because there's a lot     
11:39     
of experimental data. Uh so that's why currently I use uh C eleants as the main     
11:47     
test for my uh for my code. Uh but it's not limited to C gun. I do intend to go     
11:55     
to other organisms as well. Uh and basically what I'm do trying to     
12:02     
do is I'm just looking for some uh some     
12:08     
some kind of experimental data. Uh and then I try to simulate uh this     
12:14     
experimental data in my uh in my requisitor. And uh right now what I uh like maybe a     
12:24     
couple of months now I was working on this silic gun embryo uh uh demo     
12:30     
that u so basically there is a video and I I I'm sure you all guys saw this video     
12:40     
uh like uh sans egg uh how it or embryo     
12:45     
how it divides how the cell divides. So I'm trying to uh reproduce this behavior     
12:52     
uh in uh in the computer. Uh so this is the video right. So this     
12:58     
is uh this is egg of seligans and there are two pronuci. So there is a male     
13:05     
pronus or female and female pronucles and they kind of move together.     
13:14     
Yeah. Boom. And here they meet. Right. So this happens at about 8 minutes time. So all     
13:20     
this uh like short sequence takes about 8 8 minutes.     
13:26     
Uh and uh here is my repository.     
13:31     
Uh so and if I uh right now this is what I have.     
13:36     
So I have also two uh produci. So I have an egg. Do you see it? Yes. Yes. It's good.     
13:43     
Uh so I have uh an egg. I have two pruclei and if I zoom in here so this is a     
13:50     
centrosome. So when when uh sperm deposits this to the egg it     
13:57     
deposits the uh male prrenucleus this is male prrenucleus and there's a central     
14:03     
zone uh just near it. So they here and now uh here in the left top corner there     
14:11     
is a timer. So this is simulation time. So if I start it then uh what happens is     
14:17     
that uh this centra zone uh it starts nucleating microtubules. So those mic     
14:23     
microtubules they uh they start growing uh and uh and what happens additionally     
14:31     
is that uh in the uh uh in the uh in the cytool so in this     
14:40     
liquid that's the cell filled in there is there are dinine motors and those     
14:45     
dine motors they attach to the to those uh to those microtubules and they start     
14:54     
uh working on it uh to towards the centros zone and this creates a pulling force on the centra zone uh and this is     
15:02     
the force that's uh that's uh working in this simulation right so this is the     
15:07     
force that basically makes uh this centone moves around right and     
15:12     
additionally on the on the surface of this prucleus there are also dine motors     
15:18     
they also attach to the microtubles so if microtuble comes flow to the uh to     
15:24     
the prucleus then there is a pulling force between the nucleus and the     
15:30     
centrosome uh created by those uh dine motors and this is what's what causes     
15:36     
the uh nucleus to move as well     
15:42     
so so and then at some point uh the uh the centrosome is duplicated so there     
15:49     
there becomes two of it so I can I can move the uh the microtubles. Now you can     
15:55     
see that there is a second centone now. So let's do and now they they both kind     
16:01     
of act on this uh nucleus. Then what's supposed to happen is that     
16:08     
they are supposed to like migrate to different ends of the prucleus because     
16:14     
of the forces that act on them.     
16:19     
Uh so and now what happens is that uh is that uh those microtubules uh since     
16:27     
since there are dine motors that are dissolved in the uh in this cytoplasm     
16:34     
uh the the force on the long microtubules is greater than the force on the on the     
16:42     
short microtubules. That's because simply there are more dynam motors that that can attach to the long microtubule.     
16:48     
So this means that the uh centrosome starts migrating towards the towards the     
16:55     
uh in the direction in which it has longer microtubules. So those     
17:00     
microtubules that are near the boundary of the of the uh embryo they cannot grow     
17:07     
too long. So the the force is not as as strong but the uh force on the on those     
17:12     
ends that are long it's it's greater that's why the uh end starts sorry the     
17:19     
uh nucleus starts moving in this text.     
17:39     
So now uh I guess it's hard to see what's going on because there are too many microtubles. So I'll just hide     
17:45     
them. So now you can see just the two centrosomes and two nuclei and now     
17:53     
resume simulation. So now uh when when those uh uh two uh     
18:02     
uh nuclei they are close by what happens is that uh     
18:08     
now those microtubules they start also reaching towards this second nucleus and     
18:14     
there's also a force on the second nucleus and that's why it starts moving towards the first nucleus. So now they     
18:21     
move together. So and now they met right? So this     
18:26     
happens about about uh 472 uh seconds. Uh this is what this is uh     
18:34     
472 / by 60. So it's about 7.8 minutes.     
18:40     
So it's about the same uh time time scale that's on the video. Uh so now I     
18:45     
uh after they meet uh the simulation doesn't work correctly. Uh so I'm     
18:51     
working on the rest of the simulation. Uh     
18:57     
yeah. So that's how it works. All right. And I have uh I have this uh uh uh I     
19:05     
have this repo in the in GitHub uh where uh so I sent invite to Bradley     
19:13     
and uh I asked maybe uh I I can send invite to anyone who is interested and     
19:19     
you can guys uh you could go go over it and uh uh give me some feedback and uh     
19:26     
maybe uh tell me what what is uh how it can be improved. Right? So there's a     
19:32     
So this is the repo. It's here. Uh and I can if you want I can go over uh like     
19:41     
general architecture of this and and describe it. Yeah, that would be great if you could     
19:47     
do that. Okay. Uh can you see still the can you still see the screen?     
19:52     
Yes. Okay. Uh so basically uh there's a lot of code and uh uh to organize this code     
20:01     
I I needed some kind of hierarchy. So I created a very hierarchical representation of this right. Uh so the     
20:09     
uh first level of organization is mostly by     
20:14     
uh science. So uh everything that has to do with biology in this folder then there's chemistry uh there's geometry     
20:21     
physics uh and and that's it right uh so and basically the uh organizational idea     
20:30     
is that it has to be uh so that uh for example that it's uh like one folder can     
20:37     
include the other folder but uh not the other way around right there shouldn't be any circular dependencies so this     
20:44     
means is basically for example the uh the demos right the demos is just the uh     
20:51     
executable uh it's the this screen that you saw right now uh that I was showing     
20:58     
the warm embryo simulation right so these demos can include anything they want right so this is the top level     
21:06     
uh then uh they include the viewer the viewer is uh the uh everything     
21:12     
responsible for rendering stuff on the So uh the viewer cannot include the     
21:17     
demos but it can can include everything else right. Uh then the viewer uh uh     
21:23     
then the next level of uh uh this hierarchy is the biology. So biology uh     
21:29     
it's it's very uh like to simulate bi biology we need all the     
21:35     
sciences. So biology can include everything except except viewer and demos right. So it can include     
21:40     
chemistry, geometry, physics uh and so on, right?     
21:45     
Uh and we can discuss we can go into biology in more detail later, right? Uh     
21:51     
then uh the next level is uh chemistry. Uh so     
21:58     
chemistry can can include everything biology and viewer and     
22:04     
demos, right? Uh then after chemistry, we have physics. Uh that's the next     
22:10     
level. Uh so then geometry and uh so geometry doesn't know about anything.     
22:17     
It's just uh basically triangles and uh and points, right?     
22:24     
Uh and then there's utils. Utils is just uh utilities that everybody use for the     
22:31     
for their stuff. So this is the lowest level utils. Uh and then we can go into     
22:36     
biology. So maybe maybe this is the because this is the biggest level right.     
22:42     
So biology is organized. So there is a top level uh and this top level uh it     
22:49     
basically if uh if like all other levels for example the viewer accesses biology     
22:56     
typically it accesses this top level. So this is the level that everybody sees right. uh and uh there is a uh cell     
23:04     
simulator uh and uh organism and important class     
23:10     
it's called data collector. So this data collector it's used for debugging.     
23:15     
uh it's uh uh what it does basically it uh it can     
23:22     
uh you can say uh I I want to like per every simulation step I want to collect     
23:28     
this and this data like a concentration of some particular protein for example right uh and then uh you you get as an     
23:37     
output a CSV file uh that contains this data right and I think every time uh I     
23:44     
run the simul ation it's like so so these are uh these folders here in the     
23:50     
data sim output uh this is the uh this     
23:56     
folder is created every time I run the simulation so right now I was running this folder was created and there is the     
24:03     
CSV file that's created so you can see uh it saved some data so there is a time     
24:08     
time there is uh this is time step this is simulation time and some data here.     
24:15     
So you see there's I was looking at gamma tubuline concentration. So there's     
24:21     
gamma tubulin RNA and gamma tubulant protein. And I was looking at     
24:28     
it because uh it affects the number of uh microtubules that are u created on     
24:35     
the centrosome. Uh and this is a centrosome     
24:41     
chromosome. I'm not sure what's what's the chromosome condensation.     
24:50     
Uh so one one important thing maybe to uh note is that most of the code is     
24:55     
created uh uh using AI. So basically it's a VIP coded     
25:01     
uh repository. So uh and I don't know a lot of the stuff that's uh that's here because I     
25:09     
rely on AI to know a lot of biology instead of me. Uh     
25:14     
so like this chromosome condensation uh for example uh I I'm not sure what it     
25:20     
is. I asked the AI that I I need to debug something and it just added this uh yeah this column. That's why I don't     
25:27     
know what it is. uh uh all right so uh so I was describing     
25:36     
this this top level so this data collector is the class that save this comma separated file on every simulation     
25:44     
uh and basically what what happens is that when I debug some particular thing I I'm telling to please modify data     
25:51     
collectors we need to collect this and this data so it modifies it and then I run simulation I get a new CSV file I     
25:58     
can look Uh so then the logic here is that uh we     
26:05     
have uh bioate. So the this bioate uh     
26:11     
project it basically contains the state of the of all biology.     
26:16     
uh so the idea is that uh it I separ I     
26:22     
separate the state and the processes that have that uh occur in the uh in the     
26:28     
cell. Uh so mostly I I'm trying to have a like here in this folder bioate I try     
26:36     
to have a very small amount of fun functions. So these are just uh uh this     
26:42     
is just data right uh just data that represents the state of the cell uh and     
26:48     
all the uh functions are here in the in the folder that's called processes. Uh     
26:53     
so there is a like if we open it we can see for example there is centrosome     
26:58     
process envelope stat state process microtubial dynamics process and so on     
27:04     
right. Uh so these are all processes they go they just run every simulation step they run one after another and they     
27:12     
uh modify this bio state in some way.     
27:17     
Uh     
27:24     
and uh the last uh for the last project that's here is called forces. So this is     
27:30     
basically uh sorry this is wrong this one. Uh so these are all the forces that     
27:36     
affect the simulation and the reason it's there is because because I have physics right and normally forces should     
27:43     
be in the physics. uh the these are basically uh adapters. So this uh uh     
27:50     
project physics it it cannot know anything about biology. So it operates with stuff like uh like uh uh     
27:59     
orientational spring for example or a mesh triangle mesh. So it doesn't know anything anything about biology. So and     
28:07     
uh uh for example the force if you look at the force uh it has just a virtual     
28:13     
function here it says uh apply. So uh the force has to apply itself and then     
28:19     
this function uh would be uh would be implemented here. For example, if we uh     
28:26     
if we look at this uh so implementation of one of the     
28:32     
biological forces then it implements this function apply right so it uh it's     
28:38     
uh it says okay this is a microtubial cortex force right so and it applies it     
28:45     
and then the physics engine it doesn't know that this force is related to microtubles it just applies it to a     
28:52     
sphere which is a nucleus Right? Uh and and again it doesn't know anything about nucleus. It just sees a     
28:58     
sphere. Right? So this is just data data and function. It's just organizational     
29:04     
thing. So I guess uh that's the level I wanted to go into. So if you guys have     
29:09     
some uh feedback right now or uh want to share it later then go ahead.     
29:18     
That's great. Uh thank you Caro for that overview. Um I know we have questions.     
29:24     
So yeah, I think it's great and you know we talked a lot about how to think about     
29:29     
these processes in the meeting. So like one of the things with cell division, you know, you know, how act you know,     
29:36     
how deeply do you want to model it? Is it good enough just to have a model of cell division where you're just counting     
29:42     
cell divisions and have like this blob that divides or it you know do you need     
29:47     
to have a lot of uh biochemistry simulated underneath or what's the specificity? So I like that you have the     
29:54     
specificity there. Um, you know, we we've done things with like compy cell 3D, which has uh kind of a different     
30:02     
approach to simulating cells, cell division, but you know, it's like, you     
30:07     
know, you always have details that you want to investigate. So it's always good to have different models that can     
30:14     
simulate those things from data because that's you know we deal we usually have like secondary data     
30:21     
sets that we mine and then kind of put them into simulations or or analyze them in some     
30:27     
way. So that really does give you you know a lot of flexibility if you can     
30:32     
bring in data sets and and benchmark things like that.     
30:38     
So, so you just mentioned some uh some package that you were using. I I didn't     
30:43     
catch it. What was it? Um, so yeah, we we've used a number of packages. One that of interest to at     
30:49     
least some of us is called Compucel 3D. That's a package. I think it's written     
30:55     
in C, but you know there's a community around that where they they use it for very general purpose biology,     
31:01     
but of course you can use it for development, but it's like not specialized for development. So, you     
31:07     
know, yeah, it's Yeah, James Glazer is the uh is the head     
31:14     
person on that. Oh, yeah. I took I took I took a couple courses under him during my undergrad at IU. Yeah. Yeah.     
31:21     
Um him as well as uh I did my undergrad at Indiana University. Um so there's him, James Glazier. He was     
31:29     
there from Compell 3D. There's also uh Dr. Derer, Robert Dyer from the physics     
31:34     
department. and he would do uh pivotal research on spikes which is basically uh the the     
31:42     
book that was one of one of the seminal books of computational neuroscience.     
31:47     
Yeah. Yeah. And what kind of things did you uh did you simulate using this package     
31:53     
using um using copy cell using using Okay. So with coffee cell 3D     
31:59     
coffee 3D um we did we did some pretty basic agents     
32:04     
uh with different types of cellular structure single cellular multisellular um seeing how they themselves would     
32:11     
would uh would respond to differences in environments mostly dynamics mostly     
32:16     
biohysical dynamics given constraints on um you know the sorts of like uh     
32:21     
decision- making like decision- making matrices uh minimizing or sometimes maximizing like like resources energy     
32:30     
available to cells available to the agency also cell sorting     
32:36     
cellorting is yeah you can do cell sorting with it it's an agentbased     
32:43     
model it's um kind of what is a cell sorting I'm sorry I'm not     
32:49     
familiar with this term sorting according to uh surface um     
32:56     
surface tension Um, and you can get uh cells of the same     
33:04     
type to clump together. Um, and some cells will uh surround the other cells.     
33:10     
So there's a sort of a so you can sort of make an organel.     
33:16     
Mhm. Using it. It can start with a bunch of different     
33:21     
cells and they'll sort sort together or sort in um kind of like     
33:27     
um an organel. Is it is it still in the context of C or     
33:33     
it's a it's a physics based um and chemistry based um modeling     
33:43     
but it's still modeling of the all the organisms CD cuts or different organisms     
33:50     
um just cells yeah it's a generalized program so like a lot of the simulation platforms well     
33:56     
some of most of them are you know Well, you can model generalized processes. Of     
34:02     
course, you know, we also want to have like specialized models because we have very specific modes of development in     
34:09     
different organisms like C elegance has a lineage tree that built, you know, you're building from. So, we want to be     
34:15     
able to capture that whereas in other organisms, the lineage tree is different or there's a different kind of way that     
34:21     
that's that's done. And so having that I think having like specific models and     
34:28     
also kind of a general capability is really kind of nice in a model you're     
34:34     
you're capturing maybe some very specific things and say select     
34:42     
so you can you know switch them between models like bioysics is generally     
34:48     
similar so you know that's something that goes across development but then you have very specific specific things     
34:54     
in in a species in a in a model organism that you want to get that.     
34:59     
Yeah. It didn't help me with my project which was integrity of epithelial cells.     
35:06     
Wow. So it wasn't it wasn't um it wasn't 3D enough for me I guess you'd     
35:14     
say. Yeah. And then there again like with that it's that's a problem of     
35:20     
simulations in general that you can't get like it's not a typical biologic or     
35:26     
it's not a typical physical model.     
35:33     
But yeah I mean you know I I don't know how much you've interacted with openworm people like uh seeing what they have in     
35:39     
terms of models. Um uh well I I met with Stephen Larson. He     
35:45     
was uh he was just showing me like so they they have this website where you can look at this 3D simulation of a     
35:53     
worm. Uh and you can look at different structures inside the worm. Uh and I     
35:59     
also he showed me some package that where a worm would be simulated like so     
36:06     
it would be like wiggling in some kind of volume filled with spheres.     
36:13     
uh it's it's called cybernetic I think right uh     
36:19     
and yeah so uh I will so my my goal was to like achieve something that uh that     
36:26     
we basically from the full development of a worm like starting from from this     
36:32     
one embryo so that it develops then it has to split to two cells then more     
36:38     
cells and so on and then in the end the worm would come out Right. Right. And it it has to be has to be like uh     
36:49     
the simulations should be good enough so that it it would allow some experimenting like if you did some     
36:55     
experiment on the on the virtual worm and then you did experiment the same     
37:00     
experiment on the real worm you would see similar results. So it has to have some predictive power. Right.     
37:06     
Yeah. Uh so uh so do you know how far like     
37:12     
because you have different models right different kinds of models that you are working on. So what kind of uh     
37:19     
predictive power does your models do your models have? Uh see for C elegance in development     
37:25     
we're fortunate because C elegance has a lineage tree that is deterministic     
37:32     
meaning that we can say like we can build a model where two, you know, we divide and you know, we have     
37:38     
cell divisions which are doublings of of the cell and then we know kind of where     
37:45     
those cells are in the embryo and what those division how those divisions     
37:50     
should proceed. So you start with one you know the one cell was the two cells     
37:55     
then you get to the founder cells of which you have eight and then they start to differentiate     
38:01     
into like germ line and and that's where you get like asymmetric divisions. So     
38:06     
we've built models around like looking at the sort of the the connectedness of cells     
38:13     
uh the uh sort of categorization of cells and that whole process of you know     
38:19     
building a phenotype from the cells meaning that you're the cells are     
38:24     
dividing and differentiating and forming sort of structure.     
38:29     
um you know and and so that's kind of the way what we've focused on not had     
38:35     
something of this detail where you're looking at specific events in cell division and then kind of building up to     
38:42     
that. I like that idea of having like you know focusing on the bioysics and     
38:47     
then it's like you can ex expand that out to the entire uh process of     
38:53     
development. But we don't have anything that that scale. we just kind of focus on uh the     
39:00     
sort of the process of building a phenotype. So you have cell divisions     
39:05     
from data and it's at the level of like the cell and then where those where     
39:10     
those locations are in the embryo and how those build to a phenotype. Um yeah. So     
39:18     
um I'd say a couple of things. Um there are many many many types of models     
39:26     
out there. Like it's kind of the wild west and that's quoting James Glazner     
39:33     
saying it's the wild west. So if you're interested in Compucel 3D, it's on     
39:39     
YouTube and he gives um a course in it every summer. So you can go attend it. I     
39:46     
attended it a couple of years ago. Yeah. You will look it up.     
39:52     
Yeah. Okay. Um     
39:58     
so I was wondering if it's possible to write like uh one uh best model that     
40:04     
that like simulates everything. Well, that's that's I I my philosophy is     
40:11     
probably not. I mean there, you know, different hybrid models that I would probably advocate for, meaning like     
40:16     
you'd have like models that do different things in in development and then kind of making those work together. So it's     
40:23     
like uh like well with openworm one of the things about what they're doing uh in in the main project is they have all     
40:30     
these parts and they're simulating like the the conneto and the bioysics the cybernetic     
40:37     
model and then they have a a viewer and those are all kind of you can have those     
40:43     
and you can get you know a certain understanding of the world you can run experiments but they're still kind of     
40:48     
disconnected. In other words, you can't build like a it's really hard it's harder to build an integrated model of     
40:55     
the worm. I mean, it's it's important to focus on the different things going on like the     
41:01     
physics and and the uh you know the the the neurohysiology and things like that.     
41:07     
But then integrating all those is very hard. Not only because it's like you     
41:13     
have kind of different imperatives with each type of model, but because you know it's like you have to really kind of     
41:20     
have the same framework very exactly the same framework to get things to work. you were showing how you have different     
41:27     
modules in your model like where you have the you know you're running some process and you know for biology or for     
41:33     
physics you have a physics engine and you're you know doing that and then you have the biology and and you have that     
41:41     
you have to coordinate all that across the world. So I don't really I don't really think that I mean I think that's     
41:47     
a very long-term goal to do like an sort of a single model that integrates     
41:53     
everything to have different models giving you know different insights that's maybe better but I think it's     
41:59     
also I I like hybrid models where you know you can try to get them to work     
42:04     
together in some way or you know you have some heterogeneity in the modeling     
42:10     
but you're kind of working towards a common goal. It's uh yeah I don't     
42:16     
I don't know we we've tended in this group to why do we why do we need a single model? Hey Bradley why do we need a single     
42:22     
model? Well, you know, there is this v vision uh that people have of simulating the worm and so like that's where people     
42:30     
have said open worm is succeeded or failed even though you know that's not really the people in the project don't     
42:37     
really think that but um I mean I guess the idea of a single model is like people think of development as this     
42:43     
single thing right development of C elegance is a single thing that happens and we we observe it and underneath that     
42:50     
is all this complexity And so you know you the single model is     
42:55     
just kind of like replicating but you know replicating the parts of course is important to understanding it.     
43:02     
They don't need to be a sing, you know, they can be all sorts of different models, but I mean the uh so so uh I mean     
43:09     
fundamentally it's not I'm not I'm not actually so concerned with say um accounting for all sorts of types of     
43:16     
information like the chemistry, the biology, the physics, the genetics. Um but more so uh with whatever models     
43:22     
you're using like so like like open worm being like a dig like a digital twin. You guys know you guys know what a digital twin is, right? When you Yeah.     
43:28     
Yeah. Yeah. So like open world being like the digital twin but also like some of the more like biomechanical ones like     
43:34     
compy cell 3D very much like biophysics biomechanically. Yeah. Um     
43:39     
what I what I was um you know what I what I was uh like like     
43:45     
like to to what extent can the rules be applied from one to another and evolved in the same way? So like if x then why     
43:53     
right? So if if something then like like if if you're exposed to this uh     
43:58     
condition this environment and the cells would divide following these rules or something like that right and um from a     
44:05     
uh from a from a single model perspective or generalizing from one to another um I guess you you'd um I guess     
44:13     
I guess there's there's there's this question to what extent are the models that we are building like openworm and     
44:19     
coffee self to what extent are they themselves meant to describe things as they are or um are they more so meant to     
44:27     
be this is what we know and we in some ways are trying to see how what we know     
44:34     
would would make sense in in some sort of other environment, right? are so so so are the models themselves do they all     
44:41     
have some sort of truth to them that we're trying to combine and generalize and make and and see how they relate to one another or do they all represent     
44:48     
some sort of similar ideas similar picture similar goals similar ground truth than we are all describing in a     
44:55     
different manner you know and that that's a really interesting question to think about like that's that's what that's what I think about     
45:00     
with these models so well I think the thing here is that like when we build a model we usually build it from what we know about biology     
45:07     
ology. I mean that's what we have to do because we know and we have we put data into it. But sometimes in biology we     
45:13     
discover new things and it upends what we know about some process and sometimes     
45:19     
that's like just kind of details and doesn't necessarily matter. Sometimes it's the big picture like I don't think     
45:27     
C elegance our knowledge of C elegance development is going to be upended tomorrow but there's like we've been     
45:32     
talking in the group about um uh centia you know cential cells and so that's     
45:39     
something we don't necessarily we can do like a simulation of that that's where     
45:44     
cells what are cential cells so this is where you have cells that are born and then they merge or they fuse     
45:51     
together and so like with muscle, not in sea elegance, but muscle, you get um     
45:57     
cells that fuse together. So you get a bunch of nuclei in a cell body and that's one cell.     
46:04     
Yeah. Yeah. And um so and then the you know there's this like there are these dual processes     
46:11     
of like nuclear dynamics or nuclei are dividing and the the cell is doing you     
46:17     
know some sort of function and muscle it's it's um trying to um you know     
46:23     
establish twitch dynamics and things like that but there you know so there     
46:28     
are reasons why cential cells kind of exist but we don't exactly know why and     
46:34     
how they form. form the way they do. So we could simulate that, but we might be wrong in terms of how they form and we     
46:41     
might get the parts right. We can we can simulate the nucleus, we can simulate the cell fusion and then maybe some of     
46:48     
the functions there, but like we might not be very no have a very good handle     
46:54     
on like you know how maybe why this is happening or all the varieties of of     
47:01     
ways in which it can happen and things like that. So yeah, I think Oh, go ahead.     
47:06     
So, so my opinion is that we need to go and simulate on the lowest possible uh     
47:13     
level that we can, right? For example, uh if I if I try to simulate right now on a level of atoms, it will not be     
47:19     
practical like because there are too many of them, right? So that's why I'm I'm not doing that. But if I could, of course, I would go     
47:26     
lower. It's just I don't have compute enough compute to simulate that, right? Uh and the reason we need to     
47:32     
simulate as low as possible is because we want to get emergent behavior. Like if I just say, oh, if I have the like     
47:41     
hair for example in and I uh move this hair over the worm uh and it needs to     
47:47     
twitch in some certain way and if I just program this behavior, right? Uh I I     
47:52     
just say, oh, if the worm is touched in this place, it has to twitch. Uh then it's not emergent. I programmed this     
47:58     
behavior. So of course it it behaves this way, right? So it's not very interesting for uh understanding and for     
48:05     
like uh to getting new knowledge. So the the point of this exercise is to get     
48:10     
some new knowledge. So you level you you simulate on a very low level. Uh from     
48:17     
this you get some emerging behavior and you compare this emerging behavior to the real world. Like if you if you if     
48:24     
the real world also behaves this way, this means you got some uh detail of simulation. Right. Right. And this is     
48:31     
what guides the simulation. You uh simulate you compare. If it's not right, you say you you try to figure out what     
48:38     
you did wrong in the simulation. Uh that way uh over time the model becomes     
48:43     
better and better and better and it like becomes really uh you get a lot of new     
48:48     
knowledge of it, right? Yeah. So that's why uh and and that's     
48:54     
why I think there could be one model because uh like if if the goal is to go     
49:01     
the lowest possible level then uh it's enough essentially to have it's just one     
49:07     
goal right so you the goal is to have this lowest level so uh so there could     
49:12     
be one model that goes lower than any other model and then it could be the only model you need. So, if there is is     
49:19     
just one model or if you're going to make one, it needs to include tenseity.     
49:25     
There we go. What is tensity? It's um a structure that's made out of     
49:33     
rods and strings. It's held up by tensioning the strings. And the reason     
49:39     
why tenseity is a part of biology is because all of our cells are in     
49:46     
tension and they operate like a tense segrity on the cellular level and also     
49:53     
on multiple scales of levels in between including bone and muscle     
50:00     
are be considered a 10seg object. Yeah, for sure. Yeah, it sounds like I     
50:07     
need integrity. Yeah. Yeah. Yeah. I did a my master's thesis on     
50:15     
tensity of epithelial cells. So like uh for example the worm it would     
50:22     
have tagged right because like all when all these muscles they they Yeah.     
50:27     
Yeah. The muscle and the skin and all its organs. Yeah. Yes. basically     
50:36     
y didn't say Rudy is um a principal. Yeah, I I guess     
50:43     
well yeah I think what Carell is like I think what's interesting about segregity being a structural principle     
50:51     
so any I mean I guess anything with a structure would have its own denseity whether it's a building whether it's a     
50:56     
so yeah but I think what was interesting about what Carel just said was that you know     
51:01     
you think about like DNA as being the underpinning of all biology or proteins     
51:07     
as being the underpinning of all biology. If you had a model of DNA or proteins, you could build theoretically     
51:13     
like all these different systems and generate like the you know the structure of an organism. Now, we don't     
51:20     
necessarily know how that like all those processes are very heterogeneous, but you have that lower level model that's     
51:26     
you know at least in part predict all of those other things. So, yeah, I think in that     
51:33     
Um sorry DNA is a tenseity. Um microtubules are a tense segrity.     
51:42     
Okay. Then I already have tensity because I have microtub.     
51:48     
Yes. Yeah. Exactly. Just to just to be clear by the way not     
51:53     
not everything not everything has tense integrity like it needs to it needs so so if something if something     
52:00     
like say expands or things in form in a way that's uh um I guess I guess I guess     
52:08     
okay like something something that's like continuous right something that's continuous and doesn't rely on that sort     
52:14     
of structure on how it operates so so like I guess like um     
52:20     
uh like like integrity itself has to be isolated like like b like bones like the     
52:26     
structures of like a bones and a skeleton yeah and they have to yeah it needs um it needs It's the     
52:32     
tensioned elements and the compression elements. Yeah.     
52:38     
Yeah. And the tensioned elements aren't necessarily elastic. Like they're more like a string than an elastic to operate     
52:46     
properly properly. So it's they're otic. They're they don't     
52:51     
have proper prosonance ratio. If you're getting into mechanics,     
52:58     
they're oic. Okay. All right. Um actually     
53:04     
so returning back to this uh one model right discussion. So uh I I was thinking     
53:10     
uh uh so let's say we want to build one model that goes as low as possible right     
53:17     
and it's probably very like large amount of work right uh but I think so maybe     
53:24     
it's not achievable uh but I think it's a it's a interesting goal and uh like if     
53:32     
if we look at this repository I was just showing you right we can look at it as     
53:37     
some kind of some kind of organism that develops over time like like in like biological     
53:45     
organism they experience evolution right? Yeah. So and and we kind of uh think of     
53:50     
ourselves as the force of evolution. So we we go and do this repository. We say     
53:56     
okay we we do very small change because valition goes in very small steps right. So we we do those small changes that     
54:04     
make it better and better and closer to uh to     
54:09     
biological reality. That way this organism kind of evolves and over time     
54:14     
if we have enough time and enough persistence it will     
54:20     
it will just uh achieve some some something right it will it will become     
54:25     
great. Yeah. It will be able to simulate everything. Yeah. And and also simultaneously with this     
54:33     
the uh because it needs a lot of compute right. So essentially if you want to simulate something on low level and     
54:39     
something big then you need a lot of compute power but uh the uh right now     
54:45     
it's a great time because uh everybody is building compute for AI right so     
54:50     
there are trillions of dollars invested into uh expanding of compute uh and     
54:55     
building data centers right yeah so simult so and compute becomes cheaper     
55:01     
and cheaper and cheaper so simultaneously with model becoming uh bigger the computer will become bigger. So, it     
55:07     
kind of will all scale together. Yeah.     
55:17     
Here's a tenseity. It demonstrates a behavior. If I push on     
55:23     
it here, the whole thing contracts and if I pull on it, the whole thing     
55:29     
expands. Mhm. And if you have other types of material,     
55:36     
you'll have plus ratio where it if it thins in one direction, it expands in     
55:42     
another. This does not. It compresses all together when you push on two sides.     
55:50     
And when you pull on the sides, the whole like the whole object expands like     
55:56     
it it works to it. It's connected all together and it moves all together.     
56:06     
Okay. Yeah, that that's my best explanation right at the moment.     
56:12     
I will I will interrogate AI about how it can help me in this development.     
56:17     
Maybe. Yeah. Looks like Morgan maybe wanted to say something. Did you want to say something     
56:23     
Morgan or Sorry, I I just got back from dropping     
56:30     
my daughter at school. Okay. But I I was my my one comment I put in     
56:37     
the chat just about, you know, to     
56:44     
point about, you know, what what's the kind of scope of the models     
56:49     
that that you're interested in. I I kind of I like describing them like     
56:55     
on a chip experiments, you know, where it's like you're not you're not actually making an organ, you know, you're you're     
57:02     
actually, you know, kind of fabricating the the particular process under study,     
57:08     
right? Right. Like like what what are the important elements? What what are the pieces that you can put together? Um     
57:17     
and you know just having having succ     
57:23     
that's that's James Glazier's you know seminar series like like that those are     
57:29     
the kinds of things that they seem to focus on you know like what are the what are the important elements that I can     
57:35     
simulate and and and get at some sort of mechanistic model you know interpretable     
57:43     
model etc But I'm enjoying the conversation and     
57:49     
and now just uh yeah still still starting the day here on the west coast so     
57:54     
forgive me but I'm I'm having breakfast. All right.     
58:01     
All right. And then we have you know some people here working on u like Baron     
58:06     
and Giadra on graphs and developmental graphs. I don't know if they wanted to     
58:12     
have say anything if they had any input.     
58:17     
Uh hello are am I audible? H yeah. Yeah. Okay. Okay. So let me give a small     
58:23     
intro about myself. I'm Ban. I'm an undergrad family undergrad from Amita which is in India. So I had like all     
58:30     
this whole conversation I had a bit of doubt. So as Bradley said that there there a lot of this models which are     
58:36     
trying to simulate the war and all of them are developing it on different concepts and some on biology some on     
58:44     
physics some on chemistry. So uh under the same environment and the same conditions. So uh would they like     
58:51     
contradict each other's movement or the what they are acting upon? So that is     
58:56     
possible right? Right. all of them are contradicting whatever the other one is.     
59:03     
So maybe we could create something uh and find similarities in all of those models and uh create a better simulation     
59:12     
out of that. I think that would be a great thing to go for. I mean that's     
59:18     
just overview like idea I got. Yeah.     
59:23     
Yeah. I mean if there's something something useful uh in in uh some models     
59:29     
so models can can be combined right like if uh there's a lot of I guess code     
59:35     
that's written for simulating some particular uh thing in a worm uh maybe     
59:42     
we can just uh tell AI like get this module and integrate it into this common     
59:49     
model right so there could be common model that uh brings together everything     
59:54     
uh that's useful from all other models and uh and it's currently quite uh easy like     
1:00:03     
this AI program because uh one human could be not able to do it because it's     
1:00:09     
like a lot of work but uh AI uh we can just ask AI to do that and it's it     
1:00:15     
becomes much easier so one can do like 10 times more than she could do before.     
1:00:23     
So if there is some particular model that you want to suggest that uh that that you think is good, you could post     
1:00:30     
it and uh I I could look into it if it's uh if I can borrow something for my     
1:00:36     
model for example from that model. Yeah.     
1:00:43     
Um what is what language is your model written in or what do you usually work with in terms of     
1:00:49     
uh it's C++ C++ yeah but if if there's something on Python I     
1:00:54     
I'm sure like the uh uh like it it could     
1:01:00     
be all converted I use C++ because it's very high high performance right     
1:01:05     
and it looks like the calculation can be become a bottleneck very fast so since I     
1:01:10     
want to go as low as possible I need to language that like provides the best performance,     
1:01:16     
right? Yeah, I think C++ is a good choice for the kind of sim simulations.     
1:01:24     
Um I just wondered since you're working in Nvidia, do you have you parallelized a lot of or some of the stuff or you     
1:01:31     
Yeah, I didn't I didn't yet. So I I'm so right now the simulation runs faster     
1:01:36     
than real time, right? So I didn't have much incentive to like even use multiple     
1:01:42     
cores. Right now everything is on one just one CPU core. But I I do envision when when it gets more complicated     
1:01:49     
because right now I have like just few genes few proteins flowing around. Uh so it's not it's not a big deal to simulate     
1:01:56     
but I guess I I will start thinking about paralyzing. Yeah. So     
1:02:02     
well that's just always a question. Um yeah, no this is good because we you know we of course have limitations of     
1:02:09     
computational power and trying to figure out how to build a model with without h     
1:02:15     
I mean you know you may have access to a lot of computational power but you know     
1:02:20     
it depends on like if you want to run it locally or if you want to run it as like a you know um an individual people might     
1:02:29     
not have that kind of power to run their own simulations. So you know it's always a thing you have to think about kind of.     
1:02:36     
Yeah. Right. So the the compute it's a it still costs money. So everything like     
1:02:42     
as if I need a lot of compute then it will be expensive and and then we come     
1:02:48     
to uh the problem of financing. How to finance this? All right. So uh and to     
1:02:53     
finance this I need some some application that can be useful to     
1:02:59     
people. Like if I just show this uh uh very simple simulation like nobody can     
1:03:06     
really use it for anything right it has to become useful so I'm looking for uh for     
1:03:13     
some application where I can say oh if you do this you can earn money with it     
1:03:19     
right so and uh for example I'm looking at uh uh     
1:03:25     
there are a lot of companies that use uh that use uh uh biological organism to     
1:03:31     
synthesize things to like for example food, right? People build artificial egg     
1:03:38     
whites for example or some kind of bofuel or uh for example protein for     
1:03:44     
artificial meat, right? So if I if we can if we can use this model to uh to     
1:03:50     
somehow improve the the uh uh the output of this production like I can help these     
1:03:58     
people to optimize their production and to reach higher volumes of production     
1:04:04     
with the same money then I can I can get more money and uh this will uh open new     
1:04:11     
possibilities right? Yeah. So, so if if you have ideas how to     
1:04:16     
like how to use this for example I I want to so right now my reposi repository just has work simulation but     
1:04:24     
I want to add this simulation because a lot of companies use this for like bio     
1:04:29     
production. Yeah. And uh and then I I I want to try to like simulate some uh     
1:04:39     
some uh metabolic model basically. So there are a lot of metabolic models I want to add it to the repo so I can uh     
1:04:47     
use it to pitch to these kind of companies. Yeah. Yeah. I think that's good.     
1:04:54     
Yeah. I I'd be very interested in the yeast and metabolic models. Oh, really? Okay.     
1:05:00     
Then I continue and like when I when I have some questions.     
1:05:06     
Yeah. Yeah. Yeah. I've got some people here, you know. So, I'm in a synthetic biolab here in San Francisco.     
1:05:13     
So, those are those are the kinds of things that people would like to see. Oh, nice. Then I can drive there because     
1:05:19     
I'm I'm in Los. Oh, okay. Uh, gotcha. Gotcha. Sure.     
1:05:24     
Sure. Okay. Yeah, we're we're right downtown. Okay. Nice.     
1:05:30     
Yeah. Right. Yeah. And then I just wanted to mention that of course there's a lot of     
1:05:36     
this work being done on whole cell modeling where people are uh simulating     
1:05:42     
uh different types of bacterial cells and the idea is to get as much detail about the cell as possible the kinetics     
1:05:49     
and then simulate like a certain amount of the uh you know of time that goes by     
1:05:54     
in the dynamics. So there are a lot of you know there are a couple of groups in in the world that are working on this     
1:06:00     
sort of model where you know they model every sort of component of the cell at     
1:06:06     
sort of like the protein level DNA level and then have like an entire one entire     
1:06:11     
cell and then they simulate the dynamics and that of course takes a lot of computational power. So they're     
1:06:16     
simulating like maybe a second or two seconds of of dynamics. So that's kind     
1:06:23     
of interesting stuff because it's like you get this really high level of detail and then you can simulate it for a     
1:06:29     
little while and that's good enough to sort of understand what's going on. Um,     
1:06:34     
and then of course that contrasts with like simpler models which maybe you can     
1:06:40     
run for longer. Maybe you could run it for like a full course of like C elegance developed and but you couldn't     
1:06:47     
of course run like a whole cell model for that long to simulate the divisions because it would just be way too much.     
1:06:53     
You'd need way too much computational power. But of course as you mentioned we're kind of getting more and more     
1:06:59     
computational power online where we can do this sort of thing. Um, Yeah. So, it's really interesting um     
1:07:08     
the potential, you know, that's that's really what we want to look at. Yeah.     
1:07:13     
So, do we have any other questions? Yeah. Is there uh still some time left     
1:07:19     
to Yeah. Yeah. Go ahead. Yeah. Actually Srias have built a system. He want to share that uh his     
1:07:27     
work on how connetos of C elegant develop in time domain using graphs and     
1:07:34     
neural uh ordinary differential equation. Uh he created a generative model uh that     
1:07:41     
can show some intermediate intermediate time steps of C elegance its neural     
1:07:48     
connection development. So if uh st is ready please go ahead.     
1:07:54     
Uh yeah hi everyone um just introduce myself. Yeah. Uh I'm currently a third year     
1:08:02     
underground at IIT Madras. It's a college in India. So yeah um regarding     
1:08:07     
the presentation which K here it was like quite interesting to see the visualization of the uh of the embryo     
1:08:15     
and yeah regarding the point which we were discussing like a one model for the whole thing I kind of like I uh with I     
1:08:24     
agree that like I guess the discrete model can be good but yeah uh so uh     
1:08:30     
should I uh proceed to with the presentation? Um yeah just yeah okay so     
1:08:36     
yeah just I guess we're transitioning and so yeah thank you for uh presenting     
1:08:42     
Carl um yeah we had a lot of interesting and we'll we'll continue on if you if you'd like to attend future meetings     
1:08:49     
you're more than welcome and we'll keep talking about your model and and thinking about how to use it maybe you     
1:08:55     
know in different ways okay thank you all right thank you so you like to     
1:09:02     
present on your things. Yeah, sure. Okay. Okay. So, I'll start the presentation.     
1:09:08     
Uh, so the the data I've used is the um weight data in this GitHub repository.     
1:09:15     
It's basically uh an Excel sheet where there is like I'll just show the data.     
1:09:21     
Yeah, it's of this type. So, there's like pre post uh and a type and a synapse as the number. Um, yeah. So,     
1:09:28     
this is the data we can which we can see in this GitHub repo. So I would first like to discuss the article from which     
1:09:34     
uh I I use this data what it says. So it says that the it says it discusses many     
1:09:41     
ground truth biological facts about the conneto and how it is structured uh some of which can be related and I mentioned     
1:09:47     
below. So like the so it's a it's a symmetrical wiring. So the increase in synapses during L1 L1 is the basic like     
1:09:54     
the uh uh stance of the PP embryo. So during that the left and right wiring increases symmetrically. So this is     
1:10:01     
something which can be useful for us. So the new connections are more likely to form between neurons that are that are     
1:10:06     
shared larger physical contact areas at birth. So it sort of like forms a clusters around the larger physical     
1:10:13     
contact areas. And so each connection is classified as stable development dynamic     
1:10:18     
and variable. So like say if a connection is stable that connection would go on for the entire conneto     
1:10:24     
interpolation like across all the stages. If it's variable the connections could change in between. Right? So this     
1:10:30     
is something which I thought can be useful and can be used as a biological bias. So yeah these are the important     
1:10:36     
facts which I found from the paper and yeah I'll proceed. So yeah uh this is     
1:10:42     
something which is uh interesting. So like from this figure we can understand that there are various types of neurons     
1:10:48     
which are connected in a particular hierarchy right so the hierarchy goes like that. Uh is this visible or should     
1:10:55     
I like zoom? Yes it's fine. Yeah it's visible. It's fine. Okay. Cool. Yeah. So, so we     
1:11:01     
can see that it goes like uh uh sensory neuron then inter neuron and then motor     
1:11:07     
and then muscles and here like we have a hierarchy build up and then these are the way the neurons are connected. So     
1:11:14     
these are the four four biological facts which like these are not like inclusive of all the facts are like which I found     
1:11:20     
can be relevant. So which I which I will try to use in my model like have a biological bias because just having     
1:11:26     
mathematical bias I don't think would work that well having a biological bias would be good as we doing physics     
1:11:32     
informed networks so yeah so the data prep-processing so the initial data is     
1:11:37     
in the format shown on the right this is the initial data which I had and in total eight stages of C elegance     
1:11:43     
connects are given the initial four stages are of part of the L1 the next two are L1 L2 L3 and the uh following     
1:11:51     
Next to like the eighth and the seventh and eighth file of data are the adult uh     
1:11:56     
this is this is written in a paper and then uh so this is converted into adjacency matrix. So like what is     
1:12:02     
adjacency matrix? It's basically a n cross n say I have 295 neurons. So basically n cross n matrix and in each     
1:12:08     
cell it's written if if the neuron is connected to the particular neuron and     
1:12:13     
like what is the strength? Say it's it has five connection between those. So basically adjacency matrix is uh uh is     
1:12:20     
created where all the unique neurons uh are the rows and columns and the number of connections between each pair is     
1:12:26     
written in the corresponding cell. So later the metric scale is converted to binary for simplicity because as of now     
1:12:32     
currently I'm just modeling if a connection is there between two neurons and later I'm scaling it to the strength     
1:12:37     
like how much how uh like how many connections are there but as of now I'm just doing it finally     
1:12:43     
like if a connection is there or not and then uh a mapping graph is created for GCN. So what is a mapping graph? I'll     
1:12:50     
just show it here. Yeah. So like basically this is a mapping graph. So like say this is 0 0 1 0 15 1. So like     
1:12:58     
basically signifies that 0 is connected to neuron 1 like 0 like neuron 0 is     
1:13:03     
connected to neuron 2 and so on. So yeah so I'll just uh go through like what     
1:13:10     
this is this can be like uninterpreted as of now but I'll just go through. So for showing I showed like the mapping graph which basically maps the since it     
1:13:18     
is a graph so we need edges edges and like message passing. So to understand which neuron is beside which one and     
1:13:23     
what are the connections that mapping graph is created uh okay so I'll go to the methodology next so the methodology     
1:13:30     
is broadly divided into two parts which is the GCN the encoder and the neural decoder is just a small part of it um so     
1:13:38     
yeah so what the the GCN does is like it's a sparse graph connection uh it     
1:13:44     
basically tells who connects to whom and the adj and the input is the adjacency matrix which I previously told you So     
1:13:51     
here what here I've used the term identity adjacency matrix. So what does that mean is like all the neurons are     
1:13:57     
stacked in the rows and columns and uh uh that it's it's identity matrix. So all the 11 one in the diagonals in rest.     
1:14:04     
So why this is so so basically what we're doing is like we are having a     
1:14:09     
column for each neuron say uh say this way right so this is each neon this is     
1:14:14     
adjacency matrix the initial adjacency matrix. uh I just wrote in my iPad and     
1:14:19     
put a screenshot here because it was difficult to like type this out everything. So yeah so this is my input     
1:14:25     
adjacency matrix where uh so yeah diagonals are ones and rest     
1:14:30     
are zeros. So this the values of this will change and evolve through the graph network and it will learn its     
1:14:36     
neighborhood. So this is the moto of it. then the action. So feature sharing is     
1:14:41     
done as I told so it will look across its neighbors like all around me and it will change its uh vector we can say     
1:14:48     
like each neuron has a vector and it will change its values and then it learns how to compress the graph     
1:14:53     
topology uh each column of adjential matrix into meaningful starting coordinates Z star for the neural right     
1:15:00     
so now I'll go in the mathematical terms of a graph convolution network so yeah this is my     
1:15:06     
uh input in input to it so Again the edge index as I told you previously this is my map as to I guess this is a like I     
1:15:15     
basically wrote two here it should be it should be 15 like 0 connected to 15 neg one connected to four and so on this by     
1:15:22     
two my bad um so yeah so this is basically why we having one two three in a one one in a diagonal fashion is to     
1:15:29     
identify each of the neuron uh in a unique way it's like basically identity     
1:15:35     
of each neuron and the ID of each neuron and then These are my inputs. The tensor     
1:15:40     
this is created from the data which I have the excel sheet which I showed previously and then the left side is my     
1:15:47     
adjgency matrix. Now uh now each column of the adjacency matrix is processed individually and updated. So these are     
1:15:54     
each processed individually. So as we can see this is the initial say this is a z. Now this is my a1. So a1 is     
1:16:00     
basically this function value of feature like this this feature plus its neighborhood sum it neighborhood sum     
1:16:06     
into the weights. So w is what is learned. So this is my graph. So say for     
1:16:11     
zero zero zeroth is connected to the first one and to the 15th one and say something else later. So this will     
1:16:17     
basically model I'm connected to so and so I have a weight which I have to learn and uh it     
1:16:23     
will learn it. So this learning will be done along with the neural lo. So this is like there is not any individual loss     
1:16:29     
function on the GCN. GCN plus the neural load altogether has a loss function in the end. So I'll come to it later. So my     
1:16:36     
output is basically 16 + 225 compressed ridge matrix. So my uh vector length     
1:16:42     
decreases from 225 to 16 and the feature vector of each neuron has learned the info about its connection and it     
1:16:48     
neighborhood. Uh so yeah I'll just stop here for a second if there's any doubt in the graph convolutional network     
1:16:53     
because again on this is what neural load is built on. That looks good. Yeah.     
1:16:59     
Correct. So yeah I'll come to neural load now. So we assume that the connectto like yeah so just to come back     
1:17:07     
like this output of this is the input for the neural. So we assume that the conneto latent space which is the output     
1:17:14     
of GCN I'll call it connect latent space since we have compressed it the latent space is uh is defined is is governed by     
1:17:21     
continuous time dynamical system. So this is assumption which can be a flow of it also but yeah as of now I just     
1:17:27     
assume it and then we define an initial value problem integration like everyone know initial value problem. So this is     
1:17:34     
basically uh like I'm modeling how ZT okay so I guess I did abuse of notation     
1:17:39     
I didn't write here ZT but yeah this this this matrix which is compressed and 16 + 25 size that is the ZT that is my     
1:17:47     
ZT okay so it evolves with time so so d of zt by dt uh is I'm assuming it's a     
1:17:55     
function of zt and time and theta are my are my learnables so as you can see t     
1:18:00     
here are the continuous variables time readiness variables and f theta is a     
1:18:06     
learnable vector field parameterized by the weight theta. So it's basically neural network as you can see f theta is     
1:18:11     
parameterized as a multi-layer perceptron that operates on node feature. So what is node feature? Node     
1:18:16     
feature are basically these features. So these these learn features. It will act on those individually. Um so yeah these     
1:18:24     
are my equations which are the initial values. This is this is my initial value for solving this. So I'll go through     
1:18:32     
step by step. So the first step is temporal concatenation. So since time is the important factor here. So I'll     
1:18:38     
concatenate temporalness in each of the vectors. So Z N T is if this is my ZT     
1:18:45     
this is my ZT zed 1T zed 2T and so on. So yeah I guess I should have protection     
1:18:51     
number clear away. Yeah my bad. Uh so yeah again so this is my concatenation. So it's a dimension plus one. We adding     
1:18:57     
a dimension to it. So this is done. I do the transformation layer of MLB. So this     
1:19:03     
is basically t activation function. I'm going through two t activation functions and basically three layers but I just     
1:19:09     
did three layers because I had some limited compute uh compute but I can scale it. If I like time and compute     
1:19:15     
I'll scale this. Then then numerical integration. So since my output here as you can see is dz dzn by dt. So I'm     
1:19:24     
training for that like that is my final output from the MLP. So I just integrated using my initial value to get     
1:19:30     
ZT K which is like uh the connect to at any point of time right so now this ZT     
1:19:38     
won't be directly the connect it will be the latent space I'll show how the decoder like builds it into the connect     
1:19:43     
back to the connect so you can see this seems a bit cryptic here the f theta z     
1:19:49     
to comma to so this essentially if you look here is just the uh like yeah as     
1:19:55     
you can see this is is so this is just the essentially is the dzn by d to so if     
1:20:01     
you integrate it we'll get the we'll get dz n and then we add to this and essentially then we get solving the     
1:20:07     
initial value problem we get uh the latent space of connect at any point t in time so using this we can yield zed     
1:20:15     
at any time step so decoding is basically in inner product so if you     
1:20:20     
take zt in a product with another z like say it's a uh element wise six sigmoid     
1:20:27     
in element wise sigmoid. So like I have 225 zs and then I take inner product of     
1:20:33     
each one with each one. So like basically I'll have 225 C2 C2 as in     
1:20:38     
select I'm selecting two that mean products I'll have and then I'll uh see how the uh each of the uh vectors     
1:20:47     
uh are like correlated with each other and then that that passing it through the sigmoid this gives me a probability     
1:20:53     
matrix of n + n. So now what my output is it's a probability matrix n cross n     
1:21:00     
where each of the value is a probability if there's a connection between those neurons or not that is what is my output     
1:21:06     
right so coming to the loss function yeah so coming to the loss function so I'm using a bind okay before I go to     
1:21:13     
loss function uh is there any questions on OD the neural part     
1:21:20     
no okay cool um yeah from the loss     
1:21:25     
function. So the I'm using binary cross entropy logic loss uh because my outputs     
1:21:30     
as of now are zeros or one as to like if connection is there or not. So that's why I'm using a binary cross entropy     
1:21:36     
loss and then uh so the inter time steps so as as I said earlier like I have L1,     
1:21:42     
L2, L3, L4 and so on. So those inter time steps are considered as labels. So my input to the model is just the L1 and     
1:21:50     
to to the graph model. Uh and then that uh uh just that is the input. Others are not shown to the model. It's just used     
1:21:56     
as labels and then like enter in points and uh yeah. So this is this is my loss.     
1:22:02     
This is not quite good. I just like but fine I'll like uh I wish to improve the     
1:22:09     
architecture and better the thing better my things. So I'll tell you how I'm wishing to do it. So like the heat map     
1:22:15     
visualization which I told but this doesn't tell us much. We need something better. So yeah this is my graph     
1:22:21     
visualization which I have done. Um so only the connections with probability 0.8 is kept. Uh if if I keep all of the     
1:22:29     
connections with like say 0.5 plus probability the graph gets very messy and difficult to interpret. So I just kept connections with probability 0.8     
1:22:36     
plus uh which gives gives us some guarantee. So the colors do not explicitly are not explicitly programmed     
1:22:43     
with biological names as like sensory neurons or motor neurons which I told in the like the biological paper which I     
1:22:50     
referred in the start as like they told that each of the neurons are categorized into some as the sensory or motor and so     
1:22:55     
on and there's a hierarchy but so the colors are not programmed to be that they're determined by like communicated     
1:23:02     
detection specifically the model algorithm but the colors visualize functional modules like though we tell     
1:23:09     
it but it understands that okay here is a cluster say the yellow ones the the orange ones here is a cluster where     
1:23:16     
these neurons are a lot connected amongst themselves and they form some cluster in itself like there are not     
1:23:22     
many connections from say orange to red or so on they form a cluster in themselves they are uh they're remaining     
1:23:27     
with each other like as I told we don't tell the model like it is a motor neuron or such like for example I'm not exactly     
1:23:34     
pointing out a motor neuron as I'm just g example uh like then the math is realizing that okay these 20 neurons are     
1:23:42     
a lot like they hang out together they just are connected amongst themselves so they can be uh certain like type of     
1:23:49     
neurons so it's figuring out clusters in the uh in the connect and I'll just show a zoomed this is a larger version of     
1:23:56     
this the graph there's a zoomed version so in zoomed version each of the neuron has a name as you can see this one the     
1:24:02     
point which I'm pointing is a KVL and it's degrees 48 like it's connected to uh like there are 48 connections to it     
1:24:09     
like from some other neuron to that there are 48 sus 48 connections so I     
1:24:14     
have created the graph for all the interpolation so what I've done is t equal to 0 t equal to 0.5 1 1.5 and so     
1:24:22     
on till t equal to 3 that interpolation I've done and yeah any doubts as of now     
1:24:27     
or I'll go towards flaws and future works that's good uh yeah why don't we do the     
1:24:32     
flaws and future works I think that looks good so for the flaws what I'm thinking is     
1:24:38     
I'm not taking as I stated earlier I should take into account biology why I'm not taking into account as of now is the     
1:24:43     
biology part like I have I have just used math as of now so I intend to use     
1:24:49     
hyperbolic neural so why hyperbolic neural the paper describes a clear hierarchy as to sensory to the neuron     
1:24:56     
the motor and so on so the standard ukidian space the normal vectors is bad at representing hierarchies trees so the     
1:25:03     
map map so we can map the neurons into hyperbolic space upon their all and instead of uklidian space in the     
1:25:09     
hyperbolic geometry it naturally represents trees and a hierarchy. So if we compute in that space maybe we can     
1:25:16     
have a better representation of hierarchy and a better outcome. Second thing is BC laws only cares if there     
1:25:23     
like if if an edge exist or not in the brain which and it doesn't care for a functional circuit like if the circuit     
1:25:30     
actually function if it actually makes sense. So we can add a regular regularization term to the loss that     
1:25:35     
analyzes graph properties like what I mean here is that this can force the model to match uh the degree     
1:25:42     
distribution like preventing it from creating random edges and forcing it to create the hubs like AVA AP. So what I     
1:25:50     
have seen in paper so what I mean here is like paper states that there are hubs of neurons it's not like as I stated     
1:25:55     
like there are like physical if there is a physical connection it intends like a neuron is more uh probable to connect     
1:26:03     
there if there is a more physical contact connection and they create hubs so there are like multiple hubs in in     
1:26:08     
the connector this is what was observed from the paper so I intend to create a     
1:26:14     
uh a regularization for that part as well uh so again the flaws are the differential equation which I use is too     
1:26:19     
basic as I stated like it's just dz by dt equal to some um some some uh like     
1:26:27     
mlp but there can be like degree two correction as well so I intend to like     
1:26:32     
use a better differential equation third thing is the gcn assumes every neighbor is equally important as the paper states     
1:26:39     
that some connections are stable always present whereas some connections are stoastic uh like sorry variable which is stoastic so I need to get a way around     
1:26:47     
so this is uh in this part I didn't write here I was thinking I can use a graph attention attention layer I can     
1:26:52     
add in the graph somewhere to as to pay attention towards which uh neighbors are actually important which are not that     
1:26:58     
important is the third thing and the fourth thing is I need to find more data set to verify this idea and maybe if     
1:27:06     
possible like I don't know like uh maybe get a new organism to test this as well like if this is a generic thing or this     
1:27:12     
is limited to C elegance so basically I I need to find more data sets I did find some more data sets     
1:27:18     
um on subsets of some other organism not of this particularly and like the future     
1:27:24     
work I need to inject the anatomy into this like using the using this data set     
1:27:29     
to tell the model which neurons are sensory versus which are motor as of now I'm not telling the model which are     
1:27:35     
sensory and motor I'm trying to interpret it through the model but I can also tell the model like these are     
1:27:40     
sensory and these are motor so like my inter interpolation uh can be a bit more better so I I also     
1:27:47     
intend to add geometry like pining long connection in a loss function. So I mean from the paper it was interpreted that     
1:27:54     
long connections are not that uh you know um uh it doesn't occur that often.     
1:28:00     
So we canize long connections um and the clustering part which I said I have for     
1:28:06     
it here the clustering part as well. So we can analyze dynamics. So using the model to find the exact critical periods     
1:28:11     
of wiring changes that are missed between static snapshots. So currently the term is like we have snapshot we     
1:28:17     
don't have interpolation that was the whole problem statement. So what we can take from this uh solution like we can     
1:28:23     
what what I can try to do is that at what points of the life of the C and what what are the critical periods in     
1:28:29     
which connections are changing when is the brain developing suddenly and all so that can be uh done. So what I'm trying     
1:28:35     
to do is uh I have as I said I have those all the graphs of each of the time step like I can generate graph of any     
1:28:41     
time step. So I'm trying to create an animation now as to show how the brain is evolving and then we can observe at     
1:28:47     
what point exactly that's happening. Um I I have generated animation but it's not that good. I'm just trying to get a     
1:28:53     
good animation out of all that all that. So yeah and then finding new new finding and testing on new data sets or when you     
1:28:59     
find a new new organism. So yeah that's all from my side. These are the references. This is the paper which I     
1:29:06     
used the C C ele C elegance coron paper. This is the GCN paper. This is neod. So     
1:29:11     
basically I try to combine these two papers and create something of something new. This is not entirely new but I'm     
1:29:17     
just combining two papers and then this is the implementation code. I'll share it in Slack and you can like visit the     
1:29:24     
code and play around with it maybe. So yeah this is from my side. Uh any     
1:29:30     
questions? Yeah thank you. It was good. Um yeah uh     
1:29:38     
I'm looking forward to seeing the visualization. I think you have a lot of really interesting methods there. Um definitely     
1:29:44     
the neuro you know this is just kind of looking at the evolution of of a conneto     
1:29:50     
and kind of getting a sense of what that looks like the dynamics and that and um yeah looks like you've got some really     
1:29:58     
good work already and you know visualizing it I guess will be key. Um,     
1:30:04     
I don't know if Morgan had something to say.     
1:30:11     
Uh, no. Uh, you know, I've been playing with neural ids recently, but um, yeah,     
1:30:17     
this is interesting. And, um, I'll try and play around with this. Yeah. So, what are you using? Oh, go     
1:30:24     
ahead. So, what are you using for your neural ODS? What kind of soft just in Python or     
1:30:31     
is a toolbox that you're using? Sorry, is that for me? It's for Shreas. Well, for you too, I     
1:30:38     
guess. Go ahead.     
1:30:43     
So, uh I'm using a library neural part. uh it's a part of uh uh I guess pi uh I     
1:30:51     
guess part of py geometry the neural nodes and there uh I'm using it uh in     
1:30:57     
torch to torch only like I'm using it in torch all right but there are some mathematical     
1:31:04     
softwares like I guess sage and all which do it but I'm not quite familiar with it I was familiar with pyarch so I     
1:31:10     
did use it yeah it's best to stick with pietorch I guess Yeah, I'm more comfortable with     
1:31:17     
white out, so I stuck with that. But there are some libraries as well. Yeah. Yeah, I'm I'm I'm doing everything with     
1:31:24     
Jack's Jack's libraries. Yeah. Oh,     
1:31:29     
yeah. That's good. All right. Um Yeah. Thank you. Thank you, Shreas.     
1:31:35     
That was very impressive. Yeah. So, why don't we call it a meeting? Uh thank you again to Carill for     
1:31:41     
presenting. If you look in the chat, he's left his email address. us if you're interested in getting in touch     
1:31:47     
with them. Um, and you know, we'll probably want to do some sort of     
1:31:52     
collaboration. Uh, and so people, you know, think about what you would like to     
1:31:57     
model. Uh, the the open source repository doesn't have a link to it here, but it's not public yet. So, we'll     
1:32:04     
definitely get in touch on that. And then, Treyas, thank you for your presentation. Hopefully, you know, I     
1:32:11     
guess we might want to follow up next week on it as well. Yeah, sure. Sure. Yeah, I'll get the animation out.     
1:32:17     
Okay. And then I think Barshan probably wants to present, but we'll do that next week.     
1:32:22     
You know, we'll just keep presenting stuff that next week. Yeah. Yeah. Get some more results. Yeah.     
1:32:28     
Yeah. Yeah. Okay. All right. All right. Thank you. Have a good week.     
1:32:34     
Okay. I'm interested in in modeling, too, especially using some tengities.     
1:32:41     
Okay. in with uh the model. So, all right. Yeah. All right. Well, thank     
1:32:48     
you. Okay. Yeah. Thanks, everyone. It's great to meet you after a long     
1:32:54     
time. Thanks. Oh, yeah. Yeah. Nice seeing you again. Yeah. Thanks.     
1:32:59     
Bye.
